[[LANFRANCO_et+al_7GeneticsGenomicsDecipherPartner_2021]]

# [7 Genetics and Genomics Decipher Partner Biology in Arbuscular Mycorrhizas]()

## [[LUISA LANFRANCO]]; [[GENNARO CAROTENUTO]]; [[ANDREA GENRE]] et al.

### 2021

## Abstract
Differently from model fungi (Neurospora, Aspergillus) and plants (Arabidopsis), the genetics of arbuscular mycorrhizas, intended as the sum of interacting plants and fungi, is therefore a very recent domain of science. Parniske (2004) was one of the first to use the term genetics in his highly quoted review “Molecular genetics of the arbuscular mycorrhizal symbiosis.”

## Key concepts
#arbuscular_mycorrhizas; #symbiosis; #host_plant; #GRAS; #glomus; #metabolism; #rhizophagus_irregularis; #lotus_japonicus; #genetics; #biology; #finding/transposable_elements; #gibberellic_acid; #small_rna; #land_plant; #rna_interference_machinery; #triticum_aestivum; #small_interfering_rna

## Quote
> Together these results suggest that the karrikin receptor complex plays a role in symbiotic signaling even if the involvement of karrikin-like molecules of fungal or plant origin remains to be investigatedWaters et al 2017)


## Key points
- The aim of this chapter is to provide a review of the multiple interactions that are included in the term “arbuscular mycorrhizas” and present an updated view of our knowledge on the molecular genetics of Arbuscular mycorrhizas (AMs), covering the genomes of AM fungi, the cellular and molecular responses of the host plant, as well as the fungal and plant natural variation that contributes to the outcome of this fascinating interaction
- Even if AM host plants can survive if deprived of their fungal symbionts, this condition is virtually unknown in natural ecosystems, where AM fungi are associated as helper microorganisms in most of the environments so far investigated (Davison et al 2015, 2018)
- The expectations of the researchers involved in genome sequencing of Rhizophagus irregularis were first focused on another crucial question: why are AM fungi unculturable? At a first glance, their obligate biotrophy was not explained by genome erosion or any related loss of metabolic complexity in central metabolism
- The AM symbiosis develops in roots where extensive cellular reorganizations and specific metabolic changes occur, which are mirrored by local changes in the transcript profiles as it has been demonstrated by transcriptomic analyses carried out on several plant species
- The detailed characterization of several isolates of R. irregularis at the level of single nuclei has even opened a window on the potentials to genetically manipulate AM fungi (Chen et al 2018)

## Synopsis

### Genetics and Genomics Decipher Partner Biology in Arbuscular Mycorrhizas
From model fungi (Neurospora, Aspergillus) and plants (Arabidopsis), the genetics of arbuscular mycorrhizas, intended as the sum of interacting plants and fungi, is a very recent domain of science.
Parniske (2004) was one of the first to use the term genetics in his highly quoted review “Molecular genetics of the arbuscular mycorrhizal symbiosis.”.
In this context, the aim of this chapter is to provide a review of the multiple interactions that are included in the term “arbuscular mycorrhizas” and present an updated view of the knowledge on the molecular genetics of AMs, covering the genomes of AM fungi, the cellular and molecular responses of the host plant, as well as the fungal and plant natural variation that contributes to the outcome of this fascinating interaction

### A New Look at the Interacting Partners
Arbuscular mycorrhizas (AMs) are traditionally described as the symbiosis resulting from the interaction between the roots of land plants and soil fungi.
The colonization processes by Mucoromycotina are still to be defined: these fungi may establish different interactions with plants; some of them establish ectomycorrhizas ([^Fassi_et+al_1969_a]) and have been detected mostly by using molecular tools, while morphology suggests that they form characteristic intracellular swellings ([^Bidartondo_et+al_2011_a]).
On the basis of the current data, Glomeromycotina can be defined as a stable component of the plant microbiota, since they are found in most of the environments so far investigated (Davison et al 2015, 2018), but on the other hand, they host their own microbiota, given by the intracellular endobacteria as well as by the bacteria which are commonly associated to the surface of their extraradical hyphae (Turrini et al 2018).
Mycoviruses can be considered an additional component of the AM microbiome with the potential to influence the biology of AM fungi and their host plant (Ikeda et al 2012)

### Lessons from the Genome Sequencing of AM Fungi
The authors' knowledge of the AM symbiosis mainly mirrors a plant-centric view
This is due to (1) the obligate biotrophic status of Glomeromycotina, which cannot be cultivated in the absence of their host plants; (2) their multinuclear condition, i.e., hundreds of nuclei coexist within one continuous cytoplasm; and (3) the absence of observable sexual reproduction and a uninucleated life stage (Chen et al 2018).
All these aspects hamper the use of the classical genetic tools which have, by contrast, allowed to study model fungi like Neurospora or Aspergillus, or their host plants which offer genetically tractable systems.
Clonality still appears to be the prevalent mode of reproduction (Chen et al 2018)

### The Biotrophism of AM Fungi
The expectations of the researchers involved in genome sequencing of Rhizophagus irregularis were first focused on another crucial question: why are AM fungi unculturable? At a first glance, their obligate biotrophy was not explained by genome erosion or any related loss of metabolic complexity in central metabolism.
The genome sequencing of four Endogonaceae fungi (Chang et al 2018) has detected the symbiotic signatures already identified in the other mycorrhizal fungi such as large genome size, high repetitive DNA content, and low diversity of plant cell walldegrading enzymes but without elevated small secreted proteins/secretome ratios.
The genome sequence led to the detection of Mollicutes-related endobacteria (MREs) in D. epigea and in three of the four sequenced Endogonaceae
Their genomes can be read as “metagenomes.” By contrast, G. margarita genome confirmed the presence of Candidatus Glomeribacter gigasporarum, which was already sequenced ([^Ghignone_et+al_2012_a]).
The intimate contact between bacteria and fungi may have favored horizontal gene transfer (Torres-Cortes et al 2015; Naito et al 2015; Sun et al 2018), potentially leading to an impact on the fungal biology (Salvioli et al 2016)

### From Structure to Function
The genome sequencing of AM fungi has so far provided relevant information concerning their genome structure and evolution, even if data from some more distantly related members, such as Archeospora, would be essential to better define their ancient relationships.
A small set of secreted proteins, shared by distantly related AM fungi (Rhizophagus irregularis and Gigaspora rosea), showed similar expression patterns in different host plants ([^Kamel_et+al_2017_a]).
These genes, described as the AM symbiotic core secretome, encode proteins with unknown function or proteases.
The mechanism of action of RiCRN1 does not involve cell death processes as often described for CRNs from oomycetes
In all these three abovementioned cases, host-induced gene silencing (HIGS) has been used to silence the fungal genes during the symbiotic phase allowing the description of an impaired colonization pattern.
The success of the HIGS approach as a tool to silence fungal genes in the AM symbiosis (Helber et al 2011; Tsuzuki et al 2016) is a strong clue toward the occurrence of such a process

### Cellular and Molecular Changes in the Host Plant
In analogy to most root-microbe interactions, AM establishment depends on finely tuned recognition processes (Bonfante and Genre 2015) through signal release and perception between both partners before their physical contact ([^Gianinazzi-Pearson_1996_a]).
Rice d3 and pea rms mutants displayed important defects in AM colonization and arbuscule formation, respectively ([^Yoshida_et+al_2012_a]</a>; Foo et al 2013; [^Gutjahr_et+al_2015_a]</a></a>); a d14l/kai rice mutant does not stimulate the formation of hyphopodia ([^Gutjahr_et+al_2015_a]) and does not respond transcriptionally to AM germinating spore exudates
Together these results suggest that the karrikin receptor complex plays a role in symbiotic signaling even if the involvement of karrikin-like molecules of fungal or plant origin remains to be investigated ([^Gutjahr_et+al_2015_a]</a></a>; Waters et al 2017).
Myc-COs are active in both legumes and non-legumes at very low concentration, down to 10À8 M ([^Genre_et+al_2013_a]</a>; Sun et al 2015), and can be considered as universal AM-specific elicitors

### The Common Symbiotic Signaling Pathway
The study of Myc factor signaling mechanisms in legumes such as Medicago truncatula and Lotus japonicus has mostly come as a followup of analogous research on symbiotic nitrogen fixation (SNF; Denarieand Cullimore 1993; Maillet et al 2011).
The process of arbuscule accommodation in cortical cells is the most striking feature of AM development and requires a broad reorganization of the host cells in strict coordination with fungal development: hyphal penetration associates with nuclear movement at the center of the cell ([^Bonfante_2001_a]), engulfed by a broad PPA (Genre et al 2008)
This anticipates the formation of the arbuscule trunk and the PAM trunk domain (Pumplin and Harrison 2009), characterized by a set of proteins that is analogous to that of the plasma membrane.
In response to CSSP activation, several transcription factors are expressed during either early or later stages of arbuscule formation (Bucher et al 2014; Luginbuehl and Oldroyd 2017; [^Diedhiou_2018_a]; Pimprikar and Gutjahr 2018), in turn regulating the expression of genes involved in nutrient transfer, primary and specialized metabolism, membrane and cell wall modifications, secretion, and signal transduction (Hohnjec et al 2005; Gaude et al 2012; Hogekamp and Kuster 2013; [^Handa_et+al_2015_a]).
Arbuscule senescence is a regulated process where the host cell remains active during and after arbuscule collapse and maintains the ability to be colonized again by a new arbuscule

### Mycorrhizal Omics
The AM symbiosis develops in roots where extensive cellular reorganizations and specific metabolic changes occur, which are mirrored by local changes in the transcript profiles as it has been demonstrated by transcriptomic analyses carried out on several plant species.
An untargeted metabolomic analysis was recently performed on tomato mycorrhizal roots with the aim to identify key metabolites involved in the mycorrhiza-induced protection against osmotic stresses (Rivero et al 2018).
The analysis of putative targets of selected miRNAs revealed an involvement in P starvation, phytohormone signaling, and defense (Pandey et al 2018).
All these studies convincingly demonstrate that AM fungi have a local and systemic influence on their host plant, since they lead to a deep reorganization of the plant biology acting on multiple transcriptomic, regulatory, and metabolomic pathways

### The AM Symbiosis in the Light of Natural Variation
Despite the low morphological variation and their large host range, AM fungal species and isolates show different efficiency in promoting plant performance; on the other hand, the plant genotype has an important role in determining the extent of plant responsiveness to the AM symbiosis (Smith et al 2004).
Concerning the responsiveness to AM fungi, Sawers et al (2017) analyzed the growth response of 30 maize lines upon colonization by F. mosseae; variations in shoot dry weight and shoot Pi content were observed, and, interestingly, these correlated with the amount of extraradical mycelium, suggesting a plantfungus reciprocal effect on growth performances.
Domesticated plants reduced AM fungal colonization more strongly than did wild progenitors in response to increased P availability
On the whole these studies indicate a strong fungal genotype X plant genotype interaction in the mycorrhizal symbiosis.
This variation may have profound impact in natural populations and has to be considered in agricultural practices where AM fungi are exploited to improve plant health and productivity

### Conclusions
Genetics and genomics have recently provided crucial novel information on the biology of arbuscular mycorrhizas.
Phylogenomics analyses based on genomes from host and non-host species are emerging as powerful tools to identify conserved genes required for the AM symbiosis (Bravo et al 2016) and to trace the evolution of the underlying genetic network from basal plants to angiosperms (Delaux et al 2015).
The authors can envisage that the CRISPR/Cas-based genome editing technique will offer an efficient strategy for producing plant genotypes with mutations in genes of interest.
These genes could be selected among those responsible of the molecular dialogue between partners and among those which regulate AM functionality.
In the frame of a more friendly agriculture, these plant genes could be the targets for the development of new crop varieties more susceptible and responsive to the beneficial AM fungi


## Study subjects

### 30000 genes
- The strain DAOM-197198 was selected for several reasons: it had been hypothesized to possess a very small genome; it easily grows in association with root organ cultures, producing a large amount of noncontaminated fungal material; and – as a last key feature – it does not host endobacteria, thus representing a potentially more amenable scenario. ==The sequence of its 153-Mb haploid genome showed a repertoire of about 30,000 genes and revealed a low level of polymorphism offering for the first time a reply to the crucial question: do the nuclei of AM fungi possess multiple, highly diverged genomes? The data strongly suggested the inconsistency of such a hypothesis, which was also elegantly refuted by the whole sequence of isolated single nuclei (Lin et al 2014)==. Mating (MAT)-related genes were found to be expanded, suggesting the existence of cryptic sex-related processes and opening the possibility that a non-observable mating does not mean absence of sex

### 3 abovementioned cases
- Although not yet defined, the mechanism of action of RiCRN1 does not involve cell death processes as often described for CRNs from oomycetes. ==In all these three abovementioned cases, host-induced gene silencing (HIGS) has been used to specifically silence the fungal genes during the symbiotic phase allowing the description of an impaired colonization pattern==. These examples also highlight how, in the absence of protocols for stable genetic transformation for AM fungi, genetic manipulation tools developed for the host plants can be successfully applied to study the function of AM fungal genes at least in the in planta phase

### 3 mycorrhizal genes
- Among the CCaMK/CYCLOPS-regulated transcription factors, the GRAS-domain proteins NSP1 (Nodulation Signalling Pathway 1) and NSP2 play an essential role in Nod factor signaling (Catoira et al 2000; Kaloet al. 2005; Smit et al 2005). ==Evidence suggests direct roles of NSP1 and NSP2 also in Myc factor signaling: with NSP2 being involved in NS-LCO-induced lateral root growth (Maillet et al 2011) and NSP1 being required for the induction of three mycorrhizal genes in response to NS-LCO (Delaux et al 2013)==. NSP1 and NSP2 interaction is required for the induction of nodulationspecific promoters (Hirsch et al 2009; [^Jin_et+al_2016_a]) but not crucial for AM symbiosis, suggesting that different GRAS transcription factor complexes regulate distinct groups of genes (Pimprikar and Gutjahr 2018)

## Findings
- The genome expansion seems to be strictly correlated with the presence of transposable elements (TE) which, in the case of G. margarita, represent more than 80% of the whole genome

## Counterpoint to earlier claims
- Furthermore, Myc-LCOinduced responses were shown to be NFPdependent, indicating the involvement of this receptor in both Nod- and Myc-LCO perception (Op den Camp et al 2011). ==Although nfp mutants exhibit normal Ca2+ spiking and AM colonization== (Maillet et al 2011; [^Genre_et+al_2013_a]; [^Zhang_et+al_2015_a]</a>), such mutants do not display nuclear Ca2+ spiking in response to Myc-LCOs (Sun et al 2015)

## Contributions
- Genetics and genomics have recently provided crucial novel information on the biology of arbuscular mycorrhizas. The genome sequencing of a number of AM fungal species is allowing to identify common features such as the fatty acid auxotrophy but also dispensable species-specific components. The detailed characterization of several isolates of R. irregularis at the level of single nuclei has even opened a window on the potentials to genetically manipulate AM fungi (Chen et al 2018).<br/><br/>On the plant perspective, phylogenomics analyses based on genomes from host and non-host species are emerging as powerful tools to identify conserved genes required for the AM symbiosis (Bravo et al 2016) and to trace the evolution of the underlying genetic network from basal plants to angiosperms (Delaux et al 2015). We can also envisage that the CRISPR/Cas-based genome editing technique will offer an efficient strategy for producing plant genotypes with mutations in genes of interest. These genes could be selected among those responsible of the molecular dialogue between partners (also considering the presymbiotic steps) and among those which regulate AM functionality. In the frame of a more friendly agriculture, these plant genes could be the targets for the development of new crop varieties more susceptible and responsive to the beneficial AM fungi.


## References
[^Akiyama_2006_a]: Akiyama K, Hayashi H (2006) Strigolactones: chemical signals for fungal symbionts and parasitic weeds in plant roots. Ann Bot 97(6):925–931 [[Akiyama_StrigolactonesChemicalSignalsFungalSymbionts_2006]] [OA](https://api.scholarcy.com/oa_version?query=Akiyama%2C%20K.%20Hayashi%2C%20H.%20Strigolactones%3A%20chemical%20signals%20for%20fungal%20symbionts%20and%20parasitic%20weeds%20in%20plant%20roots%202006) [GScholar](https://scholar.google.co.uk/scholar?q=Akiyama%2C%20K.%20Hayashi%2C%20H.%20Strigolactones%3A%20chemical%20signals%20for%20fungal%20symbionts%20and%20parasitic%20weeds%20in%20plant%20roots%202006) [Scite](https://api.scholarcy.com/scite_url?query=Akiyama%2C%20K.%20Hayashi%2C%20H.%20Strigolactones%3A%20chemical%20signals%20for%20fungal%20symbionts%20and%20parasitic%20weeds%20in%20plant%20roots%202006)

[^Akiyama_2005_a]: Akiyama K, Matsuzaki K, Hayashi H (2005) Plant sesquiterpenes induce hyphal branching in arbuscular mycorrhizal fungi. Nature 435(7043):824–827 [[Akiyama_PlantSesquiterpenesInduceHyphalBranching_2005]] [OA](https://api.scholarcy.com/oa_version?query=Akiyama%20K%20Matsuzaki%20K%20Hayashi%20H%202005%20Plant%20sesquiterpenes%20induce%20hyphal%20branching%20in%20arbuscular%20mycorrhizal%20fungi%20Nature%204357043824827) [GScholar](https://scholar.google.co.uk/scholar?q=Akiyama%20K%20Matsuzaki%20K%20Hayashi%20H%202005%20Plant%20sesquiterpenes%20induce%20hyphal%20branching%20in%20arbuscular%20mycorrhizal%20fungi%20Nature%204357043824827) [Scite](https://api.scholarcy.com/scite_url?query=Akiyama%20K%20Matsuzaki%20K%20Hayashi%20H%202005%20Plant%20sesquiterpenes%20induce%20hyphal%20branching%20in%20arbuscular%20mycorrhizal%20fungi%20Nature%204357043824827)

[^Akiyama_et+al_2010_a]: Akiyama K, Ogasawara S, Ito S, Hayashi H (2010) Structural requirements of strigolactones for hyphal branching in AM fungi. Plant Cell Physiol 51(7):1104–1117 [[Akiyama_et+al_StructuralRequirementsStrigolactonesHyphalBranching_2010]] [OA](https://api.scholarcy.com/oa_version?query=Akiyama%2C%20K.%20Ogasawara%2C%20S.%20Ito%2C%20S.%20Hayashi%2C%20H.%20Structural%20requirements%20of%20strigolactones%20for%20hyphal%20branching%20in%20AM%20fungi%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Akiyama%2C%20K.%20Ogasawara%2C%20S.%20Ito%2C%20S.%20Hayashi%2C%20H.%20Structural%20requirements%20of%20strigolactones%20for%20hyphal%20branching%20in%20AM%20fungi%202010) [Scite](https://api.scholarcy.com/scite_url?query=Akiyama%2C%20K.%20Ogasawara%2C%20S.%20Ito%2C%20S.%20Hayashi%2C%20H.%20Structural%20requirements%20of%20strigolactones%20for%20hyphal%20branching%20in%20AM%20fungi%202010)

[^Alvey_2005_a]: Alvey L, Harberd NP (2005) DELLA proteins: integrators of multiple plant growth regulatory inputs? Physiol Plant 123:153–160 [[Alvey_DellaProteinsIntegratorsMultiplePlant_2005]] [OA](https://api.scholarcy.com/oa_version?query=Alvey%2C%20L.%20Harberd%2C%20N.P.%20DELLA%20proteins%3A%20integrators%20of%20multiple%20plant%20growth%20regulatory%20inputs%3F%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Alvey%2C%20L.%20Harberd%2C%20N.P.%20DELLA%20proteins%3A%20integrators%20of%20multiple%20plant%20growth%20regulatory%20inputs%3F%202005) [Scite](https://api.scholarcy.com/scite_url?query=Alvey%2C%20L.%20Harberd%2C%20N.P.%20DELLA%20proteins%3A%20integrators%20of%20multiple%20plant%20growth%20regulatory%20inputs%3F%202005)

[^Ane_2004_a]: Ane JM, Kiss GB, Riely BK, Penmetsa RV, Oldroyd GE, Ayax C, Levy J, Debelle F, Baek JM, Kalo P, Rosenberg C, Roe BA, Long SR, Denarie J, Cook DR (2004) Medicago truncatula DMI1 required for bacterial and fungal symbioses in legumes. Science 303:1364–1367 [[Ane_MedicagoTruncatulaDmi1RequiredBacterial_2004]] [OA](https://api.scholarcy.com/oa_version?query=Ane%20JM%20Kiss%20GB%20Riely%20BK%20Penmetsa%20RV%20Oldroyd%20GE%20Ayax%20C%20Levy%20J%20Debelle%20F%20Baek%20JM%20Kalo%20P%20Rosenberg%20C%20Roe%20BA%20Long%20SR%20Denarie%20J%20Cook%20DR%202004%20Medicago%20truncatula%20DMI1%20required%20for%20bacterial%20and%20fungal%20symbioses%20in%20legumes%20Science%2030313641367) [GScholar](https://scholar.google.co.uk/scholar?q=Ane%20JM%20Kiss%20GB%20Riely%20BK%20Penmetsa%20RV%20Oldroyd%20GE%20Ayax%20C%20Levy%20J%20Debelle%20F%20Baek%20JM%20Kalo%20P%20Rosenberg%20C%20Roe%20BA%20Long%20SR%20Denarie%20J%20Cook%20DR%202004%20Medicago%20truncatula%20DMI1%20required%20for%20bacterial%20and%20fungal%20symbioses%20in%20legumes%20Science%2030313641367) [Scite](https://api.scholarcy.com/scite_url?query=Ane%20JM%20Kiss%20GB%20Riely%20BK%20Penmetsa%20RV%20Oldroyd%20GE%20Ayax%20C%20Levy%20J%20Debelle%20F%20Baek%20JM%20Kalo%20P%20Rosenberg%20C%20Roe%20BA%20Long%20SR%20Denarie%20J%20Cook%20DR%202004%20Medicago%20truncatula%20DMI1%20required%20for%20bacterial%20and%20fungal%20symbioses%20in%20legumes%20Science%2030313641367)

[^Antunes_et+al_2011_a]: Antunes PM, Koch AM, Morton JB, Rillig MC, Klironomos JN (2011) Evidence for functional divergence in arbuscular mycorrhizal fungi from contrasting climatic origins. New Phytol 189:507–514 [[Antunes_et+al_EvidenceFunctionalDivergenceArbuscularMycorrhizal_2011]] [OA](https://api.scholarcy.com/oa_version?query=Antunes%2C%20P.M.%20Koch%2C%20A.M.%20Morton%2C%20J.B.%20Rillig%2C%20M.C.%20Evidence%20for%20functional%20divergence%20in%20arbuscular%20mycorrhizal%20fungi%20from%20contrasting%20climatic%20origins%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Antunes%2C%20P.M.%20Koch%2C%20A.M.%20Morton%2C%20J.B.%20Rillig%2C%20M.C.%20Evidence%20for%20functional%20divergence%20in%20arbuscular%20mycorrhizal%20fungi%20from%20contrasting%20climatic%20origins%202011) [Scite](https://api.scholarcy.com/scite_url?query=Antunes%2C%20P.M.%20Koch%2C%20A.M.%20Morton%2C%20J.B.%20Rillig%2C%20M.C.%20Evidence%20for%20functional%20divergence%20in%20arbuscular%20mycorrhizal%20fungi%20from%20contrasting%20climatic%20origins%202011)

[^Ariel_et+al_2015_a]: Ariel F, Romero-Barrios N, Jegu T, Benhamed M, Crespi M (2015) Battles and hijacks: noncoding transcription in plants. Trends Plant Sci 20:362–371 [[Ariel_et+al_BattlesHijacksNoncodingTranscriptionPlants_2015]] [OA](https://api.scholarcy.com/oa_version?query=Ariel%2C%20F.%20Romero-Barrios%2C%20N.%20Jegu%2C%20T.%20Benhamed%2C%20M.%20Battles%20and%20hijacks%3A%20noncoding%20transcription%20in%20plants%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Ariel%2C%20F.%20Romero-Barrios%2C%20N.%20Jegu%2C%20T.%20Benhamed%2C%20M.%20Battles%20and%20hijacks%3A%20noncoding%20transcription%20in%20plants%202015) [Scite](https://api.scholarcy.com/scite_url?query=Ariel%2C%20F.%20Romero-Barrios%2C%20N.%20Jegu%2C%20T.%20Benhamed%2C%20M.%20Battles%20and%20hijacks%3A%20noncoding%20transcription%20in%20plants%202015)

[^Bago_et+al_2000_a]: Bago B, Pfeffer PE, Shachar-Hill Y (2000) Carbon metabolism and transport in arbuscular mycorrhizas. Plant Physiol 124:949–957 [[Bago_et+al_CarbonMetabolismTransportArbuscularMycorrhizas_2000]] [OA](https://api.scholarcy.com/oa_version?query=Bago%2C%20B.%20Pfeffer%2C%20P.E.%20Shachar-Hill%2C%20Y.%20Carbon%20metabolism%20and%20transport%20in%20arbuscular%20mycorrhizas%202000) [GScholar](https://scholar.google.co.uk/scholar?q=Bago%2C%20B.%20Pfeffer%2C%20P.E.%20Shachar-Hill%2C%20Y.%20Carbon%20metabolism%20and%20transport%20in%20arbuscular%20mycorrhizas%202000) [Scite](https://api.scholarcy.com/scite_url?query=Bago%2C%20B.%20Pfeffer%2C%20P.E.%20Shachar-Hill%2C%20Y.%20Carbon%20metabolism%20and%20transport%20in%20arbuscular%20mycorrhizas%202000)

[^Balestrini_2005_a]: Balestrini R, Bonfante P (2005) The interface compartment in arbuscular mycorrhizae: a special type of plant cell wall? Plant Biosystems 139:8–15 [[Balestrini_InterfaceCompartmentArbuscularMycorrhizaeSpecial_2005]] [OA](https://api.scholarcy.com/oa_version?query=Balestrini%2C%20R.%20Bonfante%2C%20P.%20The%20interface%20compartment%20in%20arbuscular%20mycorrhizae%3A%20a%20special%20type%20of%20plant%20cell%20wall%3F%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Balestrini%2C%20R.%20Bonfante%2C%20P.%20The%20interface%20compartment%20in%20arbuscular%20mycorrhizae%3A%20a%20special%20type%20of%20plant%20cell%20wall%3F%202005) [Scite](https://api.scholarcy.com/scite_url?query=Balestrini%2C%20R.%20Bonfante%2C%20P.%20The%20interface%20compartment%20in%20arbuscular%20mycorrhizae%3A%20a%20special%20type%20of%20plant%20cell%20wall%3F%202005)

[^Balestrini_2014_a]: Balestrini R, Bonfante P (2014) Cell wall remodeling in mycorrhizal symbiosis: a way towards biotrophism. Front Plant Sci 5:237 [[Balestrini_CellWallRemodelingMycorrhizalSymbiosis_2014]] [OA](https://api.scholarcy.com/oa_version?query=Balestrini%2C%20R.%20Bonfante%2C%20P.%20Cell%20wall%20remodeling%20in%20mycorrhizal%20symbiosis%3A%20a%20way%20towards%20biotrophism%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Balestrini%2C%20R.%20Bonfante%2C%20P.%20Cell%20wall%20remodeling%20in%20mycorrhizal%20symbiosis%3A%20a%20way%20towards%20biotrophism%202014) [Scite](https://api.scholarcy.com/scite_url?query=Balestrini%2C%20R.%20Bonfante%2C%20P.%20Cell%20wall%20remodeling%20in%20mycorrhizal%20symbiosis%3A%20a%20way%20towards%20biotrophism%202014)

[^Banba_et+al_2008_a]: Banba M, Gutjahr C, Miyao A, Hirochika H, Paszkowski U, Kouchi H, Imaizumi-Anraku H (2008) Divergence of evolutionary ways among common sym genes: CASTOR and CCaMK show functional conservation between two symbiosis systems and constitute the root of a common signalling pathway. Plant Cell Physiol 49(11):1659–1671 [[Banba_et+al_DivergenceEvolutionaryWaysAmongCommon_2008]] [OA](https://api.scholarcy.com/oa_version?query=Banba%2C%20M.%20Gutjahr%2C%20C.%20Miyao%2C%20A.%20Hirochika%2C%20H.%20Divergence%20of%20evolutionary%20ways%20among%20common%20sym%20genes%3A%20CASTOR%20and%20CCaMK%20show%20functional%20conservation%20between%20two%20symbiosis%20systems%20and%20constitute%20the%20root%20of%20a%20common%20signalling%20pathway%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Banba%2C%20M.%20Gutjahr%2C%20C.%20Miyao%2C%20A.%20Hirochika%2C%20H.%20Divergence%20of%20evolutionary%20ways%20among%20common%20sym%20genes%3A%20CASTOR%20and%20CCaMK%20show%20functional%20conservation%20between%20two%20symbiosis%20systems%20and%20constitute%20the%20root%20of%20a%20common%20signalling%20pathway%202008) [Scite](https://api.scholarcy.com/scite_url?query=Banba%2C%20M.%20Gutjahr%2C%20C.%20Miyao%2C%20A.%20Hirochika%2C%20H.%20Divergence%20of%20evolutionary%20ways%20among%20common%20sym%20genes%3A%20CASTOR%20and%20CCaMK%20show%20functional%20conservation%20between%20two%20symbiosis%20systems%20and%20constitute%20the%20root%20of%20a%20common%20signalling%20pathway%202008)

[^Bazin_et+al_2017_a]: Bazin J, Baerenfaller K, Gosai SJ, Gregory BD, Crespi M, Bailey-Serres J (2017) Global analysis of ribosomeassociated noncoding RNAs unveils new modes of translational regulation. Proc Natl Acad Sci U S A 114(46):E10018–E10027 [[Bazin_et+al_GlobalAnalysisRibosomeassociatedNoncodingRnas_2017]] [OA](https://api.scholarcy.com/oa_version?query=Bazin%2C%20J.%20Baerenfaller%2C%20K.%20Gosai%2C%20S.J.%20Gregory%2C%20B.D.%20Global%20analysis%20of%20ribosomeassociated%20noncoding%20RNAs%20unveils%20new%20modes%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Bazin%2C%20J.%20Baerenfaller%2C%20K.%20Gosai%2C%20S.J.%20Gregory%2C%20B.D.%20Global%20analysis%20of%20ribosomeassociated%20noncoding%20RNAs%20unveils%20new%20modes%202017) [Scite](https://api.scholarcy.com/scite_url?query=Bazin%2C%20J.%20Baerenfaller%2C%20K.%20Gosai%2C%20S.J.%20Gregory%2C%20B.D.%20Global%20analysis%20of%20ribosomeassociated%20noncoding%20RNAs%20unveils%20new%20modes%202017)

[^Besserer_et+al_2006_a]: Besserer A, Puech-Pages V, Kiefer P, Gomez-Roldan V, Jauneau A, Roy S, Portais JC, Roux C, Becard G, Sejalon-Delmas N (2006) Strigolactones stimulate arbuscular mycorrhizal fungi by activating mitochondria. PLoS Biol 4(7):1239–1247 [[Besserer_et+al_StrigolactonesStimulateArbuscularMycorrhizalFungi_2006]] [OA](https://api.scholarcy.com/oa_version?query=Besserer%2C%20A.%20Puech-Pages%2C%20V.%20Kiefer%2C%20P.%20Gomez-Roldan%2C%20V.%20Strigolactones%20stimulate%20arbuscular%20mycorrhizal%20fungi%20by%20activating%20mitochondria%202006) [GScholar](https://scholar.google.co.uk/scholar?q=Besserer%2C%20A.%20Puech-Pages%2C%20V.%20Kiefer%2C%20P.%20Gomez-Roldan%2C%20V.%20Strigolactones%20stimulate%20arbuscular%20mycorrhizal%20fungi%20by%20activating%20mitochondria%202006) [Scite](https://api.scholarcy.com/scite_url?query=Besserer%2C%20A.%20Puech-Pages%2C%20V.%20Kiefer%2C%20P.%20Gomez-Roldan%2C%20V.%20Strigolactones%20stimulate%20arbuscular%20mycorrhizal%20fungi%20by%20activating%20mitochondria%202006)

[^Besserer_et+al_2008_a]: Besserer A, Becard G, Jauneau A, Roux C, SejalonDelmas N (2008) GR24, a synthetic analog of strigolactones, stimulates the mitosis and growth of the arbuscular mycorrhizal fungus Gigaspora rosea by boosting its energy metabolism. Plant Physiol 148(1):402–413 [[Besserer_et+al_Gr24SyntheticAnalogStrigolactonesStimulates_2008]] [OA](https://api.scholarcy.com/oa_version?query=Besserer%2C%20A.%20Becard%2C%20G.%20Jauneau%2C%20A.%20Roux%2C%20C.%20GR24%2C%20a%20synthetic%20analog%20of%20strigolactones%2C%20stimulates%20the%20mitosis%20and%20growth%20of%20the%20arbuscular%20mycorrhizal%20fungus%20Gigaspora%20rosea%20by%20boosting%20its%20energy%20metabolism%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Besserer%2C%20A.%20Becard%2C%20G.%20Jauneau%2C%20A.%20Roux%2C%20C.%20GR24%2C%20a%20synthetic%20analog%20of%20strigolactones%2C%20stimulates%20the%20mitosis%20and%20growth%20of%20the%20arbuscular%20mycorrhizal%20fungus%20Gigaspora%20rosea%20by%20boosting%20its%20energy%20metabolism%202008) [Scite](https://api.scholarcy.com/scite_url?query=Besserer%2C%20A.%20Becard%2C%20G.%20Jauneau%2C%20A.%20Roux%2C%20C.%20GR24%2C%20a%20synthetic%20analog%20of%20strigolactones%2C%20stimulates%20the%20mitosis%20and%20growth%20of%20the%20arbuscular%20mycorrhizal%20fungus%20Gigaspora%20rosea%20by%20boosting%20its%20energy%20metabolism%202008)

[^Bidartondo_et+al_2011_a]: Bidartondo MI, Read DJ, Trappe JM, Merckx V, Ligrone R, Duckett JG (2011) The dawn of symbiosis between plants and fungi. Biol Lett 7:574–577 [[Bidartondo_et+al_DawnSymbiosisBetweenPlantsFungi_2011]] [OA](https://api.scholarcy.com/oa_version?query=Bidartondo%2C%20M.I.%20Read%2C%20D.J.%20Trappe%2C%20J.M.%20Merckx%2C%20V.%20The%20dawn%20of%20symbiosis%20between%20plants%20and%20fungi%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Bidartondo%2C%20M.I.%20Read%2C%20D.J.%20Trappe%2C%20J.M.%20Merckx%2C%20V.%20The%20dawn%20of%20symbiosis%20between%20plants%20and%20fungi%202011) [Scite](https://api.scholarcy.com/scite_url?query=Bidartondo%2C%20M.I.%20Read%2C%20D.J.%20Trappe%2C%20J.M.%20Merckx%2C%20V.%20The%20dawn%20of%20symbiosis%20between%20plants%20and%20fungi%202011)

[^Bonfante_2001_a]: Bonfante P (2001) At the interface between mycorrhizal fungi and plants: the structural organization of cell wall, plasma membrane and cytoskeleton. In: Hock B (ed) The mycota, IX: fungal associations. Springer, Berlin, pp 45–61 [[Bonfante_InterfaceBetweenMycorrhizalFungiPlants_2001]] [OA](https://api.scholarcy.com/oa_version?query=Bonfante%2C%20P.%20At%20the%20interface%20between%20mycorrhizal%20fungi%20and%20plants%3A%20the%20structural%20organization%20of%20cell%20wall%2C%20plasma%20membrane%20and%20cytoskeleton%202001) [GScholar](https://scholar.google.co.uk/scholar?q=Bonfante%2C%20P.%20At%20the%20interface%20between%20mycorrhizal%20fungi%20and%20plants%3A%20the%20structural%20organization%20of%20cell%20wall%2C%20plasma%20membrane%20and%20cytoskeleton%202001) [Scite](https://api.scholarcy.com/scite_url?query=Bonfante%2C%20P.%20At%20the%20interface%20between%20mycorrhizal%20fungi%20and%20plants%3A%20the%20structural%20organization%20of%20cell%20wall%2C%20plasma%20membrane%20and%20cytoskeleton%202001)

[^Bonfante_2018_a]: Bonfante P (2018) The future has roots in the past: the ideas and scientists that shaped mycorrhizal research. New Phytol 220:982–995 [[Bonfante_FutureRootsPastIdeasScientists_2018]] [OA](https://api.scholarcy.com/oa_version?query=Bonfante%2C%20P.%20The%20future%20has%20roots%20in%20the%20past%3A%20the%20ideas%20and%20scientists%20that%20shaped%20mycorrhizal%20research%202018) [GScholar](https://scholar.google.co.uk/scholar?q=Bonfante%2C%20P.%20The%20future%20has%20roots%20in%20the%20past%3A%20the%20ideas%20and%20scientists%20that%20shaped%20mycorrhizal%20research%202018) [Scite](https://api.scholarcy.com/scite_url?query=Bonfante%2C%20P.%20The%20future%20has%20roots%20in%20the%20past%3A%20the%20ideas%20and%20scientists%20that%20shaped%20mycorrhizal%20research%202018)

[^Devers_et+al_2011_a]: Devers EA, Branscheid A, May P, Krajinski F (2011) Stars and symbiosis: microRNA- and microRNA*mediated transcript cleavage involved in arbuscular mycorrhizal symbiosis. Plant Physiol 156:1990– 2010 [[Devers_et+al_StarsSymbiosisMicrornaMicrornamediatedTranscript_2011]] [OA](https://api.scholarcy.com/oa_version?query=Devers%2C%20E.A.%20Branscheid%2C%20A.%20May%2C%20P.%20Krajinski%2C%20F.%20Stars%20and%20symbiosis%3A%20microRNA-%20and%20microRNA%2Amediated%20transcript%20cleavage%20involved%20in%20arbuscular%20mycorrhizal%20symbiosis%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Devers%2C%20E.A.%20Branscheid%2C%20A.%20May%2C%20P.%20Krajinski%2C%20F.%20Stars%20and%20symbiosis%3A%20microRNA-%20and%20microRNA%2Amediated%20transcript%20cleavage%20involved%20in%20arbuscular%20mycorrhizal%20symbiosis%202011) [Scite](https://api.scholarcy.com/scite_url?query=Devers%2C%20E.A.%20Branscheid%2C%20A.%20May%2C%20P.%20Krajinski%2C%20F.%20Stars%20and%20symbiosis%3A%20microRNA-%20and%20microRNA%2Amediated%20transcript%20cleavage%20involved%20in%20arbuscular%20mycorrhizal%20symbiosis%202011)

[^Devers_et+al_2013_a]: Devers E, Teply J, Reinert A, Gaude N, Krajinski F (2013) An endogenous artificial microRNA system for unraveling the function of root endosymbioses related genes in Medicago truncatula. BMC Plant Biol 13:82 [[Devers_et+al_EndogenousArtificialMicrornaSystemUnraveling_2013]] [OA](https://api.scholarcy.com/oa_version?query=Devers%2C%20E.%20Teply%2C%20J.%20Reinert%2C%20A.%20Gaude%2C%20N.%20An%20endogenous%20artificial%20microRNA%20system%20for%20unraveling%20the%20function%20of%20root%20endosymbioses%20related%20genes%20in%20Medicago%20truncatula%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Devers%2C%20E.%20Teply%2C%20J.%20Reinert%2C%20A.%20Gaude%2C%20N.%20An%20endogenous%20artificial%20microRNA%20system%20for%20unraveling%20the%20function%20of%20root%20endosymbioses%20related%20genes%20in%20Medicago%20truncatula%202013) [Scite](https://api.scholarcy.com/scite_url?query=Devers%2C%20E.%20Teply%2C%20J.%20Reinert%2C%20A.%20Gaude%2C%20N.%20An%20endogenous%20artificial%20microRNA%20system%20for%20unraveling%20the%20function%20of%20root%20endosymbioses%20related%20genes%20in%20Medicago%20truncatula%202013)

[^Diedhiou_2018_a]: Diedhiou I, Diouf D (2018) Transcription factors network in root endosymbiosis establishment and development. World J Microbiol Biotechnol 34 (3):37 [[Diedhiou_TranscriptionFactorsNetworkRootEndosymbiosis_2018]] [OA](https://api.scholarcy.com/oa_version?query=Diedhiou%2C%20I.%20Diouf%2C%20D.%20Transcription%20factors%20network%20in%20root%20endosymbiosis%20establishment%20and%20development%202018) [GScholar](https://scholar.google.co.uk/scholar?q=Diedhiou%2C%20I.%20Diouf%2C%20D.%20Transcription%20factors%20network%20in%20root%20endosymbiosis%20establishment%20and%20development%202018) [Scite](https://api.scholarcy.com/scite_url?query=Diedhiou%2C%20I.%20Diouf%2C%20D.%20Transcription%20factors%20network%20in%20root%20endosymbiosis%20establishment%20and%20development%202018)

[^Dormann_et+al_2014_a]: Dormann P, Kim H, Ott T, Schulze-Lefert P, Trujillo M, Wewer V, Huckelhoven R (2014) Cell-autonomous defense, reorganization and trafficking of membranes in plant-microbe interactions. New Phytol 204(4):815–822 [[Dormann_et+al_CellautonomousDefenseReorganizationTraffickingMembranes_2014]] [OA](https://api.scholarcy.com/oa_version?query=Dormann%2C%20P.%20Kim%2C%20H.%20Ott%2C%20T.%20Schulze-Lefert%2C%20P.%20Cell-autonomous%20defense%2C%20reorganization%20and%20trafficking%20of%20membranes%20in%20plant-microbe%20interactions%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Dormann%2C%20P.%20Kim%2C%20H.%20Ott%2C%20T.%20Schulze-Lefert%2C%20P.%20Cell-autonomous%20defense%2C%20reorganization%20and%20trafficking%20of%20membranes%20in%20plant-microbe%20interactions%202014) [Scite](https://api.scholarcy.com/scite_url?query=Dormann%2C%20P.%20Kim%2C%20H.%20Ott%2C%20T.%20Schulze-Lefert%2C%20P.%20Cell-autonomous%20defense%2C%20reorganization%20and%20trafficking%20of%20membranes%20in%20plant-microbe%20interactions%202014)

[^Douds_et+al_2000_a]: Douds DD, Pfeffer PE, Shachar-Hill Y (2000) Carbon partitioning, cost and metabolism of arbuscular mycorrhizae in arbuscular mycorrhizas: physiology and function. In: Kapulnick Y, Douds DD Jr (eds) Arbuscular mycorrhizas: molecular biology and physiology. Kluwer Academic, Dordrecht [[Douds_et+al_CarbonPartitioningCostMetabolismArbuscular_2000]] [OA](https://api.scholarcy.com/oa_version?query=Douds%2C%20D.D.%20Pfeffer%2C%20P.E.%20Shachar-Hill%2C%20Y.%20Carbon%20partitioning%2C%20cost%20and%20metabolism%20of%20arbuscular%20mycorrhizae%20in%20arbuscular%20mycorrhizas%3A%20physiology%20and%20function%202000) [GScholar](https://scholar.google.co.uk/scholar?q=Douds%2C%20D.D.%20Pfeffer%2C%20P.E.%20Shachar-Hill%2C%20Y.%20Carbon%20partitioning%2C%20cost%20and%20metabolism%20of%20arbuscular%20mycorrhizae%20in%20arbuscular%20mycorrhizas%3A%20physiology%20and%20function%202000) [Scite](https://api.scholarcy.com/scite_url?query=Douds%2C%20D.D.%20Pfeffer%2C%20P.E.%20Shachar-Hill%2C%20Y.%20Carbon%20partitioning%2C%20cost%20and%20metabolism%20of%20arbuscular%20mycorrhizae%20in%20arbuscular%20mycorrhizas%3A%20physiology%20and%20function%202000)

[^Fassi_et+al_1969_a]: Fassi B, Fontana A, Trappe JM (1969) Ectomycorrhizae formed by Endogone lactiflua with species of Pinus and Pseudotsuga. Mycologia 61:412–414 [[Fassi_et+al_EctomycorrhizaeFormedEndogoneLactifluaWith_1969]] [OA](https://api.scholarcy.com/oa_version?query=Fassi%2C%20B.%20Fontana%2C%20A.%20Trappe%2C%20J.M.%20Ectomycorrhizae%20formed%20by%20Endogone%20lactiflua%20with%20species%20of%20Pinus%20and%20Pseudotsuga%201969) [GScholar](https://scholar.google.co.uk/scholar?q=Fassi%2C%20B.%20Fontana%2C%20A.%20Trappe%2C%20J.M.%20Ectomycorrhizae%20formed%20by%20Endogone%20lactiflua%20with%20species%20of%20Pinus%20and%20Pseudotsuga%201969) [Scite](https://api.scholarcy.com/scite_url?query=Fassi%2C%20B.%20Fontana%2C%20A.%20Trappe%2C%20J.M.%20Ectomycorrhizae%20formed%20by%20Endogone%20lactiflua%20with%20species%20of%20Pinus%20and%20Pseudotsuga%201969)

[^Favre_et+al_2014_a]: Favre P, Bapaume L, Bossolini E, Delorenzi M, Falquet L, Reinhardt D (2014) A novel bioinformatics pipeline to discover genes related to arbuscular mycorrhizal symbiosis based on their evolutionary conservation pattern among higher plants. BMC Plant Biol 14:333 [[Favre_et+al_NovelBioinformaticsPipelineDiscoverGenes_2014]] [OA](https://api.scholarcy.com/oa_version?query=Favre%2C%20P.%20Bapaume%2C%20L.%20Bossolini%2C%20E.%20Delorenzi%2C%20M.%20A%20novel%20bioinformatics%20pipeline%20to%20discover%20genes%20related%20to%20arbuscular%20mycorrhizal%20symbiosis%20based%20on%20their%20evolutionary%20conservation%20pattern%20among%20higher%20plants%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Favre%2C%20P.%20Bapaume%2C%20L.%20Bossolini%2C%20E.%20Delorenzi%2C%20M.%20A%20novel%20bioinformatics%20pipeline%20to%20discover%20genes%20related%20to%20arbuscular%20mycorrhizal%20symbiosis%20based%20on%20their%20evolutionary%20conservation%20pattern%20among%20higher%20plants%202014) [Scite](https://api.scholarcy.com/scite_url?query=Favre%2C%20P.%20Bapaume%2C%20L.%20Bossolini%2C%20E.%20Delorenzi%2C%20M.%20A%20novel%20bioinformatics%20pipeline%20to%20discover%20genes%20related%20to%20arbuscular%20mycorrhizal%20symbiosis%20based%20on%20their%20evolutionary%20conservation%20pattern%20among%20higher%20plants%202014)

[^Feddermann_et+al_2008_a]: Feddermann N, Boller T, Salzer P, Elfstrand S, Wiemken A, Elfstrand M (2008) Medicago truncatula shows distinct patterns of mycorrhiza-related gene expression after inoculation with three different arbuscular mycorrhizal fungi. Planta 227:671– 680 [[Feddermann_et+al_MedicagoTruncatulaShowsDistinctPatterns_2008]] [OA](https://api.scholarcy.com/oa_version?query=Feddermann%2C%20N.%20Boller%2C%20T.%20Salzer%2C%20P.%20Elfstrand%2C%20S.%20Medicago%20truncatula%20shows%20distinct%20patterns%20of%20mycorrhiza-related%20gene%20expression%20after%20inoculation%20with%20three%20different%20arbuscular%20mycorrhizal%20fungi%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Feddermann%2C%20N.%20Boller%2C%20T.%20Salzer%2C%20P.%20Elfstrand%2C%20S.%20Medicago%20truncatula%20shows%20distinct%20patterns%20of%20mycorrhiza-related%20gene%20expression%20after%20inoculation%20with%20three%20different%20arbuscular%20mycorrhizal%20fungi%202008) [Scite](https://api.scholarcy.com/scite_url?query=Feddermann%2C%20N.%20Boller%2C%20T.%20Salzer%2C%20P.%20Elfstrand%2C%20S.%20Medicago%20truncatula%20shows%20distinct%20patterns%20of%20mycorrhiza-related%20gene%20expression%20after%20inoculation%20with%20three%20different%20arbuscular%20mycorrhizal%20fungi%202008)

[^Feddermann_et+al_2010_a]: Feddermann N, Muni RRD, Zeier T, Stuurman J, Ercolin F, Schorderet M, Reinhardt D (2010) The PAM1 gene of petunia, required for intracellular accommodation and morphogenesis of arbuscular mycorrhizal fungi, encodes a homologue of VAPYRIN. Plant J 64:470–481 [[Feddermann_et+al_Pam1GenePetuniaRequiredIntracellular_2010]] [OA](https://api.scholarcy.com/oa_version?query=Feddermann%2C%20N.%20Muni%2C%20R.R.D.%20Zeier%2C%20T.%20Stuurman%2C%20J.%20The%20PAM1%20gene%20of%20petunia%2C%20required%20for%20intracellular%20accommodation%20and%20morphogenesis%20of%20arbuscular%20mycorrhizal%20fungi%2C%20encodes%20a%20homologue%20of%20VAPYRIN%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Feddermann%2C%20N.%20Muni%2C%20R.R.D.%20Zeier%2C%20T.%20Stuurman%2C%20J.%20The%20PAM1%20gene%20of%20petunia%2C%20required%20for%20intracellular%20accommodation%20and%20morphogenesis%20of%20arbuscular%20mycorrhizal%20fungi%2C%20encodes%20a%20homologue%20of%20VAPYRIN%202010) [Scite](https://api.scholarcy.com/scite_url?query=Feddermann%2C%20N.%20Muni%2C%20R.R.D.%20Zeier%2C%20T.%20Stuurman%2C%20J.%20The%20PAM1%20gene%20of%20petunia%2C%20required%20for%20intracellular%20accommodation%20and%20morphogenesis%20of%20arbuscular%20mycorrhizal%20fungi%2C%20encodes%20a%20homologue%20of%20VAPYRIN%202010)

[^Ferlian_et+al_2018_a]: Ferlian O, Eisenhauer N, Aguirrebengoa M, Camara M, Ramirez-Rojas I, Santos F, Thakur MP (2018) Invasive earthworms erode soil biodiversity: a meta-analysis. J Anim Ecol 87:162–172 [[Ferlian_et+al_InvasiveEarthwormsErodeSoilBiodiversity_2018]] [OA](https://api.scholarcy.com/oa_version?query=Ferlian%2C%20O.%20Eisenhauer%2C%20N.%20Aguirrebengoa%2C%20M.%20Camara%2C%20M.%20Invasive%20earthworms%20erode%20soil%20biodiversity%3A%20a%20meta-analysis%202018) [GScholar](https://scholar.google.co.uk/scholar?q=Ferlian%2C%20O.%20Eisenhauer%2C%20N.%20Aguirrebengoa%2C%20M.%20Camara%2C%20M.%20Invasive%20earthworms%20erode%20soil%20biodiversity%3A%20a%20meta-analysis%202018) [Scite](https://api.scholarcy.com/scite_url?query=Ferlian%2C%20O.%20Eisenhauer%2C%20N.%20Aguirrebengoa%2C%20M.%20Camara%2C%20M.%20Invasive%20earthworms%20erode%20soil%20biodiversity%3A%20a%20meta-analysis%202018)

[^Fiorilli_et+al_2009_a]: Fiorilli V, Catoni M, Miozzi L, Novero M, Accotto GP, Lanfranco L (2009) Global and cell-type gene expression profiles in tomato plants colonized by an arbuscular mycorrhizal fungus. New Phytol 184:975–987 [[Fiorilli_et+al_GlobalCelltypeGeneExpressionProfiles_2009]] [OA](https://api.scholarcy.com/oa_version?query=Fiorilli%2C%20V.%20Catoni%2C%20M.%20Miozzi%2C%20L.%20Novero%2C%20M.%20Global%20and%20cell-type%20gene%20expression%20profiles%20in%20tomato%20plants%20colonized%20by%20an%20arbuscular%20mycorrhizal%20fungus%202009) [GScholar](https://scholar.google.co.uk/scholar?q=Fiorilli%2C%20V.%20Catoni%2C%20M.%20Miozzi%2C%20L.%20Novero%2C%20M.%20Global%20and%20cell-type%20gene%20expression%20profiles%20in%20tomato%20plants%20colonized%20by%20an%20arbuscular%20mycorrhizal%20fungus%202009) [Scite](https://api.scholarcy.com/scite_url?query=Fiorilli%2C%20V.%20Catoni%2C%20M.%20Miozzi%2C%20L.%20Novero%2C%20M.%20Global%20and%20cell-type%20gene%20expression%20profiles%20in%20tomato%20plants%20colonized%20by%20an%20arbuscular%20mycorrhizal%20fungus%202009)

[^Genre_et+al_2012_a]: Genre A, Ivanov S, Fendrych M, Faccio A, Zarsky V, Bisseling T, Bonfante P (2012) Multiple exocytotic markers accumulate at the sites of perifungal membrane biogenesis in arbuscular mycorrhizas. Plant Cell Physiol 53(1):244–255 [[Genre_et+al_MultipleExocytoticMarkersAccumulateSites_2012]] [OA](https://api.scholarcy.com/oa_version?query=Genre%2C%20A.%20Ivanov%2C%20S.%20Fendrych%2C%20M.%20Faccio%2C%20A.%20Multiple%20exocytotic%20markers%20accumulate%20at%20the%20sites%20of%20perifungal%20membrane%20biogenesis%20in%20arbuscular%20mycorrhizas%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Genre%2C%20A.%20Ivanov%2C%20S.%20Fendrych%2C%20M.%20Faccio%2C%20A.%20Multiple%20exocytotic%20markers%20accumulate%20at%20the%20sites%20of%20perifungal%20membrane%20biogenesis%20in%20arbuscular%20mycorrhizas%202012) [Scite](https://api.scholarcy.com/scite_url?query=Genre%2C%20A.%20Ivanov%2C%20S.%20Fendrych%2C%20M.%20Faccio%2C%20A.%20Multiple%20exocytotic%20markers%20accumulate%20at%20the%20sites%20of%20perifungal%20membrane%20biogenesis%20in%20arbuscular%20mycorrhizas%202012)

[^Genre_et+al_2013_a]: Genre A, Chabaud M, Balzergue C, Puech-Pages V, Novero M, Rey T, Fournier J, Rochange S, Becard G, Bonfante P, Barker DG (2013) Short-chain chitin oligomers from arbuscular mycorrhizal fungi trigger nuclear Ca2+ spiking in Medicago truncatula roots and their production is enhanced by strigolactone. New Phytol 198:190–202 [[Genre_et+al_ShortchainChitinOligomersFromArbuscular_2013]] [OA](https://api.scholarcy.com/oa_version?query=Genre%2C%20A.%20Chabaud%2C%20M.%20Balzergue%2C%20C.%20Puech-Pages%2C%20V.%20Short-chain%20chitin%20oligomers%20from%20arbuscular%20mycorrhizal%20fungi%20trigger%20nuclear%20Ca2%2B%20spiking%20in%20Medicago%20truncatula%20roots%20and%20their%20production%20is%20enhanced%20by%20strigolactone%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Genre%2C%20A.%20Chabaud%2C%20M.%20Balzergue%2C%20C.%20Puech-Pages%2C%20V.%20Short-chain%20chitin%20oligomers%20from%20arbuscular%20mycorrhizal%20fungi%20trigger%20nuclear%20Ca2%2B%20spiking%20in%20Medicago%20truncatula%20roots%20and%20their%20production%20is%20enhanced%20by%20strigolactone%202013) [Scite](https://api.scholarcy.com/scite_url?query=Genre%2C%20A.%20Chabaud%2C%20M.%20Balzergue%2C%20C.%20Puech-Pages%2C%20V.%20Short-chain%20chitin%20oligomers%20from%20arbuscular%20mycorrhizal%20fungi%20trigger%20nuclear%20Ca2%2B%20spiking%20in%20Medicago%20truncatula%20roots%20and%20their%20production%20is%20enhanced%20by%20strigolactone%202013)

[^Ghignone_et+al_2012_a]: Ghignone S, Salvioli A, Anca I, Lumini E, Ortu G, Petiti L, Cruveiller S, Bianciotto V, Piffanelli P, Lanfranco L, Bonfante P (2012) The genome of the obligate endobacterium of an AM fungus reveals an interphylum network of nutritional interactions. ISME J 6:136–145 [[Ghignone_et+al_GenomeObligateEndobacteriumAmFungus_2012]] [OA](https://api.scholarcy.com/oa_version?query=Ghignone%2C%20S.%20Salvioli%2C%20A.%20Anca%2C%20I.%20Lumini%2C%20E.%20The%20genome%20of%20the%20obligate%20endobacterium%20of%20an%20AM%20fungus%20reveals%20an%20interphylum%20network%20of%20nutritional%20interactions%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Ghignone%2C%20S.%20Salvioli%2C%20A.%20Anca%2C%20I.%20Lumini%2C%20E.%20The%20genome%20of%20the%20obligate%20endobacterium%20of%20an%20AM%20fungus%20reveals%20an%20interphylum%20network%20of%20nutritional%20interactions%202012) [Scite](https://api.scholarcy.com/scite_url?query=Ghignone%2C%20S.%20Salvioli%2C%20A.%20Anca%2C%20I.%20Lumini%2C%20E.%20The%20genome%20of%20the%20obligate%20endobacterium%20of%20an%20AM%20fungus%20reveals%20an%20interphylum%20network%20of%20nutritional%20interactions%202012)

[^Gianinazzi-Pearson_1996_a]: Gianinazzi-Pearson V (1996) Plant cell responses to arbuscular mycorrhizal fungi: getting to the roots of the symbiosis. Plant Cell 8(10):1871–1883 [[Gianinazzi-Pearson_PlantCellResponsesArbuscularMycorrhizal_1996]] [OA](https://api.scholarcy.com/oa_version?query=Gianinazzi-Pearson%2C%20V.%20Plant%20cell%20responses%20to%20arbuscular%20mycorrhizal%20fungi%3A%20getting%20to%20the%20roots%20of%20the%20symbiosis%201996) [GScholar](https://scholar.google.co.uk/scholar?q=Gianinazzi-Pearson%2C%20V.%20Plant%20cell%20responses%20to%20arbuscular%20mycorrhizal%20fungi%3A%20getting%20to%20the%20roots%20of%20the%20symbiosis%201996) [Scite](https://api.scholarcy.com/scite_url?query=Gianinazzi-Pearson%2C%20V.%20Plant%20cell%20responses%20to%20arbuscular%20mycorrhizal%20fungi%3A%20getting%20to%20the%20roots%20of%20the%20symbiosis%201996)

[^Gianinazzi-Pearson_et+al_1981_a]: Gianinazzi-Pearson V, Morandi D, Dexheimer J, Gianinazzi S (1981) Ultrastructural and ultracytochemical features of a Glomus tenuis mycorrhiza. New Phytol 88:633–639 [[Gianinazzi-Pearson_et+al_UltrastructuralUltracytochemicalFeaturesGlomusTenuis_1981]] [OA](https://api.scholarcy.com/oa_version?query=Gianinazzi-Pearson%2C%20V.%20Morandi%2C%20D.%20Dexheimer%2C%20J.%20Gianinazzi%2C%20S.%20Ultrastructural%20and%20ultracytochemical%20features%20of%20a%20Glomus%20tenuis%20mycorrhiza%201981) [GScholar](https://scholar.google.co.uk/scholar?q=Gianinazzi-Pearson%2C%20V.%20Morandi%2C%20D.%20Dexheimer%2C%20J.%20Gianinazzi%2C%20S.%20Ultrastructural%20and%20ultracytochemical%20features%20of%20a%20Glomus%20tenuis%20mycorrhiza%201981) [Scite](https://api.scholarcy.com/scite_url?query=Gianinazzi-Pearson%2C%20V.%20Morandi%2C%20D.%20Dexheimer%2C%20J.%20Gianinazzi%2C%20S.%20Ultrastructural%20and%20ultracytochemical%20features%20of%20a%20Glomus%20tenuis%20mycorrhiza%201981)

[^Giovannetti_et+al_1993_a]: Giovannetti M, Ayio L, Sbrana C, Citernesi AS (1993) Factors affecting appressorium development in the vesicular-arbuscular mycorrhizal fungus Glomus mosseae (Nicol. and Gerd.) Gerd. And Trappe. New Phytol 123:115–122 [[Giovannetti_et+al_FactorsAffectingAppressoriumDevelopmentVesiculararbuscular_1993]] [OA](https://api.scholarcy.com/oa_version?query=Giovannetti%2C%20M.%20Ayio%2C%20L.%20Sbrana%2C%20C.%20Citernesi%2C%20A.S.%20Factors%20affecting%20appressorium%20development%20in%20the%20vesicular-arbuscular%20mycorrhizal%20fungus%20Glomus%20mosseae%20%28Nicol.%20and%20Gerd.%29%20Gerd%201993) [GScholar](https://scholar.google.co.uk/scholar?q=Giovannetti%2C%20M.%20Ayio%2C%20L.%20Sbrana%2C%20C.%20Citernesi%2C%20A.S.%20Factors%20affecting%20appressorium%20development%20in%20the%20vesicular-arbuscular%20mycorrhizal%20fungus%20Glomus%20mosseae%20%28Nicol.%20and%20Gerd.%29%20Gerd%201993) [Scite](https://api.scholarcy.com/scite_url?query=Giovannetti%2C%20M.%20Ayio%2C%20L.%20Sbrana%2C%20C.%20Citernesi%2C%20A.S.%20Factors%20affecting%20appressorium%20development%20in%20the%20vesicular-arbuscular%20mycorrhizal%20fungus%20Glomus%20mosseae%20%28Nicol.%20and%20Gerd.%29%20Gerd%201993)

[^Gobbato_2015_a]: Gobbato E (2015) Recent developments in arbuscular mycorrhizal signaling. Curr Opin Plant Biol 26:1–7 [[Gobbato_RecentDevelopmentsArbuscularMycorrhizalSignaling_2015]] [OA](https://api.scholarcy.com/oa_version?query=Gobbato%2C%20E.%20Recent%20developments%20in%20arbuscular%20mycorrhizal%20signaling%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Gobbato%2C%20E.%20Recent%20developments%20in%20arbuscular%20mycorrhizal%20signaling%202015) [Scite](https://api.scholarcy.com/scite_url?query=Gobbato%2C%20E.%20Recent%20developments%20in%20arbuscular%20mycorrhizal%20signaling%202015)

[^Gobbato_et+al_2012_a]: Gobbato E, Marsh JF, Vernie T, Wang E, Maillet F, Kim J, Miller JB, Sun J, Bano SA, Ratet P, Mysore KS, Denarie J, Schultze M, Oldroyd GED (2012) A GRAS-type transcription factor with a specific function in mycorrhizal signaling. Curr Biol 22:2236–2241 [[Gobbato_et+al_GrastypeTranscriptionFactorWithSpecific_2012]] [OA](https://api.scholarcy.com/oa_version?query=Gobbato%2C%20E.%20Marsh%2C%20J.F.%20Vernie%2C%20T.%20Wang%2C%20E.%20A%20GRAS-type%20transcription%20factor%20with%20a%20specific%20function%20in%20mycorrhizal%20signaling%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Gobbato%2C%20E.%20Marsh%2C%20J.F.%20Vernie%2C%20T.%20Wang%2C%20E.%20A%20GRAS-type%20transcription%20factor%20with%20a%20specific%20function%20in%20mycorrhizal%20signaling%202012) [Scite](https://api.scholarcy.com/scite_url?query=Gobbato%2C%20E.%20Marsh%2C%20J.F.%20Vernie%2C%20T.%20Wang%2C%20E.%20A%20GRAS-type%20transcription%20factor%20with%20a%20specific%20function%20in%20mycorrhizal%20signaling%202012)

[^Gobbato_et+al_2013_a]: Gobbato E, Wang E, Higgins G, Bano SA, Henry C, Schultze M, Oldroyd GED (2013) RAM1 and RAM2 function and expression during arbuscular mycorrhizal symbiosis and Aphanomyces euteiches colonization. Plant Signal Behav 8(10): pii:e26049. https://doi.org/10.4161/psb.26049 [[Gobbato_et+al_Ram1Ram2FunctionExpressionDuring_2013]] [OA](https://doi.org/10.4161/psb.26049)  [Scite](https://scite.ai/reports/10.4161/psb.26049)

[^Gomez-Roldan_et+al_2008_a]: Gomez-Roldan V, Fermas S, Brewer PB, Puech-Pages V, Dun EA, Pillot JP, Letisse F, Matusova R, Danoun S, Portais JC, Bouwmeester H, Becard G, Beveridge CA, Rameau C, Rochange SF (2008) Strigolactone inhibition of shoot branching. Nature 455 (7210):189–194 [[Gomez-Roldan_et+al_StrigolactoneInhibitionShootBranching_2008]] [OA](https://api.scholarcy.com/oa_version?query=Gomez-Roldan%2C%20V.%20Fermas%2C%20S.%20Brewer%2C%20P.B.%20Puech-Pages%2C%20V.%20Strigolactone%20inhibition%20of%20shoot%20branching%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Gomez-Roldan%2C%20V.%20Fermas%2C%20S.%20Brewer%2C%20P.B.%20Puech-Pages%2C%20V.%20Strigolactone%20inhibition%20of%20shoot%20branching%202008) [Scite](https://api.scholarcy.com/scite_url?query=Gomez-Roldan%2C%20V.%20Fermas%2C%20S.%20Brewer%2C%20P.B.%20Puech-Pages%2C%20V.%20Strigolactone%20inhibition%20of%20shoot%20branching%202008)

[^Groth_et+al_2010_a]: Groth M, Takeda N, Perry J, Uchida H, Draxl S, Brachmann A, Sato S, Tabata S, Kawaguchi M, Wang TL, Parniske M (2010) NENA, a Lotus japonicus homolog of Sec13, is required for rhizodermal infection by arbuscular mycorrhiza fungi and rhizobia but dispensable for cortical endosymbiotic development. Plant Cell 22(7):2509–2526 [[Groth_et+al_NenaLotusJaponicusHomologSec13_2010]] [OA](https://api.scholarcy.com/oa_version?query=Groth%2C%20M.%20Takeda%2C%20N.%20Perry%2C%20J.%20Uchida%2C%20H.%20NENA%2C%20a%20Lotus%20japonicus%20homolog%20of%20Sec13%2C%20is%20required%20for%20rhizodermal%20infection%20by%20arbuscular%20mycorrhiza%20fungi%20and%20rhizobia%20but%20dispensable%20for%20cortical%20endosymbiotic%20development%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Groth%2C%20M.%20Takeda%2C%20N.%20Perry%2C%20J.%20Uchida%2C%20H.%20NENA%2C%20a%20Lotus%20japonicus%20homolog%20of%20Sec13%2C%20is%20required%20for%20rhizodermal%20infection%20by%20arbuscular%20mycorrhiza%20fungi%20and%20rhizobia%20but%20dispensable%20for%20cortical%20endosymbiotic%20development%202010) [Scite](https://api.scholarcy.com/scite_url?query=Groth%2C%20M.%20Takeda%2C%20N.%20Perry%2C%20J.%20Uchida%2C%20H.%20NENA%2C%20a%20Lotus%20japonicus%20homolog%20of%20Sec13%2C%20is%20required%20for%20rhizodermal%20infection%20by%20arbuscular%20mycorrhiza%20fungi%20and%20rhizobia%20but%20dispensable%20for%20cortical%20endosymbiotic%20development%202010)

[^Gutjahr_2013_a]: Gutjahr C, Parniske M (2013) Cell and developmental biology of arbuscular mycorrhiza symbiosis. Anna Rev Cell Dev Biol 29:593–617 [[Gutjahr_CellDevelopmentalBiologyArbuscularMycorrhiza_2013]] [OA](https://api.scholarcy.com/oa_version?query=Gutjahr%2C%20C.%20Parniske%2C%20M.%20Cell%20and%20developmental%20biology%20of%20arbuscular%20mycorrhiza%20symbiosis%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Gutjahr%2C%20C.%20Parniske%2C%20M.%20Cell%20and%20developmental%20biology%20of%20arbuscular%20mycorrhiza%20symbiosis%202013) [Scite](https://api.scholarcy.com/scite_url?query=Gutjahr%2C%20C.%20Parniske%2C%20M.%20Cell%20and%20developmental%20biology%20of%20arbuscular%20mycorrhiza%20symbiosis%202013)

[^Gutjahr_et+al_2008_a]: Gutjahr C, Banba M, Croset V, An K, Miyao A, An G, Hirochika H, Imaizumi-Anraku H, Paszkowski U (2008) Arbuscular mycorrhiza-specific signaling in rice transcends the common symbiosis signaling pathway. Plant Cell 20:2989–3005 [[Gutjahr_et+al_ArbuscularMycorrhizaspecificSignalingRiceTranscends_2008]] [OA](https://scholar.google.co.uk/scholar?q=Gutjahr%2C%20C.%20Banba%2C%20M.%20Croset%2C%20V.%20An%2C%20K.%20Arbuscular%20mycorrhiza-specific%20signaling%20in%20rice%20transcends%20the%20common%20symbiosis%20signaling%20pathway.%20Plant%20Cell%2020%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Gutjahr%2C%20C.%20Banba%2C%20M.%20Croset%2C%20V.%20An%2C%20K.%20Arbuscular%20mycorrhiza-specific%20signaling%20in%20rice%20transcends%20the%20common%20symbiosis%20signaling%20pathway.%20Plant%20Cell%2020%202008) 

[^Gutjahr_et+al_2009_a]: Gutjahr C, Novero M, Guether M, Montanari O, Udvardi M, Bonfante P (2009) Presymbiotic factors released by the arbuscular mycorrhizal fungus Gigaspora margarita induce starch accumulation in Lotus japonicus roots. New Phytol 183(1):53–61 [[Gutjahr_et+al_PresymbioticFactorsReleasedArbuscularMycorrhizal_2009]] [OA](https://api.scholarcy.com/oa_version?query=Gutjahr%2C%20C.%20Novero%2C%20M.%20Guether%2C%20M.%20Montanari%2C%20O.%20Presymbiotic%20factors%20released%20by%20the%20arbuscular%20mycorrhizal%20fungus%20Gigaspora%20margarita%20induce%20starch%20accumulation%20in%20Lotus%20japonicus%20roots%202009) [GScholar](https://scholar.google.co.uk/scholar?q=Gutjahr%2C%20C.%20Novero%2C%20M.%20Guether%2C%20M.%20Montanari%2C%20O.%20Presymbiotic%20factors%20released%20by%20the%20arbuscular%20mycorrhizal%20fungus%20Gigaspora%20margarita%20induce%20starch%20accumulation%20in%20Lotus%20japonicus%20roots%202009) [Scite](https://api.scholarcy.com/scite_url?query=Gutjahr%2C%20C.%20Novero%2C%20M.%20Guether%2C%20M.%20Montanari%2C%20O.%20Presymbiotic%20factors%20released%20by%20the%20arbuscular%20mycorrhizal%20fungus%20Gigaspora%20margarita%20induce%20starch%20accumulation%20in%20Lotus%20japonicus%20roots%202009)

[^Gutjahr_et+al_2012_a]: Gutjahr C, Radovanovic D, Geoffroy J, Zhang Q, Siegler H, Chiapello M, Casieri L, An K, An G, Guiderdoni E, Kumar CS, Sundaresan V, Harrison MJ, Paszkowski U (2012) The half- size ABC transporters STR1 and STR2 are indispensable for mycorrhizal arbuscule formation in rice. Plant J 69:906–920 [[Gutjahr_et+al_HalfSizeAbcTransportersStr1_2012]] [OA](https://api.scholarcy.com/oa_version?query=Gutjahr%2C%20C.%20Radovanovic%2C%20D.%20Geoffroy%2C%20J.%20Zhang%2C%20Q.%20The%20half-%20size%20ABC%20transporters%20STR1%20and%20STR2%20are%20indispensable%20for%20mycorrhizal%20arbuscule%20formation%20in%20rice%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Gutjahr%2C%20C.%20Radovanovic%2C%20D.%20Geoffroy%2C%20J.%20Zhang%2C%20Q.%20The%20half-%20size%20ABC%20transporters%20STR1%20and%20STR2%20are%20indispensable%20for%20mycorrhizal%20arbuscule%20formation%20in%20rice%202012) [Scite](https://api.scholarcy.com/scite_url?query=Gutjahr%2C%20C.%20Radovanovic%2C%20D.%20Geoffroy%2C%20J.%20Zhang%2C%20Q.%20The%20half-%20size%20ABC%20transporters%20STR1%20and%20STR2%20are%20indispensable%20for%20mycorrhizal%20arbuscule%20formation%20in%20rice%202012)

[^Gutjahr_et+al_2015_a]: Gutjahr C, Gobbato E, Choi J, Riemann M, Johnston MG, Summers W, Carbonnel S, Mansfield C, Yang SY, Nadal M, Acosta I, Takano M, Jiao WB, Schneeberger K, Kelly KA, Paszkowski U (2015) Rice perception of symbiotic arbuscular mycorrhizal fungi requires the karrikin receptor complex. Science 350:1521–1524 [[Gutjahr_et+al_RicePerceptionSymbioticArbuscularMycorrhizal_2015]] [OA](https://api.scholarcy.com/oa_version?query=Gutjahr%2C%20C.%20Gobbato%2C%20E.%20Choi%2C%20J.%20Riemann%2C%20M.%20Rice%20perception%20of%20symbiotic%20arbuscular%20mycorrhizal%20fungi%20requires%20the%20karrikin%20receptor%20complex%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Gutjahr%2C%20C.%20Gobbato%2C%20E.%20Choi%2C%20J.%20Riemann%2C%20M.%20Rice%20perception%20of%20symbiotic%20arbuscular%20mycorrhizal%20fungi%20requires%20the%20karrikin%20receptor%20complex%202015) [Scite](https://api.scholarcy.com/scite_url?query=Gutjahr%2C%20C.%20Gobbato%2C%20E.%20Choi%2C%20J.%20Riemann%2C%20M.%20Rice%20perception%20of%20symbiotic%20arbuscular%20mycorrhizal%20fungi%20requires%20the%20karrikin%20receptor%20complex%202015)

[^Hamiaux_et+al_2012_a]: Hamiaux C, Drummond RS, Janssen BJ, Ledger SE, Cooney JM, Newcomb RD, Snowden KC (2012) DAD2 is an a/b hydrolase likely to be involved in the perception of the plant branching hormone strigolactone. Curr Biol 22:2032–2036 [[Hamiaux_et+al_Dad2HydrolaseLikelyInvolved_2012]] [OA](https://api.scholarcy.com/oa_version?query=Hamiaux%2C%20C.%20Drummond%2C%20R.S.%20Janssen%2C%20B.J.%20Ledger%2C%20S.E.%20DAD2%20is%20an%20a/b%20hydrolase%20likely%20to%20be%20involved%20in%20the%20perception%20of%20the%20plant%20branching%20hormone%20strigolactone%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Hamiaux%2C%20C.%20Drummond%2C%20R.S.%20Janssen%2C%20B.J.%20Ledger%2C%20S.E.%20DAD2%20is%20an%20a/b%20hydrolase%20likely%20to%20be%20involved%20in%20the%20perception%20of%20the%20plant%20branching%20hormone%20strigolactone%202012) [Scite](https://api.scholarcy.com/scite_url?query=Hamiaux%2C%20C.%20Drummond%2C%20R.S.%20Janssen%2C%20B.J.%20Ledger%2C%20S.E.%20DAD2%20is%20an%20a/b%20hydrolase%20likely%20to%20be%20involved%20in%20the%20perception%20of%20the%20plant%20branching%20hormone%20strigolactone%202012)

[^Handa_et+al_2015_a]: Handa Y, Nishide H, Takeda N, Suzuki Y, Kawaguchi M, Saito K (2015) RNA-seq transcriptional profiling of an arbuscular mycorrhiza provides insights into regulated and coordinated gene expression in Lotus japonicus and Rhizophagus irregularis. Plant Cell Physiol 56:1490–1511 [[Handa_et+al_RnaTranscriptionalProfilingArbuscularMycorrhiza_2015]] [OA](https://api.scholarcy.com/oa_version?query=Handa%2C%20Y.%20Nishide%2C%20H.%20Takeda%2C%20N.%20Suzuki%2C%20Y.%20RNA-seq%20transcriptional%20profiling%20of%20an%20arbuscular%20mycorrhiza%20provides%20insights%20into%20regulated%20and%20coordinated%20gene%20expression%20in%20Lotus%20japonicus%20and%20Rhizophagus%20irregularis%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Handa%2C%20Y.%20Nishide%2C%20H.%20Takeda%2C%20N.%20Suzuki%2C%20Y.%20RNA-seq%20transcriptional%20profiling%20of%20an%20arbuscular%20mycorrhiza%20provides%20insights%20into%20regulated%20and%20coordinated%20gene%20expression%20in%20Lotus%20japonicus%20and%20Rhizophagus%20irregularis%202015) [Scite](https://api.scholarcy.com/scite_url?query=Handa%2C%20Y.%20Nishide%2C%20H.%20Takeda%2C%20N.%20Suzuki%2C%20Y.%20RNA-seq%20transcriptional%20profiling%20of%20an%20arbuscular%20mycorrhiza%20provides%20insights%20into%20regulated%20and%20coordinated%20gene%20expression%20in%20Lotus%20japonicus%20and%20Rhizophagus%20irregularis%202015)

[^Harrison_1996_a]: Harrison MJ (1996) A sugar transporter from Medicago truncatula: altered expression pattern in roots during vesicular-arbuscular (VA) mycorrhizal associations. Plant J 9:491–503 [[Harrison_SugarTransporterFromMedicagoTruncatula_1996]] [OA](https://api.scholarcy.com/oa_version?query=Harrison%2C%20M.J.%20A%20sugar%20transporter%20from%20Medicago%20truncatula%3A%20altered%20expression%20pattern%20in%20roots%20during%20vesicular-arbuscular%20%28VA%29%20mycorrhizal%20associations%201996) [GScholar](https://scholar.google.co.uk/scholar?q=Harrison%2C%20M.J.%20A%20sugar%20transporter%20from%20Medicago%20truncatula%3A%20altered%20expression%20pattern%20in%20roots%20during%20vesicular-arbuscular%20%28VA%29%20mycorrhizal%20associations%201996) [Scite](https://api.scholarcy.com/scite_url?query=Harrison%2C%20M.J.%20A%20sugar%20transporter%20from%20Medicago%20truncatula%3A%20altered%20expression%20pattern%20in%20roots%20during%20vesicular-arbuscular%20%28VA%29%20mycorrhizal%20associations%201996)

[^Harrison_2012_a]: Harrison MJ (2012) Cellular programs for arbuscular mycorrhizal symbiosis. Curr Opin Plant Biol 15 (6):691–698 [[Harrison_CellularProgramsArbuscularMycorrhizalSymbiosis_2012]] [OA](https://scholar.google.co.uk/scholar?q=Harrison%2C%20M.J.%20Cellular%20programs%20for%20arbuscular%20mycorrhizal%20symbiosis.%20Curr%20Opin%20Plant%20Biol%2015%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Harrison%2C%20M.J.%20Cellular%20programs%20for%20arbuscular%20mycorrhizal%20symbiosis.%20Curr%20Opin%20Plant%20Biol%2015%202012) 

[^Hart_2002_a]: Hart MM, Reader RJ (2002) Host plant benefit from association with arbuscular mycorrhizal fungi: variation due to differences in size of mycelium. Biol Fertil Soils 36:357–366 [[Hart_HostPlantBenefitFromAssociation_2002]] [OA](https://api.scholarcy.com/oa_version?query=Hart%2C%20M.M.%20Reader%2C%20R.J.%20Host%20plant%20benefit%20from%20association%20with%20arbuscular%20mycorrhizal%20fungi%3A%20variation%20due%20to%20differences%20in%20size%20of%20mycelium%202002) [GScholar](https://scholar.google.co.uk/scholar?q=Hart%2C%20M.M.%20Reader%2C%20R.J.%20Host%20plant%20benefit%20from%20association%20with%20arbuscular%20mycorrhizal%20fungi%3A%20variation%20due%20to%20differences%20in%20size%20of%20mycelium%202002) [Scite](https://api.scholarcy.com/scite_url?query=Hart%2C%20M.M.%20Reader%2C%20R.J.%20Host%20plant%20benefit%20from%20association%20with%20arbuscular%20mycorrhizal%20fungi%3A%20variation%20due%20to%20differences%20in%20size%20of%20mycelium%202002)

[^Havaux_2014_a]: Havaux M (2014) Carotenoid oxidation products as stress signals in plants. Plant J 79:597–606 [[Havaux_CarotenoidOxidationProductsStressSignals_2014]] [OA](https://api.scholarcy.com/oa_version?query=Havaux%2C%20M.%20Carotenoid%20oxidation%20products%20as%20stress%20signals%20in%20plants%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Havaux%2C%20M.%20Carotenoid%20oxidation%20products%20as%20stress%20signals%20in%20plants%202014) [Scite](https://api.scholarcy.com/scite_url?query=Havaux%2C%20M.%20Carotenoid%20oxidation%20products%20as%20stress%20signals%20in%20plants%202014)

[^Heard_et+al_2015_a]: Heard S, Brown NA, Hammond-Kosack K (2015) An interspecies comparative analysis of the predicted secretomes of the necrotrophic plant pathogens Sclerotinia sclerotiorum and Botrytis cinerea. PLoS One 10:e0130534. https://doi.org/10.1371/journal.pone.0130534 [[Heard_et+al_InterspeciesComparativeAnalysisPredictedSecretomes_2015]] [OA](https://doi.org/10.1371/journal.pone.0130534)  [Scite](https://scite.ai/reports/10.1371/journal.pone.0130534)

[^Ivanov_et+al_2012_a]: Ivanov S, Fedorova EE, Limpens E, De Mita S, Genre A, Bonfante P, Bisseling T (2012) Rhizobium-legume symbiosis shares an exocytotic pathway required for arbuscule formation. Proc Natl Acad Sci U S A 109(21):8316–8321 [[Ivanov_et+al_RhizobiumlegumeSymbiosisSharesExocytoticPathway_2012]] [OA](https://api.scholarcy.com/oa_version?query=Ivanov%2C%20S.%20Fedorova%2C%20E.E.%20Limpens%2C%20E.%20De%20Mita%2C%20S.%20Rhizobium-legume%20symbiosis%20shares%20an%20exocytotic%20pathway%20required%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Ivanov%2C%20S.%20Fedorova%2C%20E.E.%20Limpens%2C%20E.%20De%20Mita%2C%20S.%20Rhizobium-legume%20symbiosis%20shares%20an%20exocytotic%20pathway%20required%202012) [Scite](https://api.scholarcy.com/scite_url?query=Ivanov%2C%20S.%20Fedorova%2C%20E.E.%20Limpens%2C%20E.%20De%20Mita%2C%20S.%20Rhizobium-legume%20symbiosis%20shares%20an%20exocytotic%20pathway%20required%202012)

[^Jashni_et+al_2015_a]: Jashni MK, Mehrabi R, Collemare J, Mesarich CH, de Wit PJ (2015) The battle in the apoplast: further insights into the roles of proteases and their inhibitors in plant-pathogen interactions. Front Plant Sci 6:584 [[Jashni_et+al_BattleApoplastFurtherInsightsInto_2015]] [OA](https://api.scholarcy.com/oa_version?query=Jashni%2C%20M.K.%20Mehrabi%2C%20R.%20Collemare%2C%20J.%20Mesarich%2C%20C.H.%20The%20battle%20in%20the%20apoplast%3A%20further%20insights%20into%20the%20roles%20of%20proteases%20and%20their%20inhibitors%20in%20plant-pathogen%20interactions%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Jashni%2C%20M.K.%20Mehrabi%2C%20R.%20Collemare%2C%20J.%20Mesarich%2C%20C.H.%20The%20battle%20in%20the%20apoplast%3A%20further%20insights%20into%20the%20roles%20of%20proteases%20and%20their%20inhibitors%20in%20plant-pathogen%20interactions%202015) [Scite](https://api.scholarcy.com/scite_url?query=Jashni%2C%20M.K.%20Mehrabi%2C%20R.%20Collemare%2C%20J.%20Mesarich%2C%20C.H.%20The%20battle%20in%20the%20apoplast%3A%20further%20insights%20into%20the%20roles%20of%20proteases%20and%20their%20inhibitors%20in%20plant-pathogen%20interactions%202015)

[^Javot_et+al_2007_a]: Javot H, Penmetsa RV, Terzaghi N, Cook DR, Harrison MJ (2007) A Medicago truncatula phosphate transporter indispensable for the arbuscular mycorrhizal symbiosis. Proc Natl Acad Sci U S A 104(5):1720–1725 [[Javot_et+al_MedicagoTruncatulaPhosphateTransporterIndispensable_2007]] [OA](https://api.scholarcy.com/oa_version?query=Javot%2C%20H.%20Penmetsa%2C%20R.V.%20Terzaghi%2C%20N.%20Cook%2C%20D.R.%20A%20Medicago%20truncatula%20phosphate%20transporter%20indispensable%20for%20the%202007) [GScholar](https://scholar.google.co.uk/scholar?q=Javot%2C%20H.%20Penmetsa%2C%20R.V.%20Terzaghi%2C%20N.%20Cook%2C%20D.R.%20A%20Medicago%20truncatula%20phosphate%20transporter%20indispensable%20for%20the%202007) [Scite](https://api.scholarcy.com/scite_url?query=Javot%2C%20H.%20Penmetsa%2C%20R.V.%20Terzaghi%2C%20N.%20Cook%2C%20D.R.%20A%20Medicago%20truncatula%20phosphate%20transporter%20indispensable%20for%20the%202007)

[^Javot_et+al_2011_a]: Javot H, Penmetsa RV, Breuillin F, Bhattarai KK, Noar RD, Gomez SK, Zhang Q, Cook DR, Harrison MJ (2011) Medicago truncatula mtpt4 mutants reveal a role for nitrogen in the regulation of arbuscule degeneration in arbuscular mycorrhizal symbiosis. Plant J 68:954–965 [[Javot_et+al_MedicagoTruncatulaMtpt4MutantsReveal_2011]] [OA](https://api.scholarcy.com/oa_version?query=Javot%2C%20H.%20Penmetsa%2C%20R.V.%20Breuillin%2C%20F.%20Bhattarai%2C%20K.K.%20Medicago%20truncatula%20mtpt4%20mutants%20reveal%20a%20role%20for%20nitrogen%20in%20the%20regulation%20of%20arbuscule%20degeneration%20in%20arbuscular%20mycorrhizal%20symbiosis%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Javot%2C%20H.%20Penmetsa%2C%20R.V.%20Breuillin%2C%20F.%20Bhattarai%2C%20K.K.%20Medicago%20truncatula%20mtpt4%20mutants%20reveal%20a%20role%20for%20nitrogen%20in%20the%20regulation%20of%20arbuscule%20degeneration%20in%20arbuscular%20mycorrhizal%20symbiosis%202011) [Scite](https://api.scholarcy.com/scite_url?query=Javot%2C%20H.%20Penmetsa%2C%20R.V.%20Breuillin%2C%20F.%20Bhattarai%2C%20K.K.%20Medicago%20truncatula%20mtpt4%20mutants%20reveal%20a%20role%20for%20nitrogen%20in%20the%20regulation%20of%20arbuscule%20degeneration%20in%20arbuscular%20mycorrhizal%20symbiosis%202011)

[^Jiang_et+al_2017_a]: Jiang Y, Wang W, Xie O, Liu N, Liu L, Wang D, Zhang X, Yang C, Chen X, Tang D, Wang E (2017) Plants transfer lipids to sustain colonization by mutualistic mycorrhizal and parasitic fungi. Science 356:1172–1175 [[Jiang_et+al_PlantsTransferLipidsSustainColonization_2017]] [OA](https://api.scholarcy.com/oa_version?query=Jiang%2C%20Y.%20Wang%2C%20W.%20Xie%2C%20O.%20Liu%2C%20N.%20Plants%20transfer%20lipids%20to%20sustain%20colonization%20by%20mutualistic%20mycorrhizal%20and%20parasitic%20fungi%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Jiang%2C%20Y.%20Wang%2C%20W.%20Xie%2C%20O.%20Liu%2C%20N.%20Plants%20transfer%20lipids%20to%20sustain%20colonization%20by%20mutualistic%20mycorrhizal%20and%20parasitic%20fungi%202017) [Scite](https://api.scholarcy.com/scite_url?query=Jiang%2C%20Y.%20Wang%2C%20W.%20Xie%2C%20O.%20Liu%2C%20N.%20Plants%20transfer%20lipids%20to%20sustain%20colonization%20by%20mutualistic%20mycorrhizal%20and%20parasitic%20fungi%202017)

[^Jin_et+al_2016_a]: Jin Y, Liu H, Luo DX, Yu N, Dong WT, Wang C, Zhang XW, Dai HL, Yang J, Wang ET (2016) DELLA proteins are common components of symbiotic rhizobial and mycorrhizal signalling pathways. Nat Commun 7:12433 [[Jin_et+al_DellaProteinsCommonComponentsSymbiotic_2016]] [OA](https://api.scholarcy.com/oa_version?query=Jin%2C%20Y.%20Liu%2C%20H.%20Luo%2C%20D.X.%20Yu%2C%20N.%20DELLA%20proteins%20are%20common%20components%20of%20symbiotic%20rhizobial%20and%20mycorrhizal%20signalling%20pathways%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Jin%2C%20Y.%20Liu%2C%20H.%20Luo%2C%20D.X.%20Yu%2C%20N.%20DELLA%20proteins%20are%20common%20components%20of%20symbiotic%20rhizobial%20and%20mycorrhizal%20signalling%20pathways%202016) [Scite](https://api.scholarcy.com/scite_url?query=Jin%2C%20Y.%20Liu%2C%20H.%20Luo%2C%20D.X.%20Yu%2C%20N.%20DELLA%20proteins%20are%20common%20components%20of%20symbiotic%20rhizobial%20and%20mycorrhizal%20signalling%20pathways%202016)

[^Johnson_et+al_2013_a]: Johnson LJ, Koulman A, Christensen M, Lane GA, Fraser K, Forester N, Johnson RD, Bryan GT, Rasmussen S (2013) An extracellular siderophore is required to maintain the mutualistic interaction of Epichloefestucae with Lolium perenne. PLoS Pathog 9(5):e1003332 [[Johnson_et+al_ExtracellularSiderophoreRequiredMaintainMutualistic_2013]] [OA](https://api.scholarcy.com/oa_version?query=Johnson%2C%20L.J.%20Koulman%2C%20A.%20Christensen%2C%20M.%20Lane%2C%20G.A.%20An%20extracellular%20siderophore%20is%20required%20to%20maintain%20the%20mutualistic%20interaction%20of%20Epichloefestucae%20with%20Lolium%20perenne%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Johnson%2C%20L.J.%20Koulman%2C%20A.%20Christensen%2C%20M.%20Lane%2C%20G.A.%20An%20extracellular%20siderophore%20is%20required%20to%20maintain%20the%20mutualistic%20interaction%20of%20Epichloefestucae%20with%20Lolium%20perenne%202013) [Scite](https://api.scholarcy.com/scite_url?query=Johnson%2C%20L.J.%20Koulman%2C%20A.%20Christensen%2C%20M.%20Lane%2C%20G.A.%20An%20extracellular%20siderophore%20is%20required%20to%20maintain%20the%20mutualistic%20interaction%20of%20Epichloefestucae%20with%20Lolium%20perenne%202013)

[^Journet_et+al_2001_a]: Journet EP, El-Gachtouli N, Vernoud V, de Billy F, Pichon M, Dedieu A, Arnould C, Morandi D, Barker DG, Gianinazzi-Pearson V (2001) Medicago truncatula ENOD11: a novel RPRP-encoding early nodulin gene expressed during mycorrhization in arbuscule-containing cells. Mol Plant-Microbe Interact 14(6):737–748 [[Journet_et+al_MedicagoTruncatulaEnod11NovelRprpencoding_2001]] [OA](https://api.scholarcy.com/oa_version?query=Journet%2C%20E.P.%20El-Gachtouli%2C%20N.%20Vernoud%2C%20V.%20de%20Billy%2C%20F.%20Medicago%20truncatula%20ENOD11%3A%20a%20novel%20RPRP-encoding%20early%20nodulin%20gene%20expressed%20during%20mycorrhization%20in%20arbuscule-containing%20cells%202001) [GScholar](https://scholar.google.co.uk/scholar?q=Journet%2C%20E.P.%20El-Gachtouli%2C%20N.%20Vernoud%2C%20V.%20de%20Billy%2C%20F.%20Medicago%20truncatula%20ENOD11%3A%20a%20novel%20RPRP-encoding%20early%20nodulin%20gene%20expressed%20during%20mycorrhization%20in%20arbuscule-containing%20cells%202001) [Scite](https://api.scholarcy.com/scite_url?query=Journet%2C%20E.P.%20El-Gachtouli%2C%20N.%20Vernoud%2C%20V.%20de%20Billy%2C%20F.%20Medicago%20truncatula%20ENOD11%3A%20a%20novel%20RPRP-encoding%20early%20nodulin%20gene%20expressed%20during%20mycorrhization%20in%20arbuscule-containing%20cells%202001)

[^Jung_et+al_2012_a]: Jung SC, Martinez-Medina A, Lopez-Raez JA, Pozo MJ (2012) Mycorrhiza-induced resistance and priming of plant defenses. J Chem Ecol 38:651–664 [[Jung_et+al_MycorrhizainducedResistancePrimingPlantDefenses_2012]] [OA](https://api.scholarcy.com/oa_version?query=Jung%2C%20S.C.%20Martinez-Medina%2C%20A.%20Lopez-Raez%2C%20J.A.%20Pozo%2C%20M.J.%20Mycorrhiza-induced%20resistance%20and%20priming%20of%20plant%20defenses%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Jung%2C%20S.C.%20Martinez-Medina%2C%20A.%20Lopez-Raez%2C%20J.A.%20Pozo%2C%20M.J.%20Mycorrhiza-induced%20resistance%20and%20priming%20of%20plant%20defenses%202012) [Scite](https://api.scholarcy.com/scite_url?query=Jung%2C%20S.C.%20Martinez-Medina%2C%20A.%20Lopez-Raez%2C%20J.A.%20Pozo%2C%20M.J.%20Mycorrhiza-induced%20resistance%20and%20priming%20of%20plant%20defenses%202012)

[^Kaku_et+al_2006_a]: Kaku H, Nishizawa Y, Ishii-Minami N, AkimotoTomiyama C, Dohmae N, Takio K, Minami E, Shibuya N (2006) Plant cells recognize chitin fragments for defense signalling through a plasma membrane receptor. Proc Natl Acad Sci U S A 103(29):11086–11091 [[Kaku_et+al_PlantCellsRecognizeChitinFragments_2006]] [OA](https://api.scholarcy.com/oa_version?query=Kaku%2C%20H.%20Nishizawa%2C%20Y.%20Ishii-Minami%2C%20N.%20AkimotoTomiyama%2C%20C.%20Plant%20cells%20recognize%20chitin%20fragments%20for%20defense%20signalling%20through%20a%202006) [GScholar](https://scholar.google.co.uk/scholar?q=Kaku%2C%20H.%20Nishizawa%2C%20Y.%20Ishii-Minami%2C%20N.%20AkimotoTomiyama%2C%20C.%20Plant%20cells%20recognize%20chitin%20fragments%20for%20defense%20signalling%20through%20a%202006) [Scite](https://api.scholarcy.com/scite_url?query=Kaku%2C%20H.%20Nishizawa%2C%20Y.%20Ishii-Minami%2C%20N.%20AkimotoTomiyama%2C%20C.%20Plant%20cells%20recognize%20chitin%20fragments%20for%20defense%20signalling%20through%20a%202006)

[^Kalo_et+al_2005_a]: Kalo P, Gleason C, Edwards A, Marsh J, Mitra RM, Hirsch S, Jakab J, Sims S, Long SR, Rogers J, Kiss GB, Downie JA, Oldroyd GED (2005) Nodulation signalling in legumes requires NSP2, a member of the GRAS family of transcriptional regulators. Science 308(5729):1786–1789 [[Kalo_et+al_NodulationSignallingLegumesRequiresNsp2_2005]] [OA](https://api.scholarcy.com/oa_version?query=Kalo%2C%20P.%20Gleason%2C%20C.%20Edwards%2C%20A.%20Marsh%2C%20J.%20Nodulation%20signalling%20in%20legumes%20requires%20NSP2%2C%20a%20member%20of%20the%20GRAS%20family%20of%20transcriptional%20regulators%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Kalo%2C%20P.%20Gleason%2C%20C.%20Edwards%2C%20A.%20Marsh%2C%20J.%20Nodulation%20signalling%20in%20legumes%20requires%20NSP2%2C%20a%20member%20of%20the%20GRAS%20family%20of%20transcriptional%20regulators%202005) [Scite](https://api.scholarcy.com/scite_url?query=Kalo%2C%20P.%20Gleason%2C%20C.%20Edwards%2C%20A.%20Marsh%2C%20J.%20Nodulation%20signalling%20in%20legumes%20requires%20NSP2%2C%20a%20member%20of%20the%20GRAS%20family%20of%20transcriptional%20regulators%202005)

[^Kamel_et+al_2017_a]: Kamel L, Tang NW, Malbreil M, San Clemente H, Le Marquer M, Roux C, Frei Dit Frey N (2017) The comparison of expressed candidate secreted proteins from two arbuscular mycorrhizal fungi unravels common and specific molecular tools to invade different host plants. Front Plant Sci 8:124 [[Kamel_et+al_FreiDitFrey_2017]] [OA](https://api.scholarcy.com/oa_version?query=Kamel%2C%20L.%20Tang%2C%20N.W.%20Malbreil%2C%20M.%20San%20Clemente%2C%20H.%20Frei%20Dit%20Frey%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Kamel%2C%20L.%20Tang%2C%20N.W.%20Malbreil%2C%20M.%20San%20Clemente%2C%20H.%20Frei%20Dit%20Frey%202017) [Scite](https://api.scholarcy.com/scite_url?query=Kamel%2C%20L.%20Tang%2C%20N.W.%20Malbreil%2C%20M.%20San%20Clemente%2C%20H.%20Frei%20Dit%20Frey%202017)

[^Kanamori_et+al_2006_a]: Kanamori N, Madsen LH, Radutoiu S, Frantescu M, Quistgaard EM, Miwa H, Downie JA, James EK, Felle HH, Haaning LL, Jensen TH, Sato S, Nakamura Y, Tabata S, Sandal N, Stougaard J (2006) A nucleoporin is required for induction of Ca2+ spiking in legume nodule development and essential for rhizobial and fungal symbiosis. Proc Natl Acad Sci U S A 103:359–336 [[Kanamori_et+al_NucleoporinRequiredInductionCa2Spiking_2006]] [OA](https://api.scholarcy.com/oa_version?query=Kanamori%2C%20N.%20Madsen%2C%20L.H.%20Radutoiu%2C%20S.%20Frantescu%2C%20M.%20A%20nucleoporin%20is%20required%20for%20induction%20of%20Ca2%2B%20spiking%20in%20legume%20nodule%20development%20and%20essential%20for%20rhizobial%202006) [GScholar](https://scholar.google.co.uk/scholar?q=Kanamori%2C%20N.%20Madsen%2C%20L.H.%20Radutoiu%2C%20S.%20Frantescu%2C%20M.%20A%20nucleoporin%20is%20required%20for%20induction%20of%20Ca2%2B%20spiking%20in%20legume%20nodule%20development%20and%20essential%20for%20rhizobial%202006) [Scite](https://api.scholarcy.com/scite_url?query=Kanamori%2C%20N.%20Madsen%2C%20L.H.%20Radutoiu%2C%20S.%20Frantescu%2C%20M.%20A%20nucleoporin%20is%20required%20for%20induction%20of%20Ca2%2B%20spiking%20in%20legume%20nodule%20development%20and%20essential%20for%20rhizobial%202006)

[^Kapulnik_1991_a]: Kapulnik Y, Kushnir U (1991) Growth dependency of wild, primitive and modern cultivated wheat lines on vesicular-arbuscular mycorrhiza fungi. Euphytica 56:27–36 [[Kapulnik_GrowthDependencyWildPrimitiveModern_1991]] [OA](https://api.scholarcy.com/oa_version?query=Kapulnik%2C%20Y.%20Kushnir%2C%20U.%20Growth%20dependency%20of%20wild%2C%20primitive%20and%20modern%20cultivated%20wheat%20lines%20on%20vesicular-arbuscular%20mycorrhiza%20fungi%201991) [GScholar](https://scholar.google.co.uk/scholar?q=Kapulnik%2C%20Y.%20Kushnir%2C%20U.%20Growth%20dependency%20of%20wild%2C%20primitive%20and%20modern%20cultivated%20wheat%20lines%20on%20vesicular-arbuscular%20mycorrhiza%20fungi%201991) [Scite](https://api.scholarcy.com/scite_url?query=Kapulnik%2C%20Y.%20Kushnir%2C%20U.%20Growth%20dependency%20of%20wild%2C%20primitive%20and%20modern%20cultivated%20wheat%20lines%20on%20vesicular-arbuscular%20mycorrhiza%20fungi%201991)

[^Kevei_et+al_2007_a]: Kevei Z, Lougnon G, Mergaert P, Horvath GV, Kereszt A, Jayaraman D, Zaman N, Marcel F, Regulski K, Kiss GB, Kondorosi A, Endre G, Kondorosi E, Ane JM (2007) 3-hydroxy-3-methylglutaryl coenzyme a reductase 1 interacts with NORK and is crucial for nodulation in Medicago truncatula. Plant Cell 19 (12):3974–3989 [[Kevei_et+al_3hydroxy3methylglutarylCoenzymeReductase1Interacts_2007]] [OA](https://scholar.google.co.uk/scholar?q=Kevei%2C%20Z.%20Lougnon%2C%20G.%20Mergaert%2C%20P.%20Horvath%2C%20G.V.%203-hydroxy-3-methylglutaryl%20coenzyme%20a%20reductase%201%20interacts%20with%20NORK%20and%20is%20crucial%20for%20nodulation%20in%20Medicago%20truncatula%202007) [GScholar](https://scholar.google.co.uk/scholar?q=Kevei%2C%20Z.%20Lougnon%2C%20G.%20Mergaert%2C%20P.%20Horvath%2C%20G.V.%203-hydroxy-3-methylglutaryl%20coenzyme%20a%20reductase%201%20interacts%20with%20NORK%20and%20is%20crucial%20for%20nodulation%20in%20Medicago%20truncatula%202007) 

[^Keymer_et+al_2017_a]: Keymer A, Pimprikar P, Wewer V, Huber C, Brands M, Bucerius SL, Delaux PM, Klingl V, von RopenackLahaye E, Wang TL, Eisenreich W, Dormann P, Parniske M, Gutjahr C (2017) Lipid transfer from plants to arbuscular mycorrhiza fungi. elife 6: e29107 [[Keymer_et+al_LipidTransferFromPlantsArbuscular_2017]] [OA](https://scholar.google.co.uk/scholar?q=Keymer%2C%20A.%20Pimprikar%2C%20P.%20Wewer%2C%20V.%20Huber%2C%20C.%20Lipid%20transfer%20from%20plants%20to%20arbuscular%20mycorrhiza%20fungi%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Keymer%2C%20A.%20Pimprikar%2C%20P.%20Wewer%2C%20V.%20Huber%2C%20C.%20Lipid%20transfer%20from%20plants%20to%20arbuscular%20mycorrhiza%20fungi%202017) 

[^Kloppholz_et+al_2011_a]: Kloppholz S, Kuhn H, Requena N (2011) A secreted fungal effector of Glomus intraradices promotes symbiotic biotrophy. Curr Biol 21:1204–1209 [[Kloppholz_et+al_SecretedFungalEffectorGlomusIntraradices_2011]] [OA](https://api.scholarcy.com/oa_version?query=Kloppholz%2C%20S.%20Kuhn%2C%20H.%20Requena%2C%20N.%20A%20secreted%20fungal%20effector%20of%20Glomus%20intraradices%20promotes%20symbiotic%20biotrophy%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Kloppholz%2C%20S.%20Kuhn%2C%20H.%20Requena%2C%20N.%20A%20secreted%20fungal%20effector%20of%20Glomus%20intraradices%20promotes%20symbiotic%20biotrophy%202011) [Scite](https://api.scholarcy.com/scite_url?query=Kloppholz%2C%20S.%20Kuhn%2C%20H.%20Requena%2C%20N.%20A%20secreted%20fungal%20effector%20of%20Glomus%20intraradices%20promotes%20symbiotic%20biotrophy%202011)

[^Kobae_2010_a]: Kobae Y, Hata S (2010) Dynamics of periarbuscular membranes visualized with a fluorescent phosphate transporter in arbuscular mycorrhizal roots of rice. Plant Cell Physiol 51:341–353 [[Kobae_DynamicsPeriarbuscularMembranesVisualizedWith_2010]] [OA](https://api.scholarcy.com/oa_version?query=Kobae%2C%20Y.%20Hata%2C%20S.%20Dynamics%20of%20periarbuscular%20membranes%20visualized%20with%20a%20fluorescent%20phosphate%20transporter%20in%20arbuscular%20mycorrhizal%20roots%20of%20rice%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Kobae%2C%20Y.%20Hata%2C%20S.%20Dynamics%20of%20periarbuscular%20membranes%20visualized%20with%20a%20fluorescent%20phosphate%20transporter%20in%20arbuscular%20mycorrhizal%20roots%20of%20rice%202010) [Scite](https://api.scholarcy.com/scite_url?query=Kobae%2C%20Y.%20Hata%2C%20S.%20Dynamics%20of%20periarbuscular%20membranes%20visualized%20with%20a%20fluorescent%20phosphate%20transporter%20in%20arbuscular%20mycorrhizal%20roots%20of%20rice%202010)

[^Kobae_et+al_2010_b]: Kobae Y, Tamura Y, Takai S, Banba M, Hata S (2010) Localized expression of arbuscular mycorrhizainducible ammonium transporters in soybean. Plant Cell Physiol 51:1411–1415 [[Kobae_et+al_LocalizedExpressionArbuscularMycorrhizainducibleAmmonium_2010]] [OA](https://api.scholarcy.com/oa_version?query=Kobae%2C%20Y.%20Tamura%2C%20Y.%20Takai%2C%20S.%20Banba%2C%20M.%20Localized%20expression%20of%20arbuscular%20mycorrhizainducible%20ammonium%20transporters%20in%20soybean%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Kobae%2C%20Y.%20Tamura%2C%20Y.%20Takai%2C%20S.%20Banba%2C%20M.%20Localized%20expression%20of%20arbuscular%20mycorrhizainducible%20ammonium%20transporters%20in%20soybean%202010) [Scite](https://api.scholarcy.com/scite_url?query=Kobae%2C%20Y.%20Tamura%2C%20Y.%20Takai%2C%20S.%20Banba%2C%20M.%20Localized%20expression%20of%20arbuscular%20mycorrhizainducible%20ammonium%20transporters%20in%20soybean%202010)

[^Kobayashi_et+al_2018_a]: Kobayashi Y, Maeda T, Yamaguchi K, Kameoka H, Tanaka S, Ezawa T, Shigenobu S, Kawaguchi M (2018) The genome of Rhizophagus clarus HR1 reveals a common genetic basis for auxotrophy among arbuscular mycorrhizal fungi. BMC Genomics 19:465 [[Kobayashi_et+al_GenomeRhizophagusClarusHr1Reveals_2018]] [OA](https://api.scholarcy.com/oa_version?query=Kobayashi%2C%20Y.%20Maeda%2C%20T.%20Yamaguchi%2C%20K.%20Kameoka%2C%20H.%20The%20genome%20of%20Rhizophagus%20clarus%20HR1%20reveals%20a%20common%20genetic%20basis%20for%20auxotrophy%20among%20arbuscular%20mycorrhizal%20fungi%202018) [GScholar](https://scholar.google.co.uk/scholar?q=Kobayashi%2C%20Y.%20Maeda%2C%20T.%20Yamaguchi%2C%20K.%20Kameoka%2C%20H.%20The%20genome%20of%20Rhizophagus%20clarus%20HR1%20reveals%20a%20common%20genetic%20basis%20for%20auxotrophy%20among%20arbuscular%20mycorrhizal%20fungi%202018) [Scite](https://api.scholarcy.com/scite_url?query=Kobayashi%2C%20Y.%20Maeda%2C%20T.%20Yamaguchi%2C%20K.%20Kameoka%2C%20H.%20The%20genome%20of%20Rhizophagus%20clarus%20HR1%20reveals%20a%20common%20genetic%20basis%20for%20auxotrophy%20among%20arbuscular%20mycorrhizal%20fungi%202018)

[^Koch_et+al_2017_a]: Koch AM, Antunes PM, Maherali H, Hart MM, Klironomos JN (2017) Evolutionary asymmetry in the arbuscular mycorrhizal symbiosis; conservatism in the fungal morphology does not predict host plant growth. New Phytol 214:1330–1337 [[Koch_et+al_EvolutionaryAsymmetryArbuscularMycorrhizalSymbiosis_2017]] [OA](https://api.scholarcy.com/oa_version?query=Koch%2C%20A.M.%20Antunes%2C%20P.M.%20Maherali%2C%20H.%20Hart%2C%20M.M.%20Evolutionary%20asymmetry%20in%20the%20arbuscular%20mycorrhizal%20symbiosis%3B%20conservatism%20in%20the%20fungal%20morphology%20does%20not%20predict%20host%20plant%20growth%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Koch%2C%20A.M.%20Antunes%2C%20P.M.%20Maherali%2C%20H.%20Hart%2C%20M.M.%20Evolutionary%20asymmetry%20in%20the%20arbuscular%20mycorrhizal%20symbiosis%3B%20conservatism%20in%20the%20fungal%20morphology%20does%20not%20predict%20host%20plant%20growth%202017) [Scite](https://api.scholarcy.com/scite_url?query=Koch%2C%20A.M.%20Antunes%2C%20P.M.%20Maherali%2C%20H.%20Hart%2C%20M.M.%20Evolutionary%20asymmetry%20in%20the%20arbuscular%20mycorrhizal%20symbiosis%3B%20conservatism%20in%20the%20fungal%20morphology%20does%20not%20predict%20host%20plant%20growth%202017)

[^Wewer_et+al_2014_a]: Wewer V, Brands M, Dormann P (2014) Fatty acid synthesis and lipid metabolism in the obligate biotrophic fungus Rhizophagus irregularis during mycorrhization of Lotus japonicus. Plant J 79:398– 412 [[Wewer_et+al_FattyAcidSynthesisLipidMetabolism_2014]] [OA](https://api.scholarcy.com/oa_version?query=Wewer%2C%20V.%20Brands%2C%20M.%20Dormann%2C%20P.%20Fatty%20acid%20synthesis%20and%20lipid%20metabolism%20in%20the%20obligate%20biotrophic%20fungus%20Rhizophagus%20irregularis%20during%20mycorrhization%20of%20Lotus%20japonicus%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Wewer%2C%20V.%20Brands%2C%20M.%20Dormann%2C%20P.%20Fatty%20acid%20synthesis%20and%20lipid%20metabolism%20in%20the%20obligate%20biotrophic%20fungus%20Rhizophagus%20irregularis%20during%20mycorrhization%20of%20Lotus%20japonicus%202014) [Scite](https://api.scholarcy.com/scite_url?query=Wewer%2C%20V.%20Brands%2C%20M.%20Dormann%2C%20P.%20Fatty%20acid%20synthesis%20and%20lipid%20metabolism%20in%20the%20obligate%20biotrophic%20fungus%20Rhizophagus%20irregularis%20during%20mycorrhization%20of%20Lotus%20japonicus%202014)

[^White_et+al_1990_a]: White TJ, Bruns T, Lee S, Taylor J (1990) Amplification and direct sequencing of fungal ribosomal RNA genes for phylogenetics. In: Innis MA, Gelfand DH, Sninsky JJ, White TJ (eds) PCR protocols: a guide to methods and applications. Academic Press, New York, pp 315–322 [[White_et+al_AmplificationDirectSequencingFungalRibosomal_1990]] [OA](https://api.scholarcy.com/oa_version?query=White%2C%20T.J.%20Bruns%2C%20T.%20Lee%2C%20S.%20Taylor%2C%20J.%20Amplification%20and%20direct%20sequencing%20of%20fungal%20ribosomal%20RNA%20genes%20for%20phylogenetics%201990) [GScholar](https://scholar.google.co.uk/scholar?q=White%2C%20T.J.%20Bruns%2C%20T.%20Lee%2C%20S.%20Taylor%2C%20J.%20Amplification%20and%20direct%20sequencing%20of%20fungal%20ribosomal%20RNA%20genes%20for%20phylogenetics%201990) [Scite](https://api.scholarcy.com/scite_url?query=White%2C%20T.J.%20Bruns%2C%20T.%20Lee%2C%20S.%20Taylor%2C%20J.%20Amplification%20and%20direct%20sequencing%20of%20fungal%20ribosomal%20RNA%20genes%20for%20phylogenetics%201990)

[^Wu_et+al_2016_a]: Wu P, Wu Y, Liu C-C, Liu L-W, Ma F-F, Wu X-Y, Wu M, Hang Y-Y, Chen J-Q, Shao Z-Q, Wang B (2016) Identification of arbuscular mycorrhiza (AM)responsive microRNAs in tomato. Front Plant Sci 7:429 [[Wu_et+al_IdentificationArbuscularMycorrhizaamresponsiveMicrornas_2016]] [OA](https://api.scholarcy.com/oa_version?query=Wu%2C%20P.%20Wu%2C%20Y.%20Liu%2C%20C.-C.%20Liu%2C%20L.-W.%20Identification%20of%20arbuscular%20mycorrhiza%20%28AM%29responsive%20microRNAs%20in%20tomato%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Wu%2C%20P.%20Wu%2C%20Y.%20Liu%2C%20C.-C.%20Liu%2C%20L.-W.%20Identification%20of%20arbuscular%20mycorrhiza%20%28AM%29responsive%20microRNAs%20in%20tomato%202016) [Scite](https://api.scholarcy.com/scite_url?query=Wu%2C%20P.%20Wu%2C%20Y.%20Liu%2C%20C.-C.%20Liu%2C%20L.-W.%20Identification%20of%20arbuscular%20mycorrhiza%20%28AM%29responsive%20microRNAs%20in%20tomato%202016)

[^Xing_et+al_2012_a]: Xing X, Koch AM, Jones AMP, Ragone D, Murch S, Hart MM (2012) Mutualism breakdown in breadfruit domestication. Proc Biol Sci 279:1122–1130 [[Xing_et+al_MutualismBreakdown_2012]] [OA](https://api.scholarcy.com/oa_version?query=Xing%2C%20X.%20Koch%2C%20A.M.%20Jones%2C%20A.M.P.%20Ragone%2C%20D.%20Mutualism%20breakdown%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Xing%2C%20X.%20Koch%2C%20A.M.%20Jones%2C%20A.M.P.%20Ragone%2C%20D.%20Mutualism%20breakdown%202012) [Scite](https://api.scholarcy.com/scite_url?query=Xing%2C%20X.%20Koch%2C%20A.M.%20Jones%2C%20A.M.P.%20Ragone%2C%20D.%20Mutualism%20breakdown%202012)

[^Xue_et+al_2015_a]: Xue L, Cui H, Buer B, Vijayakumar V, Delaux P-M, Junkermann S, Bucher M (2015) Network of GRAS transcription factors involved in the control of arbuscule development in Lotus japonicus. Plant Physiol 167:854–871 [[Xue_et+al_NetworkGrasTranscriptionFactorsInvolved_2015]] [OA](https://api.scholarcy.com/oa_version?query=Xue%2C%20L.%20Cui%2C%20H.%20Buer%2C%20B.%20Vijayakumar%2C%20V.%20Network%20of%20GRAS%20transcription%20factors%20involved%20in%20the%20control%20of%20arbuscule%20development%20in%20Lotus%20japonicus%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Xue%2C%20L.%20Cui%2C%20H.%20Buer%2C%20B.%20Vijayakumar%2C%20V.%20Network%20of%20GRAS%20transcription%20factors%20involved%20in%20the%20control%20of%20arbuscule%20development%20in%20Lotus%20japonicus%202015) [Scite](https://api.scholarcy.com/scite_url?query=Xue%2C%20L.%20Cui%2C%20H.%20Buer%2C%20B.%20Vijayakumar%2C%20V.%20Network%20of%20GRAS%20transcription%20factors%20involved%20in%20the%20control%20of%20arbuscule%20development%20in%20Lotus%20japonicus%202015)

[^Yamamoto_et+al_2017_a]: Yamamoto K, Endo N, Degawa Y, Fukuda M, Yamada A (2017) First detection of Endogone ectomycorrhizas in natural oak forests. Mycorrhiza 27:295–301 [[Yamamoto_et+al_YamadaA_2017]] [OA](https://api.scholarcy.com/oa_version?query=Yamamoto%2C%20K.%20Endo%2C%20N.%20Degawa%2C%20Y.%20Fukuda%2C%20M.%20Yamada%20A%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Yamamoto%2C%20K.%20Endo%2C%20N.%20Degawa%2C%20Y.%20Fukuda%2C%20M.%20Yamada%20A%202017) [Scite](https://api.scholarcy.com/scite_url?query=Yamamoto%2C%20K.%20Endo%2C%20N.%20Degawa%2C%20Y.%20Fukuda%2C%20M.%20Yamada%20A%202017)

[^Yang_et+al_2012_a]: Yang S-Y, Grønlund M, Jakobsen I, Grotemeyer MS, Rentsch D, Miyao A, Hirochika H, Santhosh Kumar C, Sundaresan V, Salamin N, Catausan S, Mattes N, Heuer S, Paszkowski U (2012) Nonredundant regulation of rice arbuscular mycorrhizal symbiosis by two members of the phosphate transporter1 gene family. Plant Cell 24:4236–4251 [[Yang_et+al_NonredundantRegulationRiceArbuscularMycorrhizal_2012]] [OA](https://api.scholarcy.com/oa_version?query=Yang%2C%20S.-Y.%20Gr%C3%B8nlund%2C%20M.%20Jakobsen%2C%20I.%20Grotemeyer%2C%20M.S.%20Nonredundant%20regulation%20of%20rice%20arbuscular%20mycorrhizal%20symbiosis%20by%20two%20members%20of%20the%20phosphate%20transporter1%20gene%20family%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Yang%2C%20S.-Y.%20Gr%C3%B8nlund%2C%20M.%20Jakobsen%2C%20I.%20Grotemeyer%2C%20M.S.%20Nonredundant%20regulation%20of%20rice%20arbuscular%20mycorrhizal%20symbiosis%20by%20two%20members%20of%20the%20phosphate%20transporter1%20gene%20family%202012) [Scite](https://api.scholarcy.com/scite_url?query=Yang%2C%20S.-Y.%20Gr%C3%B8nlund%2C%20M.%20Jakobsen%2C%20I.%20Grotemeyer%2C%20M.S.%20Nonredundant%20regulation%20of%20rice%20arbuscular%20mycorrhizal%20symbiosis%20by%20two%20members%20of%20the%20phosphate%20transporter1%20gene%20family%202012)

[^Yano_et+al_2008_a]: Yano K, Yoshida S, Muller J, Singh S, Banba M, Vickers K, Markmann K, White C, Schuller B, Sato S, Asamizu E, Tabata S, Murooka Y, Perry J, Wang TL, Kawaguchi M, Imaizumi-Anraku H, Hayashi M, Parniske M (2008) CYCLOPS, a mediator of symbiotic intracellular accommodation. Proc Natl Acad Sci U S A 105(51):20540–20545 [[Yano_et+al_CyclopsMediator_2008]] [OA](https://api.scholarcy.com/oa_version?query=Yano%2C%20K.%20Yoshida%2C%20S.%20Muller%2C%20J.%20Singh%2C%20S.%20CYCLOPS%2C%20a%20mediator%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Yano%2C%20K.%20Yoshida%2C%20S.%20Muller%2C%20J.%20Singh%2C%20S.%20CYCLOPS%2C%20a%20mediator%202008) [Scite](https://api.scholarcy.com/scite_url?query=Yano%2C%20K.%20Yoshida%2C%20S.%20Muller%2C%20J.%20Singh%2C%20S.%20CYCLOPS%2C%20a%20mediator%202008)

[^Yoshida_et+al_2012_a]: Yoshida S, Kameoka H, Tempo M, Akiyama K, Umehara M, Yamaguchi S, Hayashi H, Kyozuka J, Shirasu K (2012) The D3 F-box protein is a key component in host strigolactone responses essential for arbuscular mycorrhizal symbiosis. New Phytol 196:1208–1216 [[Yoshida_et+al_D3FProteinComponentHost_2012]] [OA](https://api.scholarcy.com/oa_version?query=Yoshida%2C%20S.%20Kameoka%2C%20H.%20Tempo%2C%20M.%20Akiyama%2C%20K.%20The%20D3%20F-box%20protein%20is%20a%20key%20component%20in%20host%20strigolactone%20responses%20essential%20for%20arbuscular%20mycorrhizal%20symbiosis%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Yoshida%2C%20S.%20Kameoka%2C%20H.%20Tempo%2C%20M.%20Akiyama%2C%20K.%20The%20D3%20F-box%20protein%20is%20a%20key%20component%20in%20host%20strigolactone%20responses%20essential%20for%20arbuscular%20mycorrhizal%20symbiosis%202012) [Scite](https://api.scholarcy.com/scite_url?query=Yoshida%2C%20S.%20Kameoka%2C%20H.%20Tempo%2C%20M.%20Akiyama%2C%20K.%20The%20D3%20F-box%20protein%20is%20a%20key%20component%20in%20host%20strigolactone%20responses%20essential%20for%20arbuscular%20mycorrhizal%20symbiosis%202012)

[^Yu_et+al_2014_a]: Yu N, Luo D, Zhang X, Liu J, Wang W, Jin Y, Dong W, Liu J, Liu H, Yang W, Zeng L, Li Q, He Z, Oldroyd GED, Wang E (2014) A DELLA protein complex controls the arbuscular mycorrhizal symbiosis in plants. Cell Res 24:130–133 [[Yu_et+al_DellaProteinComplexControlsArbuscular_2014]] [OA](https://api.scholarcy.com/oa_version?query=Yu%2C%20N.%20Luo%2C%20D.%20Zhang%2C%20X.%20Liu%2C%20J.%20A%20DELLA%20protein%20complex%20controls%20the%20arbuscular%20mycorrhizal%20symbiosis%20in%20plants%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Yu%2C%20N.%20Luo%2C%20D.%20Zhang%2C%20X.%20Liu%2C%20J.%20A%20DELLA%20protein%20complex%20controls%20the%20arbuscular%20mycorrhizal%20symbiosis%20in%20plants%202014) [Scite](https://api.scholarcy.com/scite_url?query=Yu%2C%20N.%20Luo%2C%20D.%20Zhang%2C%20X.%20Liu%2C%20J.%20A%20DELLA%20protein%20complex%20controls%20the%20arbuscular%20mycorrhizal%20symbiosis%20in%20plants%202014)

[^Zeng_et+al_2018_a]: Zeng T, Holmer R, Hontelez J, Te Lintel HB, Marufu L, de Zeeuw T, Wu F, Schijlen E, Bisseling T, Limpens E (2018) Host- and stage-dependent secretome of the arbuscular mycorrhizal fungus Rhizophagus irregularis. Plant J 94:411–425 [[Zeng_et+al_HostStagedependentSecretomeArbuscularMycorrhizal_2018]] [OA](https://api.scholarcy.com/oa_version?query=Zeng%2C%20T.%20Holmer%2C%20R.%20Hontelez%2C%20J.%20Te%20Lintel%2C%20H.B.%20Host-%20and%20stage-dependent%20secretome%20of%20the%20arbuscular%20mycorrhizal%20fungus%20Rhizophagus%20irregularis%202018) [GScholar](https://scholar.google.co.uk/scholar?q=Zeng%2C%20T.%20Holmer%2C%20R.%20Hontelez%2C%20J.%20Te%20Lintel%2C%20H.B.%20Host-%20and%20stage-dependent%20secretome%20of%20the%20arbuscular%20mycorrhizal%20fungus%20Rhizophagus%20irregularis%202018) [Scite](https://api.scholarcy.com/scite_url?query=Zeng%2C%20T.%20Holmer%2C%20R.%20Hontelez%2C%20J.%20Te%20Lintel%2C%20H.B.%20Host-%20and%20stage-dependent%20secretome%20of%20the%20arbuscular%20mycorrhizal%20fungus%20Rhizophagus%20irregularis%202018)

[^Zhang_et+al_2015_a]: Zhang X, Dong W, Sun J, Feng F, Deng Y, He Z, Oldroyd GE, Wang E (2015) The receptor kinase CERK1 [[Zhang_et+al_ReceptorKinaseCerk1_2015]] [OA](https://scholar.google.co.uk/scholar?q=Zhang%2C%20X.%20Dong%2C%20W.%20Sun%2C%20J.%20Feng%2C%20F.%20The%20receptor%20kinase%20CERK1%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Zhang%2C%20X.%20Dong%2C%20W.%20Sun%2C%20J.%20Feng%2C%20F.%20The%20receptor%20kinase%20CERK1%202015) 

