[[MacLean_et+al_PlantSignalingMetabolicPathwaysEnabling_2017]]

# [Plant Signaling and Metabolic Pathways Enabling Arbuscular Mycorrhizal Symbiosis](https://dx.doi.org/10.1105/tpc.17.00555)

## [[Allyson M. MacLean]]; [[Armando Bravo]]; [[Maria J. Harrison]]

### 2017

## Abstract
AND CONCLUSIONS
In summary, ==research over the past few years has enhanced our understanding of the common symbiosis signaling pathway== and established direct connections between signaling and downstream events in the colonized cells. Yet, despite these advances, a full understanding of the receptor complexes and signaling molecules that generate input to the pathway, and the transcriptional regulatory networks that control downstream gene expression, remains to be achieved. Currently, GRAS factors dominate the regulatory landscape, yet spatial and temporal details are sparse, and their regulatory roles within multiprotein complexes remain to be established in a symbiotically relevant context. While the importance of the common symbiosis signaling pathway cannot be denied, ==new data from rice and maize indicate that additional signaling pathways play significant roles==. Whether signaling through the D14L/D3 pathway connects directly to the symbiosis signaling pathway or is necessary to establish an appropriate molecular environment to enable symbiosis remains to be determined. Additionally, the rice and ==maize data emphasize the importance of studies in a diversity== of plant species and provide opportunities for comparisons between hosts with different evolutionary trajectories.

## Key concepts
#arbuscular_mycorrhizal; #claim/symbiosis; #GRAS; #arabidopsis; #metabolism; #symbioses; #lysin_motif; #calcium; #gibberellic_acid; #lotus_japonicus; #land_plant; #NSP1; #gibberellin; #leucine_rich_repeat; #pisum_sativum; #solanum_lycopersicum; #arabidopsis_thaliana; #rhizophagus_irregularis

## Quote
> Phosphorylated CYCLOPS forms a complex with calcium and calmodulin-dependent kinase, which acts in concert with GRAS transcription factors such as DELLA proteins, to initiate the expression of genes such as Reduced Arbuscular Mycorrhiza 1 that are necessary to accommodate the fungal symbiont


## Key points
- Of the many associations formed between plants and microbes, arbuscular mycorrhizal (AM) symbiosis, in which plants and fungi of the Glomeromycota engage, is one of the most widespread and ancient ([^Smith_2008_a])
- Phylogenetic analyses indicate that symbiosis signaling genes are present in the genomes of the closest algal relatives to land plants and the function of the encoded proteins is conserved, which suggests that these plant ancestors were preadapted for symbiosis (Delaux et al, 2015)
- Phosphorylated CYCLOPS forms a complex with calcium and calmodulin-dependent kinase (CCaMK), which acts in concert with GRAS transcription factors such as DELLA proteins, to initiate the expression of genes such as Reduced Arbuscular Mycorrhiza 1 (RAM1) that are necessary to accommodate the fungal symbiont
- Research over the past few years has enhanced our understanding of the common symbiosis signaling pathway and established direct connections between signaling and downstream events in the colonized cells
- While the importance of the common symbiosis signaling pathway cannot be denied, new data from rice and maize indicate that additional signaling pathways play significant roles
- While this review has focused on events taking place within a plant host, a true understanding of AM symbiosis requires comprehensive knowledge of both symbiotic partners

## Synopsis
Of the many associations formed between plants and microbes, arbuscular mycorrhizal (AM) symbiosis, in which plants and fungi of the Glomeromycota engage, is one of the most widespread and ancient ([^Smith_2008_a]).
Phosphorylated CYCLOPS forms a complex with CCaMK, which acts in concert with GRAS transcription factors such as DELLA proteins, to initiate the expression of genes such as RAM1 that are necessary to accommodate the fungal symbiont.
Primacy of CCaMK at the apex of the regulatory transcriptional cascade is underscored by the observation that expression of an activated gain-of-function CCaMK is sufficient to fully complement the severe symbiotic phenotypes exhibited by mutants of genes upstream in the pathway such as dmi1/ pollux, castor, and dmi2/symrk ([^Hayashi_et+al_2010_a]), thereby uncoupling the requirement for Myc-factor perception and the resulting Ca2+ oscillations to elicit the subsequent downstream transcriptional response necessary to support AM symbiosis.
Genes that have functions outside of the symbiotic context ([^Delaux_et+al_2013_a]), for example, M. truncatula DELLA1 and DELLA2 ([^Floss_et+al_2013_a]), D14L (Gutjahr et al, 2015), and NOPE1 (Nadal et al, 2017), as well as several others indicated, are present in AM non-host and host plants and are required for AM symbiosis.
Addressing the topic from different angles and with different approaches, four groups recently provided complementary lines of evidence that collectively demonstrate that the plant provides fatty acids, most likely 16:0 b-monoacylglycerol (16:0 b-MAG) but possibly a derivative, to the fungus and that transfer occurs at the interface with the arbuscule (Figure 3) ([^Bravo_et+al_2017_a]</a>; [^Jiang_et+al_2017_a]; Keymer et al, 2017; [^Luginbuehl_et+al_2017_a]).
Support the direction of lipid flux toward 16:0 b-MAG and potentially transfer to the periarbuscular space was provided by analyses of M. truncatula loss-of-function mutants of three AM symbiosisconserved proteins, FatM, an acyl ACP-thioesterase, RAM2, a glycerol-3-phosphate acyl transferase (GPAT), and STR, a periarbuscular membrane-resident ABC transporter.
The genes encoding these three proteins are highly induced in colonized cells, and in all three mutants arbuscule development is impaired, fungal lipid levels are low, and symbiosis is not maintained ([^Zhang_et+al_2010_a]; Gobbato et al, 2012; [^Wang_et+al_2012_a]; [^Bravo_et+al_2016_a], [^Bravo_et+al_2017_a]).
Coupled with the native lipid profiles ([^Bravo_et+al_2017_a]</a>), the data collectively provide strong evidence that the colonized cell increases fatty acid biosynthesis and redirects flux through lipid metabolism to generate 16:0 b-monoacylglycerols (16:0 b-MAG) and these, or a derivative thereof, are transferred to the periarbuscular apoplast and subsequently accessed by the fungus (Figure 3).
These data reveal that during AM symbiosis, signaling through the common symbiosis signaling pathway triggers the reprogramming of lipid metabolism in the colonized cells to enable production and export of essential fatty acids for the fungus.
The rice and maize data emphasize the importance of studies in a diversity of plant species and provide opportunities for comparisons between hosts with different evolutionary trajectories


## Study subjects

### 1000 plant genes
- The latter two proteins are also required for arbuscule development, which suggests that changes in composition of a transcription factor complex may regulate the transition between the development and degeneration phases of the accommodation program. ==With several thousand plant genes showing differential expression during AM symbiosis, genetic dissection of the symbiotic program is a daunting task==. However, the early single origin of AM symbiosis, the broad taxonomic distribution within the vascular plant lineage, and the observation that all mycorrhizal plants contain the same set of genes for AM symbiosis provided a unique opportunity to use phylogenomics to identify genes conserved for AM symbiosis, which provides a point of focus for reverse genetics analyses

### 138 AM symbiosisconserved genes
- This evolutionary pattern of conservation in hosts and loss in non-host plants was visualized by constructing phylogenies and exploited to identify genes conserved for AM symbiosis ([^Delaux_et+al_2014_a]; [^Favre_et+al_2014_a]; [^Bravo_et+al_2016_a]</a></a>). ==The most stringent analysis identified 138 AM symbiosisconserved genes, of which 15 had known roles in AM symbiosis and mutants in an additional six also revealed their involvement ([^Bravo_et+al_2016_a]</a></a>)==. The 138 AM symbiosis-conserved genes show a variety of molecular functions, but in several cases, they were found to interact or to function at different points of a cellular process or single metabolic pathway, leading to a proposal that the AM conserved genes function in small modules to fine-tune cellular processes for symbiosis ([^Bravo_et+al_2016_a]</a></a>)

### 138 AM symbiosis-conserved genes
- The most stringent analysis identified 138 AM symbiosisconserved genes, of which 15 had known roles in AM symbiosis and mutants in an additional six also revealed their involvement ([^Bravo_et+al_2016_a]</a>). ==The 138 AM symbiosis-conserved genes show a variety of molecular functions, but in several cases, they were found to interact or to function at different points of a cellular process or single metabolic pathway, leading to a proposal that the AM conserved genes function in small modules to fine-tune cellular processes for symbiosis ([^Bravo_et+al_2016_a]</a>)==. For example, EXO70I, Vapyrin, and SYP132 are AM symbiosis-conserved proteins that modulate exocytosis to enable deposition of the periarbuscular membrane ([^Feddermann_et+al_2010_a]; [^Pumplin_et+al_2010_a]; Murray et al, 2011; [^Zhang_et+al_2015_b]; [^Huisman_et+al_2016_a]; [^Pan_et+al_2016_a])

## Findings
- AM fungi obtain their entire carbon supply from the plant, and it is estimated that they acquire up to 20% of the carbon fixed during photosynthesis ([^Bago_et+al_2000_a])

##  Builds on previous research
- The developmental phenotypes exhibited by kai2 mutants that are not related to karrikin perception per se, and broad conservation of KAI2 in basal land plants and species not associated with fire-prone habitats, have led to a hypothesis that the receptor KAI2 recognizes and binds to an as yet unidentified endogenous ligand, presumably a phytohormone that is structurally related to karrikins and strigolactones ([^Waters_et+al_2014_a], [^Waters_et+al_2015_a]; [^Conn_2016_a]</a>). The observation that D14L is essential for AM symbiosis in rice (Gutjahr et al, 2015), coupled with an ==earlier report of a rice d3 mutant== (homolog of MAX2) that is likewise unable to support AM symbiosis ([^Yoshida_et+al_2012_a]), suggests this signaling pathway may be involved in AM symbiosis

## Contributions
- In summary, research over the past few years has enhanced our understanding of the common symbiosis signaling pathway and established direct connections between signaling and downstream events in the colonized cells. Yet, despite these advances, a full understanding of the receptor complexes and signaling molecules that generate input to the pathway, and the transcriptional regulatory networks that control downstream gene expression, remains to be achieved. Currently, GRAS factors dominate the regulatory landscape, yet spatial and temporal details are sparse, and their regulatory roles within multiprotein complexes remain to be established in a symbiotically relevant context. While the importance of the common symbiosis signaling pathway cannot be denied, new data from rice and maize indicate that additional signaling pathways play significant roles. Whether signaling through the D14L/D3 pathway connects directly to the symbiosis signaling pathway or is necessary to establish an appropriate molecular environment to enable symbiosis remains to be determined. Additionally, the rice and maize data emphasize the importance of studies in a diversity of plant species and provide opportunities for comparisons between hosts with different evolutionary trajectories.


## References
[^RNAi:_2008_a]: RNAi: Increased number of degenerating (Floss et al., 2008a) arbuscules [[RNAi:_RnaiIncreasedNumberDegeneratingfloss_2008]] [OA](https://scholar.google.co.uk/scholar?q=RNAi%20Increased%20number%20of%20degenerating%20Floss%20et%20al%202008a%20arbuscules) [GScholar](https://scholar.google.co.uk/scholar?q=RNAi%20Increased%20number%20of%20degenerating%20Floss%20et%20al%202008a%20arbuscules) 

[^Carotenoid_2008_b]: Carotenoid cleavage dioxygenase RNAi: Increased number of degenerating (Floss et al., 2008b) arbuscules [[Carotenoid_CarotenoidCleavageDioxygenaseRnaiIncreased_2008]] [OA](https://scholar.google.co.uk/scholar?q=Carotenoid%20cleavage%20dioxygenase%20RNAi%20Increased%20number%20of%20degenerating%20Floss%20et%20al%202008b%20arbuscules) [GScholar](https://scholar.google.co.uk/scholar?q=Carotenoid%20cleavage%20dioxygenase%20RNAi%20Increased%20number%20of%20degenerating%20Floss%20et%20al%202008b%20arbuscules) 

[^Carotenoid_2008_a]: Carotenoid cleavage dioxygenase Reduced colonization (Gomez-Roldan et al., 2008) [[Carotenoid_CarotenoidCleavageDioxygenaseReducedColonization_2008]] [OA](https://scholar.google.co.uk/scholar?q=Carotenoid%20cleavage%20dioxygenase%20Reduced%20colonization%20GomezRoldan%20et%20al%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Carotenoid%20cleavage%20dioxygenase%20Reduced%20colonization%20GomezRoldan%20et%20al%202008) 

[^Reduced_2012_a]: Reduced colonization (Kretzschmar et al., 2012) [[Reduced_ReducedColonization_2012]] [OA](https://scholar.google.co.uk/scholar?q=Reduced%20colonization%20Kretzschmar%20et%20al%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Reduced%20colonization%20Kretzschmar%20et%20al%202012) 

[^No_2002_a]: No epidermal penetration (Endre et al., 2002; Stracke et al., 2002) [[No_EpidermalPenetrationendre_2002]] [OA](https://scholar.google.co.uk/scholar?q=No%20epidermal%20penetration%20Endre%20et%20al%202002%20Stracke%20et%20al%202002) [GScholar](https://scholar.google.co.uk/scholar?q=No%20epidermal%20penetration%20Endre%20et%20al%202002%20Stracke%20et%20al%202002) 

[^VIGS:_2011_a]: VIGS: No epidermal penetration in S. lycopersicum (Op den Camp et al., 2011; Maillet et al., 2011; Buendia et al., 2016; Madsen et al., 2003) [[VIGS:_VigsEpidermalPenetrationSLycopersicum_2011]] [OA](https://api.scholarcy.com/oa_version?query=VIGS%20No%20epidermal%20penetration%20in%20S%20lycopersicum%20Op%20den%20Camp%20et%20al%202011%20Maillet%20et%20al%202011%20Buendia%20et%20al%202016%20Madsen%20et%20al%202003) [GScholar](https://scholar.google.co.uk/scholar?q=VIGS%20No%20epidermal%20penetration%20in%20S%20lycopersicum%20Op%20den%20Camp%20et%20al%202011%20Maillet%20et%20al%202011%20Buendia%20et%20al%202016%20Madsen%20et%20al%202003) [Scite](https://api.scholarcy.com/scite_url?query=VIGS%20No%20epidermal%20penetration%20in%20S%20lycopersicum%20Op%20den%20Camp%20et%20al%202011%20Maillet%20et%20al%202011%20Buendia%20et%20al%202016%20Madsen%20et%20al%202003)

[^Increased_2012_b]: Increased colonization (Yoshida et al., 2012) [[Increased_IncreasedColonization_2012]] [OA](https://scholar.google.co.uk/scholar?q=Increased%20colonization%20Yoshida%20et%20al%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Increased%20colonization%20Yoshida%20et%20al%202012) 

[^No_2015_a]: No colonization (Gutjahr et al., 2015) [[No_Colonization_2015]] [OA](https://scholar.google.co.uk/scholar?q=No%20colonization%20Gutjahr%20et%20al%202015) [GScholar](https://scholar.google.co.uk/scholar?q=No%20colonization%20Gutjahr%20et%20al%202015) 

[^Reduced_2012_c]: Reduced colonization and arbuscule formation (Yoshida et al., 2012) [[Reduced_ReducedColonizationArbusculeFormation_2012]] [OA](https://scholar.google.co.uk/scholar?q=Reduced%20colonization%20and%20arbuscule%20formation%20Yoshida%20et%20al%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Reduced%20colonization%20and%20arbuscule%20formation%20Yoshida%20et%20al%202012) 

[^colonization_2005_a]: Delayed colonization (Kistner et al., 2005; Kanamori et al., 2006) [[colonization_Kanamori_2005]] [OA](https://api.scholarcy.com/oa_version?query=colonization%20%28Kistner%2C%20Delayed%20Kanamori%202005) [GScholar](https://scholar.google.co.uk/scholar?q=colonization%20%28Kistner%2C%20Delayed%20Kanamori%202005) [Scite](https://api.scholarcy.com/scite_url?query=colonization%20%28Kistner%2C%20Delayed%20Kanamori%202005)

[^Reduced_2011_b]: Reduced epidermal penetration (Capoen et al., 2011) [[Reduced_ReducedEpidermalPenetration_2011]] [OA](https://scholar.google.co.uk/scholar?q=Reduced%20epidermal%20penetration%20Capoen%20et%20al%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Reduced%20epidermal%20penetration%20Capoen%20et%20al%202011) 

[^Nuclear-localized_2016_a]: Nuclear-localized cyclic nucleotide- Reduced colonization gated channel (Charpentier et al., 2016) [[Nuclear-localized_NuclearlocalizedCyclicNucleotideReducedColonization_2016]] [OA](https://scholar.google.co.uk/scholar?q=Nuclearlocalized%20cyclic%20nucleotide%20Reduced%20colonization%20gated%20channel%20Charpentier%20et%20al%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Nuclearlocalized%20cyclic%20nucleotide%20Reduced%20colonization%20gated%20channel%20Charpentier%20et%20al%202016) 

[^M_2005_a]: Reduced numbers of arbuscules in M. truncatula, P. sativum, and S. lycopersicum (Kistner et al., 2005; Messinese et al., 2007; Yano et al., 2008; Horváth et al., 2011; Ovchinnikova et al., 2011; Larkan et al., 2013) [[M_ReducedNumbersArbuscules_2005]] [OA](https://api.scholarcy.com/oa_version?query=Reduced%20numbers%20of%20arbuscules%20in%20M%20truncatula%20P%20sativum%20and%20S%20lycopersicum%20Kistner%20et%20al%202005%20Messinese%20et%20al%202007%20Yano%20et%20al%202008%20Horv%C3%A1th%20et%20al%202011%20Ovchinnikova%20et%20al%202011%20Larkan%20et%20al%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Reduced%20numbers%20of%20arbuscules%20in%20M%20truncatula%20P%20sativum%20and%20S%20lycopersicum%20Kistner%20et%20al%202005%20Messinese%20et%20al%202007%20Yano%20et%20al%202008%20Horv%C3%A1th%20et%20al%202011%20Ovchinnikova%20et%20al%202011%20Larkan%20et%20al%202013) [Scite](https://api.scholarcy.com/scite_url?query=Reduced%20numbers%20of%20arbuscules%20in%20M%20truncatula%20P%20sativum%20and%20S%20lycopersicum%20Kistner%20et%20al%202005%20Messinese%20et%20al%202007%20Yano%20et%20al%202008%20Horv%C3%A1th%20et%20al%202011%20Ovchinnikova%20et%20al%202011%20Larkan%20et%20al%202013)

[^Intraradical_2013_a]: Intraradical colonization but very limited formation of arbuscules (Floss et al., 2013; Foo et al., 2013; Yu et al., 2014) [[Intraradical_IntraradicalColonizationVeryLimitedFormation_2013]] [OA](https://api.scholarcy.com/oa_version?query=Intraradical%20colonization%20but%20very%20limited%20formation%20of%20arbuscules%20Floss%20et%20al%202013%20Foo%20et%20al%202013%20Yu%20et%20al%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Intraradical%20colonization%20but%20very%20limited%20formation%20of%20arbuscules%20Floss%20et%20al%202013%20Foo%20et%20al%202013%20Yu%20et%20al%202014) [Scite](https://api.scholarcy.com/scite_url?query=Intraradical%20colonization%20but%20very%20limited%20formation%20of%20arbuscules%20Floss%20et%20al%202013%20Foo%20et%20al%202013%20Yu%20et%20al%202014)

[^Cortical_2012_d]: Cortical cell penetration and trunk formation but almost no hyphal branching (Gobbato et al., 2012, 2013; Rich et al., 2015; Park et al., 2015) [[Cortical_CorticalCellPenetrationTrunkFormation_2012]] [OA](https://api.scholarcy.com/oa_version?query=Cortical%20cell%20penetration%20and%20trunk%20formation%20but%20almost%20no%20hyphal%20branching%20Gobbato%20et%20al%202012%202013%20Rich%20et%20al%202015%20Park%20et%20al%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Cortical%20cell%20penetration%20and%20trunk%20formation%20but%20almost%20no%20hyphal%20branching%20Gobbato%20et%20al%202012%202013%20Rich%20et%20al%202015%20Park%20et%20al%202015) [Scite](https://api.scholarcy.com/scite_url?query=Cortical%20cell%20penetration%20and%20trunk%20formation%20but%20almost%20no%20hyphal%20branching%20Gobbato%20et%20al%202012%202013%20Rich%20et%20al%202015%20Park%20et%20al%202015)

[^Stunted_2015_b]: Stunted arbuscules in Lj-rad1 Reduced colonization in Mt-rad1; reduced number of arbuscules and increased arbuscule degeneration (Xue et al., 2015; Park et al., 2015) [[Stunted_StuntedArbusculesLjrad1ReducedColonization_2015]] [OA](https://api.scholarcy.com/oa_version?query=Stunted%20arbuscules%20in%20Ljrad1%20Reduced%20colonization%20in%20Mtrad1%20reduced%20number%20of%20arbuscules%20and%20increased%20arbuscule%20degeneration%20Xue%20et%20al%202015%20Park%20et%20al%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Stunted%20arbuscules%20in%20Ljrad1%20Reduced%20colonization%20in%20Mtrad1%20reduced%20number%20of%20arbuscules%20and%20increased%20arbuscule%20degeneration%20Xue%20et%20al%202015%20Park%20et%20al%202015) [Scite](https://api.scholarcy.com/scite_url?query=Stunted%20arbuscules%20in%20Ljrad1%20Reduced%20colonization%20in%20Mtrad1%20reduced%20number%20of%20arbuscules%20and%20increased%20arbuscule%20degeneration%20Xue%20et%20al%202015%20Park%20et%20al%202015)

[^RNAi:_2014_a]: RNAi: Reduced colonization (Yu et al., 2014) [[RNAi:_RnaiReducedColonization_2014]] [OA](https://scholar.google.co.uk/scholar?q=RNAi%20Reduced%20colonization%20Yu%20et%20al%202014) [GScholar](https://scholar.google.co.uk/scholar?q=RNAi%20Reduced%20colonization%20Yu%20et%20al%202014) 

[^RNAi:_2016_b]: RNAi: Small and malformed arbuscules (Heck et al., 2016) [[RNAi:_RnaiSmallMalformedArbuscules_2016]] [OA](https://scholar.google.co.uk/scholar?q=RNAi%20Small%20and%20malformed%20arbuscules%20Heck%20et%20al%202016) [GScholar](https://scholar.google.co.uk/scholar?q=RNAi%20Small%20and%20malformed%20arbuscules%20Heck%20et%20al%202016) 

[^Gm-NF-YA1a/b_2013_b]: Gm-NF-YA1a/b CCAAT-binding transcription factor RNAi: Reduced colonization (Schaarschmidt et al., 2013) [[Gm-NF-YA1a/b_nfya1aCcaatbindingTranscriptionFactorRnai_2013]] [OA](https://scholar.google.co.uk/scholar?q=GmNFYA1ab%20CCAATbinding%20transcription%20factor%20RNAi%20Reduced%20colonization%20Schaarschmidt%20et%20al%202013) [GScholar](https://scholar.google.co.uk/scholar?q=GmNFYA1ab%20CCAATbinding%20transcription%20factor%20RNAi%20Reduced%20colonization%20Schaarschmidt%20et%20al%202013) 

[^AP2_2013_c]: AP2 transcription factor amiR: Collapsed arbuscules (Devers et al., 2013) [[AP2_Ap2TranscriptionFactorAmirCollapsed_2013]] [OA](https://scholar.google.co.uk/scholar?q=AP2%20transcription%20factor%20amiR%20Collapsed%20arbuscules%20Devers%20et%20al%202013) [GScholar](https://scholar.google.co.uk/scholar?q=AP2%20transcription%20factor%20amiR%20Collapsed%20arbuscules%20Devers%20et%20al%202013) 

[^Suppression_2017_a]: Suppression of premature arbuscule degeneration in Mt-pt4/myb1 (Floss et al., 2017) [[Suppression_SuppressionPrematureArbusculeDegenerationMtpt4myb1_2017]] [OA](https://scholar.google.co.uk/scholar?q=Suppression%20of%20premature%20arbuscule%20degeneration%20in%20Mtpt4myb1%20Floss%20et%20al%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Suppression%20of%20premature%20arbuscule%20degeneration%20in%20Mtpt4myb1%20Floss%20et%20al%202017) 

[^Small_2010_a]: Small protrusions into cortical cells in P. hybrida (Pumplin et al., 2010; Feddermann et al., 2010) [[Small_SmallProtrusionsIntoCorticalCells_2010]] [OA](https://api.scholarcy.com/oa_version?query=Small%20protrusions%20into%20cortical%20cells%20in%20P%20hybrida%20Pumplin%20et%20al%202010%20Feddermann%20et%20al%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Small%20protrusions%20into%20cortical%20cells%20in%20P%20hybrida%20Pumplin%20et%20al%202010%20Feddermann%20et%20al%202010) [Scite](https://api.scholarcy.com/scite_url?query=Small%20protrusions%20into%20cortical%20cells%20in%20P%20hybrida%20Pumplin%20et%20al%202010%20Feddermann%20et%20al%202010)

[^RNAi:_2012_e]: RNAi: Stunted arbuscules (Ivanov et al., 2012) [[RNAi:_RnaiStuntedArbuscules_2012]] [OA](https://scholar.google.co.uk/scholar?q=RNAi%20Stunted%20arbuscules%20Ivanov%20et%20al%202012) [GScholar](https://scholar.google.co.uk/scholar?q=RNAi%20Stunted%20arbuscules%20Ivanov%20et%20al%202012) 

[^Stunted_2015_c]: Stunted arbuscules (Zhang et al., 2015b) [[Stunted_StuntedArbuscules_2015]] [OA](https://scholar.google.co.uk/scholar?q=Stunted%20arbuscules%20Zhang%20et%20al%202015b) [GScholar](https://scholar.google.co.uk/scholar?q=Stunted%20arbuscules%20Zhang%20et%20al%202015b) 

[^RNAi:_2016_c]: RNAi: Collapsed arbuscules (Pan et al., 2016) [[RNAi:_RnaiCollapsedArbuscules_2016]] [OA](https://scholar.google.co.uk/scholar?q=RNAi%20Collapsed%20arbuscules%20Pan%20et%20al%202016) [GScholar](https://scholar.google.co.uk/scholar?q=RNAi%20Collapsed%20arbuscules%20Pan%20et%20al%202016) 

[^amiR:_2013_d]: amiR: Increased number of collapsed arbuscules (Lota et al., 2013) [[amiR:_AmirIncreasedNumberCollapsedArbuscules_2013]] [OA](https://scholar.google.co.uk/scholar?q=amiR%20Increased%20number%20of%20collapsed%20arbuscules%20Lota%20et%20al%202013) [GScholar](https://scholar.google.co.uk/scholar?q=amiR%20Increased%20number%20of%20collapsed%20arbuscules%20Lota%20et%20al%202013) 

[^Membrane-bound_2010_b]: Membrane-bound steroid-binding RNAi: Aberrant arbuscule development protein (Kuhn et al., 2010) [[Membrane-bound_MembraneboundSteroidbindingRnaiAberrantArbuscule_2010]] [OA](https://scholar.google.co.uk/scholar?q=Membranebound%20steroidbinding%20RNAi%20Aberrant%20arbuscule%20development%20protein%20Kuhn%20et%20al%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Membranebound%20steroidbinding%20RNAi%20Aberrant%20arbuscule%20development%20protein%20Kuhn%20et%20al%202010) 

[^References_2010_a]: References (Zhang et al., 2010; Gutjahr et al., 2012; Kojima et al., 2014) [[References_Gutjahr_2010]] [OA](https://api.scholarcy.com/oa_version?query=%28Zhang%2C%20References%20Gutjahr%202010) [GScholar](https://scholar.google.co.uk/scholar?q=%28Zhang%2C%20References%20Gutjahr%202010) [Scite](https://api.scholarcy.com/scite_url?query=%28Zhang%2C%20References%20Gutjahr%202010)

[^arbuscules_2012_a]: Collapsed arbuscules (Wang et al., 2012; Keymer et al., 2017) [[arbuscules_Keymer_2012]] [OA](https://api.scholarcy.com/oa_version?query=arbuscules%20%28Wang%2C%20Collapsed%20Keymer%202012) [GScholar](https://scholar.google.co.uk/scholar?q=arbuscules%20%28Wang%2C%20Collapsed%20Keymer%202012) [Scite](https://api.scholarcy.com/scite_url?query=arbuscules%20%28Wang%2C%20Collapsed%20Keymer%202012)

[^Reduced_2013_e]: Reduced intercellular hyphal elongation (Takeda et al., 2013) [[Reduced_ReducedIntercellularHyphalElongation_2013]] [OA](https://scholar.google.co.uk/scholar?q=Reduced%20intercellular%20hyphal%20elongation%20Takeda%20et%20al%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Reduced%20intercellular%20hyphal%20elongation%20Takeda%20et%20al%202013) 

[^NRAMP-like_2008_b]: NRAMP-like integral membrane protein Increased number of infections (Penmetsa et al., 2008) [[NRAMP-like_NramplikeIntegralMembraneProteinIncreased_2008]] [OA](https://scholar.google.co.uk/scholar?q=NRAMPlike%20integral%20membrane%20protein%20Increased%20number%20of%20infections%20Penmetsa%20et%20al%202008) [GScholar](https://scholar.google.co.uk/scholar?q=NRAMPlike%20integral%20membrane%20protein%20Increased%20number%20of%20infections%20Penmetsa%20et%20al%202008) 

[^Morandi_2000_a]: Increased arbuscule formation in P. sativum (Morandi et al., 2000; Krusell et al., 2002; Nishimura et al., 2002; Searle et al., 2003; Schnabel et al., 2005) [[Morandi_IncreasedArbusculeFormationP_2000]] [OA](https://api.scholarcy.com/oa_version?query=Morandi%20Increased%20arbuscule%20formation%20in%20P%202000) [GScholar](https://scholar.google.co.uk/scholar?q=Morandi%20Increased%20arbuscule%20formation%20in%20P%202000) [Scite](https://api.scholarcy.com/scite_url?query=Morandi%20Increased%20arbuscule%20formation%20in%20P%202000)

[^RNAi:_2014_b]: RNAi: increased mycorrhizal colonization (Bitterlich et al., 2014) [[RNAi:_RnaiIncreasedMycorrhizalColonization_2014]] [OA](https://scholar.google.co.uk/scholar?q=RNAi%20increased%20mycorrhizal%20colonization%20Bitterlich%20et%20al%202014) [GScholar](https://scholar.google.co.uk/scholar?q=RNAi%20increased%20mycorrhizal%20colonization%20Bitterlich%20et%20al%202014) 

[^RNAi:_2009_a]: RNAi: reduced colonization and decrease in (Takeda et al., 2009) arbuscule formation [[RNAi:_RnaiReducedColonizationDecrease_2009]] [OA](https://api.scholarcy.com/oa_version?query=RNAi%20reduced%20colonization%20and%20decrease%20in%20Takeda%20et%20al%202009%20arbuscule%20formation) [GScholar](https://scholar.google.co.uk/scholar?q=RNAi%20reduced%20colonization%20and%20decrease%20in%20Takeda%20et%20al%202009%20arbuscule%20formation) [Scite](https://api.scholarcy.com/scite_url?query=RNAi%20reduced%20colonization%20and%20decrease%20in%20Takeda%20et%20al%202009%20arbuscule%20formation)

[^RNAi:_2013_f]: RNAi: malformed arbuscules (Rech et al., 2013) [[RNAi:_RnaiMalformedArbuscules_2013]] [OA](https://scholar.google.co.uk/scholar?q=RNAi%20malformed%20arbuscules%20Rech%20et%20al%202013) [GScholar](https://scholar.google.co.uk/scholar?q=RNAi%20malformed%20arbuscules%20Rech%20et%20al%202013) 

[^Premature_2007_a]: Premature arbuscule degeneration (Javot et al., 2007; Yang et al., 2012) [[Premature_Yang_2007]] [OA](https://api.scholarcy.com/oa_version?query=arbuscule%20degeneration%20%28Javot%2C%20Premature%20Yang%202007) [GScholar](https://scholar.google.co.uk/scholar?q=arbuscule%20degeneration%20%28Javot%2C%20Premature%20Yang%202007) [Scite](https://api.scholarcy.com/scite_url?query=arbuscule%20degeneration%20%28Javot%2C%20Premature%20Yang%202007)

[^Small_2012_f]: Small arbuscules (Yang et al., 2012) [[Small_SmallArbuscules_2012]] [OA](https://scholar.google.co.uk/scholar?q=Small%20arbuscules%20Yang%20et%20al%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Small%20arbuscules%20Yang%20et%20al%202012) 

[^Degenerating_2014_a]: Degenerating arbuscules (Krajinski et al., 2014; Wang et al., 2014) [[Degenerating__2014]] [OA](https://api.scholarcy.com/oa_version?query=Degenerating%20arbuscules%20Krajinski%20et%20al%202014%20Wang%20et%20al%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Degenerating%20arbuscules%20Krajinski%20et%20al%202014%20Wang%20et%20al%202014) [Scite](https://api.scholarcy.com/scite_url?query=Degenerating%20arbuscules%20Krajinski%20et%20al%202014%20Wang%20et%20al%202014)

[^Suppression_2015_d]: Suppression of premature arbuscule degeneration in Mt-pt4/amt2;3 (Breuillin-Sessoms et al., 2015) [[Suppression_SuppressionPrematureArbusculeDegenerationMtpt4amt2_2015]] [OA](https://scholar.google.co.uk/scholar?q=Suppression%20of%20premature%20arbuscule%20degeneration%20in%20Mtpt4amt23%20BreuillinSessoms%20et%20al%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Suppression%20of%20premature%20arbuscule%20degeneration%20in%20Mtpt4amt23%20BreuillinSessoms%20et%20al%202015) 

[^Several_2008_c]: Several profiling studies had documented increases in lipid biosynthesis during symbiosis (Schliemann et al., 2008; Wewer et al., 2014), but evidence for increased de novo fatty acid production in the colonized cells and its significance for symbiosis was provided by the L. japonicus dis mutant. DIS encodes a Keto-acyl ACP synthase 1 (KAS1), part of the fatty acid biosynthetic machinery responsible for the elongation of fatty acid chains specifically between C4:0 and C16:0 carbon chain length. In dis mutants, arbuscule development is impaired and fungal fatty acid levels are low (Groth et al., 2013; Keymer et al., 2017). Thus, DIS initiates increased fatty acid biosynthesis in the colonized cell. Evidence to [[Several_SeveralProfilingStudiesDocumentedIncreases_2008]] [OA](https://api.scholarcy.com/oa_version?query=Several%20profiling%20studies%20had%20documented%20increases%20in%20lipid%20biosynthesis%20during%20symbiosis%20Schliemann%20et%20al%202008%20Wewer%20et%20al%202014%20but%20evidence%20for%20increased%20de%20novo%20fatty%20acid%20production%20in%20the%20colonized%20cells%20and%20its%20significance%20for%20symbiosis%20was%20provided%20by%20the%20L%20japonicus%20dis%20mutant%20DIS%20encodes%20a%20Ketoacyl%20ACP%20synthase%201%20KAS1%20part%20of%20the%20fatty%20acid%20biosynthetic%20machinery%20responsible%20for%20the%20elongation%20of%20fatty%20acid%20chains%20specifically%20between%20C40%20and%20C160%20carbon%20chain%20length%20In%20dis%20mutants%20arbuscule%20development%20is%20impaired%20and%20fungal%20fatty%20acid%20levels%20are%20low%20Groth%20et%20al%202013%20Keymer%20et%20al%202017%20Thus%20DIS%20initiates%20increased%20fatty%20acid%20biosynthesis%20in%20the%20colonized%20cell%20Evidence%20to) [GScholar](https://scholar.google.co.uk/scholar?q=Several%20profiling%20studies%20had%20documented%20increases%20in%20lipid%20biosynthesis%20during%20symbiosis%20Schliemann%20et%20al%202008%20Wewer%20et%20al%202014%20but%20evidence%20for%20increased%20de%20novo%20fatty%20acid%20production%20in%20the%20colonized%20cells%20and%20its%20significance%20for%20symbiosis%20was%20provided%20by%20the%20L%20japonicus%20dis%20mutant%20DIS%20encodes%20a%20Ketoacyl%20ACP%20synthase%201%20KAS1%20part%20of%20the%20fatty%20acid%20biosynthetic%20machinery%20responsible%20for%20the%20elongation%20of%20fatty%20acid%20chains%20specifically%20between%20C40%20and%20C160%20carbon%20chain%20length%20In%20dis%20mutants%20arbuscule%20development%20is%20impaired%20and%20fungal%20fatty%20acid%20levels%20are%20low%20Groth%20et%20al%202013%20Keymer%20et%20al%202017%20Thus%20DIS%20initiates%20increased%20fatty%20acid%20biosynthesis%20in%20the%20colonized%20cell%20Evidence%20to) [Scite](https://api.scholarcy.com/scite_url?query=Several%20profiling%20studies%20had%20documented%20increases%20in%20lipid%20biosynthesis%20during%20symbiosis%20Schliemann%20et%20al%202008%20Wewer%20et%20al%202014%20but%20evidence%20for%20increased%20de%20novo%20fatty%20acid%20production%20in%20the%20colonized%20cells%20and%20its%20significance%20for%20symbiosis%20was%20provided%20by%20the%20L%20japonicus%20dis%20mutant%20DIS%20encodes%20a%20Ketoacyl%20ACP%20synthase%201%20KAS1%20part%20of%20the%20fatty%20acid%20biosynthetic%20machinery%20responsible%20for%20the%20elongation%20of%20fatty%20acid%20chains%20specifically%20between%20C40%20and%20C160%20carbon%20chain%20length%20In%20dis%20mutants%20arbuscule%20development%20is%20impaired%20and%20fungal%20fatty%20acid%20levels%20are%20low%20Groth%20et%20al%202013%20Keymer%20et%20al%202017%20Thus%20DIS%20initiates%20increased%20fatty%20acid%20biosynthesis%20in%20the%20colonized%20cell%20Evidence%20to)

[^Akiyama_2006_a]: Akiyama, K., and Hayashi, H. (2006). Strigolactones: chemical signals for fungal symbionts and parasitic weeds in plant roots. Ann. Bot. (Lond.) 97: 925–931. [[Akiyama_StrigolactonesChemicalSignalsFungalSymbionts_2006]] [OA](https://api.scholarcy.com/oa_version?query=Akiyama%2C%20K.%20Hayashi%2C%20H.%20Strigolactones%3A%20chemical%20signals%20for%20fungal%20symbionts%20and%20parasitic%20weeds%20in%20plant%20roots%202006) [GScholar](https://scholar.google.co.uk/scholar?q=Akiyama%2C%20K.%20Hayashi%2C%20H.%20Strigolactones%3A%20chemical%20signals%20for%20fungal%20symbionts%20and%20parasitic%20weeds%20in%20plant%20roots%202006) [Scite](https://api.scholarcy.com/scite_url?query=Akiyama%2C%20K.%20Hayashi%2C%20H.%20Strigolactones%3A%20chemical%20signals%20for%20fungal%20symbionts%20and%20parasitic%20weeds%20in%20plant%20roots%202006)

[^Akiyama_et+al_2005_a]: Akiyama, K., Matsuzaki, K., and Hayashi, H. (2005). Plant sesquiterpenes induce hyphal branching in arbuscular mycorrhizal fungi. Nature 435: 824–827. [[Akiyama_et+al_PlantSesquiterpenesInduceHyphalBranching_2005]] [OA](https://api.scholarcy.com/oa_version?query=Akiyama%2C%20K.%20Matsuzaki%2C%20K.%20Hayashi%2C%20H.%20Plant%20sesquiterpenes%20induce%20hyphal%20branching%20in%20arbuscular%20mycorrhizal%20fungi%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Akiyama%2C%20K.%20Matsuzaki%2C%20K.%20Hayashi%2C%20H.%20Plant%20sesquiterpenes%20induce%20hyphal%20branching%20in%20arbuscular%20mycorrhizal%20fungi%202005) [Scite](https://api.scholarcy.com/scite_url?query=Akiyama%2C%20K.%20Matsuzaki%2C%20K.%20Hayashi%2C%20H.%20Plant%20sesquiterpenes%20induce%20hyphal%20branching%20in%20arbuscular%20mycorrhizal%20fungi%202005)

[^Alvey_2005_a]: Alvey, L., and Harberd, N.P. (2005). DELLA proteins: integrators of multiple plant growth regulatory inputs? Physiol. Plant. 123: 153–160. [[Alvey_DellaProteinsIntegratorsMultiplePlant_2005]] [OA](https://api.scholarcy.com/oa_version?query=Alvey%2C%20L.%20Harberd%2C%20N.P.%20DELLA%20proteins%3A%20integrators%20of%20multiple%20plant%20growth%20regulatory%20inputs%3F%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Alvey%2C%20L.%20Harberd%2C%20N.P.%20DELLA%20proteins%3A%20integrators%20of%20multiple%20plant%20growth%20regulatory%20inputs%3F%202005) [Scite](https://api.scholarcy.com/scite_url?query=Alvey%2C%20L.%20Harberd%2C%20N.P.%20DELLA%20proteins%3A%20integrators%20of%20multiple%20plant%20growth%20regulatory%20inputs%3F%202005)

[^An_2004_a]: Ané, J.-M., et al. (2004). Medicago truncatula DMI1 required for bacterial and fungal symbioses in legumes. Science 303: 1364–1367. [[An_MedicagoTruncatulaDmi1RequiredBacterial_2004]] [OA](https://api.scholarcy.com/oa_version?query=An%C3%A9%2C%20J.-M.%20Medicago%20truncatula%20DMI1%20required%20for%20bacterial%20and%20fungal%20symbioses%20in%20legumes%202004) [GScholar](https://scholar.google.co.uk/scholar?q=An%C3%A9%2C%20J.-M.%20Medicago%20truncatula%20DMI1%20required%20for%20bacterial%20and%20fungal%20symbioses%20in%20legumes%202004) [Scite](https://api.scholarcy.com/scite_url?query=An%C3%A9%2C%20J.-M.%20Medicago%20truncatula%20DMI1%20required%20for%20bacterial%20and%20fungal%20symbioses%20in%20legumes%202004)

[^Antolín-Llovera_et+al_2014_a]: Antolín-Llovera, M., Ried, M.K., and Parniske, M. (2014). Cleavage of the SYMBIOSIS RECEPTOR-LIKE KINASE ectodomain promotes complex formation with Nod factor receptor 5. Curr. Biol. 24: 422–427. [[Antolín-Llovera_et+al_CleavageSymbiosisReceptorlikeKinaseEctodomain_2014]] [OA](https://api.scholarcy.com/oa_version?query=Antol%C3%ADn-Llovera%2C%20M.%20Ried%2C%20M.K.%20Parniske%2C%20M.%20Cleavage%20of%20the%20SYMBIOSIS%20RECEPTOR-LIKE%20KINASE%20ectodomain%20promotes%20complex%20formation%20with%20Nod%20factor%20receptor%205.%20Curr%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Antol%C3%ADn-Llovera%2C%20M.%20Ried%2C%20M.K.%20Parniske%2C%20M.%20Cleavage%20of%20the%20SYMBIOSIS%20RECEPTOR-LIKE%20KINASE%20ectodomain%20promotes%20complex%20formation%20with%20Nod%20factor%20receptor%205.%20Curr%202014) [Scite](https://api.scholarcy.com/scite_url?query=Antol%C3%ADn-Llovera%2C%20M.%20Ried%2C%20M.K.%20Parniske%2C%20M.%20Cleavage%20of%20the%20SYMBIOSIS%20RECEPTOR-LIKE%20KINASE%20ectodomain%20promotes%20complex%20formation%20with%20Nod%20factor%20receptor%205.%20Curr%202014)

[^Bago_et+al_2000_a]: Bago, B., Pfeffer, P.E., and Shachar-Hill, Y. (2000). Carbon metabolism and transport in arbuscular mycorrhizas. Plant Physiol. 124: 949–958. [[Bago_et+al_CarbonMetabolismTransportArbuscularMycorrhizas_2000]] [OA](https://api.scholarcy.com/oa_version?query=Bago%2C%20B.%20Pfeffer%2C%20P.E.%20Shachar-Hill%2C%20Y.%20Carbon%20metabolism%20and%20transport%20in%20arbuscular%20mycorrhizas%202000) [GScholar](https://scholar.google.co.uk/scholar?q=Bago%2C%20B.%20Pfeffer%2C%20P.E.%20Shachar-Hill%2C%20Y.%20Carbon%20metabolism%20and%20transport%20in%20arbuscular%20mycorrhizas%202000) [Scite](https://api.scholarcy.com/scite_url?query=Bago%2C%20B.%20Pfeffer%2C%20P.E.%20Shachar-Hill%2C%20Y.%20Carbon%20metabolism%20and%20transport%20in%20arbuscular%20mycorrhizas%202000)

[^Bago_et+al_2002_a]: Bago, B., Pfeffer, P.E., Zipfel, W., Lammers, P., and Shachar-Hill, Y. (2002). Tracking metabolism and imaging transport in arbuscular mycorrhizal fungi. Metabolism and transport in AM fungi. Plant Soil 244: 189–197. [[Bago_et+al_TrackingMetabolismImagingTransportArbuscular_2002]] [OA](https://scholar.google.co.uk/scholar?q=Bago%2C%20B.%20Pfeffer%2C%20P.E.%20Zipfel%2C%20W.%20Lammers%2C%20P.%20Tracking%20metabolism%20and%20imaging%20transport%20in%20arbuscular%20mycorrhizal%20fungi.%20Metabolism%20and%20transport%20in%20AM%20fungi.%20Plant%20Soil%20244%202002) [GScholar](https://scholar.google.co.uk/scholar?q=Bago%2C%20B.%20Pfeffer%2C%20P.E.%20Zipfel%2C%20W.%20Lammers%2C%20P.%20Tracking%20metabolism%20and%20imaging%20transport%20in%20arbuscular%20mycorrhizal%20fungi.%20Metabolism%20and%20transport%20in%20AM%20fungi.%20Plant%20Soil%20244%202002) 

[^Banba_et+al_2008_a]: Banba, M., Gutjahr, C., Miyao, A., Hirochika, H., Paszkowski, U., Kouchi, H., and Imaizumi-Anraku, H. (2008). Divergence of evolutionary ways among common sym genes: CASTOR and CCaMK show functional conservation between two symbiosis systems and constitute the root of a common signaling pathway. Plant Cell Physiol. 49: 1659–1671. [[Banba_et+al_DivergenceEvolutionaryWaysAmongCommon_2008]] [OA](https://api.scholarcy.com/oa_version?query=Banba%2C%20M.%20Gutjahr%2C%20C.%20Miyao%2C%20A.%20Hirochika%2C%20H.%20Divergence%20of%20evolutionary%20ways%20among%20common%20sym%20genes%3A%20CASTOR%20and%20CCaMK%20show%20functional%20conservation%20between%20two%20symbiosis%20systems%20and%20constitute%20the%20root%20of%20a%20common%20signaling%20pathway%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Banba%2C%20M.%20Gutjahr%2C%20C.%20Miyao%2C%20A.%20Hirochika%2C%20H.%20Divergence%20of%20evolutionary%20ways%20among%20common%20sym%20genes%3A%20CASTOR%20and%20CCaMK%20show%20functional%20conservation%20between%20two%20symbiosis%20systems%20and%20constitute%20the%20root%20of%20a%20common%20signaling%20pathway%202008) [Scite](https://api.scholarcy.com/scite_url?query=Banba%2C%20M.%20Gutjahr%2C%20C.%20Miyao%2C%20A.%20Hirochika%2C%20H.%20Divergence%20of%20evolutionary%20ways%20among%20common%20sym%20genes%3A%20CASTOR%20and%20CCaMK%20show%20functional%20conservation%20between%20two%20symbiosis%20systems%20and%20constitute%20the%20root%20of%20a%20common%20signaling%20pathway%202008)

[^Benfey_et+al_1993_a]: Benfey, P.N., Linstead, P.J., Roberts, K., Schiefelbein, J.W., Hauser, M.T., and Aeschbacher, R.A. (1993). Root development in Arabidopsis: four mutants with dramatically altered root morphogenesis. Development 119: 57–70. [[Benfey_et+al_RootDevelopmentArabidopsisFourMutants_1993]] [OA](https://api.scholarcy.com/oa_version?query=Benfey%2C%20P.N.%20Linstead%2C%20P.J.%20Roberts%2C%20K.%20Schiefelbein%2C%20J.W.%20Root%20development%20in%20Arabidopsis%3A%20four%20mutants%20with%20dramatically%20altered%20root%20morphogenesis%201993) [GScholar](https://scholar.google.co.uk/scholar?q=Benfey%2C%20P.N.%20Linstead%2C%20P.J.%20Roberts%2C%20K.%20Schiefelbein%2C%20J.W.%20Root%20development%20in%20Arabidopsis%3A%20four%20mutants%20with%20dramatically%20altered%20root%20morphogenesis%201993) [Scite](https://api.scholarcy.com/scite_url?query=Benfey%2C%20P.N.%20Linstead%2C%20P.J.%20Roberts%2C%20K.%20Schiefelbein%2C%20J.W.%20Root%20development%20in%20Arabidopsis%3A%20four%20mutants%20with%20dramatically%20altered%20root%20morphogenesis%201993)

[^Besserer_et+al_2008_a]: Besserer, A., Bécard, G., Jauneau, A., Roux, C., and SéjalonDelmas, N. (2008). GR24, a synthetic analog of strigolactones, stimulates the mitosis and growth of the arbuscular mycorrhizal fungus Gigaspora rosea by boosting its energy metabolism. Plant Physiol. 148: 402–413. [[Besserer_et+al_Gr24SyntheticAnalogStrigolactonesStimulates_2008]] [OA](https://api.scholarcy.com/oa_version?query=Besserer%2C%20A.%20B%C3%A9card%2C%20G.%20Jauneau%2C%20A.%20Roux%2C%20C.%20GR24%2C%20a%20synthetic%20analog%20of%20strigolactones%2C%20stimulates%20the%20mitosis%20and%20growth%20of%20the%20arbuscular%20mycorrhizal%20fungus%20Gigaspora%20rosea%20by%20boosting%20its%20energy%20metabolism%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Besserer%2C%20A.%20B%C3%A9card%2C%20G.%20Jauneau%2C%20A.%20Roux%2C%20C.%20GR24%2C%20a%20synthetic%20analog%20of%20strigolactones%2C%20stimulates%20the%20mitosis%20and%20growth%20of%20the%20arbuscular%20mycorrhizal%20fungus%20Gigaspora%20rosea%20by%20boosting%20its%20energy%20metabolism%202008) [Scite](https://api.scholarcy.com/scite_url?query=Besserer%2C%20A.%20B%C3%A9card%2C%20G.%20Jauneau%2C%20A.%20Roux%2C%20C.%20GR24%2C%20a%20synthetic%20analog%20of%20strigolactones%2C%20stimulates%20the%20mitosis%20and%20growth%20of%20the%20arbuscular%20mycorrhizal%20fungus%20Gigaspora%20rosea%20by%20boosting%20its%20energy%20metabolism%202008)

[^Besserer_et+al_2006_a]: Besserer, A., Puech-Pagès, V., Kiefer, P., Gomez-Roldan, V., Jauneau, A., Roy, S., Portais, J.C., Roux, C., Bécard, G., and Séjalon-Delmas, N. (2006). Strigolactones stimulate arbuscular mycorrhizal fungi by activating mitochondria. PLoS Biol. 4: e226. [[Besserer_et+al_StrigolactonesStimulateArbuscularMycorrhizalFungi_2006]] [OA](https://api.scholarcy.com/oa_version?query=Besserer%2C%20A.%20Puech-Pag%C3%A8s%2C%20V.%20Kiefer%2C%20P.%20Gomez-Roldan%2C%20V.%20Strigolactones%20stimulate%20arbuscular%20mycorrhizal%20fungi%20by%20activating%20mitochondria%202006) [GScholar](https://scholar.google.co.uk/scholar?q=Besserer%2C%20A.%20Puech-Pag%C3%A8s%2C%20V.%20Kiefer%2C%20P.%20Gomez-Roldan%2C%20V.%20Strigolactones%20stimulate%20arbuscular%20mycorrhizal%20fungi%20by%20activating%20mitochondria%202006) [Scite](https://api.scholarcy.com/scite_url?query=Besserer%2C%20A.%20Puech-Pag%C3%A8s%2C%20V.%20Kiefer%2C%20P.%20Gomez-Roldan%2C%20V.%20Strigolactones%20stimulate%20arbuscular%20mycorrhizal%20fungi%20by%20activating%20mitochondria%202006)

[^Bitterlich_et+al_2014_a]: Bitterlich, M., Krügel, U., Boldt-Burisch, K., Franken, P., and Kühn, C. (2014). The sucrose transporter SlSUT2 from tomato interacts with brassinosteroid functioning and affects arbuscular mycorrhiza formation. Plant J. 78: 877–889. [[Bitterlich_et+al_SucroseTransporterSlsut2FromTomato_2014]] [OA](https://api.scholarcy.com/oa_version?query=Bitterlich%2C%20M.%20Kr%C3%BCgel%2C%20U.%20Boldt-Burisch%2C%20K.%20Franken%2C%20P.%20The%20sucrose%20transporter%20SlSUT2%20from%20tomato%20interacts%20with%20brassinosteroid%20functioning%20and%20affects%20arbuscular%20mycorrhiza%20formation%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Bitterlich%2C%20M.%20Kr%C3%BCgel%2C%20U.%20Boldt-Burisch%2C%20K.%20Franken%2C%20P.%20The%20sucrose%20transporter%20SlSUT2%20from%20tomato%20interacts%20with%20brassinosteroid%20functioning%20and%20affects%20arbuscular%20mycorrhiza%20formation%202014) [Scite](https://api.scholarcy.com/scite_url?query=Bitterlich%2C%20M.%20Kr%C3%BCgel%2C%20U.%20Boldt-Burisch%2C%20K.%20Franken%2C%20P.%20The%20sucrose%20transporter%20SlSUT2%20from%20tomato%20interacts%20with%20brassinosteroid%20functioning%20and%20affects%20arbuscular%20mycorrhiza%20formation%202014)

[^Bolle_et+al_2000_a]: Bolle, C., Koncz, C., and Chua, N.H. (2000). PAT1, a new member of the GRAS family, is involved in phytochrome A signal transduction. Genes Dev. 14: 1269–1278. [[Bolle_et+al_Pat1MemberGrasFamilyInvolved_2000]] [OA](https://api.scholarcy.com/oa_version?query=Bolle%2C%20C.%20Koncz%2C%20C.%20Chua%2C%20N.H.%20PAT1%2C%20a%20new%20member%20of%20the%20GRAS%20family%2C%20is%20involved%20in%20phytochrome%20A%20signal%20transduction%202000) [GScholar](https://scholar.google.co.uk/scholar?q=Bolle%2C%20C.%20Koncz%2C%20C.%20Chua%2C%20N.H.%20PAT1%2C%20a%20new%20member%20of%20the%20GRAS%20family%2C%20is%20involved%20in%20phytochrome%20A%20signal%20transduction%202000) [Scite](https://api.scholarcy.com/scite_url?query=Bolle%2C%20C.%20Koncz%2C%20C.%20Chua%2C%20N.H.%20PAT1%2C%20a%20new%20member%20of%20the%20GRAS%20family%2C%20is%20involved%20in%20phytochrome%20A%20signal%20transduction%202000)

[^Bonfante-Fasolo_1984_a]: Bonfante-Fasolo, P. (1984). Anatomy and morphology of VA mycorrhizae. V.A. Mycorrhizae, C.L. Powell and D.J. Bagyaraj, eds (Boca Raton, FL: CRC Press), pp. 5–33. [[Bonfante-Fasolo_AnatomyMorphology_1984]] [OA](https://scholar.google.co.uk/scholar?q=Bonfante-Fasolo%2C%20P.%20Anatomy%20and%20morphology%20of%201984) [GScholar](https://scholar.google.co.uk/scholar?q=Bonfante-Fasolo%2C%20P.%20Anatomy%20and%20morphology%20of%201984) 

[^Bravo_et+al_2016_a]: Bravo, A., York, T., Pumplin, N., Mueller, L.A., and Harrison, M.J. (2016). Genes conserved for arbuscular mycorrhizal symbiosis identified through phylogenomics. Nat. Plants 2: 15208 [[Bravo_et+al_GenesConservedArbuscularMycorrhizalSymbiosis_2016]] [OA](https://api.scholarcy.com/oa_version?query=Bravo%2C%20A.%20York%2C%20T.%20Pumplin%2C%20N.%20Mueller%2C%20L.A.%20Genes%20conserved%20for%20arbuscular%20mycorrhizal%20symbiosis%20identified%20through%20phylogenomics%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Bravo%2C%20A.%20York%2C%20T.%20Pumplin%2C%20N.%20Mueller%2C%20L.A.%20Genes%20conserved%20for%20arbuscular%20mycorrhizal%20symbiosis%20identified%20through%20phylogenomics%202016) [Scite](https://api.scholarcy.com/scite_url?query=Bravo%2C%20A.%20York%2C%20T.%20Pumplin%2C%20N.%20Mueller%2C%20L.A.%20Genes%20conserved%20for%20arbuscular%20mycorrhizal%20symbiosis%20identified%20through%20phylogenomics%202016)

[^Bravo_et+al_2017_a]: Bravo, A., Brands, M., Wewer, V., Dörmann, P., and Harrison, M.J. (2017). Arbuscular mycorrhiza-specific enzymes FatM and RAM2 fine-tune lipid biosynthesis to promote development of arbuscular mycorrhiza. New Phytol. 214: 1631–1645. [[Bravo_et+al_ArbuscularMycorrhizaspecificEnzymesFatmRam2_2017]] [OA](https://api.scholarcy.com/oa_version?query=Bravo%2C%20A.%20Brands%2C%20M.%20Wewer%2C%20V.%20D%C3%B6rmann%2C%20P.%20Arbuscular%20mycorrhiza-specific%20enzymes%20FatM%20and%20RAM2%20fine-tune%20lipid%20biosynthesis%20to%20promote%20development%20of%20arbuscular%20mycorrhiza%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Bravo%2C%20A.%20Brands%2C%20M.%20Wewer%2C%20V.%20D%C3%B6rmann%2C%20P.%20Arbuscular%20mycorrhiza-specific%20enzymes%20FatM%20and%20RAM2%20fine-tune%20lipid%20biosynthesis%20to%20promote%20development%20of%20arbuscular%20mycorrhiza%202017) [Scite](https://api.scholarcy.com/scite_url?query=Bravo%2C%20A.%20Brands%2C%20M.%20Wewer%2C%20V.%20D%C3%B6rmann%2C%20P.%20Arbuscular%20mycorrhiza-specific%20enzymes%20FatM%20and%20RAM2%20fine-tune%20lipid%20biosynthesis%20to%20promote%20development%20of%20arbuscular%20mycorrhiza%202017)

[^Breuillin-Sessoms_2015_a]: Breuillin-Sessoms, F., et al. (2015). Suppression of arbuscule degeneration in Medicago truncatula phosphate transporter4 mutants is dependent on the Ammonium Transporter 2 family protein AMT2; 3. Plant Cell 27: 1352–1366. [[Breuillin-Sessoms_SuppressionArbusculeDegenerationMedicagoTruncatula_2015]] [OA](https://scholar.google.co.uk/scholar?q=Breuillin-Sessoms%2C%20F.%20Suppression%20of%20arbuscule%20degeneration%20in%20Medicago%20truncatula%20phosphate%20transporter4%20mutants%20is%20dependent%20on%20the%20Ammonium%20Transporter%202%20family%20protein%20AMT2%3B%203.%20Plant%20Cell%2027%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Breuillin-Sessoms%2C%20F.%20Suppression%20of%20arbuscule%20degeneration%20in%20Medicago%20truncatula%20phosphate%20transporter4%20mutants%20is%20dependent%20on%20the%20Ammonium%20Transporter%202%20family%20protein%20AMT2%3B%203.%20Plant%20Cell%2027%202015) 

[^Brundrett_2002_a]: Brundrett, M.C. (2002). Coevolution of roots and mycorrhiza of land plants. New Phytol. 154: 275–304. [[Brundrett_CoevolutionRootsMycorrhizaLandPlants_2002]] [OA](https://api.scholarcy.com/oa_version?query=Brundrett%2C%20M.C.%20Coevolution%20of%20roots%20and%20mycorrhiza%20of%20land%20plants%202002) [GScholar](https://scholar.google.co.uk/scholar?q=Brundrett%2C%20M.C.%20Coevolution%20of%20roots%20and%20mycorrhiza%20of%20land%20plants%202002) [Scite](https://api.scholarcy.com/scite_url?query=Brundrett%2C%20M.C.%20Coevolution%20of%20roots%20and%20mycorrhiza%20of%20land%20plants%202002)

[^Bucher_et+al_2014_a]: Bucher, M., Hause, B., Krajinski, F., and Küster, H. (2014). Through the doors of perception to function in arbuscular mycorrhizal symbioses. New Phytol. 204: 833–840. [[Bucher_et+al_ThroughDoorsPerceptionFunctionArbuscular_2014]] [OA](https://api.scholarcy.com/oa_version?query=Bucher%2C%20M.%20Hause%2C%20B.%20Krajinski%2C%20F.%20K%C3%BCster%2C%20H.%20Through%20the%20doors%20of%20perception%20to%20function%20in%20arbuscular%20mycorrhizal%20symbioses%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Bucher%2C%20M.%20Hause%2C%20B.%20Krajinski%2C%20F.%20K%C3%BCster%2C%20H.%20Through%20the%20doors%20of%20perception%20to%20function%20in%20arbuscular%20mycorrhizal%20symbioses%202014) [Scite](https://api.scholarcy.com/scite_url?query=Bucher%2C%20M.%20Hause%2C%20B.%20Krajinski%2C%20F.%20K%C3%BCster%2C%20H.%20Through%20the%20doors%20of%20perception%20to%20function%20in%20arbuscular%20mycorrhizal%20symbioses%202014)

[^Buee_et+al_2000_a]: Buee, M., Rossignol, M., Jauneau, A., Ranjeva, R., and Bécard, G. (2000). The pre-symbiotic growth of arbuscular mycorrhizal fungi is induced by a branching factor partially purified from plant root exudates. Mol. Plant Microbe Interact. 13: 693–698. [[Buee_et+al_symbioticGrowthArbuscularMycorrhizalFungi_2000]] [OA](https://api.scholarcy.com/oa_version?query=Buee%2C%20M.%20Rossignol%2C%20M.%20Jauneau%2C%20A.%20Ranjeva%2C%20R.%20The%20pre-symbiotic%20growth%20of%20arbuscular%20mycorrhizal%20fungi%20is%20induced%20by%20a%20branching%20factor%20partially%20purified%20from%20plant%20root%20exudates%202000) [GScholar](https://scholar.google.co.uk/scholar?q=Buee%2C%20M.%20Rossignol%2C%20M.%20Jauneau%2C%20A.%20Ranjeva%2C%20R.%20The%20pre-symbiotic%20growth%20of%20arbuscular%20mycorrhizal%20fungi%20is%20induced%20by%20a%20branching%20factor%20partially%20purified%20from%20plant%20root%20exudates%202000) [Scite](https://api.scholarcy.com/scite_url?query=Buee%2C%20M.%20Rossignol%2C%20M.%20Jauneau%2C%20A.%20Ranjeva%2C%20R.%20The%20pre-symbiotic%20growth%20of%20arbuscular%20mycorrhizal%20fungi%20is%20induced%20by%20a%20branching%20factor%20partially%20purified%20from%20plant%20root%20exudates%202000)

[^Buendia_et+al_2016_a]: Buendia, L., Wang, T., Girardin, A., and Lefebvre, B. (2016). The LysM receptor-like kinase SlLYK10 regulates the arbuscular mycorrhizal symbiosis in tomato. New Phytol. 210: 184–195. [[Buendia_et+al_LysmReceptorlikeKinaseSllyk10Regulates_2016]] [OA](https://api.scholarcy.com/oa_version?query=Buendia%2C%20L.%20Wang%2C%20T.%20Girardin%2C%20A.%20Lefebvre%2C%20B.%20The%20LysM%20receptor-like%20kinase%20SlLYK10%20regulates%20the%20arbuscular%20mycorrhizal%20symbiosis%20in%20tomato%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Buendia%2C%20L.%20Wang%2C%20T.%20Girardin%2C%20A.%20Lefebvre%2C%20B.%20The%20LysM%20receptor-like%20kinase%20SlLYK10%20regulates%20the%20arbuscular%20mycorrhizal%20symbiosis%20in%20tomato%202016) [Scite](https://api.scholarcy.com/scite_url?query=Buendia%2C%20L.%20Wang%2C%20T.%20Girardin%2C%20A.%20Lefebvre%2C%20B.%20The%20LysM%20receptor-like%20kinase%20SlLYK10%20regulates%20the%20arbuscular%20mycorrhizal%20symbiosis%20in%20tomato%202016)

[^Capoen_et+al_2011_a]: Capoen, W., Sun, J., Wysham, D., Otegui, M.S., Venkateshwaran, M., Hirsch, S., Miwa, H., Downie, J.A., Morris, R.J., Ané, J.M., and Oldroyd, G.E.D. (2011). Nuclear membranes control symbiotic calcium signaling of legumes. Proc. Natl. Acad. Sci. USA 108: 14348–14353. [[Capoen_et+al_NuclearMembranesControlSymbioticCalcium_2011]] [OA](https://api.scholarcy.com/oa_version?query=Capoen%2C%20W.%20Sun%2C%20J.%20Wysham%2C%20D.%20Otegui%2C%20M.S.%20Nuclear%20membranes%20control%20symbiotic%20calcium%20signaling%20of%20legumes%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Capoen%2C%20W.%20Sun%2C%20J.%20Wysham%2C%20D.%20Otegui%2C%20M.S.%20Nuclear%20membranes%20control%20symbiotic%20calcium%20signaling%20of%20legumes%202011) [Scite](https://api.scholarcy.com/scite_url?query=Capoen%2C%20W.%20Sun%2C%20J.%20Wysham%2C%20D.%20Otegui%2C%20M.S.%20Nuclear%20membranes%20control%20symbiotic%20calcium%20signaling%20of%20legumes%202011)

[^Carotenuto_et+al_2017_a]: Carotenuto, G., Chabaud, M., Miyata, K., Capozzi, M., Takeda, N., Kaku, H., Shibuya, N., Nakagawa, T., Barker, D.G., and Genre, A. (2017). The rice LysM receptor-like kinase OsCERK1 is required for the perception of short-chain chitin oligomers in arbuscular mycorrhizal signaling. New Phytol. 214: 1440–1446. [[Carotenuto_et+al_RiceLysmReceptorlikeKinaseOscerk1_2017]] [OA](https://api.scholarcy.com/oa_version?query=Carotenuto%2C%20G.%20Chabaud%2C%20M.%20Miyata%2C%20K.%20Capozzi%2C%20M.%20The%20rice%20LysM%20receptor-like%20kinase%20OsCERK1%20is%20required%20for%20the%20perception%20of%20short-chain%20chitin%20oligomers%20in%20arbuscular%20mycorrhizal%20signaling%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Carotenuto%2C%20G.%20Chabaud%2C%20M.%20Miyata%2C%20K.%20Capozzi%2C%20M.%20The%20rice%20LysM%20receptor-like%20kinase%20OsCERK1%20is%20required%20for%20the%20perception%20of%20short-chain%20chitin%20oligomers%20in%20arbuscular%20mycorrhizal%20signaling%202017) [Scite](https://api.scholarcy.com/scite_url?query=Carotenuto%2C%20G.%20Chabaud%2C%20M.%20Miyata%2C%20K.%20Capozzi%2C%20M.%20The%20rice%20LysM%20receptor-like%20kinase%20OsCERK1%20is%20required%20for%20the%20perception%20of%20short-chain%20chitin%20oligomers%20in%20arbuscular%20mycorrhizal%20signaling%202017)

[^Chabaud_et+al_2011_a]: Chabaud, M., Genre, A., Sieberer, B.J., Faccio, A., Fournier, J., Novero, M., Barker, D.G., and Bonfante, P. (2011). Arbuscular mycorrhizal hyphopodia and germinated spore exudates trigger Ca2+ spiking in the legume and nonlegume root epidermis. New Phytol. 189: 347–355. [[Chabaud_et+al_ArbuscularMycorrhizalHyphopodiaGerminatedSpore_2011]] [OA](https://api.scholarcy.com/oa_version?query=Chabaud%2C%20M.%20Genre%2C%20A.%20Sieberer%2C%20B.J.%20Faccio%2C%20A.%20Arbuscular%20mycorrhizal%20hyphopodia%20and%20germinated%20spore%20exudates%20trigger%20Ca2%2B%20spiking%20in%20the%20legume%20and%20nonlegume%20root%20epidermis%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Chabaud%2C%20M.%20Genre%2C%20A.%20Sieberer%2C%20B.J.%20Faccio%2C%20A.%20Arbuscular%20mycorrhizal%20hyphopodia%20and%20germinated%20spore%20exudates%20trigger%20Ca2%2B%20spiking%20in%20the%20legume%20and%20nonlegume%20root%20epidermis%202011) [Scite](https://api.scholarcy.com/scite_url?query=Chabaud%2C%20M.%20Genre%2C%20A.%20Sieberer%2C%20B.J.%20Faccio%2C%20A.%20Arbuscular%20mycorrhizal%20hyphopodia%20and%20germinated%20spore%20exudates%20trigger%20Ca2%2B%20spiking%20in%20the%20legume%20and%20nonlegume%20root%20epidermis%202011)

[^Charpentier_et+al_2014_a]: Charpentier, M., Sun, J., Wen, J., Mysore, K.S., and Oldroyd, G.E.D. (2014). Abscisic acid promotion of arbuscular mycorrhizal colonization requires a component of the PROTEIN PHOSPHATASE 2A complex. Plant Physiol. 166: 2077–2090. [[Charpentier_et+al_AbscisicAcidPromotionArbuscularMycorrhizal_2014]] [OA](https://api.scholarcy.com/oa_version?query=Charpentier%2C%20M.%20Sun%2C%20J.%20Wen%2C%20J.%20Mysore%2C%20K.S.%20Abscisic%20acid%20promotion%20of%20arbuscular%20mycorrhizal%20colonization%20requires%20a%20component%20of%20the%20PROTEIN%20PHOSPHATASE%202A%20complex%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Charpentier%2C%20M.%20Sun%2C%20J.%20Wen%2C%20J.%20Mysore%2C%20K.S.%20Abscisic%20acid%20promotion%20of%20arbuscular%20mycorrhizal%20colonization%20requires%20a%20component%20of%20the%20PROTEIN%20PHOSPHATASE%202A%20complex%202014) [Scite](https://api.scholarcy.com/scite_url?query=Charpentier%2C%20M.%20Sun%2C%20J.%20Wen%2C%20J.%20Mysore%2C%20K.S.%20Abscisic%20acid%20promotion%20of%20arbuscular%20mycorrhizal%20colonization%20requires%20a%20component%20of%20the%20PROTEIN%20PHOSPHATASE%202A%20complex%202014)

[^Charpentier_et+al_2016_a]: Charpentier, M., Sun, J., Vaz Martins, T., Radhakrishnan, G.V., Findlay, K., Soumpourou, E., Thouin, J., Véry, A.A., Sanders, D., Morris, R.J., and Oldroyd, G.E.D. (2016). Nuclear-localized cyclic nucleotide-gated channels mediate symbiotic calcium oscillations. Science 352: 1102–1105. [[Charpentier_et+al_NuclearlocalizedCyclicNucleotidegatedChannelsMediate_2016]] [OA](https://api.scholarcy.com/oa_version?query=Charpentier%2C%20M.%20Sun%2C%20J.%20Vaz%20Martins%2C%20T.%20Radhakrishnan%2C%20G.V.%20Nuclear-localized%20cyclic%20nucleotide-gated%20channels%20mediate%20symbiotic%20calcium%20oscillations%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Charpentier%2C%20M.%20Sun%2C%20J.%20Vaz%20Martins%2C%20T.%20Radhakrishnan%2C%20G.V.%20Nuclear-localized%20cyclic%20nucleotide-gated%20channels%20mediate%20symbiotic%20calcium%20oscillations%202016) [Scite](https://api.scholarcy.com/scite_url?query=Charpentier%2C%20M.%20Sun%2C%20J.%20Vaz%20Martins%2C%20T.%20Radhakrishnan%2C%20G.V.%20Nuclear-localized%20cyclic%20nucleotide-gated%20channels%20mediate%20symbiotic%20calcium%20oscillations%202016)

[^Chen_et+al_2007_a]: Chen, C., Gao, M., Liu, J., and Zhu, H. (2007). Fungal symbiosis in rice requires an ortholog of a legume common symbiosis gene encoding a Ca2+/calmodulin-dependent protein kinase. Plant Physiol. 145: 1619–1628. [[Chen_et+al_FungalSymbiosisRiceRequiresOrtholog_2007]] [OA](https://api.scholarcy.com/oa_version?query=Chen%2C%20C.%20Gao%2C%20M.%20Liu%2C%20J.%20Zhu%2C%20H.%20Fungal%20symbiosis%20in%20rice%20requires%20an%20ortholog%20of%20a%20legume%20common%20symbiosis%20gene%20encoding%20a%20Ca2%2B/calmodulin-dependent%20protein%20kinase%202007) [GScholar](https://scholar.google.co.uk/scholar?q=Chen%2C%20C.%20Gao%2C%20M.%20Liu%2C%20J.%20Zhu%2C%20H.%20Fungal%20symbiosis%20in%20rice%20requires%20an%20ortholog%20of%20a%20legume%20common%20symbiosis%20gene%20encoding%20a%20Ca2%2B/calmodulin-dependent%20protein%20kinase%202007) [Scite](https://api.scholarcy.com/scite_url?query=Chen%2C%20C.%20Gao%2C%20M.%20Liu%2C%20J.%20Zhu%2C%20H.%20Fungal%20symbiosis%20in%20rice%20requires%20an%20ortholog%20of%20a%20legume%20common%20symbiosis%20gene%20encoding%20a%20Ca2%2B/calmodulin-dependent%20protein%20kinase%202007)

[^Chen_et+al_2008_a]: Chen, C., Ané, J.M., and Zhu, H. (2008). OsIPD3, an ortholog of the Medicago truncatula DMI3 interacting protein IPD3, is required for mycorrhizal symbiosis in rice. New Phytol. 180: 311–315. [[Chen_et+al_Osipd3OrthologMedicagoTruncatulaDmi3_2008]] [OA](https://api.scholarcy.com/oa_version?query=Chen%2C%20C.%20An%C3%A9%2C%20J.M.%20Zhu%2C%20H.%20OsIPD3%2C%20an%20ortholog%20of%20the%20Medicago%20truncatula%20DMI3%20interacting%20protein%20IPD3%2C%20is%20required%20for%20mycorrhizal%20symbiosis%20in%20rice%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Chen%2C%20C.%20An%C3%A9%2C%20J.M.%20Zhu%2C%20H.%20OsIPD3%2C%20an%20ortholog%20of%20the%20Medicago%20truncatula%20DMI3%20interacting%20protein%20IPD3%2C%20is%20required%20for%20mycorrhizal%20symbiosis%20in%20rice%202008) [Scite](https://api.scholarcy.com/scite_url?query=Chen%2C%20C.%20An%C3%A9%2C%20J.M.%20Zhu%2C%20H.%20OsIPD3%2C%20an%20ortholog%20of%20the%20Medicago%20truncatula%20DMI3%20interacting%20protein%20IPD3%2C%20is%20required%20for%20mycorrhizal%20symbiosis%20in%20rice%202008)

[^Conn_2016_a]: Conn, C.E., and Nelson, D.C. (2016). Evidence that KARRIKIN-INSENSITIVE2 (KAI2) receptors may perceive an unknown signal that is not karrikin or strigolactone. Front. Plant Sci. 6: 1219. [[Conn_EvidenceThatKarrikininsensitive2kai2Receptors_2016]] [OA](https://api.scholarcy.com/oa_version?query=Conn%2C%20C.E.%20Nelson%2C%20D.C.%20Evidence%20that%20KARRIKIN-INSENSITIVE2%20%28KAI2%29%20receptors%20may%20perceive%20an%20unknown%20signal%20that%20is%20not%20karrikin%20or%20strigolactone%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Conn%2C%20C.E.%20Nelson%2C%20D.C.%20Evidence%20that%20KARRIKIN-INSENSITIVE2%20%28KAI2%29%20receptors%20may%20perceive%20an%20unknown%20signal%20that%20is%20not%20karrikin%20or%20strigolactone%202016) [Scite](https://api.scholarcy.com/scite_url?query=Conn%2C%20C.E.%20Nelson%2C%20D.C.%20Evidence%20that%20KARRIKIN-INSENSITIVE2%20%28KAI2%29%20receptors%20may%20perceive%20an%20unknown%20signal%20that%20is%20not%20karrikin%20or%20strigolactone%202016)

[^Couzigou_et+al_2017_a]: Couzigou, J.M., Lauressergues, D., André, O., Gutjahr, C., Guillotin, B., Bécard, G., and Combier, J.P. (2017). Positive gene regulation by a natural protective miRNA enables arbuscular mycorrhizal symbiosis. Cell Host Microbe 21: 106–112. [[Couzigou_et+al_PositiveGeneRegulationNaturalProtective_2017]] [OA](https://api.scholarcy.com/oa_version?query=Couzigou%2C%20J.M.%20Lauressergues%2C%20D.%20Andr%C3%A9%2C%20O.%20Gutjahr%2C%20C.%20Positive%20gene%20regulation%20by%20a%20natural%20protective%20miRNA%20enables%20arbuscular%20mycorrhizal%20symbiosis%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Couzigou%2C%20J.M.%20Lauressergues%2C%20D.%20Andr%C3%A9%2C%20O.%20Gutjahr%2C%20C.%20Positive%20gene%20regulation%20by%20a%20natural%20protective%20miRNA%20enables%20arbuscular%20mycorrhizal%20symbiosis%202017) [Scite](https://api.scholarcy.com/scite_url?query=Couzigou%2C%20J.M.%20Lauressergues%2C%20D.%20Andr%C3%A9%2C%20O.%20Gutjahr%2C%20C.%20Positive%20gene%20regulation%20by%20a%20natural%20protective%20miRNA%20enables%20arbuscular%20mycorrhizal%20symbiosis%202017)

[^Davière_2013_a]: Davière, J.M., and Achard, P. (2013). Gibberellin signaling in plants. Development 140: 1147–1151. [[Davière_GibberellinSignalingPlants_2013]] [OA](https://api.scholarcy.com/oa_version?query=Davi%C3%A8re%2C%20J.M.%20Achard%2C%20P.%20Gibberellin%20signaling%20in%20plants%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Davi%C3%A8re%2C%20J.M.%20Achard%2C%20P.%20Gibberellin%20signaling%20in%20plants%202013) [Scite](https://api.scholarcy.com/scite_url?query=Davi%C3%A8re%2C%20J.M.%20Achard%2C%20P.%20Gibberellin%20signaling%20in%20plants%202013)

[^Delaux_et+al_2013_a]: Delaux, P.-M., Séjalon-Delmas, N., Bécard, G., and Ané, J.-M. (2013a). Evolution of the plant-microbe symbiotic ‘toolkit’. Trends Plant Sci. 18: 298–304. [[Delaux_et+al_EvolutionPlantmicrobeSymbiotictoolkit_2013]] [OA](https://api.scholarcy.com/oa_version?query=Delaux%2C%20P.-M.%20S%C3%A9jalon-Delmas%2C%20N.%20B%C3%A9card%2C%20G.%20An%C3%A9%2C%20J.-M.%20Evolution%20of%20the%20plant-microbe%20symbiotic%20%E2%80%98toolkit%E2%80%99%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Delaux%2C%20P.-M.%20S%C3%A9jalon-Delmas%2C%20N.%20B%C3%A9card%2C%20G.%20An%C3%A9%2C%20J.-M.%20Evolution%20of%20the%20plant-microbe%20symbiotic%20%E2%80%98toolkit%E2%80%99%202013) [Scite](https://api.scholarcy.com/scite_url?query=Delaux%2C%20P.-M.%20S%C3%A9jalon-Delmas%2C%20N.%20B%C3%A9card%2C%20G.%20An%C3%A9%2C%20J.-M.%20Evolution%20of%20the%20plant-microbe%20symbiotic%20%E2%80%98toolkit%E2%80%99%202013)

[^Delaux_et+al_2014_a]: Delaux, P.-M., Varala, K., Edger, P.P., Coruzzi, G.M., Pires, J.C., and Ané, J.-M. (2014). Comparative phylogenomics uncovers the impact of symbiotic associations on host genome evolution. PLoS Genet. 10: e1004487. [[Delaux_et+al_ComparativePhylogenomicsUncoversImpactSymbiotic_2014]] [OA](https://api.scholarcy.com/oa_version?query=Delaux%2C%20P.-M.%20Varala%2C%20K.%20Edger%2C%20P.P.%20Coruzzi%2C%20G.M.%20Comparative%20phylogenomics%20uncovers%20the%20impact%20of%20symbiotic%20associations%20on%20host%20genome%20evolution%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Delaux%2C%20P.-M.%20Varala%2C%20K.%20Edger%2C%20P.P.%20Coruzzi%2C%20G.M.%20Comparative%20phylogenomics%20uncovers%20the%20impact%20of%20symbiotic%20associations%20on%20host%20genome%20evolution%202014) [Scite](https://api.scholarcy.com/scite_url?query=Delaux%2C%20P.-M.%20Varala%2C%20K.%20Edger%2C%20P.P.%20Coruzzi%2C%20G.M.%20Comparative%20phylogenomics%20uncovers%20the%20impact%20of%20symbiotic%20associations%20on%20host%20genome%20evolution%202014)

[^Delaux_et+al_2013_b]: Delaux, P.M., Bécard, G., and Combier, J.P. (2013b). NSP1 is a component of the Myc signaling pathway. New Phytol. 199: 59–65. [[Delaux_et+al_Nsp1ComponentMycSignalingPathway_2013]] [OA](https://scholar.google.co.uk/scholar?q=Delaux%2C%20P.M.%20B%C3%A9card%2C%20G.%20Combier%2C%20J.P.%20NSP1%20is%20a%20component%20of%20the%20Myc%20signaling%20pathway%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Delaux%2C%20P.M.%20B%C3%A9card%2C%20G.%20Combier%2C%20J.P.%20NSP1%20is%20a%20component%20of%20the%20Myc%20signaling%20pathway%202013) 

[^Delaux_2015_a]: Delaux, P.M., et al. (2015). Algal ancestor of land plants was preadapted for symbiosis. Proc. Natl. Acad. Sci. USA 112: 13390– 13395. [[Delaux_AlgalAncestorLandPlantsPreadapted_2015]] [OA](https://api.scholarcy.com/oa_version?query=Delaux%2C%20P.M.%20Algal%20ancestor%20of%20land%20plants%20was%20preadapted%20for%20symbiosis%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Delaux%2C%20P.M.%20Algal%20ancestor%20of%20land%20plants%20was%20preadapted%20for%20symbiosis%202015) [Scite](https://api.scholarcy.com/scite_url?query=Delaux%2C%20P.M.%20Algal%20ancestor%20of%20land%20plants%20was%20preadapted%20for%20symbiosis%202015)

[^Devers_et+al_2013_a]: Devers, E.A., Teply, J., Reinert, A., Gaude, N., and Krajinski, F. (2013). An endogenous artificial microRNA system for unraveling the function of root endosymbioses related genes in Medicago truncatula. BMC Plant Biol. 13: 82. [[Devers_et+al_EndogenousArtificialMicrornaSystemUnraveling_2013]] [OA](https://api.scholarcy.com/oa_version?query=Devers%2C%20E.A.%20Teply%2C%20J.%20Reinert%2C%20A.%20Gaude%2C%20N.%20An%20endogenous%20artificial%20microRNA%20system%20for%20unraveling%20the%20function%20of%20root%20endosymbioses%20related%20genes%20in%20Medicago%20truncatula%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Devers%2C%20E.A.%20Teply%2C%20J.%20Reinert%2C%20A.%20Gaude%2C%20N.%20An%20endogenous%20artificial%20microRNA%20system%20for%20unraveling%20the%20function%20of%20root%20endosymbioses%20related%20genes%20in%20Medicago%20truncatula%202013) [Scite](https://api.scholarcy.com/scite_url?query=Devers%2C%20E.A.%20Teply%2C%20J.%20Reinert%2C%20A.%20Gaude%2C%20N.%20An%20endogenous%20artificial%20microRNA%20system%20for%20unraveling%20the%20function%20of%20root%20endosymbioses%20related%20genes%20in%20Medicago%20truncatula%202013)

[^di_Laurenzio_et+al_1996_a]: Di Laurenzio, L., Wysocka-Diller, J., Malamy, J.E., Pysh, L., Helariutta, Y., Freshour, G., Hahn, M.G., Feldmann, K.A., and Benfey, P.N. (1996). The SCARECROW gene regulates an asymmetric cell division that is essential for generating the radial organization of the Arabidopsis root. Cell 86: 423–433. [[di_Laurenzio_et+al_ScarecrowGeneRegulatesAsymmetricCell_1996]] [OA](https://api.scholarcy.com/oa_version?query=Di%20Laurenzio%2C%20L.%20Wysocka-Diller%2C%20J.%20Malamy%2C%20J.E.%20Pysh%2C%20L.%20The%20SCARECROW%20gene%20regulates%20an%20asymmetric%20cell%20division%20that%20is%20essential%20for%20generating%20the%20radial%20organization%20of%20the%20Arabidopsis%20root%201996) [GScholar](https://scholar.google.co.uk/scholar?q=Di%20Laurenzio%2C%20L.%20Wysocka-Diller%2C%20J.%20Malamy%2C%20J.E.%20Pysh%2C%20L.%20The%20SCARECROW%20gene%20regulates%20an%20asymmetric%20cell%20division%20that%20is%20essential%20for%20generating%20the%20radial%20organization%20of%20the%20Arabidopsis%20root%201996) [Scite](https://api.scholarcy.com/scite_url?query=Di%20Laurenzio%2C%20L.%20Wysocka-Diller%2C%20J.%20Malamy%2C%20J.E.%20Pysh%2C%20L.%20The%20SCARECROW%20gene%20regulates%20an%20asymmetric%20cell%20division%20that%20is%20essential%20for%20generating%20the%20radial%20organization%20of%20the%20Arabidopsis%20root%201996)

[^Endre_et+al_2002_a]: Endre, G., Kereszt, A., Kevei, Z., Mihacea, S., Kaló, P., and Kiss, G.B. (2002). A receptor kinase gene regulating symbiotic nodule development. Nature 417: 962–966. [[Endre_et+al_ReceptorKinaseGeneRegulatingSymbiotic_2002]] [OA](https://api.scholarcy.com/oa_version?query=Endre%2C%20G.%20Kereszt%2C%20A.%20Kevei%2C%20Z.%20Mihacea%2C%20S.%20A%20receptor%20kinase%20gene%20regulating%20symbiotic%20nodule%20development%202002) [GScholar](https://scholar.google.co.uk/scholar?q=Endre%2C%20G.%20Kereszt%2C%20A.%20Kevei%2C%20Z.%20Mihacea%2C%20S.%20A%20receptor%20kinase%20gene%20regulating%20symbiotic%20nodule%20development%202002) [Scite](https://api.scholarcy.com/scite_url?query=Endre%2C%20G.%20Kereszt%2C%20A.%20Kevei%2C%20Z.%20Mihacea%2C%20S.%20A%20receptor%20kinase%20gene%20regulating%20symbiotic%20nodule%20development%202002)

[^Ezawa_et+al_2002_a]: Ezawa, T., Smith, S.E., and Smith, F.A. (2002). P metabolism and transport in AM fungi. Plant Soil 244: 221–230. [[Ezawa_et+al_MetabolismTransportAmFungiPlant_2002]] [OA](https://scholar.google.co.uk/scholar?q=Ezawa%2C%20T.%20Smith%2C%20S.E.%20Smith%2C%20F.A.%20P%20metabolism%20and%20transport%20in%20AM%20fungi.%20Plant%20Soil%20244%202002) [GScholar](https://scholar.google.co.uk/scholar?q=Ezawa%2C%20T.%20Smith%2C%20S.E.%20Smith%2C%20F.A.%20P%20metabolism%20and%20transport%20in%20AM%20fungi.%20Plant%20Soil%20244%202002) 

[^Favre_et+al_2014_a]: Favre, P., Bapaume, L., Bossolini, E., Delorenzi, M., Falquet, L., and Reinhardt, D. (2014). A novel bioinformatics pipeline to discover genes related to arbuscular mycorrhizal symbiosis based on their evolutionary conservation pattern among higher plants. BMC Plant Biol. 14: 333. [[Favre_et+al_NovelBioinformaticsPipelineDiscoverGenes_2014]] [OA](https://api.scholarcy.com/oa_version?query=Favre%2C%20P.%20Bapaume%2C%20L.%20Bossolini%2C%20E.%20Delorenzi%2C%20M.%20A%20novel%20bioinformatics%20pipeline%20to%20discover%20genes%20related%20to%20arbuscular%20mycorrhizal%20symbiosis%20based%20on%20their%20evolutionary%20conservation%20pattern%20among%20higher%20plants%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Favre%2C%20P.%20Bapaume%2C%20L.%20Bossolini%2C%20E.%20Delorenzi%2C%20M.%20A%20novel%20bioinformatics%20pipeline%20to%20discover%20genes%20related%20to%20arbuscular%20mycorrhizal%20symbiosis%20based%20on%20their%20evolutionary%20conservation%20pattern%20among%20higher%20plants%202014) [Scite](https://api.scholarcy.com/scite_url?query=Favre%2C%20P.%20Bapaume%2C%20L.%20Bossolini%2C%20E.%20Delorenzi%2C%20M.%20A%20novel%20bioinformatics%20pipeline%20to%20discover%20genes%20related%20to%20arbuscular%20mycorrhizal%20symbiosis%20based%20on%20their%20evolutionary%20conservation%20pattern%20among%20higher%20plants%202014)

[^Feddermann_et+al_2010_a]: Feddermann, N., Muni, R.R.D., Zeier, T., Stuurman, J., Ercolin, F., Schorderet, M., and Reinhardt, D. (2010). The PAM1 gene of petunia, required for intracellular accommodation and morphogenesis of arbuscular mycorrhizal fungi, encodes a homologue of VAPYRIN. Plant J. 64: 470–481. [[Feddermann_et+al_Pam1GenePetuniaRequiredIntracellular_2010]] [OA](https://api.scholarcy.com/oa_version?query=Feddermann%2C%20N.%20Muni%2C%20R.R.D.%20Zeier%2C%20T.%20Stuurman%2C%20J.%20The%20PAM1%20gene%20of%20petunia%2C%20required%20for%20intracellular%20accommodation%20and%20morphogenesis%20of%20arbuscular%20mycorrhizal%20fungi%2C%20encodes%20a%20homologue%20of%20VAPYRIN%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Feddermann%2C%20N.%20Muni%2C%20R.R.D.%20Zeier%2C%20T.%20Stuurman%2C%20J.%20The%20PAM1%20gene%20of%20petunia%2C%20required%20for%20intracellular%20accommodation%20and%20morphogenesis%20of%20arbuscular%20mycorrhizal%20fungi%2C%20encodes%20a%20homologue%20of%20VAPYRIN%202010) [Scite](https://api.scholarcy.com/scite_url?query=Feddermann%2C%20N.%20Muni%2C%20R.R.D.%20Zeier%2C%20T.%20Stuurman%2C%20J.%20The%20PAM1%20gene%20of%20petunia%2C%20required%20for%20intracellular%20accommodation%20and%20morphogenesis%20of%20arbuscular%20mycorrhizal%20fungi%2C%20encodes%20a%20homologue%20of%20VAPYRIN%202010)

[^Field_et+al_2012_a]: Field, K.J., Cameron, D.D., Leake, J.R., Tille, S., Bidartondo, M.I., and Beerling, D.J. (2012). Contrasting arbuscular mycorrhizal responses of vascular and non-vascular plants to a simulated Palaeozoic CO2 decline. Nat. Commun. 3: 835. [[Field_et+al_ContrastingArbuscularMycorrhizalResponsesVascular_2012]] [OA](https://api.scholarcy.com/oa_version?query=Field%2C%20K.J.%20Cameron%2C%20D.D.%20Leake%2C%20J.R.%20Tille%2C%20S.%20Contrasting%20arbuscular%20mycorrhizal%20responses%20of%20vascular%20and%20non-vascular%20plants%20to%20a%20simulated%20Palaeozoic%20CO2%20decline%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Field%2C%20K.J.%20Cameron%2C%20D.D.%20Leake%2C%20J.R.%20Tille%2C%20S.%20Contrasting%20arbuscular%20mycorrhizal%20responses%20of%20vascular%20and%20non-vascular%20plants%20to%20a%20simulated%20Palaeozoic%20CO2%20decline%202012) [Scite](https://api.scholarcy.com/scite_url?query=Field%2C%20K.J.%20Cameron%2C%20D.D.%20Leake%2C%20J.R.%20Tille%2C%20S.%20Contrasting%20arbuscular%20mycorrhizal%20responses%20of%20vascular%20and%20non-vascular%20plants%20to%20a%20simulated%20Palaeozoic%20CO2%20decline%202012)

[^Fiorilli_et+al_2015_a]: Fiorilli, V., Volpe, V., Zanini, S., Vallino, M., Abbà, S., and Bonfante, P. (2015). A rice GRAS gene has an impact on the success of arbuscular mycorrhizal colonization. Am. J. Plant Sci. 6: 1905–1915. [[Fiorilli_et+al_RiceGrasGeneImpactSuccess_2015]] [OA](https://api.scholarcy.com/oa_version?query=Fiorilli%2C%20V.%20Volpe%2C%20V.%20Zanini%2C%20S.%20Vallino%2C%20M.%20A%20rice%20GRAS%20gene%20has%20an%20impact%20on%20the%20success%20of%20arbuscular%20mycorrhizal%20colonization%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Fiorilli%2C%20V.%20Volpe%2C%20V.%20Zanini%2C%20S.%20Vallino%2C%20M.%20A%20rice%20GRAS%20gene%20has%20an%20impact%20on%20the%20success%20of%20arbuscular%20mycorrhizal%20colonization%202015) [Scite](https://api.scholarcy.com/scite_url?query=Fiorilli%2C%20V.%20Volpe%2C%20V.%20Zanini%2C%20S.%20Vallino%2C%20M.%20A%20rice%20GRAS%20gene%20has%20an%20impact%20on%20the%20success%20of%20arbuscular%20mycorrhizal%20colonization%202015)

[^Flematti_et+al_2015_a]: Flematti, G.R., Dixon, K.W., and Smith, S.M. (2015). What are karrikins and how were they ‘discovered’ by plants? BMC Biol. 13: 108. [[Flematti_et+al_WhatKarrikinsWereTheydiscovered_2015]] [OA](https://api.scholarcy.com/oa_version?query=Flematti%2C%20G.R.%20Dixon%2C%20K.W.%20Smith%2C%20S.M.%20What%20are%20karrikins%20and%20how%20were%20they%20%E2%80%98discovered%E2%80%99%20by%20plants%3F%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Flematti%2C%20G.R.%20Dixon%2C%20K.W.%20Smith%2C%20S.M.%20What%20are%20karrikins%20and%20how%20were%20they%20%E2%80%98discovered%E2%80%99%20by%20plants%3F%202015) [Scite](https://api.scholarcy.com/scite_url?query=Flematti%2C%20G.R.%20Dixon%2C%20K.W.%20Smith%2C%20S.M.%20What%20are%20karrikins%20and%20how%20were%20they%20%E2%80%98discovered%E2%80%99%20by%20plants%3F%202015)

[^Floss_et+al_2008_a]: Floss, D.S., Schliemann, W., Schmidt, J., Strack, D., and Walter, M.H. (2008a). RNA interference-mediated repression of MtCCD1 in mycorrhizal roots of Medicago truncatula causes accumulation of C27 apocarotenoids, shedding light on the functional role of CCD1. Plant Physiol. 148: 1267–1282. [[Floss_et+al_RnaInterferencemediatedRepressionMtccd1Mycorrhizal_2008]] [OA](https://api.scholarcy.com/oa_version?query=Floss%2C%20D.S.%20Schliemann%2C%20W.%20Schmidt%2C%20J.%20Strack%2C%20D.%20RNA%20interference-mediated%20repression%20of%20MtCCD1%20in%20mycorrhizal%20roots%20of%20Medicago%20truncatula%20causes%20accumulation%20of%20C27%20apocarotenoids%2C%20shedding%20light%20on%20the%20functional%20role%20of%20CCD1%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Floss%2C%20D.S.%20Schliemann%2C%20W.%20Schmidt%2C%20J.%20Strack%2C%20D.%20RNA%20interference-mediated%20repression%20of%20MtCCD1%20in%20mycorrhizal%20roots%20of%20Medicago%20truncatula%20causes%20accumulation%20of%20C27%20apocarotenoids%2C%20shedding%20light%20on%20the%20functional%20role%20of%20CCD1%202008) [Scite](https://api.scholarcy.com/scite_url?query=Floss%2C%20D.S.%20Schliemann%2C%20W.%20Schmidt%2C%20J.%20Strack%2C%20D.%20RNA%20interference-mediated%20repression%20of%20MtCCD1%20in%20mycorrhizal%20roots%20of%20Medicago%20truncatula%20causes%20accumulation%20of%20C27%20apocarotenoids%2C%20shedding%20light%20on%20the%20functional%20role%20of%20CCD1%202008)

[^Floss_et+al_2013_a]: Floss, D.S., Levy, J.G., Levesque-Tremblay, V., Pumplin, N., and Harrison, M.J. (2013). DELLA proteins regulate arbuscule formation in arbuscular mycorrhizal symbiosis. Proc. Natl. Acad. Sci. USA 110: E5025–E5034. [[Floss_et+al_DellaProteinsRegulateArbusculeFormation_2013]] [OA](https://api.scholarcy.com/oa_version?query=Floss%2C%20D.S.%20Levy%2C%20J.G.%20Levesque-Tremblay%2C%20V.%20Pumplin%2C%20N.%20DELLA%20proteins%20regulate%20arbuscule%20formation%20in%20arbuscular%20mycorrhizal%20symbiosis%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Floss%2C%20D.S.%20Levy%2C%20J.G.%20Levesque-Tremblay%2C%20V.%20Pumplin%2C%20N.%20DELLA%20proteins%20regulate%20arbuscule%20formation%20in%20arbuscular%20mycorrhizal%20symbiosis%202013) [Scite](https://api.scholarcy.com/scite_url?query=Floss%2C%20D.S.%20Levy%2C%20J.G.%20Levesque-Tremblay%2C%20V.%20Pumplin%2C%20N.%20DELLA%20proteins%20regulate%20arbuscule%20formation%20in%20arbuscular%20mycorrhizal%20symbiosis%202013)

[^Floss_et+al_2008_b]: Floss, D.S., Hause, B., Lange, P.R., Küster, H., Strack, D., and Walter, M.H. (2008b). Knock-down of the MEP pathway isogene [[Floss_et+al_KnockdownMepPathwayIsogene_2008]] [OA](https://scholar.google.co.uk/scholar?q=Floss%2C%20D.S.%20Hause%2C%20B.%20Lange%2C%20P.R.%20K%C3%BCster%2C%20H.%20Knock-down%20of%20the%20MEP%20pathway%20isogene%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Floss%2C%20D.S.%20Hause%2C%20B.%20Lange%2C%20P.R.%20K%C3%BCster%2C%20H.%20Knock-down%20of%20the%20MEP%20pathway%20isogene%202008) 

[^Floss_et+al_2017_a]: Floss, D.S., Gomez, S.K., Park, H.J., MacLean, A.M., Müller, L.M., Bhattarai, K.K., Lévesque-Tremblay, V., Maldonado-Mendoza, I.E., and Harrison, M.J. (2017). A transcriptional program for arbuscule degeneration during AM symbiosisis regulated by MYB1. Curr. Biol. 27: 1206–1212. [[Floss_et+al_TranscriptionalProgramArbusculeDegenerationDuring_2017]] [OA](https://api.scholarcy.com/oa_version?query=Floss%2C%20D.S.%20Gomez%2C%20S.K.%20Park%2C%20H.J.%20MacLean%2C%20A.M.%20A%20transcriptional%20program%20for%20arbuscule%20degeneration%20during%20AM%20symbiosisis%20regulated%20by%20MYB1.%20Curr%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Floss%2C%20D.S.%20Gomez%2C%20S.K.%20Park%2C%20H.J.%20MacLean%2C%20A.M.%20A%20transcriptional%20program%20for%20arbuscule%20degeneration%20during%20AM%20symbiosisis%20regulated%20by%20MYB1.%20Curr%202017) [Scite](https://api.scholarcy.com/scite_url?query=Floss%2C%20D.S.%20Gomez%2C%20S.K.%20Park%2C%20H.J.%20MacLean%2C%20A.M.%20A%20transcriptional%20program%20for%20arbuscule%20degeneration%20during%20AM%20symbiosisis%20regulated%20by%20MYB1.%20Curr%202017)

[^Foo_et+al_2013_a]: Foo, E., Ross, J.J., Jones, W.T., and Reid, J.B. (2013). Plant hormones in arbuscular mycorrhizal symbioses: an emerging role for gibberellins. Ann. Bot. (Lond.) 111: 769–779. [[Foo_et+al_PlantHormonesArbuscularMycorrhizalSymbioses_2013]] [OA](https://api.scholarcy.com/oa_version?query=Foo%2C%20E.%20Ross%2C%20J.J.%20Jones%2C%20W.T.%20Reid%2C%20J.B.%20Plant%20hormones%20in%20arbuscular%20mycorrhizal%20symbioses%3A%20an%20emerging%20role%20for%20gibberellins%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Foo%2C%20E.%20Ross%2C%20J.J.%20Jones%2C%20W.T.%20Reid%2C%20J.B.%20Plant%20hormones%20in%20arbuscular%20mycorrhizal%20symbioses%3A%20an%20emerging%20role%20for%20gibberellins%202013) [Scite](https://api.scholarcy.com/scite_url?query=Foo%2C%20E.%20Ross%2C%20J.J.%20Jones%2C%20W.T.%20Reid%2C%20J.B.%20Plant%20hormones%20in%20arbuscular%20mycorrhizal%20symbioses%3A%20an%20emerging%20role%20for%20gibberellins%202013)

[^Gallego-Bartolomé_et+al_2012_a]: Gallego-Bartolomé, J., Minguet, E.G., Grau-Enguix, F., Abbas, M., Locascio, A., Thomas, S.G., Alabadí, D., and Blázquez, M.A. (2012). Molecular mechanism for the interaction between gibberellin and brassinosteroid signaling pathways in Arabidopsis. Proc. Natl. Acad. Sci. USA 109: 13446–13451. [[Gallego-Bartolomé_et+al_MolecularMechanismInteractionBetweenGibberellin_2012]] [OA](https://api.scholarcy.com/oa_version?query=Gallego-Bartolom%C3%A9%2C%20J.%20Minguet%2C%20E.G.%20Grau-Enguix%2C%20F.%20Abbas%2C%20M.%20Molecular%20mechanism%20for%20the%20interaction%20between%20gibberellin%20and%20brassinosteroid%20signaling%20pathways%20in%20Arabidopsis%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Gallego-Bartolom%C3%A9%2C%20J.%20Minguet%2C%20E.G.%20Grau-Enguix%2C%20F.%20Abbas%2C%20M.%20Molecular%20mechanism%20for%20the%20interaction%20between%20gibberellin%20and%20brassinosteroid%20signaling%20pathways%20in%20Arabidopsis%202012) [Scite](https://api.scholarcy.com/scite_url?query=Gallego-Bartolom%C3%A9%2C%20J.%20Minguet%2C%20E.G.%20Grau-Enguix%2C%20F.%20Abbas%2C%20M.%20Molecular%20mechanism%20for%20the%20interaction%20between%20gibberellin%20and%20brassinosteroid%20signaling%20pathways%20in%20Arabidopsis%202012)

[^Garcia_et+al_2016_a]: Garcia, K., Doidy, J., Zimmermann, S.D., Wipf, D., and Courty, P.E. (2016). Take a trip through the plant and fungal transportome of mycorrhiza. Trends Plant Sci. 21: 937–950. [[Garcia_et+al_TakeTripThroughPlantFungal_2016]] [OA](https://api.scholarcy.com/oa_version?query=Garcia%2C%20K.%20Doidy%2C%20J.%20Zimmermann%2C%20S.D.%20Wipf%2C%20D.%20Take%20a%20trip%20through%20the%20plant%20and%20fungal%20transportome%20of%20mycorrhiza%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Garcia%2C%20K.%20Doidy%2C%20J.%20Zimmermann%2C%20S.D.%20Wipf%2C%20D.%20Take%20a%20trip%20through%20the%20plant%20and%20fungal%20transportome%20of%20mycorrhiza%202016) [Scite](https://api.scholarcy.com/scite_url?query=Garcia%2C%20K.%20Doidy%2C%20J.%20Zimmermann%2C%20S.D.%20Wipf%2C%20D.%20Take%20a%20trip%20through%20the%20plant%20and%20fungal%20transportome%20of%20mycorrhiza%202016)

[^Gaude_et+al_2012_a]: Gaude, N., Bortfeld, S., Duensing, N., Lohse, M., and Krajinski, F. (2012). Arbuscule-containing and non-colonized cortical cells of mycorrhizal roots undergo extensive and specific reprogramming during arbuscular mycorrhizal development. Plant J. 69: 510–528. [[Gaude_et+al_ArbusculecontainingcolonizedCorticalCellsMycorrhizal_2012]] [OA](https://api.scholarcy.com/oa_version?query=Gaude%2C%20N.%20Bortfeld%2C%20S.%20Duensing%2C%20N.%20Lohse%2C%20M.%20Arbuscule-containing%20and%20non-colonized%20cortical%20cells%20of%20mycorrhizal%20roots%20undergo%20extensive%20and%20specific%20reprogramming%20during%20arbuscular%20mycorrhizal%20development%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Gaude%2C%20N.%20Bortfeld%2C%20S.%20Duensing%2C%20N.%20Lohse%2C%20M.%20Arbuscule-containing%20and%20non-colonized%20cortical%20cells%20of%20mycorrhizal%20roots%20undergo%20extensive%20and%20specific%20reprogramming%20during%20arbuscular%20mycorrhizal%20development%202012) [Scite](https://api.scholarcy.com/scite_url?query=Gaude%2C%20N.%20Bortfeld%2C%20S.%20Duensing%2C%20N.%20Lohse%2C%20M.%20Arbuscule-containing%20and%20non-colonized%20cortical%20cells%20of%20mycorrhizal%20roots%20undergo%20extensive%20and%20specific%20reprogramming%20during%20arbuscular%20mycorrhizal%20development%202012)

[^Genre_et+al_2013_a]: Genre, A., Chabaud, M., Balzergue, C., Puech-Pagès, V., Novero, M., Rey, T., Fournier, J., Rochange, S., Bécard, G., Bonfante, P., and Barker, D.G. (2013). Short-chain chitin oligomers from arbuscular mycorrhizal fungi trigger nuclear Ca2+ spiking in Medicago truncatula roots and their production is enhanced by strigolactone. New Phytol. 198: 190–202. [[Genre_et+al_ShortchainChitinOligomersFromArbuscular_2013]] [OA](https://api.scholarcy.com/oa_version?query=Genre%2C%20A.%20Chabaud%2C%20M.%20Balzergue%2C%20C.%20Puech-Pag%C3%A8s%2C%20V.%20Short-chain%20chitin%20oligomers%20from%20arbuscular%20mycorrhizal%20fungi%20trigger%20nuclear%20Ca2%2B%20spiking%20in%20Medicago%20truncatula%20roots%20and%20their%20production%20is%20enhanced%20by%20strigolactone%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Genre%2C%20A.%20Chabaud%2C%20M.%20Balzergue%2C%20C.%20Puech-Pag%C3%A8s%2C%20V.%20Short-chain%20chitin%20oligomers%20from%20arbuscular%20mycorrhizal%20fungi%20trigger%20nuclear%20Ca2%2B%20spiking%20in%20Medicago%20truncatula%20roots%20and%20their%20production%20is%20enhanced%20by%20strigolactone%202013) [Scite](https://api.scholarcy.com/scite_url?query=Genre%2C%20A.%20Chabaud%2C%20M.%20Balzergue%2C%20C.%20Puech-Pag%C3%A8s%2C%20V.%20Short-chain%20chitin%20oligomers%20from%20arbuscular%20mycorrhizal%20fungi%20trigger%20nuclear%20Ca2%2B%20spiking%20in%20Medicago%20truncatula%20roots%20and%20their%20production%20is%20enhanced%20by%20strigolactone%202013)

[^Gobbato_et+al_2013_a]: Gobbato, E., Wang, E., Higgins, G., Bano, S.A., Henry, C., Schultze, M., and Oldroyd, G.E.D. (2013). RAM1 and RAM2 function and expression during arbuscular mycorrhizal symbiosis and Aphanomyces euteiches colonization. Plant Signal. Behav. 8: e26049. [[Gobbato_et+al_Ram1Ram2FunctionExpressionDuring_2013]] [OA](https://api.scholarcy.com/oa_version?query=Gobbato%2C%20E.%20Wang%2C%20E.%20Higgins%2C%20G.%20Bano%2C%20S.A.%20RAM1%20and%20RAM2%20function%20and%20expression%20during%20arbuscular%20mycorrhizal%20symbiosis%20and%20Aphanomyces%20euteiches%20colonization%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Gobbato%2C%20E.%20Wang%2C%20E.%20Higgins%2C%20G.%20Bano%2C%20S.A.%20RAM1%20and%20RAM2%20function%20and%20expression%20during%20arbuscular%20mycorrhizal%20symbiosis%20and%20Aphanomyces%20euteiches%20colonization%202013) [Scite](https://api.scholarcy.com/scite_url?query=Gobbato%2C%20E.%20Wang%2C%20E.%20Higgins%2C%20G.%20Bano%2C%20S.A.%20RAM1%20and%20RAM2%20function%20and%20expression%20during%20arbuscular%20mycorrhizal%20symbiosis%20and%20Aphanomyces%20euteiches%20colonization%202013)

[^Gobbato_2012_a]: Gobbato, E., et al. (2012). A GRAS-type transcription factor with a specific function in mycorrhizal signaling. Curr. Biol. 22: 2236–2241. [[Gobbato_GrastypeTranscriptionFactorWithSpecific_2012]] [OA](https://api.scholarcy.com/oa_version?query=Gobbato%2C%20E.%20A%20GRAS-type%20transcription%20factor%20with%20a%20specific%20function%20in%20mycorrhizal%20signaling%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Gobbato%2C%20E.%20A%20GRAS-type%20transcription%20factor%20with%20a%20specific%20function%20in%20mycorrhizal%20signaling%202012) [Scite](https://api.scholarcy.com/scite_url?query=Gobbato%2C%20E.%20A%20GRAS-type%20transcription%20factor%20with%20a%20specific%20function%20in%20mycorrhizal%20signaling%202012)

[^Gomez-Roldan_2008_a]: Gomez-Roldan, V., et al. (2008). Strigolactone inhibition of shoot branching. Nature 455: 189–194. [[Gomez-Roldan_StrigolactoneInhibitionShootBranching_2008]] [OA](https://api.scholarcy.com/oa_version?query=Gomez-Roldan%2C%20V.%20Strigolactone%20inhibition%20of%20shoot%20branching%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Gomez-Roldan%2C%20V.%20Strigolactone%20inhibition%20of%20shoot%20branching%202008) [Scite](https://api.scholarcy.com/scite_url?query=Gomez-Roldan%2C%20V.%20Strigolactone%20inhibition%20of%20shoot%20branching%202008)

[^Groth_et+al_2010_a]: Groth, M., Takeda, N., Perry, J., Uchida, H., Dräxl, S., Brachmann, A., Sato, S., Tabata, S., Kawaguchi, M., Wang, T.L., and Parniske, M. (2010). NENA, a Lotus japonicus homolog of Sec13, is required for rhizodermal infection by arbuscular mycorrhiza fungi and rhizobia but dispensable for cortical endosymbiotic development. Plant Cell 22: 2509–2526. [[Groth_et+al_NenaLotusJaponicusHomologSec13_2010]] [OA](https://api.scholarcy.com/oa_version?query=Groth%2C%20M.%20Takeda%2C%20N.%20Perry%2C%20J.%20Uchida%2C%20H.%20NENA%2C%20a%20Lotus%20japonicus%20homolog%20of%20Sec13%2C%20is%20required%20for%20rhizodermal%20infection%20by%20arbuscular%20mycorrhiza%20fungi%20and%20rhizobia%20but%20dispensable%20for%20cortical%20endosymbiotic%20development%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Groth%2C%20M.%20Takeda%2C%20N.%20Perry%2C%20J.%20Uchida%2C%20H.%20NENA%2C%20a%20Lotus%20japonicus%20homolog%20of%20Sec13%2C%20is%20required%20for%20rhizodermal%20infection%20by%20arbuscular%20mycorrhiza%20fungi%20and%20rhizobia%20but%20dispensable%20for%20cortical%20endosymbiotic%20development%202010) [Scite](https://api.scholarcy.com/scite_url?query=Groth%2C%20M.%20Takeda%2C%20N.%20Perry%2C%20J.%20Uchida%2C%20H.%20NENA%2C%20a%20Lotus%20japonicus%20homolog%20of%20Sec13%2C%20is%20required%20for%20rhizodermal%20infection%20by%20arbuscular%20mycorrhiza%20fungi%20and%20rhizobia%20but%20dispensable%20for%20cortical%20endosymbiotic%20development%202010)

[^Groth_et+al_2013_a]: Groth, M., Kosuta, S., Gutjahr, C., Haage, K., Hardel, S.L., Schaub, M., Brachmann, A., Sato, S., Tabata, S., Findlay, K., Wang, T.L., and Parniske, M. (2013). Two Lotus japonicus symbiosis mutants impaired at distinct steps of arbuscule development. Plant J. 75: 117–129. [[Groth_et+al_LotusJaponicusSymbiosisMutantsImpaired_2013]] [OA](https://api.scholarcy.com/oa_version?query=Groth%2C%20M.%20Kosuta%2C%20S.%20Gutjahr%2C%20C.%20Haage%2C%20K.%20Two%20Lotus%20japonicus%20symbiosis%20mutants%20impaired%20at%20distinct%20steps%20of%20arbuscule%20development%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Groth%2C%20M.%20Kosuta%2C%20S.%20Gutjahr%2C%20C.%20Haage%2C%20K.%20Two%20Lotus%20japonicus%20symbiosis%20mutants%20impaired%20at%20distinct%20steps%20of%20arbuscule%20development%202013) [Scite](https://api.scholarcy.com/scite_url?query=Groth%2C%20M.%20Kosuta%2C%20S.%20Gutjahr%2C%20C.%20Haage%2C%20K.%20Two%20Lotus%20japonicus%20symbiosis%20mutants%20impaired%20at%20distinct%20steps%20of%20arbuscule%20development%202013)

[^Gust_et+al_2012_a]: Gust, A.A., Willmann, R., Desaki, Y., Grabherr, H.M., and Nürnberger, T. (2012). Plant LysM proteins: modules mediating symbiosis and immunity. Trends Plant Sci. 17: 495–502. [[Gust_et+al_PlantLysmProteinsModulesMediating_2012]] [OA](https://api.scholarcy.com/oa_version?query=Gust%2C%20A.A.%20Willmann%2C%20R.%20Desaki%2C%20Y.%20Grabherr%2C%20H.M.%20Plant%20LysM%20proteins%3A%20modules%20mediating%20symbiosis%20and%20immunity%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Gust%2C%20A.A.%20Willmann%2C%20R.%20Desaki%2C%20Y.%20Grabherr%2C%20H.M.%20Plant%20LysM%20proteins%3A%20modules%20mediating%20symbiosis%20and%20immunity%202012) [Scite](https://api.scholarcy.com/scite_url?query=Gust%2C%20A.A.%20Willmann%2C%20R.%20Desaki%2C%20Y.%20Grabherr%2C%20H.M.%20Plant%20LysM%20proteins%3A%20modules%20mediating%20symbiosis%20and%20immunity%202012)

[^Gutjahr_2013_a]: Gutjahr, C., and Parniske, M. (2013). Cell and developmental biology of arbuscular mycorrhiza symbiosis. In Annual Review of Cell and Developmental Biology, Vol. 29, R. Schekman, ed (Palo Alto, CA: Annual Reviews), pp. 593–617. [[Gutjahr_CellDevelopmentalBiologyArbuscularMycorrhiza_2013]] [OA](https://api.scholarcy.com/oa_version?query=Gutjahr%2C%20C.%20Parniske%2C%20M.%20Cell%20and%20developmental%20biology%20of%20arbuscular%20mycorrhiza%20symbiosis%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Gutjahr%2C%20C.%20Parniske%2C%20M.%20Cell%20and%20developmental%20biology%20of%20arbuscular%20mycorrhiza%20symbiosis%202013) [Scite](https://api.scholarcy.com/scite_url?query=Gutjahr%2C%20C.%20Parniske%2C%20M.%20Cell%20and%20developmental%20biology%20of%20arbuscular%20mycorrhiza%20symbiosis%202013)

[^Gutjahr_et+al_2008_a]: Gutjahr, C., Banba, M., Croset, V., An, K., Miyao, A., An, G., Hirochika, H., Imaizumi-Anraku, H., and Paszkowski, U. (2008). Arbuscular mycorrhiza-specific signaling in rice transcends the common symbiosis signaling pathway. Plant Cell 20: 2989–3005. [[Gutjahr_et+al_ArbuscularMycorrhizaspecificSignalingRiceTranscends_2008]] [OA](https://scholar.google.co.uk/scholar?q=Gutjahr%2C%20C.%20Banba%2C%20M.%20Croset%2C%20V.%20An%2C%20K.%20Arbuscular%20mycorrhiza-specific%20signaling%20in%20rice%20transcends%20the%20common%20symbiosis%20signaling%20pathway.%20Plant%20Cell%2020%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Gutjahr%2C%20C.%20Banba%2C%20M.%20Croset%2C%20V.%20An%2C%20K.%20Arbuscular%20mycorrhiza-specific%20signaling%20in%20rice%20transcends%20the%20common%20symbiosis%20signaling%20pathway.%20Plant%20Cell%2020%202008) 

[^Gutjahr_2012_a]: Gutjahr, C., et al. (2012). The half-size ABC transporters STR1 and STR2 are indispensable for mycorrhizal arbuscule formation in rice. Plant J. 69: 906–920. [[Gutjahr_HalfsizeAbcTransportersStr1Str2_2012]] [OA](https://api.scholarcy.com/oa_version?query=Gutjahr%2C%20C.%20The%20half-size%20ABC%20transporters%20STR1%20and%20STR2%20are%20indispensable%20for%20mycorrhizal%20arbuscule%20formation%20in%20rice%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Gutjahr%2C%20C.%20The%20half-size%20ABC%20transporters%20STR1%20and%20STR2%20are%20indispensable%20for%20mycorrhizal%20arbuscule%20formation%20in%20rice%202012) [Scite](https://api.scholarcy.com/scite_url?query=Gutjahr%2C%20C.%20The%20half-size%20ABC%20transporters%20STR1%20and%20STR2%20are%20indispensable%20for%20mycorrhizal%20arbuscule%20formation%20in%20rice%202012)

[^Gutjahr_2015_a]: Gutjahr, C., et al. (2015). Rice perception of symbiotic arbuscular mycorrhizal fungi requires the karrikin receptor complex. Science 350: 1521–1524. [[Gutjahr_RicePerceptionSymbioticArbuscularMycorrhizal_2015]] [OA](https://api.scholarcy.com/oa_version?query=Gutjahr%2C%20C.%20Rice%20perception%20of%20symbiotic%20arbuscular%20mycorrhizal%20fungi%20requires%20the%20karrikin%20receptor%20complex%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Gutjahr%2C%20C.%20Rice%20perception%20of%20symbiotic%20arbuscular%20mycorrhizal%20fungi%20requires%20the%20karrikin%20receptor%20complex%202015) [Scite](https://api.scholarcy.com/scite_url?query=Gutjahr%2C%20C.%20Rice%20perception%20of%20symbiotic%20arbuscular%20mycorrhizal%20fungi%20requires%20the%20karrikin%20receptor%20complex%202015)

[^Harrison_et+al_2002_a]: Harrison, M.J., Dewbre, G.R., and Liu, J. (2002). A phosphate transporter from Medicago truncatula involved in the acquisition of phosphate released by arbuscular mycorrhizal fungi. Plant Cell 14: 2413–2429. [[Harrison_et+al_PhosphateTransporterFromMedicagoTruncatula_2002]] [OA](https://api.scholarcy.com/oa_version?query=Harrison%2C%20M.J.%20Dewbre%2C%20G.R.%20Liu%2C%20J.%20A%20phosphate%20transporter%20from%20Medicago%20truncatula%20involved%20in%20the%20acquisition%20of%20phosphate%20released%20by%20arbuscular%20mycorrhizal%20fungi%202002) [GScholar](https://scholar.google.co.uk/scholar?q=Harrison%2C%20M.J.%20Dewbre%2C%20G.R.%20Liu%2C%20J.%20A%20phosphate%20transporter%20from%20Medicago%20truncatula%20involved%20in%20the%20acquisition%20of%20phosphate%20released%20by%20arbuscular%20mycorrhizal%20fungi%202002) [Scite](https://api.scholarcy.com/scite_url?query=Harrison%2C%20M.J.%20Dewbre%2C%20G.R.%20Liu%2C%20J.%20A%20phosphate%20transporter%20from%20Medicago%20truncatula%20involved%20in%20the%20acquisition%20of%20phosphate%20released%20by%20arbuscular%20mycorrhizal%20fungi%202002)

[^Hayashi_et+al_2010_a]: Hayashi, T., Banba, M., Shimoda, Y., Kouchi, H., Hayashi, M., and Imaizumi-Anraku, H. (2010). A dominant function of CCaMK in intracellular accommodation of bacterial and fungal endosymbionts. Plant J. 63: 141–154. [[Hayashi_et+al_DominantFunctionCcamkIntracellularAccommodation_2010]] [OA](https://api.scholarcy.com/oa_version?query=Hayashi%2C%20T.%20Banba%2C%20M.%20Shimoda%2C%20Y.%20Kouchi%2C%20H.%20A%20dominant%20function%20of%20CCaMK%20in%20intracellular%20accommodation%20of%20bacterial%20and%20fungal%20endosymbionts%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Hayashi%2C%20T.%20Banba%2C%20M.%20Shimoda%2C%20Y.%20Kouchi%2C%20H.%20A%20dominant%20function%20of%20CCaMK%20in%20intracellular%20accommodation%20of%20bacterial%20and%20fungal%20endosymbionts%202010) [Scite](https://api.scholarcy.com/scite_url?query=Hayashi%2C%20T.%20Banba%2C%20M.%20Shimoda%2C%20Y.%20Kouchi%2C%20H.%20A%20dominant%20function%20of%20CCaMK%20in%20intracellular%20accommodation%20of%20bacterial%20and%20fungal%20endosymbionts%202010)

[^Heck_et+al_2016_a]: Heck, C., Kuhn, H., Heidt, S., Walter, S., Rieger, N., and Requena, N. (2016). Symbiotic fungi control plant root cortex development through the novel GRAS transcription factor MIG1. Curr. Biol. 26: 2770–2778. [[Heck_et+al_SymbioticFungiControlPlantRoot_2016]] [OA](https://api.scholarcy.com/oa_version?query=Heck%2C%20C.%20Kuhn%2C%20H.%20Heidt%2C%20S.%20Walter%2C%20S.%20Symbiotic%20fungi%20control%20plant%20root%20cortex%20development%20through%20the%20novel%20GRAS%20transcription%20factor%20MIG1%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Heck%2C%20C.%20Kuhn%2C%20H.%20Heidt%2C%20S.%20Walter%2C%20S.%20Symbiotic%20fungi%20control%20plant%20root%20cortex%20development%20through%20the%20novel%20GRAS%20transcription%20factor%20MIG1%202016) [Scite](https://api.scholarcy.com/scite_url?query=Heck%2C%20C.%20Kuhn%2C%20H.%20Heidt%2C%20S.%20Walter%2C%20S.%20Symbiotic%20fungi%20control%20plant%20root%20cortex%20development%20through%20the%20novel%20GRAS%20transcription%20factor%20MIG1%202016)

[^Heckman_et+al_2001_a]: Heckman, D.S., Geiser, D.M., Eidell, B.R., Stauffer, R.L., Kardos, N.L., and Hedges, S.B. (2001). Molecular evidence for the early colonization of land by fungi and plants. Science 293: 1129–1133. [[Heckman_et+al_MolecularEvidenceEarlyColonizationLand_2001]] [OA](https://api.scholarcy.com/oa_version?query=Heckman%2C%20D.S.%20Geiser%2C%20D.M.%20Eidell%2C%20B.R.%20Stauffer%2C%20R.L.%20Molecular%20evidence%20for%20the%20early%20colonization%20of%20land%20by%20fungi%20and%20plants%202001) [GScholar](https://scholar.google.co.uk/scholar?q=Heckman%2C%20D.S.%20Geiser%2C%20D.M.%20Eidell%2C%20B.R.%20Stauffer%2C%20R.L.%20Molecular%20evidence%20for%20the%20early%20colonization%20of%20land%20by%20fungi%20and%20plants%202001) [Scite](https://api.scholarcy.com/scite_url?query=Heckman%2C%20D.S.%20Geiser%2C%20D.M.%20Eidell%2C%20B.R.%20Stauffer%2C%20R.L.%20Molecular%20evidence%20for%20the%20early%20colonization%20of%20land%20by%20fungi%20and%20plants%202001)

[^Helber_et+al_2011_a]: Helber, N., Wippel, K., Sauer, N., Schaarschmidt, S., Hause, B., and Requena, N. (2011). A versatile monosaccharide transporter that operates in the arbuscular mycorrhizal fungus Glomus sp is crucial for the symbiotic relationship with plants. Plant Cell 23: 3812–3823. [[Helber_et+al_VersatileMonosaccharideTransporterThatOperates_2011]] [OA](https://api.scholarcy.com/oa_version?query=Helber%2C%20N.%20Wippel%2C%20K.%20Sauer%2C%20N.%20Schaarschmidt%2C%20S.%20A%20versatile%20monosaccharide%20transporter%20that%20operates%20in%20the%20arbuscular%20mycorrhizal%20fungus%20Glomus%20sp%20is%20crucial%20for%20the%20symbiotic%20relationship%20with%20plants%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Helber%2C%20N.%20Wippel%2C%20K.%20Sauer%2C%20N.%20Schaarschmidt%2C%20S.%20A%20versatile%20monosaccharide%20transporter%20that%20operates%20in%20the%20arbuscular%20mycorrhizal%20fungus%20Glomus%20sp%20is%20crucial%20for%20the%20symbiotic%20relationship%20with%20plants%202011) [Scite](https://api.scholarcy.com/scite_url?query=Helber%2C%20N.%20Wippel%2C%20K.%20Sauer%2C%20N.%20Schaarschmidt%2C%20S.%20A%20versatile%20monosaccharide%20transporter%20that%20operates%20in%20the%20arbuscular%20mycorrhizal%20fungus%20Glomus%20sp%20is%20crucial%20for%20the%20symbiotic%20relationship%20with%20plants%202011)

[^Hijikata_et+al_2010_a]: Hijikata, N., Murase, M., Tani, C., Ohtomo, R., Osaki, M., and Ezawa, T. (2010). Polyphosphate has a central role in the rapid and massive accumulation of phosphorus in extraradical mycelium of an arbuscular mycorrhizal fungus. New Phytol. 186: 285–289. [[Hijikata_et+al_PolyphosphateCentralRoleRapidMassive_2010]] [OA](https://api.scholarcy.com/oa_version?query=Hijikata%2C%20N.%20Murase%2C%20M.%20Tani%2C%20C.%20Ohtomo%2C%20R.%20Polyphosphate%20has%20a%20central%20role%20in%20the%20rapid%20and%20massive%20accumulation%20of%20phosphorus%20in%20extraradical%20mycelium%20of%20an%20arbuscular%20mycorrhizal%20fungus%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Hijikata%2C%20N.%20Murase%2C%20M.%20Tani%2C%20C.%20Ohtomo%2C%20R.%20Polyphosphate%20has%20a%20central%20role%20in%20the%20rapid%20and%20massive%20accumulation%20of%20phosphorus%20in%20extraradical%20mycelium%20of%20an%20arbuscular%20mycorrhizal%20fungus%202010) [Scite](https://api.scholarcy.com/scite_url?query=Hijikata%2C%20N.%20Murase%2C%20M.%20Tani%2C%20C.%20Ohtomo%2C%20R.%20Polyphosphate%20has%20a%20central%20role%20in%20the%20rapid%20and%20massive%20accumulation%20of%20phosphorus%20in%20extraradical%20mycelium%20of%20an%20arbuscular%20mycorrhizal%20fungus%202010)

[^Hirano_et+al_2017_a]: Hirano, Y., Nakagawa, M., Suyama, T., Murase, K., Shirakawa, M., Takayama, S., Sun, T.P., and Hakoshima, T. (2017). Structure of the SHR-SCR heterodimer bound to the BIRD/IDD transcriptional factor JKD. Nat. Plants 3: 17010. [[Hirano_et+al_StructureShrscrHeterodimerBoundBirdidd_2017]] [OA](https://api.scholarcy.com/oa_version?query=Hirano%2C%20Y.%20Nakagawa%2C%20M.%20Suyama%2C%20T.%20Murase%2C%20K.%20Structure%20of%20the%20SHR-SCR%20heterodimer%20bound%20to%20the%20BIRD/IDD%20transcriptional%20factor%20JKD%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Hirano%2C%20Y.%20Nakagawa%2C%20M.%20Suyama%2C%20T.%20Murase%2C%20K.%20Structure%20of%20the%20SHR-SCR%20heterodimer%20bound%20to%20the%20BIRD/IDD%20transcriptional%20factor%20JKD%202017) [Scite](https://api.scholarcy.com/scite_url?query=Hirano%2C%20Y.%20Nakagawa%2C%20M.%20Suyama%2C%20T.%20Murase%2C%20K.%20Structure%20of%20the%20SHR-SCR%20heterodimer%20bound%20to%20the%20BIRD/IDD%20transcriptional%20factor%20JKD%202017)

[^Hirsch_et+al_2009_a]: Hirsch, S., Kim, J., Muñoz, A., Heckmann, A.B., Downie, J.A., and Oldroyd, G.E.D. (2009). GRAS proteins form a DNA binding complex to induce gene expression during nodulation signaling in Medicago truncatula. Plant Cell 21: 545–557. [[Hirsch_et+al_GrasProteinsFormDnaBinding_2009]] [OA](https://api.scholarcy.com/oa_version?query=Hirsch%2C%20S.%20Kim%2C%20J.%20Mu%C3%B1oz%2C%20A.%20Heckmann%2C%20A.B.%20GRAS%20proteins%20form%20a%20DNA%20binding%20complex%20to%20induce%20gene%20expression%20during%20nodulation%20signaling%20in%20Medicago%20truncatula%202009) [GScholar](https://scholar.google.co.uk/scholar?q=Hirsch%2C%20S.%20Kim%2C%20J.%20Mu%C3%B1oz%2C%20A.%20Heckmann%2C%20A.B.%20GRAS%20proteins%20form%20a%20DNA%20binding%20complex%20to%20induce%20gene%20expression%20during%20nodulation%20signaling%20in%20Medicago%20truncatula%202009) [Scite](https://api.scholarcy.com/scite_url?query=Hirsch%2C%20S.%20Kim%2C%20J.%20Mu%C3%B1oz%2C%20A.%20Heckmann%2C%20A.B.%20GRAS%20proteins%20form%20a%20DNA%20binding%20complex%20to%20induce%20gene%20expression%20during%20nodulation%20signaling%20in%20Medicago%20truncatula%202009)

[^Hogekamp_2013_a]: Hogekamp, C., and Küster, H. (2013). A roadmap of cell-type specific gene expression during sequential stages of the arbuscular mycorrhiza symbiosis. BMC Genomics 14: 306. [[Hogekamp_RoadmapCelltypeSpecificGeneExpression_2013]] [OA](https://api.scholarcy.com/oa_version?query=Hogekamp%2C%20C.%20K%C3%BCster%2C%20H.%20A%20roadmap%20of%20cell-type%20specific%20gene%20expression%20during%20sequential%20stages%20of%20the%20arbuscular%20mycorrhiza%20symbiosis%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Hogekamp%2C%20C.%20K%C3%BCster%2C%20H.%20A%20roadmap%20of%20cell-type%20specific%20gene%20expression%20during%20sequential%20stages%20of%20the%20arbuscular%20mycorrhiza%20symbiosis%202013) [Scite](https://api.scholarcy.com/scite_url?query=Hogekamp%2C%20C.%20K%C3%BCster%2C%20H.%20A%20roadmap%20of%20cell-type%20specific%20gene%20expression%20during%20sequential%20stages%20of%20the%20arbuscular%20mycorrhiza%20symbiosis%202013)

[^Hohnjec_et+al_2015_a]: Hohnjec, N., Czaja-Hasse, L.F., Hogekamp, C., and Küster, H. (2015). Pre-announcement of symbiotic guests: transcriptional reprogramming by mycorrhizal lipochitooligosaccharides shows a strict co-dependency on the GRAS transcription factors NSP1 and RAM1. BMC Genomics 16: 994. [[Hohnjec_et+al_announcementSymbioticGuestsTranscriptionalReprogramming_2015]] [OA](https://api.scholarcy.com/oa_version?query=Hohnjec%2C%20N.%20Czaja-Hasse%2C%20L.F.%20Hogekamp%2C%20C.%20K%C3%BCster%2C%20H.%20Pre-announcement%20of%20symbiotic%20guests%3A%20transcriptional%20reprogramming%20by%20mycorrhizal%20lipochitooligosaccharides%20shows%20a%20strict%20co-dependency%20on%20the%20GRAS%20transcription%20factors%20NSP1%20and%20RAM1%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Hohnjec%2C%20N.%20Czaja-Hasse%2C%20L.F.%20Hogekamp%2C%20C.%20K%C3%BCster%2C%20H.%20Pre-announcement%20of%20symbiotic%20guests%3A%20transcriptional%20reprogramming%20by%20mycorrhizal%20lipochitooligosaccharides%20shows%20a%20strict%20co-dependency%20on%20the%20GRAS%20transcription%20factors%20NSP1%20and%20RAM1%202015) [Scite](https://api.scholarcy.com/scite_url?query=Hohnjec%2C%20N.%20Czaja-Hasse%2C%20L.F.%20Hogekamp%2C%20C.%20K%C3%BCster%2C%20H.%20Pre-announcement%20of%20symbiotic%20guests%3A%20transcriptional%20reprogramming%20by%20mycorrhizal%20lipochitooligosaccharides%20shows%20a%20strict%20co-dependency%20on%20the%20GRAS%20transcription%20factors%20NSP1%20and%20RAM1%202015)

[^Horváth_2011_a]: Horváth, B., et al. (2011). Medicago truncatula IPD3 is a member of the common symbiotic signaling pathway required for rhizobial and mycorrhizal symbioses. Mol. Plant Microbe Interact. 24: 1345–1358. [[Horváth_MedicagoTruncatulaIpd3MemberCommon_2011]] [OA](https://api.scholarcy.com/oa_version?query=Horv%C3%A1th%2C%20B.%20Medicago%20truncatula%20IPD3%20is%20a%20member%20of%20the%20common%20symbiotic%20signaling%20pathway%20required%20for%20rhizobial%20and%20mycorrhizal%20symbioses%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Horv%C3%A1th%2C%20B.%20Medicago%20truncatula%20IPD3%20is%20a%20member%20of%20the%20common%20symbiotic%20signaling%20pathway%20required%20for%20rhizobial%20and%20mycorrhizal%20symbioses%202011) [Scite](https://api.scholarcy.com/scite_url?query=Horv%C3%A1th%2C%20B.%20Medicago%20truncatula%20IPD3%20is%20a%20member%20of%20the%20common%20symbiotic%20signaling%20pathway%20required%20for%20rhizobial%20and%20mycorrhizal%20symbioses%202011)

[^Huisman_et+al_2016_a]: Huisman, R., Hontelez, J., Mysore, K.S., Wen, J., Bisseling, T., and Limpens, E. (2016). A symbiosis-dedicated SYNTAXIN OF PLANTS 13II isoform controls the formation of a stable host-microbe interface in symbiosis. New Phytol. 211: 1338–1351. [[Huisman_et+al_SymbiosisdedicatedSyntaxinOfPlants13ii_2016]] [OA](https://api.scholarcy.com/oa_version?query=Huisman%2C%20R.%20Hontelez%2C%20J.%20Mysore%2C%20K.S.%20Wen%2C%20J.%20A%20symbiosis-dedicated%20SYNTAXIN%20OF%20PLANTS%2013II%20isoform%20controls%20the%20formation%20of%20a%20stable%20host-microbe%20interface%20in%20symbiosis%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Huisman%2C%20R.%20Hontelez%2C%20J.%20Mysore%2C%20K.S.%20Wen%2C%20J.%20A%20symbiosis-dedicated%20SYNTAXIN%20OF%20PLANTS%2013II%20isoform%20controls%20the%20formation%20of%20a%20stable%20host-microbe%20interface%20in%20symbiosis%202016) [Scite](https://api.scholarcy.com/scite_url?query=Huisman%2C%20R.%20Hontelez%2C%20J.%20Mysore%2C%20K.S.%20Wen%2C%20J.%20A%20symbiosis-dedicated%20SYNTAXIN%20OF%20PLANTS%2013II%20isoform%20controls%20the%20formation%20of%20a%20stable%20host-microbe%20interface%20in%20symbiosis%202016)

[^Imaizumi-Anraku_2005_a]: Imaizumi-Anraku, H., et al. (2005). Plastid proteins crucial for symbiotic fungal and bacterial entry into plant roots. Nature 433: 527– 531. [[Imaizumi-Anraku_PlastidProteinsCrucialSymbioticFungal_2005]] [OA](https://api.scholarcy.com/oa_version?query=Imaizumi-Anraku%2C%20H.%20Plastid%20proteins%20crucial%20for%20symbiotic%20fungal%20and%20bacterial%20entry%20into%20plant%20roots%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Imaizumi-Anraku%2C%20H.%20Plastid%20proteins%20crucial%20for%20symbiotic%20fungal%20and%20bacterial%20entry%20into%20plant%20roots%202005) [Scite](https://api.scholarcy.com/scite_url?query=Imaizumi-Anraku%2C%20H.%20Plastid%20proteins%20crucial%20for%20symbiotic%20fungal%20and%20bacterial%20entry%20into%20plant%20roots%202005)

[^Ivanov_et+al_2012_a]: Ivanov, S., Fedorova, E.E., Limpens, E., De Mita, S., Genre, A., Bonfante, P., and Bisseling, T. (2012). Rhizobium-legume symbiosis shares an exocytotic pathway required for arbuscule formation. Proc. Natl. Acad. Sci. USA 109: 8316–8321. [[Ivanov_et+al_RhizobiumlegumeSymbiosisSharesExocytoticPathway_2012]] [OA](https://api.scholarcy.com/oa_version?query=Ivanov%2C%20S.%20Fedorova%2C%20E.E.%20Limpens%2C%20E.%20De%20Mita%2C%20S.%20Rhizobium-legume%20symbiosis%20shares%20an%20exocytotic%20pathway%20required%20for%20arbuscule%20formation%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Ivanov%2C%20S.%20Fedorova%2C%20E.E.%20Limpens%2C%20E.%20De%20Mita%2C%20S.%20Rhizobium-legume%20symbiosis%20shares%20an%20exocytotic%20pathway%20required%20for%20arbuscule%20formation%202012) [Scite](https://api.scholarcy.com/scite_url?query=Ivanov%2C%20S.%20Fedorova%2C%20E.E.%20Limpens%2C%20E.%20De%20Mita%2C%20S.%20Rhizobium-legume%20symbiosis%20shares%20an%20exocytotic%20pathway%20required%20for%20arbuscule%20formation%202012)

[^Javot_et+al_2007_a]: Javot, H., Penmetsa, R.V., Terzaghi, N., Cook, D.R., and Harrison, M.J. (2007). A Medicago truncatula phosphate transporter indispensable for the arbuscular mycorrhizal symbiosis. Proc. Natl. Acad. Sci. USA 104: 1720–1725. [[Javot_et+al_MedicagoTruncatulaPhosphateTransporterIndispensable_2007]] [OA](https://api.scholarcy.com/oa_version?query=Javot%2C%20H.%20Penmetsa%2C%20R.V.%20Terzaghi%2C%20N.%20Cook%2C%20D.R.%20A%20Medicago%20truncatula%20phosphate%20transporter%20indispensable%20for%20the%20arbuscular%20mycorrhizal%20symbiosis%202007) [GScholar](https://scholar.google.co.uk/scholar?q=Javot%2C%20H.%20Penmetsa%2C%20R.V.%20Terzaghi%2C%20N.%20Cook%2C%20D.R.%20A%20Medicago%20truncatula%20phosphate%20transporter%20indispensable%20for%20the%20arbuscular%20mycorrhizal%20symbiosis%202007) [Scite](https://api.scholarcy.com/scite_url?query=Javot%2C%20H.%20Penmetsa%2C%20R.V.%20Terzaghi%2C%20N.%20Cook%2C%20D.R.%20A%20Medicago%20truncatula%20phosphate%20transporter%20indispensable%20for%20the%20arbuscular%20mycorrhizal%20symbiosis%202007)

[^Jiang_et+al_2007_a]: Jiang, C., Gao, X., Liao, L., Harberd, N.P., and Fu, X. (2007). Phosphate starvation root architecture and anthocyanin accumulation responses are modulated by the gibberellin-DELLA signaling pathway in Arabidopsis. Plant Physiol. 145: 1460–1470. [[Jiang_et+al_PhosphateStarvationRootArchitectureAnthocyanin_2007]] [OA](https://api.scholarcy.com/oa_version?query=Jiang%2C%20C.%20Gao%2C%20X.%20Liao%2C%20L.%20Harberd%2C%20N.P.%20Phosphate%20starvation%20root%20architecture%20and%20anthocyanin%20accumulation%20responses%20are%20modulated%20by%20the%20gibberellin-DELLA%20signaling%20pathway%20in%20Arabidopsis%202007) [GScholar](https://scholar.google.co.uk/scholar?q=Jiang%2C%20C.%20Gao%2C%20X.%20Liao%2C%20L.%20Harberd%2C%20N.P.%20Phosphate%20starvation%20root%20architecture%20and%20anthocyanin%20accumulation%20responses%20are%20modulated%20by%20the%20gibberellin-DELLA%20signaling%20pathway%20in%20Arabidopsis%202007) [Scite](https://api.scholarcy.com/scite_url?query=Jiang%2C%20C.%20Gao%2C%20X.%20Liao%2C%20L.%20Harberd%2C%20N.P.%20Phosphate%20starvation%20root%20architecture%20and%20anthocyanin%20accumulation%20responses%20are%20modulated%20by%20the%20gibberellin-DELLA%20signaling%20pathway%20in%20Arabidopsis%202007)

[^Jiang_et+al_2017_a]: Jiang, Y., Wang, W., Xie, Q., Liu, N., Liu, L., Wang, D., Zhang, X., Yang, C., Chen, X., Tang, D., and Wang, E. (2017). Plants transfer lipids to sustain colonization by mutualistic mycorrhizal and parasitic fungi. Science 356: 1172–1175. [[Jiang_et+al_PlantsTransferLipidsSustainColonization_2017]] [OA](https://api.scholarcy.com/oa_version?query=Jiang%2C%20Y.%20Wang%2C%20W.%20Xie%2C%20Q.%20Liu%2C%20N.%20Plants%20transfer%20lipids%20to%20sustain%20colonization%20by%20mutualistic%20mycorrhizal%20and%20parasitic%20fungi%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Jiang%2C%20Y.%20Wang%2C%20W.%20Xie%2C%20Q.%20Liu%2C%20N.%20Plants%20transfer%20lipids%20to%20sustain%20colonization%20by%20mutualistic%20mycorrhizal%20and%20parasitic%20fungi%202017) [Scite](https://api.scholarcy.com/scite_url?query=Jiang%2C%20Y.%20Wang%2C%20W.%20Xie%2C%20Q.%20Liu%2C%20N.%20Plants%20transfer%20lipids%20to%20sustain%20colonization%20by%20mutualistic%20mycorrhizal%20and%20parasitic%20fungi%202017)

[^Jin_et+al_2016_a]: Jin, Y., Liu, H., Luo, D., Yu, N., Dong, W., Wang, C., Zhang, X., Dai, H., Yang, J., and Wang, E. (2016). DELLA proteins are common components of symbiotic rhizobial and mycorrhizal signalling pathways. Nat. Commun. 7: 12433. [[Jin_et+al_DellaProteinsCommonComponentsSymbiotic_2016]] [OA](https://api.scholarcy.com/oa_version?query=Jin%2C%20Y.%20Liu%2C%20H.%20Luo%2C%20D.%20Yu%2C%20N.%20DELLA%20proteins%20are%20common%20components%20of%20symbiotic%20rhizobial%20and%20mycorrhizal%20signalling%20pathways%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Jin%2C%20Y.%20Liu%2C%20H.%20Luo%2C%20D.%20Yu%2C%20N.%20DELLA%20proteins%20are%20common%20components%20of%20symbiotic%20rhizobial%20and%20mycorrhizal%20signalling%20pathways%202016) [Scite](https://api.scholarcy.com/scite_url?query=Jin%2C%20Y.%20Liu%2C%20H.%20Luo%2C%20D.%20Yu%2C%20N.%20DELLA%20proteins%20are%20common%20components%20of%20symbiotic%20rhizobial%20and%20mycorrhizal%20signalling%20pathways%202016)

[^Kaku_et+al_2006_a]: Kaku, H., Nishizawa, Y., Ishii-Minami, N., Akimoto-Tomiyama, C., Dohmae, N., Takio, K., Minami, E., and Shibuya, N. (2006). Plant cells recognize chitin fragments for defense signaling through a plasma membrane receptor. Proc. Natl. Acad. Sci. USA 103: 11086–11091. [[Kaku_et+al_PlantCellsRecognizeChitinFragments_2006]] [OA](https://api.scholarcy.com/oa_version?query=Kaku%2C%20H.%20Nishizawa%2C%20Y.%20Ishii-Minami%2C%20N.%20Akimoto-Tomiyama%2C%20C.%20Plant%20cells%20recognize%20chitin%20fragments%20for%20defense%20signaling%20through%20a%20plasma%20membrane%20receptor%202006) [GScholar](https://scholar.google.co.uk/scholar?q=Kaku%2C%20H.%20Nishizawa%2C%20Y.%20Ishii-Minami%2C%20N.%20Akimoto-Tomiyama%2C%20C.%20Plant%20cells%20recognize%20chitin%20fragments%20for%20defense%20signaling%20through%20a%20plasma%20membrane%20receptor%202006) [Scite](https://api.scholarcy.com/scite_url?query=Kaku%2C%20H.%20Nishizawa%2C%20Y.%20Ishii-Minami%2C%20N.%20Akimoto-Tomiyama%2C%20C.%20Plant%20cells%20recognize%20chitin%20fragments%20for%20defense%20signaling%20through%20a%20plasma%20membrane%20receptor%202006)

[^Kaló_2005_a]: Kaló, P., et al. (2005). Nodulation signaling in legumes requires NSP2, a member of the GRAS family of transcriptional regulators. Science 308: 1786–1789. [[Kaló_NodulationSignalingLegumesRequiresNsp2_2005]] [OA](https://api.scholarcy.com/oa_version?query=Kal%C3%B3%2C%20P.%20Nodulation%20signaling%20in%20legumes%20requires%20NSP2%2C%20a%20member%20of%20the%20GRAS%20family%20of%20transcriptional%20regulators%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Kal%C3%B3%2C%20P.%20Nodulation%20signaling%20in%20legumes%20requires%20NSP2%2C%20a%20member%20of%20the%20GRAS%20family%20of%20transcriptional%20regulators%202005) [Scite](https://api.scholarcy.com/scite_url?query=Kal%C3%B3%2C%20P.%20Nodulation%20signaling%20in%20legumes%20requires%20NSP2%2C%20a%20member%20of%20the%20GRAS%20family%20of%20transcriptional%20regulators%202005)

[^Kamel_et+al_2017_a]: Kamel, L., Keller-Pearson, M., Roux, C., and Ané, J.M. (2017). Biology and evolution of arbuscular mycorrhizal symbiosis in the light of genomics. New Phytol. 213: 531–536. [[Kamel_et+al_BiologyEvolutionArbuscularMycorrhizalSymbiosis_2017]] [OA](https://api.scholarcy.com/oa_version?query=Kamel%2C%20L.%20Keller-Pearson%2C%20M.%20Roux%2C%20C.%20An%C3%A9%2C%20J.M.%20Biology%20and%20evolution%20of%20arbuscular%20mycorrhizal%20symbiosis%20in%20the%20light%20of%20genomics%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Kamel%2C%20L.%20Keller-Pearson%2C%20M.%20Roux%2C%20C.%20An%C3%A9%2C%20J.M.%20Biology%20and%20evolution%20of%20arbuscular%20mycorrhizal%20symbiosis%20in%20the%20light%20of%20genomics%202017) [Scite](https://api.scholarcy.com/scite_url?query=Kamel%2C%20L.%20Keller-Pearson%2C%20M.%20Roux%2C%20C.%20An%C3%A9%2C%20J.M.%20Biology%20and%20evolution%20of%20arbuscular%20mycorrhizal%20symbiosis%20in%20the%20light%20of%20genomics%202017)

[^Kanamori_2006_a]: Kanamori, N., et al. (2006). A nucleoporin is required for induction of Ca2+ spiking in legume nodule development and essential for rhizobial and fungal symbiosis. Proc. Natl. Acad. Sci. USA 103: 359– 364. [[Kanamori_NucleoporinRequiredInductionCa2Spiking_2006]] [OA](https://api.scholarcy.com/oa_version?query=Kanamori%2C%20N.%20A%20nucleoporin%20is%20required%20for%20induction%20of%20Ca2%2B%20spiking%20in%20legume%20nodule%20development%20and%20essential%20for%20rhizobial%20and%20fungal%20symbiosis%202006) [GScholar](https://scholar.google.co.uk/scholar?q=Kanamori%2C%20N.%20A%20nucleoporin%20is%20required%20for%20induction%20of%20Ca2%2B%20spiking%20in%20legume%20nodule%20development%20and%20essential%20for%20rhizobial%20and%20fungal%20symbiosis%202006) [Scite](https://api.scholarcy.com/scite_url?query=Kanamori%2C%20N.%20A%20nucleoporin%20is%20required%20for%20induction%20of%20Ca2%2B%20spiking%20in%20legume%20nodule%20development%20and%20essential%20for%20rhizobial%20and%20fungal%20symbiosis%202006)

[^Kevei_2007_a]: Kevei, Z., et al. (2007). 3-Hydroxy-3-Methylglutaryl Coenzyme A Reductase1 interacts with NORK and is crucial for nodulation in Medicago truncatula. Plant Cell 19: 3974–3989. [[Kevei_3hydroxy3methylglutarylCoenzymeAReductase1Interacts_2007]] [OA](https://api.scholarcy.com/oa_version?query=Kevei%2C%20Z.%203-Hydroxy-3-Methylglutaryl%20Coenzyme%20A%20Reductase1%20interacts%20with%20NORK%20and%20is%20crucial%20for%20nodulation%20in%20Medicago%20truncatula%202007) [GScholar](https://scholar.google.co.uk/scholar?q=Kevei%2C%20Z.%203-Hydroxy-3-Methylglutaryl%20Coenzyme%20A%20Reductase1%20interacts%20with%20NORK%20and%20is%20crucial%20for%20nodulation%20in%20Medicago%20truncatula%202007) [Scite](https://api.scholarcy.com/scite_url?query=Kevei%2C%20Z.%203-Hydroxy-3-Methylglutaryl%20Coenzyme%20A%20Reductase1%20interacts%20with%20NORK%20and%20is%20crucial%20for%20nodulation%20in%20Medicago%20truncatula%202007)

[^Keymer_2017_a]: Keymer, A., et al. (2017). Lipid transfer from plants to arbuscular mycorrhiza fungi. eLife 6: e29107. [[Keymer_LipidTransferFromPlantsArbuscular_2017]] [OA](https://api.scholarcy.com/oa_version?query=Keymer%2C%20A.%20Lipid%20transfer%20from%20plants%20to%20arbuscular%20mycorrhiza%20fungi%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Keymer%2C%20A.%20Lipid%20transfer%20from%20plants%20to%20arbuscular%20mycorrhiza%20fungi%202017) [Scite](https://api.scholarcy.com/scite_url?query=Keymer%2C%20A.%20Lipid%20transfer%20from%20plants%20to%20arbuscular%20mycorrhiza%20fungi%202017)

[^Kistner_et+al_2005_a]: Kistner, C., Winzer, T., Pitzschke, A., Mulder, L., Sato, S., Kaneko, T., Tabata, S., Sandal, N., Stougaard, J., Webb, K.J., Szczyglowski, K., and Parniske, M. (2005). Seven Lotus japonicus genes required for transcriptional reprogramming of the root during fungal and bacterial symbiosis. Plant Cell 17: 2217–2229. [[Kistner_et+al_SevenLotusJaponicusGenesRequired_2005]] [OA](https://api.scholarcy.com/oa_version?query=Kistner%2C%20C.%20Winzer%2C%20T.%20Pitzschke%2C%20A.%20Mulder%2C%20L.%20Seven%20Lotus%20japonicus%20genes%20required%20for%20transcriptional%20reprogramming%20of%20the%20root%20during%20fungal%20and%20bacterial%20symbiosis%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Kistner%2C%20C.%20Winzer%2C%20T.%20Pitzschke%2C%20A.%20Mulder%2C%20L.%20Seven%20Lotus%20japonicus%20genes%20required%20for%20transcriptional%20reprogramming%20of%20the%20root%20during%20fungal%20and%20bacterial%20symbiosis%202005) [Scite](https://api.scholarcy.com/scite_url?query=Kistner%2C%20C.%20Winzer%2C%20T.%20Pitzschke%2C%20A.%20Mulder%2C%20L.%20Seven%20Lotus%20japonicus%20genes%20required%20for%20transcriptional%20reprogramming%20of%20the%20root%20during%20fungal%20and%20bacterial%20symbiosis%202005)

[^Kobae_2010_a]: Kobae, Y., and Hata, S. (2010). Dynamics of periarbuscular membranes visualized with a fluorescent phosphate transporter in arbuscular mycorrhizal roots of rice. Plant Cell Physiol. 51: 341–353. [[Kobae_DynamicsPeriarbuscularMembranesVisualizedWith_2010]] [OA](https://api.scholarcy.com/oa_version?query=Kobae%2C%20Y.%20Hata%2C%20S.%20Dynamics%20of%20periarbuscular%20membranes%20visualized%20with%20a%20fluorescent%20phosphate%20transporter%20in%20arbuscular%20mycorrhizal%20roots%20of%20rice%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Kobae%2C%20Y.%20Hata%2C%20S.%20Dynamics%20of%20periarbuscular%20membranes%20visualized%20with%20a%20fluorescent%20phosphate%20transporter%20in%20arbuscular%20mycorrhizal%20roots%20of%20rice%202010) [Scite](https://api.scholarcy.com/scite_url?query=Kobae%2C%20Y.%20Hata%2C%20S.%20Dynamics%20of%20periarbuscular%20membranes%20visualized%20with%20a%20fluorescent%20phosphate%20transporter%20in%20arbuscular%20mycorrhizal%20roots%20of%20rice%202010)

[^Kobae_et+al_2010_b]: Kobae, Y., Tamura, Y., Takai, S., Banba, M., and Hata, S. (2010). Localized expression of arbuscular mycorrhiza-inducible ammonium transporters in soybean. Plant Cell Physiol. 51: 1411–1415. [[Kobae_et+al_LocalizedExpressionArbuscularMycorrhizainducibleAmmonium_2010]] [OA](https://api.scholarcy.com/oa_version?query=Kobae%2C%20Y.%20Tamura%2C%20Y.%20Takai%2C%20S.%20Banba%2C%20M.%20Localized%20expression%20of%20arbuscular%20mycorrhiza-inducible%20ammonium%20transporters%20in%20soybean%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Kobae%2C%20Y.%20Tamura%2C%20Y.%20Takai%2C%20S.%20Banba%2C%20M.%20Localized%20expression%20of%20arbuscular%20mycorrhiza-inducible%20ammonium%20transporters%20in%20soybean%202010) [Scite](https://api.scholarcy.com/scite_url?query=Kobae%2C%20Y.%20Tamura%2C%20Y.%20Takai%2C%20S.%20Banba%2C%20M.%20Localized%20expression%20of%20arbuscular%20mycorrhiza-inducible%20ammonium%20transporters%20in%20soybean%202010)

[^Kojima_et+al_2014_a]: Kojima, T., Saito, K., Oba, H., Yoshida, Y., Terasawa, J., Umehara, Y., Suganuma, N., Kawaguchi, M., and Ohtomo, R. (2014). Isolation and phenotypic characterization of Lotus japonicus mutants specifically defective in arbuscular mycorrhizal formation. Plant Cell Physiol. 55: 928–941. [[Kojima_et+al_IsolationPhenotypicCharacterizationLotusJaponicus_2014]] [OA](https://api.scholarcy.com/oa_version?query=Kojima%2C%20T.%20Saito%2C%20K.%20Oba%2C%20H.%20Yoshida%2C%20Y.%20Isolation%20and%20phenotypic%20characterization%20of%20Lotus%20japonicus%20mutants%20specifically%20defective%20in%20arbuscular%20mycorrhizal%20formation%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Kojima%2C%20T.%20Saito%2C%20K.%20Oba%2C%20H.%20Yoshida%2C%20Y.%20Isolation%20and%20phenotypic%20characterization%20of%20Lotus%20japonicus%20mutants%20specifically%20defective%20in%20arbuscular%20mycorrhizal%20formation%202014) [Scite](https://api.scholarcy.com/scite_url?query=Kojima%2C%20T.%20Saito%2C%20K.%20Oba%2C%20H.%20Yoshida%2C%20Y.%20Isolation%20and%20phenotypic%20characterization%20of%20Lotus%20japonicus%20mutants%20specifically%20defective%20in%20arbuscular%20mycorrhizal%20formation%202014)

[^Koltai_et+al_2010_a]: Koltai, H., LekKala, S.P., Bhattacharya, C., Mayzlish-Gati, E., Resnick, N., Wininger, S., Dor, E., Yoneyama, K., Yoneyama, K., Hershenhorn, J., Joel, D.M., and Kapulnik, Y. (2010). A tomato strigolactone-impaired mutant displays aberrant shoot morphology and plant interactions. J. Exp. Bot. 61: 1739–1749. [[Koltai_et+al_TomatoStrigolactoneimpairedMutantDisplaysAberrant_2010]] [OA](https://api.scholarcy.com/oa_version?query=Koltai%2C%20H.%20LekKala%2C%20S.P.%20Bhattacharya%2C%20C.%20Mayzlish-Gati%2C%20E.%20A%20tomato%20strigolactone-impaired%20mutant%20displays%20aberrant%20shoot%20morphology%20and%20plant%20interactions%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Koltai%2C%20H.%20LekKala%2C%20S.P.%20Bhattacharya%2C%20C.%20Mayzlish-Gati%2C%20E.%20A%20tomato%20strigolactone-impaired%20mutant%20displays%20aberrant%20shoot%20morphology%20and%20plant%20interactions%202010) [Scite](https://api.scholarcy.com/scite_url?query=Koltai%2C%20H.%20LekKala%2C%20S.P.%20Bhattacharya%2C%20C.%20Mayzlish-Gati%2C%20E.%20A%20tomato%20strigolactone-impaired%20mutant%20displays%20aberrant%20shoot%20morphology%20and%20plant%20interactions%202010)

[^Krajinski_et+al_2014_a]: Krajinski, F., Courty, P.E., Sieh, D., Franken, P., Zhang, H., Bucher, M., Gerlach, N., Kryvoruchko, I., Zoeller, D., Udvardi, M., and Hause, B. (2014). The H+-ATPase HA1 of Medicago truncatula is essential for phosphate transport and plant growth during arbuscular mycorrhizal symbiosis. Plant Cell 26: 1808–1817. [[Krajinski_et+al_HatpaseHa1MedicagoTruncatulaEssential_2014]] [OA](https://api.scholarcy.com/oa_version?query=Krajinski%2C%20F.%20Courty%2C%20P.E.%20Sieh%2C%20D.%20Franken%2C%20P.%20The%20H%2B-ATPase%20HA1%20of%20Medicago%20truncatula%20is%20essential%20for%20phosphate%20transport%20and%20plant%20growth%20during%20arbuscular%20mycorrhizal%20symbiosis%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Krajinski%2C%20F.%20Courty%2C%20P.E.%20Sieh%2C%20D.%20Franken%2C%20P.%20The%20H%2B-ATPase%20HA1%20of%20Medicago%20truncatula%20is%20essential%20for%20phosphate%20transport%20and%20plant%20growth%20during%20arbuscular%20mycorrhizal%20symbiosis%202014) [Scite](https://api.scholarcy.com/scite_url?query=Krajinski%2C%20F.%20Courty%2C%20P.E.%20Sieh%2C%20D.%20Franken%2C%20P.%20The%20H%2B-ATPase%20HA1%20of%20Medicago%20truncatula%20is%20essential%20for%20phosphate%20transport%20and%20plant%20growth%20during%20arbuscular%20mycorrhizal%20symbiosis%202014)

[^Kretzschmar_et+al_2012_a]: Kretzschmar, T., Kohlen, W., Sasse, J., Borghi, L., Schlegel, M., Bachelier, J.B., Reinhardt, D., Bours, R., Bouwmeester, H.J., and Martinoia, E. (2012). A petunia ABC protein controls strigolactone-dependent symbiotic signalling and branching. Nature 483: 341–344. [[Kretzschmar_et+al_PetuniaAbcProteinControlsStrigolactonedependent_2012]] [OA](https://api.scholarcy.com/oa_version?query=Kretzschmar%2C%20T.%20Kohlen%2C%20W.%20Sasse%2C%20J.%20Borghi%2C%20L.%20A%20petunia%20ABC%20protein%20controls%20strigolactone-dependent%20symbiotic%20signalling%20and%20branching%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Kretzschmar%2C%20T.%20Kohlen%2C%20W.%20Sasse%2C%20J.%20Borghi%2C%20L.%20A%20petunia%20ABC%20protein%20controls%20strigolactone-dependent%20symbiotic%20signalling%20and%20branching%202012) [Scite](https://api.scholarcy.com/scite_url?query=Kretzschmar%2C%20T.%20Kohlen%2C%20W.%20Sasse%2C%20J.%20Borghi%2C%20L.%20A%20petunia%20ABC%20protein%20controls%20strigolactone-dependent%20symbiotic%20signalling%20and%20branching%202012)

[^Krusell_2002_a]: Krusell, L., et al. (2002). Shoot control of root development and nodulation is mediated by a receptor-like kinase. Nature 420: 422–426. [[Krusell_ShootControlRootDevelopmentNodulation_2002]] [OA](https://api.scholarcy.com/oa_version?query=Krusell%2C%20L.%20Shoot%20control%20of%20root%20development%20and%20nodulation%20is%20mediated%20by%20a%20receptor-like%20kinase%202002) [GScholar](https://scholar.google.co.uk/scholar?q=Krusell%2C%20L.%20Shoot%20control%20of%20root%20development%20and%20nodulation%20is%20mediated%20by%20a%20receptor-like%20kinase%202002) [Scite](https://api.scholarcy.com/scite_url?query=Krusell%2C%20L.%20Shoot%20control%20of%20root%20development%20and%20nodulation%20is%20mediated%20by%20a%20receptor-like%20kinase%202002)

[^Kuhn_et+al_2010_a]: Kuhn, H., Küster, H., and Requena, N. (2010). Membrane steroidbinding protein 1 induced by a diffusible fungal signal is critical for mycorrhization in Medicago truncatula. New Phytol. 185: 716–733. [[Kuhn_et+al_MembraneSteroidbindingProtein1Induced_2010]] [OA](https://api.scholarcy.com/oa_version?query=Kuhn%2C%20H.%20K%C3%BCster%2C%20H.%20Requena%2C%20N.%20Membrane%20steroidbinding%20protein%201%20induced%20by%20a%20diffusible%20fungal%20signal%20is%20critical%20for%20mycorrhization%20in%20Medicago%20truncatula%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Kuhn%2C%20H.%20K%C3%BCster%2C%20H.%20Requena%2C%20N.%20Membrane%20steroidbinding%20protein%201%20induced%20by%20a%20diffusible%20fungal%20signal%20is%20critical%20for%20mycorrhization%20in%20Medicago%20truncatula%202010) [Scite](https://api.scholarcy.com/scite_url?query=Kuhn%2C%20H.%20K%C3%BCster%2C%20H.%20Requena%2C%20N.%20Membrane%20steroidbinding%20protein%201%20induced%20by%20a%20diffusible%20fungal%20signal%20is%20critical%20for%20mycorrhization%20in%20Medicago%20truncatula%202010)

[^Lanfranco_et+al_2016_a]: Lanfranco, L., Bonfante, P., and Genre, A. (2016). The mutualistic interaction between plants and arbuscular mycorrhizal fungi. Microbiol. Spectr. 4: 4. [[Lanfranco_et+al_MutualisticInteractionBetweenPlantsArbuscular_2016]] [OA](https://api.scholarcy.com/oa_version?query=Lanfranco%2C%20L.%20Bonfante%2C%20P.%20Genre%2C%20A.%20The%20mutualistic%20interaction%20between%20plants%20and%20arbuscular%20mycorrhizal%20fungi%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Lanfranco%2C%20L.%20Bonfante%2C%20P.%20Genre%2C%20A.%20The%20mutualistic%20interaction%20between%20plants%20and%20arbuscular%20mycorrhizal%20fungi%202016) [Scite](https://api.scholarcy.com/scite_url?query=Lanfranco%2C%20L.%20Bonfante%2C%20P.%20Genre%2C%20A.%20The%20mutualistic%20interaction%20between%20plants%20and%20arbuscular%20mycorrhizal%20fungi%202016)

[^Larkan_et+al_2013_a]: Larkan, N.J., Ruzicka, D.R., Edmonds-Tibbett, T., Durkin, J.M.H., Jackson, L.E., Smith, F.A., Schachtman, D.P., Smith, S.E., and Barker, S.J. (2013). The reduced mycorrhizal colonisation (rmc) mutation of tomato disrupts five gene sequences including the CYCLOPS/IPD3 homologue. Mycorrhiza 23: 573–584. [[Larkan_et+al_ReducedMycorrhizalColonisationMutation_2013]] [OA](https://api.scholarcy.com/oa_version?query=Larkan%2C%20N.J.%20Ruzicka%2C%20D.R.%20Edmonds-Tibbett%2C%20T.%20Durkin%2C%20J.M.H.%20The%20reduced%20mycorrhizal%20colonisation%20%28rmc%29%20mutation%20of%20tomato%20disrupts%20five%20gene%20sequences%20including%20the%20CYCLOPS/IPD3%20homologue%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Larkan%2C%20N.J.%20Ruzicka%2C%20D.R.%20Edmonds-Tibbett%2C%20T.%20Durkin%2C%20J.M.H.%20The%20reduced%20mycorrhizal%20colonisation%20%28rmc%29%20mutation%20of%20tomato%20disrupts%20five%20gene%20sequences%20including%20the%20CYCLOPS/IPD3%20homologue%202013) [Scite](https://api.scholarcy.com/scite_url?query=Larkan%2C%20N.J.%20Ruzicka%2C%20D.R.%20Edmonds-Tibbett%2C%20T.%20Durkin%2C%20J.M.H.%20The%20reduced%20mycorrhizal%20colonisation%20%28rmc%29%20mutation%20of%20tomato%20disrupts%20five%20gene%20sequences%20including%20the%20CYCLOPS/IPD3%20homologue%202013)

[^Lauressergues_et+al_2012_a]: Lauressergues, D., Delaux, P.M., Formey, D., Lelandais-Brière, C., Fort, S., Cottaz, S., Bécard, G., Niebel, A., Roux, C., and Combier, J.P. (2012). The microRNA miR171h modulates arbuscular mycorrhizal colonization of Medicago truncatula by targeting NSP2. Plant J. 72: 512–522. [[Lauressergues_et+al_MicrornaMir171hModulatesArbuscularMycorrhizal_2012]] [OA](https://api.scholarcy.com/oa_version?query=Lauressergues%2C%20D.%20Delaux%2C%20P.M.%20Formey%2C%20D.%20Lelandais-Bri%C3%A8re%2C%20C.%20The%20microRNA%20miR171h%20modulates%20arbuscular%20mycorrhizal%20colonization%20of%20Medicago%20truncatula%20by%20targeting%20NSP2%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Lauressergues%2C%20D.%20Delaux%2C%20P.M.%20Formey%2C%20D.%20Lelandais-Bri%C3%A8re%2C%20C.%20The%20microRNA%20miR171h%20modulates%20arbuscular%20mycorrhizal%20colonization%20of%20Medicago%20truncatula%20by%20targeting%20NSP2%202012) [Scite](https://api.scholarcy.com/scite_url?query=Lauressergues%2C%20D.%20Delaux%2C%20P.M.%20Formey%2C%20D.%20Lelandais-Bri%C3%A8re%2C%20C.%20The%20microRNA%20miR171h%20modulates%20arbuscular%20mycorrhizal%20colonization%20of%20Medicago%20truncatula%20by%20targeting%20NSP2%202012)

[^L_2004_a]: Lévy, J., et al. (2004). A putative Ca2+ and calmodulin-dependent protein kinase required for bacterial and fungal symbioses. Science 303: 1361–1364. [[L_PutativeCa2CalmodulindependentProteinKinase_2004]] [OA](https://api.scholarcy.com/oa_version?query=L%C3%A9vy%2C%20J.%20A%20putative%20Ca2%2B%20and%20calmodulin-dependent%20protein%20kinase%20required%20for%20bacterial%20and%20fungal%20symbioses%202004) [GScholar](https://scholar.google.co.uk/scholar?q=L%C3%A9vy%2C%20J.%20A%20putative%20Ca2%2B%20and%20calmodulin-dependent%20protein%20kinase%20required%20for%20bacterial%20and%20fungal%20symbioses%202004) [Scite](https://api.scholarcy.com/scite_url?query=L%C3%A9vy%2C%20J.%20A%20putative%20Ca2%2B%20and%20calmodulin-dependent%20protein%20kinase%20required%20for%20bacterial%20and%20fungal%20symbioses%202004)

[^Li_et+al_2016_a]: Li, S., Zhao, Y., Zhao, Z., Wu, X., Sun, L., Liu, Q., and Wu, Y. (2016). Crystal structure of the GRAS domain of SCARECROW-LIKE7 in Oryza sativa. Plant Cell 28: 1025–1034. [[Li_et+al_CrystalStructureGrasDomainScarecrowlike7_2016]] [OA](https://api.scholarcy.com/oa_version?query=Li%2C%20S.%20Zhao%2C%20Y.%20Zhao%2C%20Z.%20Wu%2C%20X.%20Crystal%20structure%20of%20the%20GRAS%20domain%20of%20SCARECROW-LIKE7%20in%20Oryza%20sativa%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Li%2C%20S.%20Zhao%2C%20Y.%20Zhao%2C%20Z.%20Wu%2C%20X.%20Crystal%20structure%20of%20the%20GRAS%20domain%20of%20SCARECROW-LIKE7%20in%20Oryza%20sativa%202016) [Scite](https://api.scholarcy.com/scite_url?query=Li%2C%20S.%20Zhao%2C%20Y.%20Zhao%2C%20Z.%20Wu%2C%20X.%20Crystal%20structure%20of%20the%20GRAS%20domain%20of%20SCARECROW-LIKE7%20in%20Oryza%20sativa%202016)

[^Lin_2014_a]: Lin, K., et al. (2014). Single nucleus genome sequencing reveals high similarity among nuclei of an endomycorrhizal fungus. PLoS Genet. 10: e1004078. [[Lin_SingleNucleusGenomeSequencingReveals_2014]] [OA](https://api.scholarcy.com/oa_version?query=Lin%2C%20K.%20Single%20nucleus%20genome%20sequencing%20reveals%20high%20similarity%20among%20nuclei%20of%20an%20endomycorrhizal%20fungus%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Lin%2C%20K.%20Single%20nucleus%20genome%20sequencing%20reveals%20high%20similarity%20among%20nuclei%20of%20an%20endomycorrhizal%20fungus%202014) [Scite](https://api.scholarcy.com/scite_url?query=Lin%2C%20K.%20Single%20nucleus%20genome%20sequencing%20reveals%20high%20similarity%20among%20nuclei%20of%20an%20endomycorrhizal%20fungus%202014)

[^Liu_2011_a]: Liu, W., et al. (2011). Strigolactone biosynthesis in Medicago truncatula and rice requires the symbiotic GRAS-type transcription factors NSP1 and NSP2. Plant Cell 23: 3853–3865. [[Liu_StrigolactoneBiosynthesisMedicagoTruncatulaRice_2011]] [OA](https://api.scholarcy.com/oa_version?query=Liu%2C%20W.%20Strigolactone%20biosynthesis%20in%20Medicago%20truncatula%20and%20rice%20requires%20the%20symbiotic%20GRAS-type%20transcription%20factors%20NSP1%20and%20NSP2%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Liu%2C%20W.%20Strigolactone%20biosynthesis%20in%20Medicago%20truncatula%20and%20rice%20requires%20the%20symbiotic%20GRAS-type%20transcription%20factors%20NSP1%20and%20NSP2%202011) [Scite](https://api.scholarcy.com/scite_url?query=Liu%2C%20W.%20Strigolactone%20biosynthesis%20in%20Medicago%20truncatula%20and%20rice%20requires%20the%20symbiotic%20GRAS-type%20transcription%20factors%20NSP1%20and%20NSP2%202011)

[^Lopez-Obando_et+al_2015_a]: Lopez-Obando, M., Ligerot, Y., Bonhomme, S., Boyer, F.D., and Rameau, C. (2015). Strigolactone biosynthesis and signaling in plant development. Development 142: 3615–3619. [[Lopez-Obando_et+al_StrigolactoneBiosynthesisSignalingPlantDevelopment_2015]] [OA](https://api.scholarcy.com/oa_version?query=Lopez-Obando%2C%20M.%20Ligerot%2C%20Y.%20Bonhomme%2C%20S.%20Boyer%2C%20F.D.%20Strigolactone%20biosynthesis%20and%20signaling%20in%20plant%20development%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Lopez-Obando%2C%20M.%20Ligerot%2C%20Y.%20Bonhomme%2C%20S.%20Boyer%2C%20F.D.%20Strigolactone%20biosynthesis%20and%20signaling%20in%20plant%20development%202015) [Scite](https://api.scholarcy.com/scite_url?query=Lopez-Obando%2C%20M.%20Ligerot%2C%20Y.%20Bonhomme%2C%20S.%20Boyer%2C%20F.D.%20Strigolactone%20biosynthesis%20and%20signaling%20in%20plant%20development%202015)

[^Lota_et+al_2013_a]: Lota, F., Wegmüller, S., Buer, B., Sato, S., Bräutigam, A., Hanf, B., and Bucher, M. (2013). The cis-acting CTTC-P1BS module is indicative for gene function of LjVTI12, a Qb-SNARE protein gene that is required for arbuscule formation in Lotus japonicus. Plant J. 74: 280–293. [[Lota_et+al_actingCttcp1bsModuleIndicativeGene_2013]] [OA](https://api.scholarcy.com/oa_version?query=Lota%2C%20F.%20Wegm%C3%BCller%2C%20S.%20Buer%2C%20B.%20Sato%2C%20S.%20The%20cis-acting%20CTTC-P1BS%20module%20is%20indicative%20for%20gene%20function%20of%20LjVTI12%2C%20a%20Qb-SNARE%20protein%20gene%20that%20is%20required%20for%20arbuscule%20formation%20in%20Lotus%20japonicus%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Lota%2C%20F.%20Wegm%C3%BCller%2C%20S.%20Buer%2C%20B.%20Sato%2C%20S.%20The%20cis-acting%20CTTC-P1BS%20module%20is%20indicative%20for%20gene%20function%20of%20LjVTI12%2C%20a%20Qb-SNARE%20protein%20gene%20that%20is%20required%20for%20arbuscule%20formation%20in%20Lotus%20japonicus%202013) [Scite](https://api.scholarcy.com/scite_url?query=Lota%2C%20F.%20Wegm%C3%BCller%2C%20S.%20Buer%2C%20B.%20Sato%2C%20S.%20The%20cis-acting%20CTTC-P1BS%20module%20is%20indicative%20for%20gene%20function%20of%20LjVTI12%2C%20a%20Qb-SNARE%20protein%20gene%20that%20is%20required%20for%20arbuscule%20formation%20in%20Lotus%20japonicus%202013)

[^Luginbuehl_et+al_2017_a]: Luginbuehl, L.H., Menard, G.N., Kurup, S., Van Erp, H., Radhakrishnan, G.V., Breakspear, A., Oldroyd, G.E.D., and Eastmond, P.J. (2017). Fatty acids in arbuscular mycorrhizal fungi are synthesized by the host plant. Science 356: 1175–1178. [[Luginbuehl_et+al_FattyAcidsArbuscularMycorrhizalFungi_2017]] [OA](https://api.scholarcy.com/oa_version?query=Luginbuehl%2C%20L.H.%20Menard%2C%20G.N.%20Kurup%2C%20S.%20Van%20Erp%2C%20H.%20Fatty%20acids%20in%20arbuscular%20mycorrhizal%20fungi%20are%20synthesized%20by%20the%20host%20plant%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Luginbuehl%2C%20L.H.%20Menard%2C%20G.N.%20Kurup%2C%20S.%20Van%20Erp%2C%20H.%20Fatty%20acids%20in%20arbuscular%20mycorrhizal%20fungi%20are%20synthesized%20by%20the%20host%20plant%202017) [Scite](https://api.scholarcy.com/scite_url?query=Luginbuehl%2C%20L.H.%20Menard%2C%20G.N.%20Kurup%2C%20S.%20Van%20Erp%2C%20H.%20Fatty%20acids%20in%20arbuscular%20mycorrhizal%20fungi%20are%20synthesized%20by%20the%20host%20plant%202017)

[^Madsen_et+al_2003_a]: Madsen, E.B., Madsen, L.H., Radutoiu, S., Olbryt, M., Rakwalska, M., Szczyglowski, K., Sato, S., Kaneko, T., Tabata, S., Sandal, N., and Stougaard, J. (2003). A receptor kinase gene of the LysM type is involved in legume perception of rhizobial signals. Nature 425: 637–640. [[Madsen_et+al_ReceptorKinaseGeneLysmType_2003]] [OA](https://api.scholarcy.com/oa_version?query=Madsen%2C%20E.B.%20Madsen%2C%20L.H.%20Radutoiu%2C%20S.%20Olbryt%2C%20M.%20A%20receptor%20kinase%20gene%20of%20the%20LysM%20type%20is%20involved%20in%20legume%20perception%20of%20rhizobial%20signals%202003) [GScholar](https://scholar.google.co.uk/scholar?q=Madsen%2C%20E.B.%20Madsen%2C%20L.H.%20Radutoiu%2C%20S.%20Olbryt%2C%20M.%20A%20receptor%20kinase%20gene%20of%20the%20LysM%20type%20is%20involved%20in%20legume%20perception%20of%20rhizobial%20signals%202003) [Scite](https://api.scholarcy.com/scite_url?query=Madsen%2C%20E.B.%20Madsen%2C%20L.H.%20Radutoiu%2C%20S.%20Olbryt%2C%20M.%20A%20receptor%20kinase%20gene%20of%20the%20LysM%20type%20is%20involved%20in%20legume%20perception%20of%20rhizobial%20signals%202003)

[^Maillet_2011_a]: Maillet, F., et al. (2011). Fungal lipochitooligosaccharide symbiotic signals in arbuscular mycorrhiza. Nature 469: 58–63. [[Maillet_FungalLipochitooligosaccharideSymbioticSignalsArbuscular_2011]] [OA](https://api.scholarcy.com/oa_version?query=Maillet%2C%20F.%20Fungal%20lipochitooligosaccharide%20symbiotic%20signals%20in%20arbuscular%20mycorrhiza%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Maillet%2C%20F.%20Fungal%20lipochitooligosaccharide%20symbiotic%20signals%20in%20arbuscular%20mycorrhiza%202011) [Scite](https://api.scholarcy.com/scite_url?query=Maillet%2C%20F.%20Fungal%20lipochitooligosaccharide%20symbiotic%20signals%20in%20arbuscular%20mycorrhiza%202011)

[^Messinese_et+al_2007_a]: Messinese, E., Mun, J.H., Yeun, L.H., Jayaraman, D., Rougé, P., Barre, A., Lougnon, G., Schornack, S., Bono, J.J., Cook, D.R., and Ané, J.M. (2007). A novel nuclear protein interacts with the symbiotic DMI3 calcium- and calmodulin-dependent protein kinase of Medicago truncatula. Mol. Plant Microbe Interact. 20: 912–921. [[Messinese_et+al_NovelNuclearProteinInteractsWith_2007]] [OA](https://api.scholarcy.com/oa_version?query=Messinese%2C%20E.%20Mun%2C%20J.H.%20Yeun%2C%20L.H.%20Jayaraman%2C%20D.%20A%20novel%20nuclear%20protein%20interacts%20with%20the%20symbiotic%20DMI3%20calcium-%20and%20calmodulin-dependent%20protein%20kinase%20of%20Medicago%20truncatula%202007) [GScholar](https://scholar.google.co.uk/scholar?q=Messinese%2C%20E.%20Mun%2C%20J.H.%20Yeun%2C%20L.H.%20Jayaraman%2C%20D.%20A%20novel%20nuclear%20protein%20interacts%20with%20the%20symbiotic%20DMI3%20calcium-%20and%20calmodulin-dependent%20protein%20kinase%20of%20Medicago%20truncatula%202007) [Scite](https://api.scholarcy.com/scite_url?query=Messinese%2C%20E.%20Mun%2C%20J.H.%20Yeun%2C%20L.H.%20Jayaraman%2C%20D.%20A%20novel%20nuclear%20protein%20interacts%20with%20the%20symbiotic%20DMI3%20calcium-%20and%20calmodulin-dependent%20protein%20kinase%20of%20Medicago%20truncatula%202007)

[^Miller_et+al_2013_a]: Miller, J.B., Pratap, A., Miyahara, A., Zhou, L., Bornemann, S., Morris, R.J., and Oldroyd, G.E.D. (2013). Calcium/calmodulindependent protein kinase is negatively and positively regulated by calcium, providing a mechanism for decoding calcium responses during symbiosis signaling. Plant Cell 25: 5053–5066. [[Miller_et+al_CalciumcalmodulindependentProteinKinaseNegativelyPositively_2013]] [OA](https://scholar.google.co.uk/scholar?q=Miller%2C%20J.B.%20Pratap%2C%20A.%20Miyahara%2C%20A.%20Zhou%2C%20L.%20Calcium/calmodulindependent%20protein%20kinase%20is%20negatively%20and%20positively%20regulated%20by%20calcium%2C%20providing%20a%20mechanism%20for%20decoding%20calcium%20responses%20during%20symbiosis%20signaling.%20Plant%20Cell%2025%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Miller%2C%20J.B.%20Pratap%2C%20A.%20Miyahara%2C%20A.%20Zhou%2C%20L.%20Calcium/calmodulindependent%20protein%20kinase%20is%20negatively%20and%20positively%20regulated%20by%20calcium%2C%20providing%20a%20mechanism%20for%20decoding%20calcium%20responses%20during%20symbiosis%20signaling.%20Plant%20Cell%2025%202013) 

[^Miya_et+al_2007_a]: Miya, A., Albert, P., Shinya, T., Desaki, Y., Ichimura, K., Shirasu, K., Narusaka, Y., Kawakami, N., Kaku, H., and Shibuya, N. (2007). CERK1, a LysM receptor kinase, is essential for chitin elicitor signaling in Arabidopsis. Proc. Natl. Acad. Sci. USA 104: 19613– 19618. [[Miya_et+al_Cerk1LysmReceptorKinaseEssential_2007]] [OA](https://api.scholarcy.com/oa_version?query=Miya%2C%20A.%20Albert%2C%20P.%20Shinya%2C%20T.%20Desaki%2C%20Y.%20CERK1%2C%20a%20LysM%20receptor%20kinase%2C%20is%20essential%20for%20chitin%20elicitor%20signaling%20in%20Arabidopsis%202007) [GScholar](https://scholar.google.co.uk/scholar?q=Miya%2C%20A.%20Albert%2C%20P.%20Shinya%2C%20T.%20Desaki%2C%20Y.%20CERK1%2C%20a%20LysM%20receptor%20kinase%2C%20is%20essential%20for%20chitin%20elicitor%20signaling%20in%20Arabidopsis%202007) [Scite](https://api.scholarcy.com/scite_url?query=Miya%2C%20A.%20Albert%2C%20P.%20Shinya%2C%20T.%20Desaki%2C%20Y.%20CERK1%2C%20a%20LysM%20receptor%20kinase%2C%20is%20essential%20for%20chitin%20elicitor%20signaling%20in%20Arabidopsis%202007)

[^Miyata_2014_a]: Miyata, K., et al. (2014). The bifunctional plant receptor, OsCERK1, regulates both chitin-triggered immunity and arbuscular mycorrhizal symbiosis in rice. Plant Cell Physiol. 55: 1864–1872. [[Miyata_BifunctionalPlantReceptorOscerk1Regulates_2014]] [OA](https://api.scholarcy.com/oa_version?query=Miyata%2C%20K.%20The%20bifunctional%20plant%20receptor%2C%20OsCERK1%2C%20regulates%20both%20chitin-triggered%20immunity%20and%20arbuscular%20mycorrhizal%20symbiosis%20in%20rice%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Miyata%2C%20K.%20The%20bifunctional%20plant%20receptor%2C%20OsCERK1%2C%20regulates%20both%20chitin-triggered%20immunity%20and%20arbuscular%20mycorrhizal%20symbiosis%20in%20rice%202014) [Scite](https://api.scholarcy.com/scite_url?query=Miyata%2C%20K.%20The%20bifunctional%20plant%20receptor%2C%20OsCERK1%2C%20regulates%20both%20chitin-triggered%20immunity%20and%20arbuscular%20mycorrhizal%20symbiosis%20in%20rice%202014)

[^Morandi_et+al_2000_b]: Morandi, D., Sagan, M., Prado-Vivant, E., and Duc, G. (2000). Influence of genes determining supernodulation on root colonization by the mycorrhizal fungus Glomus mosseae in Pisum sativum and Medicago truncatula mutants. Mycorrhiza 10: 37–42. [[Morandi_et+al_InfluenceGenesDeterminingSupernodulationRoot_2000]] [OA](https://api.scholarcy.com/oa_version?query=Morandi%2C%20D.%20Sagan%2C%20M.%20Prado-Vivant%2C%20E.%20Duc%2C%20G.%20Influence%20of%20genes%20determining%20supernodulation%20on%20root%20colonization%20by%20the%20mycorrhizal%20fungus%20Glomus%20mosseae%20in%20Pisum%20sativum%20and%20Medicago%20truncatula%20mutants%202000) [GScholar](https://scholar.google.co.uk/scholar?q=Morandi%2C%20D.%20Sagan%2C%20M.%20Prado-Vivant%2C%20E.%20Duc%2C%20G.%20Influence%20of%20genes%20determining%20supernodulation%20on%20root%20colonization%20by%20the%20mycorrhizal%20fungus%20Glomus%20mosseae%20in%20Pisum%20sativum%20and%20Medicago%20truncatula%20mutants%202000) [Scite](https://api.scholarcy.com/scite_url?query=Morandi%2C%20D.%20Sagan%2C%20M.%20Prado-Vivant%2C%20E.%20Duc%2C%20G.%20Influence%20of%20genes%20determining%20supernodulation%20on%20root%20colonization%20by%20the%20mycorrhizal%20fungus%20Glomus%20mosseae%20in%20Pisum%20sativum%20and%20Medicago%20truncatula%20mutants%202000)

[^Murray_2011_a]: Murray, J.D., et al. (2011). Vapyrin, a gene essential for intracellular progression of arbuscular mycorrhizal symbiosis, is also essential for infection by rhizobia in the nodule symbiosis of Medicago truncatula. Plant J. 65: 244–252. [[Murray_VapyrinGeneEssentialIntracellularProgression_2011]] [OA](https://api.scholarcy.com/oa_version?query=Murray%2C%20J.D.%20Vapyrin%2C%20a%20gene%20essential%20for%20intracellular%20progression%20of%20arbuscular%20mycorrhizal%20symbiosis%2C%20is%20also%20essential%20for%20infection%20by%20rhizobia%20in%20the%20nodule%20symbiosis%20of%20Medicago%20truncatula%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Murray%2C%20J.D.%20Vapyrin%2C%20a%20gene%20essential%20for%20intracellular%20progression%20of%20arbuscular%20mycorrhizal%20symbiosis%2C%20is%20also%20essential%20for%20infection%20by%20rhizobia%20in%20the%20nodule%20symbiosis%20of%20Medicago%20truncatula%202011) [Scite](https://api.scholarcy.com/scite_url?query=Murray%2C%20J.D.%20Vapyrin%2C%20a%20gene%20essential%20for%20intracellular%20progression%20of%20arbuscular%20mycorrhizal%20symbiosis%2C%20is%20also%20essential%20for%20infection%20by%20rhizobia%20in%20the%20nodule%20symbiosis%20of%20Medicago%20truncatula%202011)

[^Nadal_2017_a]: Nadal, M., et al. (2017). An N-acetylglucosamine transporter required for arbuscular mycorrhizal symbioses in rice and maize. Nat. Plants 3: 17073. [[Nadal_NacetylglucosamineTransporterRequiredArbuscularMycorrhizal_2017]] [OA](https://api.scholarcy.com/oa_version?query=Nadal%2C%20M.%20An%20N-acetylglucosamine%20transporter%20required%20for%20arbuscular%20mycorrhizal%20symbioses%20in%20rice%20and%20maize%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Nadal%2C%20M.%20An%20N-acetylglucosamine%20transporter%20required%20for%20arbuscular%20mycorrhizal%20symbioses%20in%20rice%20and%20maize%202017) [Scite](https://api.scholarcy.com/scite_url?query=Nadal%2C%20M.%20An%20N-acetylglucosamine%20transporter%20required%20for%20arbuscular%20mycorrhizal%20symbioses%20in%20rice%20and%20maize%202017)

[^Nelson_et+al_2011_a]: Nelson, D.C., Scaffidi, A., Dun, E.A., Waters, M.T., Flematti, G.R., Dixon, K.W., Beveridge, C.A., Ghisalberti, E.L., and Smith, S.M. (2011). F-box protein MAX2 has dual roles in karrikin and strigolactone signaling in Arabidopsis thaliana. Proc. Natl. Acad. Sci. USA 108: 8897–8902. [[Nelson_et+al_ProteinMax2DualRoles_2011]] [OA](https://api.scholarcy.com/oa_version?query=Nelson%2C%20D.C.%20Scaffidi%2C%20A.%20Dun%2C%20E.A.%20Waters%2C%20M.T.%20F-box%20protein%20MAX2%20has%20dual%20roles%20in%20karrikin%20and%20strigolactone%20signaling%20in%20Arabidopsis%20thaliana%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Nelson%2C%20D.C.%20Scaffidi%2C%20A.%20Dun%2C%20E.A.%20Waters%2C%20M.T.%20F-box%20protein%20MAX2%20has%20dual%20roles%20in%20karrikin%20and%20strigolactone%20signaling%20in%20Arabidopsis%20thaliana%202011) [Scite](https://api.scholarcy.com/scite_url?query=Nelson%2C%20D.C.%20Scaffidi%2C%20A.%20Dun%2C%20E.A.%20Waters%2C%20M.T.%20F-box%20protein%20MAX2%20has%20dual%20roles%20in%20karrikin%20and%20strigolactone%20signaling%20in%20Arabidopsis%20thaliana%202011)

[^Nishimura_et+al_2002_a]: Nishimura, R., Hayashi, M., Wu, G.J., Kouchi, H., Imaizumi-Anraku, H., Murakami, Y., Kawasaki, S., Akao, S., Ohmori, M., Nagasawa, M., Harada, K., and Kawaguchi, M. (2002). HAR1 mediates systemic regulation of symbiotic organ development. Nature 420: 426–429. [[Nishimura_et+al_Har1MediatesSystemicRegulationSymbiotic_2002]] [OA](https://api.scholarcy.com/oa_version?query=Nishimura%2C%20R.%20Hayashi%2C%20M.%20Wu%2C%20G.J.%20Kouchi%2C%20H.%20HAR1%20mediates%20systemic%20regulation%20of%20symbiotic%20organ%20development%202002) [GScholar](https://scholar.google.co.uk/scholar?q=Nishimura%2C%20R.%20Hayashi%2C%20M.%20Wu%2C%20G.J.%20Kouchi%2C%20H.%20HAR1%20mediates%20systemic%20regulation%20of%20symbiotic%20organ%20development%202002) [Scite](https://api.scholarcy.com/scite_url?query=Nishimura%2C%20R.%20Hayashi%2C%20M.%20Wu%2C%20G.J.%20Kouchi%2C%20H.%20HAR1%20mediates%20systemic%20regulation%20of%20symbiotic%20organ%20development%202002)

[^Ogawa_et+al_2000_a]: Ogawa, M., Kusano, T., Katsumi, M., and Sano, H. (2000). Rice gibberellin-insensitive gene homolog, OsGAI, encodes a nuclearlocalized protein capable of gene activation at transcriptional level. Gene 245: 21–29. [[Ogawa_et+al_RiceGibberellininsensitiveGeneHomologOsgai_2000]] [OA](https://api.scholarcy.com/oa_version?query=Ogawa%2C%20M.%20Kusano%2C%20T.%20Katsumi%2C%20M.%20Sano%2C%20H.%20Rice%20gibberellin-insensitive%20gene%20homolog%2C%20OsGAI%2C%20encodes%20a%20nuclearlocalized%20protein%20capable%20of%20gene%20activation%20at%20transcriptional%20level%202000) [GScholar](https://scholar.google.co.uk/scholar?q=Ogawa%2C%20M.%20Kusano%2C%20T.%20Katsumi%2C%20M.%20Sano%2C%20H.%20Rice%20gibberellin-insensitive%20gene%20homolog%2C%20OsGAI%2C%20encodes%20a%20nuclearlocalized%20protein%20capable%20of%20gene%20activation%20at%20transcriptional%20level%202000) [Scite](https://api.scholarcy.com/scite_url?query=Ogawa%2C%20M.%20Kusano%2C%20T.%20Katsumi%2C%20M.%20Sano%2C%20H.%20Rice%20gibberellin-insensitive%20gene%20homolog%2C%20OsGAI%2C%20encodes%20a%20nuclearlocalized%20protein%20capable%20of%20gene%20activation%20at%20transcriptional%20level%202000)

[^Oldroyd_2013_a]: Oldroyd, G.E.D. (2013). Speak, friend, and enter: signalling systems that promote beneficial symbiotic associations in plants. Nat. Rev. Microbiol. 11: 252–263. [[Oldroyd_SpeakFriendEnterSignallingSystems_2013]] [OA](https://api.scholarcy.com/oa_version?query=Oldroyd%2C%20G.E.D.%20Speak%2C%20friend%2C%20and%20enter%3A%20signalling%20systems%20that%20promote%20beneficial%20symbiotic%20associations%20in%20plants%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Oldroyd%2C%20G.E.D.%20Speak%2C%20friend%2C%20and%20enter%3A%20signalling%20systems%20that%20promote%20beneficial%20symbiotic%20associations%20in%20plants%202013) [Scite](https://api.scholarcy.com/scite_url?query=Oldroyd%2C%20G.E.D.%20Speak%2C%20friend%2C%20and%20enter%3A%20signalling%20systems%20that%20promote%20beneficial%20symbiotic%20associations%20in%20plants%202013)

[^Op_et+al_2011_a]: Op den Camp, R., Streng, A., De Mita, S., Cao, Q., Polone, E., Liu, W., Ammiraju, J.S.S., Kudrna, D., Wing, R., Untergasser, A., Bisseling, T., and Geurts, R. (2011). LysM-type mycorrhizal receptor recruited for rhizobium symbiosis in nonlegume Parasponia. Science 331: 909–912. [[Op_et+al_LysmtypeMycorrhizalReceptorRecruitedRhizobium_2011]] [OA](https://api.scholarcy.com/oa_version?query=Op%20den%20Camp%2C%20R.%20Streng%2C%20A.%20De%20Mita%2C%20S.%20Cao%2C%20Q.%20LysM-type%20mycorrhizal%20receptor%20recruited%20for%20rhizobium%20symbiosis%20in%20nonlegume%20Parasponia%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Op%20den%20Camp%2C%20R.%20Streng%2C%20A.%20De%20Mita%2C%20S.%20Cao%2C%20Q.%20LysM-type%20mycorrhizal%20receptor%20recruited%20for%20rhizobium%20symbiosis%20in%20nonlegume%20Parasponia%202011) [Scite](https://api.scholarcy.com/scite_url?query=Op%20den%20Camp%2C%20R.%20Streng%2C%20A.%20De%20Mita%2C%20S.%20Cao%2C%20Q.%20LysM-type%20mycorrhizal%20receptor%20recruited%20for%20rhizobium%20symbiosis%20in%20nonlegume%20Parasponia%202011)

[^Ovchinnikova_2011_a]: Ovchinnikova, E., et al. (2011). IPD3 controls the formation of nitrogenfixing symbiosomes in pea and Medicago spp. Mol. Plant Microbe Interact. 24: 1333–1344. [[Ovchinnikova_Ipd3ControlsFormationNitrogenfixingSymbiosomes_2011]] [OA](https://api.scholarcy.com/oa_version?query=Ovchinnikova%2C%20E.%20IPD3%20controls%20the%20formation%20of%20nitrogenfixing%20symbiosomes%20in%20pea%20and%20Medicago%20spp%202011) [GScholar](https://scholar.google.co.uk/scholar?q=Ovchinnikova%2C%20E.%20IPD3%20controls%20the%20formation%20of%20nitrogenfixing%20symbiosomes%20in%20pea%20and%20Medicago%20spp%202011) [Scite](https://api.scholarcy.com/scite_url?query=Ovchinnikova%2C%20E.%20IPD3%20controls%20the%20formation%20of%20nitrogenfixing%20symbiosomes%20in%20pea%20and%20Medicago%20spp%202011)

[^Pan_et+al_2016_a]: Pan, H., Oztas, O., Zhang, X., Wu, X., Stonoha, C., Wang, E., Wang, B., and Wang, D. (2016). A symbiotic SNARE protein generated by alternative termination of transcription. Nat. Plants 2: 15197. [[Pan_et+al_SymbioticSnareProteinGeneratedAlternative_2016]] [OA](https://api.scholarcy.com/oa_version?query=Pan%2C%20H.%20Oztas%2C%20O.%20Zhang%2C%20X.%20Wu%2C%20X.%20A%20symbiotic%20SNARE%20protein%20generated%20by%20alternative%20termination%20of%20transcription%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Pan%2C%20H.%20Oztas%2C%20O.%20Zhang%2C%20X.%20Wu%2C%20X.%20A%20symbiotic%20SNARE%20protein%20generated%20by%20alternative%20termination%20of%20transcription%202016) [Scite](https://api.scholarcy.com/scite_url?query=Pan%2C%20H.%20Oztas%2C%20O.%20Zhang%2C%20X.%20Wu%2C%20X.%20A%20symbiotic%20SNARE%20protein%20generated%20by%20alternative%20termination%20of%20transcription%202016)

[^Park_et+al_2015_a]: Park, H.J., Floss, D.S., Levesque-Tremblay, V., Bravo, A., and Harrison, M.J. (2015). Hyphal branching during arbuscule development requires Reduced Arbuscular Mycorrhiza 1. Plant Physiol. 169: 2774–2788. [[Park_et+al_HyphalBranchingDuringArbusculeDevelopment_2015]] [OA](https://api.scholarcy.com/oa_version?query=Park%2C%20H.J.%20Floss%2C%20D.S.%20Levesque-Tremblay%2C%20V.%20Bravo%2C%20A.%20Hyphal%20branching%20during%20arbuscule%20development%20requires%20Reduced%20Arbuscular%20Mycorrhiza%201.%20Plant%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Park%2C%20H.J.%20Floss%2C%20D.S.%20Levesque-Tremblay%2C%20V.%20Bravo%2C%20A.%20Hyphal%20branching%20during%20arbuscule%20development%20requires%20Reduced%20Arbuscular%20Mycorrhiza%201.%20Plant%202015) [Scite](https://api.scholarcy.com/scite_url?query=Park%2C%20H.J.%20Floss%2C%20D.S.%20Levesque-Tremblay%2C%20V.%20Bravo%2C%20A.%20Hyphal%20branching%20during%20arbuscule%20development%20requires%20Reduced%20Arbuscular%20Mycorrhiza%201.%20Plant%202015)

[^Parniske_2000_a]: Parniske, M. (2000). Intracellular accommodation of microbes by plants: a common developmental program for symbiosis and disease? Curr. Opin. Plant Biol. 3: 320–328. [[Parniske_IntracellularAccommodationMicrobesPlantsCommon_2000]] [OA](https://api.scholarcy.com/oa_version?query=Parniske%2C%20M.%20Intracellular%20accommodation%20of%20microbes%20by%20plants%3A%20a%20common%20developmental%20program%20for%20symbiosis%20and%20disease%3F%202000) [GScholar](https://scholar.google.co.uk/scholar?q=Parniske%2C%20M.%20Intracellular%20accommodation%20of%20microbes%20by%20plants%3A%20a%20common%20developmental%20program%20for%20symbiosis%20and%20disease%3F%202000) [Scite](https://api.scholarcy.com/scite_url?query=Parniske%2C%20M.%20Intracellular%20accommodation%20of%20microbes%20by%20plants%3A%20a%20common%20developmental%20program%20for%20symbiosis%20and%20disease%3F%202000)

[^Peng_1993_a]: Peng, J., and Harberd, N.P. (1993). Derivative alleles of the Arabidopsis gibberellin-insensitive (GAI) mutation confer a wild-type phenotype. Plant Cell 5: 351–360. [[Peng_DerivativeAllelesArabidopsisGibberellininsensitivegai_1993]] [OA](https://api.scholarcy.com/oa_version?query=Peng%2C%20J.%20Harberd%2C%20N.P.%20Derivative%20alleles%20of%20the%20Arabidopsis%20gibberellin-insensitive%20%28GAI%29%20mutation%20confer%20a%20wild-type%20phenotype%201993) [GScholar](https://scholar.google.co.uk/scholar?q=Peng%2C%20J.%20Harberd%2C%20N.P.%20Derivative%20alleles%20of%20the%20Arabidopsis%20gibberellin-insensitive%20%28GAI%29%20mutation%20confer%20a%20wild-type%20phenotype%201993) [Scite](https://api.scholarcy.com/scite_url?query=Peng%2C%20J.%20Harberd%2C%20N.P.%20Derivative%20alleles%20of%20the%20Arabidopsis%20gibberellin-insensitive%20%28GAI%29%20mutation%20confer%20a%20wild-type%20phenotype%201993)

[^Peng_et+al_1997_a]: Peng, J., Carol, P., Richards, D.E., King, K.E., Cowling, R.J., Murphy, G.P., and Harberd, N.P. (1997). The Arabidopsis GAI gene defines a signaling pathway that negatively regulates gibberellin responses. Genes Dev. 11: 3194–3205. [[Peng_et+al_ArabidopsisGaiGeneDefinesSignaling_1997]] [OA](https://api.scholarcy.com/oa_version?query=Peng%2C%20J.%20Carol%2C%20P.%20Richards%2C%20D.E.%20King%2C%20K.E.%20The%20Arabidopsis%20GAI%20gene%20defines%20a%20signaling%20pathway%20that%20negatively%20regulates%20gibberellin%20responses%201997) [GScholar](https://scholar.google.co.uk/scholar?q=Peng%2C%20J.%20Carol%2C%20P.%20Richards%2C%20D.E.%20King%2C%20K.E.%20The%20Arabidopsis%20GAI%20gene%20defines%20a%20signaling%20pathway%20that%20negatively%20regulates%20gibberellin%20responses%201997) [Scite](https://api.scholarcy.com/scite_url?query=Peng%2C%20J.%20Carol%2C%20P.%20Richards%2C%20D.E.%20King%2C%20K.E.%20The%20Arabidopsis%20GAI%20gene%20defines%20a%20signaling%20pathway%20that%20negatively%20regulates%20gibberellin%20responses%201997)

[^Penmetsa_2008_a]: Penmetsa, R.V., et al. (2008). The Medicago truncatula ortholog of Arabidopsis EIN2, sickle, is a negative regulator of symbiotic and pathogenic microbial associations. Plant J. 55: 580–595. [[Penmetsa_MedicagoTruncatulaOrthologArabidopsisEin2_2008]] [OA](https://api.scholarcy.com/oa_version?query=Penmetsa%2C%20R.V.%20The%20Medicago%20truncatula%20ortholog%20of%20Arabidopsis%20EIN2%2C%20sickle%2C%20is%20a%20negative%20regulator%20of%20symbiotic%20and%20pathogenic%20microbial%20associations%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Penmetsa%2C%20R.V.%20The%20Medicago%20truncatula%20ortholog%20of%20Arabidopsis%20EIN2%2C%20sickle%2C%20is%20a%20negative%20regulator%20of%20symbiotic%20and%20pathogenic%20microbial%20associations%202008) [Scite](https://api.scholarcy.com/scite_url?query=Penmetsa%2C%20R.V.%20The%20Medicago%20truncatula%20ortholog%20of%20Arabidopsis%20EIN2%2C%20sickle%2C%20is%20a%20negative%20regulator%20of%20symbiotic%20and%20pathogenic%20microbial%20associations%202008)

[^Pfeffer_et+al_1999_a]: Pfeffer, P.E., Douds, D.D., Becard, G., and Shachar-Hill, Y. (1999). Carbon uptake and the metabolism and transport of lipids in an arbuscular mycorrhiza. Plant Physiol. 120: 587–598. [[Pfeffer_et+al_CarbonUptakeMetabolismTransportLipids_1999]] [OA](https://api.scholarcy.com/oa_version?query=Pfeffer%2C%20P.E.%20Douds%2C%20D.D.%20Becard%2C%20G.%20Shachar-Hill%2C%20Y.%20Carbon%20uptake%20and%20the%20metabolism%20and%20transport%20of%20lipids%20in%20an%20arbuscular%20mycorrhiza%201999) [GScholar](https://scholar.google.co.uk/scholar?q=Pfeffer%2C%20P.E.%20Douds%2C%20D.D.%20Becard%2C%20G.%20Shachar-Hill%2C%20Y.%20Carbon%20uptake%20and%20the%20metabolism%20and%20transport%20of%20lipids%20in%20an%20arbuscular%20mycorrhiza%201999) [Scite](https://api.scholarcy.com/scite_url?query=Pfeffer%2C%20P.E.%20Douds%2C%20D.D.%20Becard%2C%20G.%20Shachar-Hill%2C%20Y.%20Carbon%20uptake%20and%20the%20metabolism%20and%20transport%20of%20lipids%20in%20an%20arbuscular%20mycorrhiza%201999)

[^Pimprikar_et+al_2016_a]: Pimprikar, P., Carbonnel, S., Paries, M., Katzer, K., Klingl, V., Bohmer, M.J., Karl, L., Floss, D.S., Harrison, M.J., Parniske, M., and Gutjahr, C. (2016). A CCaMK-CYCLOPS-DELLA complex activates transcription of RAM1 to regulate arbuscule branching. Curr. Biol. 26: 987–998. [[Pimprikar_et+al_CcamkcyclopsdellaComplexActivatesTranscriptionRam1_2016]] [OA](https://api.scholarcy.com/oa_version?query=Pimprikar%2C%20P.%20Carbonnel%2C%20S.%20Paries%2C%20M.%20Katzer%2C%20K.%20A%20CCaMK-CYCLOPS-DELLA%20complex%20activates%20transcription%20of%20RAM1%20to%20regulate%20arbuscule%20branching%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Pimprikar%2C%20P.%20Carbonnel%2C%20S.%20Paries%2C%20M.%20Katzer%2C%20K.%20A%20CCaMK-CYCLOPS-DELLA%20complex%20activates%20transcription%20of%20RAM1%20to%20regulate%20arbuscule%20branching%202016) [Scite](https://api.scholarcy.com/scite_url?query=Pimprikar%2C%20P.%20Carbonnel%2C%20S.%20Paries%2C%20M.%20Katzer%2C%20K.%20A%20CCaMK-CYCLOPS-DELLA%20complex%20activates%20transcription%20of%20RAM1%20to%20regulate%20arbuscule%20branching%202016)

[^Pirozynski_1975_a]: Pirozynski, K.A., and Malloch, D.W. (1975). The origin of land plants: a matter of mycotrophism. Biosystems 6: 153–164. [[Pirozynski_OriginLandPlantsMatterMycotrophism_1975]] [OA](https://api.scholarcy.com/oa_version?query=Pirozynski%2C%20K.A.%20Malloch%2C%20D.W.%20The%20origin%20of%20land%20plants%3A%20a%20matter%20of%20mycotrophism%201975) [GScholar](https://scholar.google.co.uk/scholar?q=Pirozynski%2C%20K.A.%20Malloch%2C%20D.W.%20The%20origin%20of%20land%20plants%3A%20a%20matter%20of%20mycotrophism%201975) [Scite](https://api.scholarcy.com/scite_url?query=Pirozynski%2C%20K.A.%20Malloch%2C%20D.W.%20The%20origin%20of%20land%20plants%3A%20a%20matter%20of%20mycotrophism%201975)

[^Pumplin_et+al_2012_a]: Pumplin, N., Zhang, X., Noar, R.D., and Harrison, M.J. (2012). Polar localization of a symbiosis-specific phosphate transporter is mediated by a transient reorientation of secretion. Proc. Natl. Acad. Sci. USA 109: E665–E672. [[Pumplin_et+al_PolarLocalizationSymbiosisspecificPhosphateTransporter_2012]] [OA](https://api.scholarcy.com/oa_version?query=Pumplin%2C%20N.%20Zhang%2C%20X.%20Noar%2C%20R.D.%20Harrison%2C%20M.J.%20Polar%20localization%20of%20a%20symbiosis-specific%20phosphate%20transporter%20is%20mediated%20by%20a%20transient%20reorientation%20of%20secretion%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Pumplin%2C%20N.%20Zhang%2C%20X.%20Noar%2C%20R.D.%20Harrison%2C%20M.J.%20Polar%20localization%20of%20a%20symbiosis-specific%20phosphate%20transporter%20is%20mediated%20by%20a%20transient%20reorientation%20of%20secretion%202012) [Scite](https://api.scholarcy.com/scite_url?query=Pumplin%2C%20N.%20Zhang%2C%20X.%20Noar%2C%20R.D.%20Harrison%2C%20M.J.%20Polar%20localization%20of%20a%20symbiosis-specific%20phosphate%20transporter%20is%20mediated%20by%20a%20transient%20reorientation%20of%20secretion%202012)

[^Pumplin_et+al_2010_a]: Pumplin, N., Mondo, S.J., Topp, S., Starker, C.G., Gantt, J.S., and Harrison, M.J. (2010). Medicago truncatula Vapyrin is a novel protein required for arbuscular mycorrhizal symbiosis. Plant J. 61: 482–494. [[Pumplin_et+al_MedicagoTruncatulaVapyrinNovelProtein_2010]] [OA](https://api.scholarcy.com/oa_version?query=Pumplin%2C%20N.%20Mondo%2C%20S.J.%20Topp%2C%20S.%20Starker%2C%20C.G.%20Medicago%20truncatula%20Vapyrin%20is%20a%20novel%20protein%20required%20for%20arbuscular%20mycorrhizal%20symbiosis%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Pumplin%2C%20N.%20Mondo%2C%20S.J.%20Topp%2C%20S.%20Starker%2C%20C.G.%20Medicago%20truncatula%20Vapyrin%20is%20a%20novel%20protein%20required%20for%20arbuscular%20mycorrhizal%20symbiosis%202010) [Scite](https://api.scholarcy.com/scite_url?query=Pumplin%2C%20N.%20Mondo%2C%20S.J.%20Topp%2C%20S.%20Starker%2C%20C.G.%20Medicago%20truncatula%20Vapyrin%20is%20a%20novel%20protein%20required%20for%20arbuscular%20mycorrhizal%20symbiosis%202010)

[^Rech_et+al_2013_a]: Rech, S.S., Heidt, S., and Requena, N. (2013). A tandem Kunitz protease inhibitor (KPI106)-serine carboxypeptidase (SCP1) controls mycorrhiza establishment and arbuscule development in Medicago truncatula. Plant J. 75: 711–725. [[Rech_et+al_TandemKunitzProteaseInhibitorkpi106serine_2013]] [OA](https://api.scholarcy.com/oa_version?query=Rech%2C%20S.S.%20Heidt%2C%20S.%20Requena%2C%20N.%20A%20tandem%20Kunitz%20protease%20inhibitor%20%28KPI106%29-serine%20carboxypeptidase%20%28SCP1%29%20controls%20mycorrhiza%20establishment%20and%20arbuscule%20development%20in%20Medicago%20truncatula%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Rech%2C%20S.S.%20Heidt%2C%20S.%20Requena%2C%20N.%20A%20tandem%20Kunitz%20protease%20inhibitor%20%28KPI106%29-serine%20carboxypeptidase%20%28SCP1%29%20controls%20mycorrhiza%20establishment%20and%20arbuscule%20development%20in%20Medicago%20truncatula%202013) [Scite](https://api.scholarcy.com/scite_url?query=Rech%2C%20S.S.%20Heidt%2C%20S.%20Requena%2C%20N.%20A%20tandem%20Kunitz%20protease%20inhibitor%20%28KPI106%29-serine%20carboxypeptidase%20%28SCP1%29%20controls%20mycorrhiza%20establishment%20and%20arbuscule%20development%20in%20Medicago%20truncatula%202013)

[^Remy_et+al_1994_a]: Remy, W., Taylor, T.N., Hass, H., and Kerp, H. (1994). Four hundredmillion-year-old vesicular arbuscular mycorrhizae. Proc. Natl. Acad. Sci. USA 91: 11841–11843. [[Remy_et+al_FourHundredmillionyearVesicularArbuscularMycorrhizae_1994]] [OA](https://api.scholarcy.com/oa_version?query=Remy%2C%20W.%20Taylor%2C%20T.N.%20Hass%2C%20H.%20Kerp%2C%20H.%20Four%20hundredmillion-year-old%20vesicular%20arbuscular%20mycorrhizae%201994) [GScholar](https://scholar.google.co.uk/scholar?q=Remy%2C%20W.%20Taylor%2C%20T.N.%20Hass%2C%20H.%20Kerp%2C%20H.%20Four%20hundredmillion-year-old%20vesicular%20arbuscular%20mycorrhizae%201994) [Scite](https://api.scholarcy.com/scite_url?query=Remy%2C%20W.%20Taylor%2C%20T.N.%20Hass%2C%20H.%20Kerp%2C%20H.%20Four%20hundredmillion-year-old%20vesicular%20arbuscular%20mycorrhizae%201994)

[^Rich_et+al_2015_a]: Rich, M.K., Schorderet, M., Bapaume, L., Falquet, L., Morel, P., Vandenbussche, M., and Reinhardt, D. (2015). The petunia GRAS transcription factor ATA/RAM1 regulates symbiotic gene expression and fungal morphogenesis in arbuscular mycorrhiza. Plant Physiol. 168: 788–797. [[Rich_et+al_PetuniaGrasTranscriptionFactorAtaram1_2015]] [OA](https://api.scholarcy.com/oa_version?query=Rich%2C%20M.K.%20Schorderet%2C%20M.%20Bapaume%2C%20L.%20Falquet%2C%20L.%20The%20petunia%20GRAS%20transcription%20factor%20ATA/RAM1%20regulates%20symbiotic%20gene%20expression%20and%20fungal%20morphogenesis%20in%20arbuscular%20mycorrhiza%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Rich%2C%20M.K.%20Schorderet%2C%20M.%20Bapaume%2C%20L.%20Falquet%2C%20L.%20The%20petunia%20GRAS%20transcription%20factor%20ATA/RAM1%20regulates%20symbiotic%20gene%20expression%20and%20fungal%20morphogenesis%20in%20arbuscular%20mycorrhiza%202015) [Scite](https://api.scholarcy.com/scite_url?query=Rich%2C%20M.K.%20Schorderet%2C%20M.%20Bapaume%2C%20L.%20Falquet%2C%20L.%20The%20petunia%20GRAS%20transcription%20factor%20ATA/RAM1%20regulates%20symbiotic%20gene%20expression%20and%20fungal%20morphogenesis%20in%20arbuscular%20mycorrhiza%202015)

[^Ried_et+al_2014_a]: Ried, M.K., Antolín-Llovera, M., and Parniske, M. (2014). Spontaneous symbiotic reprogramming of plant roots triggered by receptor-like kinases. eLife 3: 3. [[Ried_et+al_SpontaneousSymbioticReprogrammingPlantRoots_2014]] [OA](https://api.scholarcy.com/oa_version?query=Ried%2C%20M.K.%20Antol%C3%ADn-Llovera%2C%20M.%20Parniske%2C%20M.%20Spontaneous%20symbiotic%20reprogramming%20of%20plant%20roots%20triggered%20by%20receptor-like%20kinases%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Ried%2C%20M.K.%20Antol%C3%ADn-Llovera%2C%20M.%20Parniske%2C%20M.%20Spontaneous%20symbiotic%20reprogramming%20of%20plant%20roots%20triggered%20by%20receptor-like%20kinases%202014) [Scite](https://api.scholarcy.com/scite_url?query=Ried%2C%20M.K.%20Antol%C3%ADn-Llovera%2C%20M.%20Parniske%2C%20M.%20Spontaneous%20symbiotic%20reprogramming%20of%20plant%20roots%20triggered%20by%20receptor-like%20kinases%202014)

[^Roth_2017_a]: Roth, R., and Paszkowski, U. (2017). Plant carbon nourishment of arbuscular mycorrhizal fungi. Curr. Opin. Plant Biol. 39: 50–56. [[Roth_PlantCarbonNourishmentArbuscularMycorrhizal_2017]] [OA](https://api.scholarcy.com/oa_version?query=Roth%2C%20R.%20Paszkowski%2C%20U.%20Plant%20carbon%20nourishment%20of%20arbuscular%20mycorrhizal%20fungi%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Roth%2C%20R.%20Paszkowski%2C%20U.%20Plant%20carbon%20nourishment%20of%20arbuscular%20mycorrhizal%20fungi%202017) [Scite](https://api.scholarcy.com/scite_url?query=Roth%2C%20R.%20Paszkowski%2C%20U.%20Plant%20carbon%20nourishment%20of%20arbuscular%20mycorrhizal%20fungi%202017)

[^Saito_2007_a]: Saito, K., et al. (2007). NUCLEOPORIN85 is required for calcium spiking, fungal and bacterial symbioses, and seed production in Lotus japonicus. Plant Cell 19: 610–624. [[Saito_Nucleoporin85RequiredCalciumSpikingFungal_2007]] [OA](https://api.scholarcy.com/oa_version?query=Saito%2C%20K.%20NUCLEOPORIN85%20is%20required%20for%20calcium%20spiking%2C%20fungal%20and%20bacterial%20symbioses%2C%20and%20seed%20production%20in%20Lotus%20japonicus%202007) [GScholar](https://scholar.google.co.uk/scholar?q=Saito%2C%20K.%20NUCLEOPORIN85%20is%20required%20for%20calcium%20spiking%2C%20fungal%20and%20bacterial%20symbioses%2C%20and%20seed%20production%20in%20Lotus%20japonicus%202007) [Scite](https://api.scholarcy.com/scite_url?query=Saito%2C%20K.%20NUCLEOPORIN85%20is%20required%20for%20calcium%20spiking%2C%20fungal%20and%20bacterial%20symbioses%2C%20and%20seed%20production%20in%20Lotus%20japonicus%202007)

[^Schaarschmidt_et+al_2013_a]: Schaarschmidt, S., Gresshoff, P.M., and Hause, B. (2013). Analyzing the soybean transcriptome during autoregulation of mycorrhization identifies the transcription factors GmNF-YA1a/b as positive regulators of arbuscular mycorrhization. Genome Biol. 14: R62. [[Schaarschmidt_et+al_AnalyzingSoybeanTranscriptomeDuringAutoregulation_2013]] [OA](https://api.scholarcy.com/oa_version?query=Schaarschmidt%2C%20S.%20Gresshoff%2C%20P.M.%20Hause%2C%20B.%20Analyzing%20the%20soybean%20transcriptome%20during%20autoregulation%20of%20mycorrhization%20identifies%20the%20transcription%20factors%20GmNF-YA1a/b%20as%20positive%20regulators%20of%20arbuscular%20mycorrhization%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Schaarschmidt%2C%20S.%20Gresshoff%2C%20P.M.%20Hause%2C%20B.%20Analyzing%20the%20soybean%20transcriptome%20during%20autoregulation%20of%20mycorrhization%20identifies%20the%20transcription%20factors%20GmNF-YA1a/b%20as%20positive%20regulators%20of%20arbuscular%20mycorrhization%202013) [Scite](https://api.scholarcy.com/scite_url?query=Schaarschmidt%2C%20S.%20Gresshoff%2C%20P.M.%20Hause%2C%20B.%20Analyzing%20the%20soybean%20transcriptome%20during%20autoregulation%20of%20mycorrhization%20identifies%20the%20transcription%20factors%20GmNF-YA1a/b%20as%20positive%20regulators%20of%20arbuscular%20mycorrhization%202013)

[^Schliemann_et+al_2008_a]: Schliemann, W., Ammer, C., and Strack, D. (2008). Metabolite profiling of mycorrhizal roots of Medicago truncatula. Phytochemistry 69: 112–146. [[Schliemann_et+al_MetaboliteProfilingMycorrhizalRootsMedicago_2008]] [OA](https://api.scholarcy.com/oa_version?query=Schliemann%2C%20W.%20Ammer%2C%20C.%20Strack%2C%20D.%20Metabolite%20profiling%20of%20mycorrhizal%20roots%20of%20Medicago%20truncatula%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Schliemann%2C%20W.%20Ammer%2C%20C.%20Strack%2C%20D.%20Metabolite%20profiling%20of%20mycorrhizal%20roots%20of%20Medicago%20truncatula%202008) [Scite](https://api.scholarcy.com/scite_url?query=Schliemann%2C%20W.%20Ammer%2C%20C.%20Strack%2C%20D.%20Metabolite%20profiling%20of%20mycorrhizal%20roots%20of%20Medicago%20truncatula%202008)

[^Schnabel_et+al_2005_a]: Schnabel, E., Journet, E.P., de Carvalho-Niebel, F., Duc, G., and Frugoli, J. (2005). The Medicago truncatula SUNN gene encodes a CLV1-like leucine-rich repeat receptor kinase that regulates nodule number and root length. Plant Mol. Biol. 58: 809–822. [[Schnabel_et+al_MedicagoTruncatulaSunnGeneEncodes_2005]] [OA](https://api.scholarcy.com/oa_version?query=Schnabel%2C%20E.%20Journet%2C%20E.P.%20de%20Carvalho-Niebel%2C%20F.%20Duc%2C%20G.%20The%20Medicago%20truncatula%20SUNN%20gene%20encodes%20a%20CLV1-like%20leucine-rich%20repeat%20receptor%20kinase%20that%20regulates%20nodule%20number%20and%20root%20length%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Schnabel%2C%20E.%20Journet%2C%20E.P.%20de%20Carvalho-Niebel%2C%20F.%20Duc%2C%20G.%20The%20Medicago%20truncatula%20SUNN%20gene%20encodes%20a%20CLV1-like%20leucine-rich%20repeat%20receptor%20kinase%20that%20regulates%20nodule%20number%20and%20root%20length%202005) [Scite](https://api.scholarcy.com/scite_url?query=Schnabel%2C%20E.%20Journet%2C%20E.P.%20de%20Carvalho-Niebel%2C%20F.%20Duc%2C%20G.%20The%20Medicago%20truncatula%20SUNN%20gene%20encodes%20a%20CLV1-like%20leucine-rich%20repeat%20receptor%20kinase%20that%20regulates%20nodule%20number%20and%20root%20length%202005)

[^Searle_et+al_2003_a]: Searle, I.R., Men, A.E., Laniya, T.S., Buzas, D.M., Iturbe-Ormaetxe, I., Carroll, B.J., and Gresshoff, P.M. (2003). Long-distance signaling in nodulation directed by a CLAVATA1-like receptor kinase. Science 299: 109–112. [[Searle_et+al_LongdistanceSignalingNodulationDirectedClavata1like_2003]] [OA](https://api.scholarcy.com/oa_version?query=Searle%2C%20I.R.%20Men%2C%20A.E.%20Laniya%2C%20T.S.%20Buzas%2C%20D.M.%20Long-distance%20signaling%20in%20nodulation%20directed%20by%20a%20CLAVATA1-like%20receptor%20kinase%202003) [GScholar](https://scholar.google.co.uk/scholar?q=Searle%2C%20I.R.%20Men%2C%20A.E.%20Laniya%2C%20T.S.%20Buzas%2C%20D.M.%20Long-distance%20signaling%20in%20nodulation%20directed%20by%20a%20CLAVATA1-like%20receptor%20kinase%202003) [Scite](https://api.scholarcy.com/scite_url?query=Searle%2C%20I.R.%20Men%2C%20A.E.%20Laniya%2C%20T.S.%20Buzas%2C%20D.M.%20Long-distance%20signaling%20in%20nodulation%20directed%20by%20a%20CLAVATA1-like%20receptor%20kinase%202003)

[^Shachar-Hill_et+al_1995_a]: Shachar-Hill, Y., Pfeffer, P.E., Douds, D., Osman, S.F., Doner, L.W., and Ratcliffe, R.G. (1995). Partitioning of intermediary carbon metabolism in vesicular-arbuscular mycorrhizal leek. Plant Physiol. 108: 7–15. [[Shachar-Hill_et+al_PartitioningIntermediaryCarbonMetabolismVesiculararbuscular_1995]] [OA](https://api.scholarcy.com/oa_version?query=Shachar-Hill%2C%20Y.%20Pfeffer%2C%20P.E.%20Douds%2C%20D.%20Osman%2C%20S.F.%20Partitioning%20of%20intermediary%20carbon%20metabolism%20in%20vesicular-arbuscular%20mycorrhizal%20leek%201995) [GScholar](https://scholar.google.co.uk/scholar?q=Shachar-Hill%2C%20Y.%20Pfeffer%2C%20P.E.%20Douds%2C%20D.%20Osman%2C%20S.F.%20Partitioning%20of%20intermediary%20carbon%20metabolism%20in%20vesicular-arbuscular%20mycorrhizal%20leek%201995) [Scite](https://api.scholarcy.com/scite_url?query=Shachar-Hill%2C%20Y.%20Pfeffer%2C%20P.E.%20Douds%2C%20D.%20Osman%2C%20S.F.%20Partitioning%20of%20intermediary%20carbon%20metabolism%20in%20vesicular-arbuscular%20mycorrhizal%20leek%201995)

[^Shimizu_et+al_2010_a]: Shimizu, T., Nakano, T., Takamizawa, D., Desaki, Y., Ishii-Minami, N., Nishizawa, Y., Minami, E., Okada, K., Yamane, H., Kaku, H., and Shibuya, N. (2010). Two LysM receptor molecules, CEBiP and OsCERK1, cooperatively regulate chitin elicitor signaling in rice. Plant J. 64: 204–214. [[Shimizu_et+al_LysmReceptorMoleculesCebipOscerk1_2010]] [OA](https://api.scholarcy.com/oa_version?query=Shimizu%2C%20T.%20Nakano%2C%20T.%20Takamizawa%2C%20D.%20Desaki%2C%20Y.%20Two%20LysM%20receptor%20molecules%2C%20CEBiP%20and%20OsCERK1%2C%20cooperatively%20regulate%20chitin%20elicitor%20signaling%20in%20rice%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Shimizu%2C%20T.%20Nakano%2C%20T.%20Takamizawa%2C%20D.%20Desaki%2C%20Y.%20Two%20LysM%20receptor%20molecules%2C%20CEBiP%20and%20OsCERK1%2C%20cooperatively%20regulate%20chitin%20elicitor%20signaling%20in%20rice%202010) [Scite](https://api.scholarcy.com/scite_url?query=Shimizu%2C%20T.%20Nakano%2C%20T.%20Takamizawa%2C%20D.%20Desaki%2C%20Y.%20Two%20LysM%20receptor%20molecules%2C%20CEBiP%20and%20OsCERK1%2C%20cooperatively%20regulate%20chitin%20elicitor%20signaling%20in%20rice%202010)

[^Silverstone_et+al_1998_a]: Silverstone, A.L., Ciampaglio, C.N., and Sun, T. (1998). The Arabidopsis RGA gene encodes a transcriptional regulator repressing the gibberellin signal transduction pathway. Plant Cell 10: 155–169. [[Silverstone_et+al_ArabidopsisRgaGeneEncodesTranscriptional_1998]] [OA](https://api.scholarcy.com/oa_version?query=Silverstone%2C%20A.L.%20Ciampaglio%2C%20C.N.%20Sun%2C%20T.%20The%20Arabidopsis%20RGA%20gene%20encodes%20a%20transcriptional%20regulator%20repressing%20the%20gibberellin%20signal%20transduction%20pathway%201998) [GScholar](https://scholar.google.co.uk/scholar?q=Silverstone%2C%20A.L.%20Ciampaglio%2C%20C.N.%20Sun%2C%20T.%20The%20Arabidopsis%20RGA%20gene%20encodes%20a%20transcriptional%20regulator%20repressing%20the%20gibberellin%20signal%20transduction%20pathway%201998) [Scite](https://api.scholarcy.com/scite_url?query=Silverstone%2C%20A.L.%20Ciampaglio%2C%20C.N.%20Sun%2C%20T.%20The%20Arabidopsis%20RGA%20gene%20encodes%20a%20transcriptional%20regulator%20repressing%20the%20gibberellin%20signal%20transduction%20pathway%201998)

[^Singh_et+al_2014_a]: Singh, S., Katzer, K., Lambert, J., Cerri, M., and Parniske, M. (2014). CYCLOPS, a DNA-binding transcriptional activator, orchestrates symbiotic root nodule development. Cell Host Microbe 15: 139–152. [[Singh_et+al_CyclopsDnabindingTranscriptionalActivatorOrchestrates_2014]] [OA](https://api.scholarcy.com/oa_version?query=Singh%2C%20S.%20Katzer%2C%20K.%20Lambert%2C%20J.%20Cerri%2C%20M.%20CYCLOPS%2C%20a%20DNA-binding%20transcriptional%20activator%2C%20orchestrates%20symbiotic%20root%20nodule%20development%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Singh%2C%20S.%20Katzer%2C%20K.%20Lambert%2C%20J.%20Cerri%2C%20M.%20CYCLOPS%2C%20a%20DNA-binding%20transcriptional%20activator%2C%20orchestrates%20symbiotic%20root%20nodule%20development%202014) [Scite](https://api.scholarcy.com/scite_url?query=Singh%2C%20S.%20Katzer%2C%20K.%20Lambert%2C%20J.%20Cerri%2C%20M.%20CYCLOPS%2C%20a%20DNA-binding%20transcriptional%20activator%2C%20orchestrates%20symbiotic%20root%20nodule%20development%202014)

[^Smit_et+al_2005_a]: Smit, P., Raedts, J., Portyanko, V., Debellé, F., Gough, C., Bisseling, T., and Geurts, R. (2005). NSP1 of the GRAS protein family is essential for rhizobial Nod factor-induced transcription. Science 308: 1789–1791. [[Smit_et+al_Nsp1GrasProteinFamilyEssential_2005]] [OA](https://api.scholarcy.com/oa_version?query=Smit%2C%20P.%20Raedts%2C%20J.%20Portyanko%2C%20V.%20Debell%C3%A9%2C%20F.%20NSP1%20of%20the%20GRAS%20protein%20family%20is%20essential%20for%20rhizobial%20Nod%20factor-induced%20transcription%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Smit%2C%20P.%20Raedts%2C%20J.%20Portyanko%2C%20V.%20Debell%C3%A9%2C%20F.%20NSP1%20of%20the%20GRAS%20protein%20family%20is%20essential%20for%20rhizobial%20Nod%20factor-induced%20transcription%202005) [Scite](https://api.scholarcy.com/scite_url?query=Smit%2C%20P.%20Raedts%2C%20J.%20Portyanko%2C%20V.%20Debell%C3%A9%2C%20F.%20NSP1%20of%20the%20GRAS%20protein%20family%20is%20essential%20for%20rhizobial%20Nod%20factor-induced%20transcription%202005)

[^Smith_2008_a]: Smith, S.E., and Read, D.J. (2008). Mycorrhizal Symbiosis. (San Diego, CA: Academic Press). [[Smith_MycorrhizalSymbiosis_2008]] [OA](https://scholar.google.co.uk/scholar?q=Smith%2C%20S.E.%20Read%2C%20D.J.%20Mycorrhizal%20Symbiosis%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Smith%2C%20S.E.%20Read%2C%20D.J.%20Mycorrhizal%20Symbiosis%202008) 

[^Stracke_et+al_2002_a]: Stracke, S., Kistner, C., Yoshida, S., Mulder, L., Sato, S., Kaneko, T., Tabata, S., Sandal, N., Stougaard, J., Szczyglowski, K., and Parniske, M. (2002). A plant receptor-like kinase required for both bacterial and fungal symbiosis. Nature 417: 959–962. [[Stracke_et+al_PlantReceptorlikeKinaseRequiredBoth_2002]] [OA](https://api.scholarcy.com/oa_version?query=Stracke%2C%20S.%20Kistner%2C%20C.%20Yoshida%2C%20S.%20Mulder%2C%20L.%20A%20plant%20receptor-like%20kinase%20required%20for%20both%20bacterial%20and%20fungal%20symbiosis%202002) [GScholar](https://scholar.google.co.uk/scholar?q=Stracke%2C%20S.%20Kistner%2C%20C.%20Yoshida%2C%20S.%20Mulder%2C%20L.%20A%20plant%20receptor-like%20kinase%20required%20for%20both%20bacterial%20and%20fungal%20symbiosis%202002) [Scite](https://api.scholarcy.com/scite_url?query=Stracke%2C%20S.%20Kistner%2C%20C.%20Yoshida%2C%20S.%20Mulder%2C%20L.%20A%20plant%20receptor-like%20kinase%20required%20for%20both%20bacterial%20and%20fungal%20symbiosis%202002)

[^Takeda_et+al_2009_a]: Takeda, N., Sato, S., Asamizu, E., Tabata, S., and Parniske, M. (2009). Apoplastic plant subtilases support arbuscular mycorrhiza development in Lotus japonicus. Plant J. 58: 766–777. [[Takeda_et+al_ApoplasticPlantSubtilasesSupportArbuscular_2009]] [OA](https://api.scholarcy.com/oa_version?query=Takeda%2C%20N.%20Sato%2C%20S.%20Asamizu%2C%20E.%20Tabata%2C%20S.%20Apoplastic%20plant%20subtilases%20support%20arbuscular%20mycorrhiza%20development%20in%20Lotus%20japonicus%202009) [GScholar](https://scholar.google.co.uk/scholar?q=Takeda%2C%20N.%20Sato%2C%20S.%20Asamizu%2C%20E.%20Tabata%2C%20S.%20Apoplastic%20plant%20subtilases%20support%20arbuscular%20mycorrhiza%20development%20in%20Lotus%20japonicus%202009) [Scite](https://api.scholarcy.com/scite_url?query=Takeda%2C%20N.%20Sato%2C%20S.%20Asamizu%2C%20E.%20Tabata%2C%20S.%20Apoplastic%20plant%20subtilases%20support%20arbuscular%20mycorrhiza%20development%20in%20Lotus%20japonicus%202009)

[^Takeda_et+al_2013_a]: Takeda, N., Tsuzuki, S., Suzaki, T., Parniske, M., and Kawaguchi, M. (2013). CERBERUS and NSP1 of Lotus japonicus are common symbiosis genes that modulate arbuscular mycorrhiza development. Plant Cell Physiol. 54: 1711–1723. [[Takeda_et+al_CerberusNsp1LotusJaponicusCommon_2013]] [OA](https://api.scholarcy.com/oa_version?query=Takeda%2C%20N.%20Tsuzuki%2C%20S.%20Suzaki%2C%20T.%20Parniske%2C%20M.%20CERBERUS%20and%20NSP1%20of%20Lotus%20japonicus%20are%20common%20symbiosis%20genes%20that%20modulate%20arbuscular%20mycorrhiza%20development%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Takeda%2C%20N.%20Tsuzuki%2C%20S.%20Suzaki%2C%20T.%20Parniske%2C%20M.%20CERBERUS%20and%20NSP1%20of%20Lotus%20japonicus%20are%20common%20symbiosis%20genes%20that%20modulate%20arbuscular%20mycorrhiza%20development%202013) [Scite](https://api.scholarcy.com/scite_url?query=Takeda%2C%20N.%20Tsuzuki%2C%20S.%20Suzaki%2C%20T.%20Parniske%2C%20M.%20CERBERUS%20and%20NSP1%20of%20Lotus%20japonicus%20are%20common%20symbiosis%20genes%20that%20modulate%20arbuscular%20mycorrhiza%20development%202013)

[^Tani_et+al_2009_a]: Tani, C., Ohtomo, R., Osaki, M., Kuga, Y., and Ezawa, T. (2009). ATP-dependent but proton gradient-independent polyphosphatesynthesizing activity in extraradical hyphae of an arbuscular mycorrhizal fungus. Appl. Environ. Microbiol. 75: 7044–7050. [[Tani_et+al_AtpdependentProtonGradientindependentPolyphosphatesynthesizingActivity_2009]] [OA](https://api.scholarcy.com/oa_version?query=Tani%2C%20C.%20Ohtomo%2C%20R.%20Osaki%2C%20M.%20Kuga%2C%20Y.%20ATP-dependent%20but%20proton%20gradient-independent%20polyphosphatesynthesizing%20activity%20in%20extraradical%20hyphae%20of%20an%20arbuscular%20mycorrhizal%20fungus%202009) [GScholar](https://scholar.google.co.uk/scholar?q=Tani%2C%20C.%20Ohtomo%2C%20R.%20Osaki%2C%20M.%20Kuga%2C%20Y.%20ATP-dependent%20but%20proton%20gradient-independent%20polyphosphatesynthesizing%20activity%20in%20extraradical%20hyphae%20of%20an%20arbuscular%20mycorrhizal%20fungus%202009) [Scite](https://api.scholarcy.com/scite_url?query=Tani%2C%20C.%20Ohtomo%2C%20R.%20Osaki%2C%20M.%20Kuga%2C%20Y.%20ATP-dependent%20but%20proton%20gradient-independent%20polyphosphatesynthesizing%20activity%20in%20extraradical%20hyphae%20of%20an%20arbuscular%20mycorrhizal%20fungus%202009)

[^Tirichine_2006_a]: Tirichine, L., et al. (2006). Deregulation of a Ca2+/calmodulindependent kinase leads to spontaneous nodule development. Nature 441: 1153–1156. [[Tirichine_DeregulationCa2calmodulindependentKinaseLeadsSpontaneous_2006]] [OA](https://api.scholarcy.com/oa_version?query=Tirichine%2C%20L.%20Deregulation%20of%20a%20Ca2%2B/calmodulindependent%20kinase%20leads%20to%20spontaneous%20nodule%20development%202006) [GScholar](https://scholar.google.co.uk/scholar?q=Tirichine%2C%20L.%20Deregulation%20of%20a%20Ca2%2B/calmodulindependent%20kinase%20leads%20to%20spontaneous%20nodule%20development%202006) [Scite](https://api.scholarcy.com/scite_url?query=Tirichine%2C%20L.%20Deregulation%20of%20a%20Ca2%2B/calmodulindependent%20kinase%20leads%20to%20spontaneous%20nodule%20development%202006)

[^Tisserant_2013_a]: Tisserant, E., et al. (2013). Genome of an arbuscular mycorrhizal fungus provides insight into the oldest plant symbiosis. Proc. Natl. Acad. Sci. USA 110: 20117–20122,. Erratum. Proc. Natl. Acad. Sci. USA 111: 563. [[Tisserant_GenomeArbuscularMycorrhizalFungusProvides_2013]] [OA](https://api.scholarcy.com/oa_version?query=Tisserant%2C%20E.%20Genome%20of%20an%20arbuscular%20mycorrhizal%20fungus%20provides%20insight%20into%20the%20oldest%20plant%20symbiosis%202013) [GScholar](https://scholar.google.co.uk/scholar?q=Tisserant%2C%20E.%20Genome%20of%20an%20arbuscular%20mycorrhizal%20fungus%20provides%20insight%20into%20the%20oldest%20plant%20symbiosis%202013) [Scite](https://api.scholarcy.com/scite_url?query=Tisserant%2C%20E.%20Genome%20of%20an%20arbuscular%20mycorrhizal%20fungus%20provides%20insight%20into%20the%20oldest%20plant%20symbiosis%202013)

[^Tong_et+al_2009_a]: Tong, H., Jin, Y., Liu, W., Li, F., Fang, J., Yin, Y., Qian, Q., Zhu, L., and Chu, C. (2009). DWARF AND LOW-TILLERING, a new member of the GRAS family, plays positive roles in brassinosteroid signaling in rice. Plant J. 58: 803–816. [[Tong_et+al_DwarfAndLowtilleringMemberGras_2009]] [OA](https://api.scholarcy.com/oa_version?query=Tong%2C%20H.%20Jin%2C%20Y.%20Liu%2C%20W.%20Li%2C%20F.%20DWARF%20AND%20LOW-TILLERING%2C%20a%20new%20member%20of%20the%20GRAS%20family%2C%20plays%20positive%20roles%20in%20brassinosteroid%20signaling%20in%20rice%202009) [GScholar](https://scholar.google.co.uk/scholar?q=Tong%2C%20H.%20Jin%2C%20Y.%20Liu%2C%20W.%20Li%2C%20F.%20DWARF%20AND%20LOW-TILLERING%2C%20a%20new%20member%20of%20the%20GRAS%20family%2C%20plays%20positive%20roles%20in%20brassinosteroid%20signaling%20in%20rice%202009) [Scite](https://api.scholarcy.com/scite_url?query=Tong%2C%20H.%20Jin%2C%20Y.%20Liu%2C%20W.%20Li%2C%20F.%20DWARF%20AND%20LOW-TILLERING%2C%20a%20new%20member%20of%20the%20GRAS%20family%2C%20plays%20positive%20roles%20in%20brassinosteroid%20signaling%20in%20rice%202009)

[^Tr_et+al_2005_a]: Trépanier, M., Bécard, G., Moutoglis, P., Willemot, C., Gagné, S., Avis, T.J., and Rioux, J.A. (2005). Dependence of arbuscularmycorrhizal fungi on their plant host for palmitic acid synthesis. Appl. Environ. Microbiol. 71: 5341–5347. [[Tr_et+al_DependenceArbuscularmycorrhizalFungiTheirPlant_2005]] [OA](https://api.scholarcy.com/oa_version?query=Tr%C3%A9panier%2C%20M.%20B%C3%A9card%2C%20G.%20Moutoglis%2C%20P.%20Willemot%2C%20C.%20Dependence%20of%20arbuscularmycorrhizal%20fungi%20on%20their%20plant%20host%20for%20palmitic%20acid%20synthesis%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Tr%C3%A9panier%2C%20M.%20B%C3%A9card%2C%20G.%20Moutoglis%2C%20P.%20Willemot%2C%20C.%20Dependence%20of%20arbuscularmycorrhizal%20fungi%20on%20their%20plant%20host%20for%20palmitic%20acid%20synthesis%202005) [Scite](https://api.scholarcy.com/scite_url?query=Tr%C3%A9panier%2C%20M.%20B%C3%A9card%2C%20G.%20Moutoglis%2C%20P.%20Willemot%2C%20C.%20Dependence%20of%20arbuscularmycorrhizal%20fungi%20on%20their%20plant%20host%20for%20palmitic%20acid%20synthesis%202005)

[^Venkateshwaran_et+al_2015_a]: Venkateshwaran, M., Jayaraman, D., Chabaud, M., Genre, A., Balloon, A.J., Maeda, J., Forshey, K., den Os, D., Kwiecien, N.W., Coon, J.J., Barker, D.G., and Ané, J.M. (2015). A role for the mevalonate pathway in early plant symbiotic signaling. Proc. Natl. Acad. Sci. USA 112: 9781–9786. [[Venkateshwaran_et+al_RoleMevalonatePathwayEarlyPlant_2015]] [OA](https://api.scholarcy.com/oa_version?query=Venkateshwaran%2C%20M.%20Jayaraman%2C%20D.%20Chabaud%2C%20M.%20Genre%2C%20A.%20A%20role%20for%20the%20mevalonate%20pathway%20in%20early%20plant%20symbiotic%20signaling%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Venkateshwaran%2C%20M.%20Jayaraman%2C%20D.%20Chabaud%2C%20M.%20Genre%2C%20A.%20A%20role%20for%20the%20mevalonate%20pathway%20in%20early%20plant%20symbiotic%20signaling%202015) [Scite](https://api.scholarcy.com/scite_url?query=Venkateshwaran%2C%20M.%20Jayaraman%2C%20D.%20Chabaud%2C%20M.%20Genre%2C%20A.%20A%20role%20for%20the%20mevalonate%20pathway%20in%20early%20plant%20symbiotic%20signaling%202015)

[^Vernié_2016_a]: Vernié, T., et al. (2016). PUB1 interacts with the receptor kinase DMI2 and negatively regulates rhizobial and arbuscular mycorrhizal symbioses through its ubiquitination activity in Medicago truncatula. Plant Physiol. 170: 2312–2324. [[Vernié_Pub1InteractsWithReceptorKinase_2016]] [OA](https://api.scholarcy.com/oa_version?query=Verni%C3%A9%2C%20T.%20PUB1%20interacts%20with%20the%20receptor%20kinase%20DMI2%20and%20negatively%20regulates%20rhizobial%20and%20arbuscular%20mycorrhizal%20symbioses%20through%20its%20ubiquitination%20activity%20in%20Medicago%20truncatula%202016) [GScholar](https://scholar.google.co.uk/scholar?q=Verni%C3%A9%2C%20T.%20PUB1%20interacts%20with%20the%20receptor%20kinase%20DMI2%20and%20negatively%20regulates%20rhizobial%20and%20arbuscular%20mycorrhizal%20symbioses%20through%20its%20ubiquitination%20activity%20in%20Medicago%20truncatula%202016) [Scite](https://api.scholarcy.com/scite_url?query=Verni%C3%A9%2C%20T.%20PUB1%20interacts%20with%20the%20receptor%20kinase%20DMI2%20and%20negatively%20regulates%20rhizobial%20and%20arbuscular%20mycorrhizal%20symbioses%20through%20its%20ubiquitination%20activity%20in%20Medicago%20truncatula%202016)

[^Wang_2006_a]: Wang, B., and Qiu, Y.L. (2006). Phylogenetic distribution and evolution of mycorrhizas in land plants. Mycorrhiza 16: 299–363. [[Wang_PhylogeneticDistributionEvolutionMycorrhizasLand_2006]] [OA](https://api.scholarcy.com/oa_version?query=Wang%2C%20B.%20Qiu%2C%20Y.L.%20Phylogenetic%20distribution%20and%20evolution%20of%20mycorrhizas%20in%20land%20plants%202006) [GScholar](https://scholar.google.co.uk/scholar?q=Wang%2C%20B.%20Qiu%2C%20Y.L.%20Phylogenetic%20distribution%20and%20evolution%20of%20mycorrhizas%20in%20land%20plants%202006) [Scite](https://api.scholarcy.com/scite_url?query=Wang%2C%20B.%20Qiu%2C%20Y.L.%20Phylogenetic%20distribution%20and%20evolution%20of%20mycorrhizas%20in%20land%20plants%202006)

[^Wang_et+al_2012_a]: Wang, E., Schornack, S., Marsh, J.F., Gobbato, E., Schwessinger, B., Eastmond, P., Schultze, M., Kamoun, S., and Oldroyd, G.E.D. (2012). A common signaling process that promotes mycorrhizal and oomycete colonization of plants. Curr. Biol. 22: 2242–2246. [[Wang_et+al_CommonSignalingProcessThatPromotes_2012]] [OA](https://api.scholarcy.com/oa_version?query=Wang%2C%20E.%20Schornack%2C%20S.%20Marsh%2C%20J.F.%20Gobbato%2C%20E.%20A%20common%20signaling%20process%20that%20promotes%20mycorrhizal%20and%20oomycete%20colonization%20of%20plants%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Wang%2C%20E.%20Schornack%2C%20S.%20Marsh%2C%20J.F.%20Gobbato%2C%20E.%20A%20common%20signaling%20process%20that%20promotes%20mycorrhizal%20and%20oomycete%20colonization%20of%20plants%202012) [Scite](https://api.scholarcy.com/scite_url?query=Wang%2C%20E.%20Schornack%2C%20S.%20Marsh%2C%20J.F.%20Gobbato%2C%20E.%20A%20common%20signaling%20process%20that%20promotes%20mycorrhizal%20and%20oomycete%20colonization%20of%20plants%202012)

[^Wang_2014_a]: Wang, E., et al. (2014). A H+-ATPase that energizes nutrient uptake during mycorrhizal symbioses in rice and Medicago truncatula. Plant Cell 26: 1818–1830. [[Wang_HatpaseThatEnergizesNutrientUptake_2014]] [OA](https://api.scholarcy.com/oa_version?query=Wang%2C%20E.%20A%20H%2B-ATPase%20that%20energizes%20nutrient%20uptake%20during%20mycorrhizal%20symbioses%20in%20rice%20and%20Medicago%20truncatula%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Wang%2C%20E.%20A%20H%2B-ATPase%20that%20energizes%20nutrient%20uptake%20during%20mycorrhizal%20symbioses%20in%20rice%20and%20Medicago%20truncatula%202014) [Scite](https://api.scholarcy.com/scite_url?query=Wang%2C%20E.%20A%20H%2B-ATPase%20that%20energizes%20nutrient%20uptake%20during%20mycorrhizal%20symbioses%20in%20rice%20and%20Medicago%20truncatula%202014)

[^Waters_et+al_2015_a]: Waters, M.T., Scaffidi, A., Flematti, G., and Smith, S.M. (2015). Substrate-induced degradation of the a/b-fold hydrolase KARRIKIN INSENSITIVE2 requires a functional catalytic triad but is independent of MAX2. Mol. Plant 8: 814–817. [[Waters_et+al_SubstrateinducedDegradationfoldHydrolaseKarrikin_2015]] [OA](https://api.scholarcy.com/oa_version?query=Waters%2C%20M.T.%20Scaffidi%2C%20A.%20Flematti%2C%20G.%20Smith%2C%20S.M.%20Substrate-induced%20degradation%20of%20the%20a/b-fold%20hydrolase%20KARRIKIN%20INSENSITIVE2%20requires%20a%20functional%20catalytic%20triad%20but%20is%20independent%20of%20MAX2%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Waters%2C%20M.T.%20Scaffidi%2C%20A.%20Flematti%2C%20G.%20Smith%2C%20S.M.%20Substrate-induced%20degradation%20of%20the%20a/b-fold%20hydrolase%20KARRIKIN%20INSENSITIVE2%20requires%20a%20functional%20catalytic%20triad%20but%20is%20independent%20of%20MAX2%202015) [Scite](https://api.scholarcy.com/scite_url?query=Waters%2C%20M.T.%20Scaffidi%2C%20A.%20Flematti%2C%20G.%20Smith%2C%20S.M.%20Substrate-induced%20degradation%20of%20the%20a/b-fold%20hydrolase%20KARRIKIN%20INSENSITIVE2%20requires%20a%20functional%20catalytic%20triad%20but%20is%20independent%20of%20MAX2%202015)

[^Waters_et+al_2014_a]: Waters, M.T., Scaffidi, A., Sun, Y.K., Flematti, G.R., and Smith, S.M. (2014). The karrikin response system of Arabidopsis. Plant J. 79: 623–631. [[Waters_et+al_KarrikinResponseSystemArabidopsis_2014]] [OA](https://api.scholarcy.com/oa_version?query=Waters%2C%20M.T.%20Scaffidi%2C%20A.%20Sun%2C%20Y.K.%20Flematti%2C%20G.R.%20The%20karrikin%20response%20system%20of%20Arabidopsis%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Waters%2C%20M.T.%20Scaffidi%2C%20A.%20Sun%2C%20Y.K.%20Flematti%2C%20G.R.%20The%20karrikin%20response%20system%20of%20Arabidopsis%202014) [Scite](https://api.scholarcy.com/scite_url?query=Waters%2C%20M.T.%20Scaffidi%2C%20A.%20Sun%2C%20Y.K.%20Flematti%2C%20G.R.%20The%20karrikin%20response%20system%20of%20Arabidopsis%202014)

[^Waters_et+al_2012_a]: Waters, M.T., Nelson, D.C., Scaffidi, A., Flematti, G.R., Sun, Y.K.M., Dixon, K.W., and Smith, S.M. (2012). Specialisation within the DWARF14 protein family confers distinct responses to karrikins and strigolactones in Arabidopsis. Development 139: 1285–1295. [[Waters_et+al_SpecialisationWithinDwarf14ProteinFamily_2012]] [OA](https://api.scholarcy.com/oa_version?query=Waters%2C%20M.T.%20Nelson%2C%20D.C.%20Scaffidi%2C%20A.%20Flematti%2C%20G.R.%20Specialisation%20within%20the%20DWARF14%20protein%20family%20confers%20distinct%20responses%20to%20karrikins%20and%20strigolactones%20in%20Arabidopsis%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Waters%2C%20M.T.%20Nelson%2C%20D.C.%20Scaffidi%2C%20A.%20Flematti%2C%20G.R.%20Specialisation%20within%20the%20DWARF14%20protein%20family%20confers%20distinct%20responses%20to%20karrikins%20and%20strigolactones%20in%20Arabidopsis%202012) [Scite](https://api.scholarcy.com/scite_url?query=Waters%2C%20M.T.%20Nelson%2C%20D.C.%20Scaffidi%2C%20A.%20Flematti%2C%20G.R.%20Specialisation%20within%20the%20DWARF14%20protein%20family%20confers%20distinct%20responses%20to%20karrikins%20and%20strigolactones%20in%20Arabidopsis%202012)

[^Wewer_et+al_2014_a]: Wewer, V., Brands, M., and Dörmann, P. (2014). Fatty acid synthesis and lipid metabolism in the obligate biotrophic fungus Rhizophagus irregularis during mycorrhization of Lotus japonicus. Plant J. 79: 398–412. [[Wewer_et+al_FattyAcidSynthesisLipidMetabolism_2014]] [OA](https://api.scholarcy.com/oa_version?query=Wewer%2C%20V.%20Brands%2C%20M.%20D%C3%B6rmann%2C%20P.%20Fatty%20acid%20synthesis%20and%20lipid%20metabolism%20in%20the%20obligate%20biotrophic%20fungus%20Rhizophagus%20irregularis%20during%20mycorrhization%20of%20Lotus%20japonicus%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Wewer%2C%20V.%20Brands%2C%20M.%20D%C3%B6rmann%2C%20P.%20Fatty%20acid%20synthesis%20and%20lipid%20metabolism%20in%20the%20obligate%20biotrophic%20fungus%20Rhizophagus%20irregularis%20during%20mycorrhization%20of%20Lotus%20japonicus%202014) [Scite](https://api.scholarcy.com/scite_url?query=Wewer%2C%20V.%20Brands%2C%20M.%20D%C3%B6rmann%2C%20P.%20Fatty%20acid%20synthesis%20and%20lipid%20metabolism%20in%20the%20obligate%20biotrophic%20fungus%20Rhizophagus%20irregularis%20during%20mycorrhization%20of%20Lotus%20japonicus%202014)

[^Xue_2015_a]: Xue, L., Cui, H., Buer, B., Vijayakumar, V.,, P.-M., Junkermann, S., and Bucher, M. (2015). Network of GRAS transcription factors involved in the control of arbuscule development in Lotus japonicus. Plant Physiol. 167: 854–871. [[Xue_NetworkGrasTranscriptionFactorsInvolved_2015]] [OA](https://api.scholarcy.com/oa_version?query=Xue%20L%20Cui%20H%20Buer%20B%20Vijayakumar%20V%20PM%20Junkermann%20S%20and%20Bucher%20M%202015%20Network%20of%20GRAS%20transcription%20factors%20involved%20in%20the%20control%20of%20arbuscule%20development%20in%20Lotus%20japonicus%20Plant%20Physiol%20167%20854871) [GScholar](https://scholar.google.co.uk/scholar?q=Xue%20L%20Cui%20H%20Buer%20B%20Vijayakumar%20V%20PM%20Junkermann%20S%20and%20Bucher%20M%202015%20Network%20of%20GRAS%20transcription%20factors%20involved%20in%20the%20control%20of%20arbuscule%20development%20in%20Lotus%20japonicus%20Plant%20Physiol%20167%20854871) [Scite](https://api.scholarcy.com/scite_url?query=Xue%20L%20Cui%20H%20Buer%20B%20Vijayakumar%20V%20PM%20Junkermann%20S%20and%20Bucher%20M%202015%20Network%20of%20GRAS%20transcription%20factors%20involved%20in%20the%20control%20of%20arbuscule%20development%20in%20Lotus%20japonicus%20Plant%20Physiol%20167%20854871)

[^Yang_2012_a]: Yang, S.Y., et al. (2012). Nonredundant regulation of rice arbuscular mycorrhizal symbiosis by two members of the phosphate transporter1 gene family. Plant Cell 24: 4236–4251. [[Yang_NonredundantRegulationRiceArbuscularMycorrhizal_2012]] [OA](https://api.scholarcy.com/oa_version?query=Yang%2C%20S.Y.%20Nonredundant%20regulation%20of%20rice%20arbuscular%20mycorrhizal%20symbiosis%20by%20two%20members%20of%20the%20phosphate%20transporter1%20gene%20family%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Yang%2C%20S.Y.%20Nonredundant%20regulation%20of%20rice%20arbuscular%20mycorrhizal%20symbiosis%20by%20two%20members%20of%20the%20phosphate%20transporter1%20gene%20family%202012) [Scite](https://api.scholarcy.com/scite_url?query=Yang%2C%20S.Y.%20Nonredundant%20regulation%20of%20rice%20arbuscular%20mycorrhizal%20symbiosis%20by%20two%20members%20of%20the%20phosphate%20transporter1%20gene%20family%202012)

[^Yano_2008_a]: Yano, K., et al. (2008). CYCLOPS, a mediator of symbiotic intracellular accommodation. Proc. Natl. Acad. Sci. USA 105: 20540–20545. [[Yano_CyclopsMediatorSymbioticIntracellularAccommodation_2008]] [OA](https://api.scholarcy.com/oa_version?query=Yano%2C%20K.%20CYCLOPS%2C%20a%20mediator%20of%20symbiotic%20intracellular%20accommodation%202008) [GScholar](https://scholar.google.co.uk/scholar?q=Yano%2C%20K.%20CYCLOPS%2C%20a%20mediator%20of%20symbiotic%20intracellular%20accommodation%202008) [Scite](https://api.scholarcy.com/scite_url?query=Yano%2C%20K.%20CYCLOPS%2C%20a%20mediator%20of%20symbiotic%20intracellular%20accommodation%202008)

[^Yoneyama_et+al_2007_a]: Yoneyama, K., Yoneyama, K., Takeuchi, Y., and Sekimoto, H. (2007a). Phosphorus deficiency in red clover promotes exudation of orobanchol, the signal for mycorrhizal symbionts and germination stimulant for root parasites. Planta 225: 1031–1038. [[Yoneyama_et+al_PhosphorusDeficiencyCloverPromotesExudation_2007]] [OA](https://api.scholarcy.com/oa_version?query=Yoneyama%2C%20K.%20Yoneyama%2C%20K.%20Takeuchi%2C%20Y.%20Sekimoto%2C%20H.%20Phosphorus%20deficiency%20in%20red%20clover%20promotes%20exudation%20of%20orobanchol%2C%20the%20signal%20for%20mycorrhizal%20symbionts%20and%20germination%20stimulant%20for%20root%20parasites%202007) [GScholar](https://scholar.google.co.uk/scholar?q=Yoneyama%2C%20K.%20Yoneyama%2C%20K.%20Takeuchi%2C%20Y.%20Sekimoto%2C%20H.%20Phosphorus%20deficiency%20in%20red%20clover%20promotes%20exudation%20of%20orobanchol%2C%20the%20signal%20for%20mycorrhizal%20symbionts%20and%20germination%20stimulant%20for%20root%20parasites%202007) [Scite](https://api.scholarcy.com/scite_url?query=Yoneyama%2C%20K.%20Yoneyama%2C%20K.%20Takeuchi%2C%20Y.%20Sekimoto%2C%20H.%20Phosphorus%20deficiency%20in%20red%20clover%20promotes%20exudation%20of%20orobanchol%2C%20the%20signal%20for%20mycorrhizal%20symbionts%20and%20germination%20stimulant%20for%20root%20parasites%202007)

[^Yoneyama_et+al_2007_b]: Yoneyama, K., Xie, X., Kusumoto, D., Sekimoto, H., Sugimoto, Y., Takeuchi, Y., and Yoneyama, K. (2007b). Nitrogen deficiency as well as phosphorus deficiency in sorghum promotes the production and exudation of 5-deoxystrigol, the host recognition signal for arbuscular mycorrhizal fungi and root parasites. Planta 227: 125–132. [[Yoneyama_et+al_NitrogenDeficiencyWellPhosphorusDeficiency_2007]] [OA](https://api.scholarcy.com/oa_version?query=Yoneyama%2C%20K.%20Xie%2C%20X.%20Kusumoto%2C%20D.%20Sekimoto%2C%20H.%20Nitrogen%20deficiency%20as%20well%20as%20phosphorus%20deficiency%20in%20sorghum%20promotes%20the%20production%20and%20exudation%20of%205-deoxystrigol%2C%20the%20host%20recognition%20signal%20for%20arbuscular%20mycorrhizal%20fungi%20and%20root%20parasites%202007) [GScholar](https://scholar.google.co.uk/scholar?q=Yoneyama%2C%20K.%20Xie%2C%20X.%20Kusumoto%2C%20D.%20Sekimoto%2C%20H.%20Nitrogen%20deficiency%20as%20well%20as%20phosphorus%20deficiency%20in%20sorghum%20promotes%20the%20production%20and%20exudation%20of%205-deoxystrigol%2C%20the%20host%20recognition%20signal%20for%20arbuscular%20mycorrhizal%20fungi%20and%20root%20parasites%202007) [Scite](https://api.scholarcy.com/scite_url?query=Yoneyama%2C%20K.%20Xie%2C%20X.%20Kusumoto%2C%20D.%20Sekimoto%2C%20H.%20Nitrogen%20deficiency%20as%20well%20as%20phosphorus%20deficiency%20in%20sorghum%20promotes%20the%20production%20and%20exudation%20of%205-deoxystrigol%2C%20the%20host%20recognition%20signal%20for%20arbuscular%20mycorrhizal%20fungi%20and%20root%20parasites%202007)

[^Yoshida_2014_a]: Yoshida, H., et al. (2014). DELLA protein functions as a transcriptional activator through the DNA binding of the indeterminate domain family proteins. Proc. Natl. Acad. Sci. USA 111: 7861–7866. [[Yoshida_DellaProteinFunctionsTranscriptionalActivator_2014]] [OA](https://api.scholarcy.com/oa_version?query=Yoshida%2C%20H.%20DELLA%20protein%20functions%20as%20a%20transcriptional%20activator%20through%20the%20DNA%20binding%20of%20the%20indeterminate%20domain%20family%20proteins%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Yoshida%2C%20H.%20DELLA%20protein%20functions%20as%20a%20transcriptional%20activator%20through%20the%20DNA%20binding%20of%20the%20indeterminate%20domain%20family%20proteins%202014) [Scite](https://api.scholarcy.com/scite_url?query=Yoshida%2C%20H.%20DELLA%20protein%20functions%20as%20a%20transcriptional%20activator%20through%20the%20DNA%20binding%20of%20the%20indeterminate%20domain%20family%20proteins%202014)

[^Yoshida_et+al_2012_a]: Yoshida, S., Kameoka, H., Tempo, M., Akiyama, K., Umehara, M., Yamaguchi, S., Hayashi, H., Kyozuka, J., and Shirasu, K. (2012). The D3 F-box protein is a key component in host strigolactone responses essential for arbuscular mycorrhizal symbiosis. New Phytol. 196: 1208–1216. [[Yoshida_et+al_D3FProteinComponentHost_2012]] [OA](https://api.scholarcy.com/oa_version?query=Yoshida%2C%20S.%20Kameoka%2C%20H.%20Tempo%2C%20M.%20Akiyama%2C%20K.%20The%20D3%20F-box%20protein%20is%20a%20key%20component%20in%20host%20strigolactone%20responses%20essential%20for%20arbuscular%20mycorrhizal%20symbiosis%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Yoshida%2C%20S.%20Kameoka%2C%20H.%20Tempo%2C%20M.%20Akiyama%2C%20K.%20The%20D3%20F-box%20protein%20is%20a%20key%20component%20in%20host%20strigolactone%20responses%20essential%20for%20arbuscular%20mycorrhizal%20symbiosis%202012) [Scite](https://api.scholarcy.com/scite_url?query=Yoshida%2C%20S.%20Kameoka%2C%20H.%20Tempo%2C%20M.%20Akiyama%2C%20K.%20The%20D3%20F-box%20protein%20is%20a%20key%20component%20in%20host%20strigolactone%20responses%20essential%20for%20arbuscular%20mycorrhizal%20symbiosis%202012)

[^Yu_2014_a]: Yu, N., et al. (2014). A DELLA protein complex controls the arbuscular mycorrhizal symbiosis in plants. Cell Res. 24: 130–133. [[Yu_DellaProteinComplexControlsArbuscular_2014]] [OA](https://api.scholarcy.com/oa_version?query=Yu%2C%20N.%20A%20DELLA%20protein%20complex%20controls%20the%20arbuscular%20mycorrhizal%20symbiosis%20in%20plants%202014) [GScholar](https://scholar.google.co.uk/scholar?q=Yu%2C%20N.%20A%20DELLA%20protein%20complex%20controls%20the%20arbuscular%20mycorrhizal%20symbiosis%20in%20plants%202014) [Scite](https://api.scholarcy.com/scite_url?query=Yu%2C%20N.%20A%20DELLA%20protein%20complex%20controls%20the%20arbuscular%20mycorrhizal%20symbiosis%20in%20plants%202014)

[^Zhang_et+al_2012_a]: Zhang, D., Iyer, L.M., and Aravind, L. (2012). Bacterial GRAS domain proteins throw new light on gibberellic acid response mechanisms. Bioinformatics 28: 2407–2411. [[Zhang_et+al_BacterialGrasDomainProteinsThrow_2012]] [OA](https://api.scholarcy.com/oa_version?query=Zhang%2C%20D.%20Iyer%2C%20L.M.%20Aravind%2C%20L.%20Bacterial%20GRAS%20domain%20proteins%20throw%20new%20light%20on%20gibberellic%20acid%20response%20mechanisms%202012) [GScholar](https://scholar.google.co.uk/scholar?q=Zhang%2C%20D.%20Iyer%2C%20L.M.%20Aravind%2C%20L.%20Bacterial%20GRAS%20domain%20proteins%20throw%20new%20light%20on%20gibberellic%20acid%20response%20mechanisms%202012) [Scite](https://api.scholarcy.com/scite_url?query=Zhang%2C%20D.%20Iyer%2C%20L.M.%20Aravind%2C%20L.%20Bacterial%20GRAS%20domain%20proteins%20throw%20new%20light%20on%20gibberellic%20acid%20response%20mechanisms%202012)

[^Zhang_et+al_2010_a]: Zhang, Q., Blaylock, L.A., and Harrison, M.J. (2010). Two Medicago truncatula half-ABC transporters are essential for arbuscule development in arbuscular mycorrhizal symbiosis. Plant Cell 22: 1483– 1497. [[Zhang_et+al_MedicagoTruncatulaHalfabcTransportersEssential_2010]] [OA](https://api.scholarcy.com/oa_version?query=Zhang%2C%20Q.%20Blaylock%2C%20L.A.%20Harrison%2C%20M.J.%20Two%20Medicago%20truncatula%20half-ABC%20transporters%20are%20essential%20for%20arbuscule%20development%20in%20arbuscular%20mycorrhizal%20symbiosis%202010) [GScholar](https://scholar.google.co.uk/scholar?q=Zhang%2C%20Q.%20Blaylock%2C%20L.A.%20Harrison%2C%20M.J.%20Two%20Medicago%20truncatula%20half-ABC%20transporters%20are%20essential%20for%20arbuscule%20development%20in%20arbuscular%20mycorrhizal%20symbiosis%202010) [Scite](https://api.scholarcy.com/scite_url?query=Zhang%2C%20Q.%20Blaylock%2C%20L.A.%20Harrison%2C%20M.J.%20Two%20Medicago%20truncatula%20half-ABC%20transporters%20are%20essential%20for%20arbuscule%20development%20in%20arbuscular%20mycorrhizal%20symbiosis%202010)

[^Zhang_et+al_2015_a]: Zhang, X., Pumplin, N., Ivanov, S., and Harrison, M.J. (2015a). EXO70I is essential for development of a sub-domain of the periarbuscular membrane during arbuscular mycorrhizal symbiosis. Curr. Biol. 25: 2189–2195. [[Zhang_et+al_Exo70iEssentialDevelopmentdomainPeriarbuscular_2015]] [OA](https://api.scholarcy.com/oa_version?query=Zhang%2C%20X.%20Pumplin%2C%20N.%20Ivanov%2C%20S.%20Harrison%2C%20M.J.%20EXO70I%20is%20essential%20for%20development%20of%20a%20sub-domain%20of%20the%20periarbuscular%20membrane%20during%20arbuscular%20mycorrhizal%20symbiosis%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Zhang%2C%20X.%20Pumplin%2C%20N.%20Ivanov%2C%20S.%20Harrison%2C%20M.J.%20EXO70I%20is%20essential%20for%20development%20of%20a%20sub-domain%20of%20the%20periarbuscular%20membrane%20during%20arbuscular%20mycorrhizal%20symbiosis%202015) [Scite](https://api.scholarcy.com/scite_url?query=Zhang%2C%20X.%20Pumplin%2C%20N.%20Ivanov%2C%20S.%20Harrison%2C%20M.J.%20EXO70I%20is%20essential%20for%20development%20of%20a%20sub-domain%20of%20the%20periarbuscular%20membrane%20during%20arbuscular%20mycorrhizal%20symbiosis%202015)

[^Zhang_et+al_2015_b]: Zhang, X., Pumplin, N., Ivanov, S., and Harrison, M.J. (2015b). EXO70I is required for development of a sub-domain of the periarbuscular membrane during arbuscular mycorrhizal symbiosis. Curr. Biol. 25: 2189–2195. [[Zhang_et+al_Exo70iRequiredDevelopmentdomainPeriarbuscular_2015]] [OA](https://api.scholarcy.com/oa_version?query=Zhang%2C%20X.%20Pumplin%2C%20N.%20Ivanov%2C%20S.%20Harrison%2C%20M.J.%20EXO70I%20is%20required%20for%20development%20of%20a%20sub-domain%20of%20the%20periarbuscular%20membrane%20during%20arbuscular%20mycorrhizal%20symbiosis%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Zhang%2C%20X.%20Pumplin%2C%20N.%20Ivanov%2C%20S.%20Harrison%2C%20M.J.%20EXO70I%20is%20required%20for%20development%20of%20a%20sub-domain%20of%20the%20periarbuscular%20membrane%20during%20arbuscular%20mycorrhizal%20symbiosis%202015) [Scite](https://api.scholarcy.com/scite_url?query=Zhang%2C%20X.%20Pumplin%2C%20N.%20Ivanov%2C%20S.%20Harrison%2C%20M.J.%20EXO70I%20is%20required%20for%20development%20of%20a%20sub-domain%20of%20the%20periarbuscular%20membrane%20during%20arbuscular%20mycorrhizal%20symbiosis%202015)

[^Zhang_et+al_2015_c]: Zhang, X., Dong, W., Sun, J., Feng, F., Deng, Y., He, Z., Oldroyd, G.E.D., and Wang, E. (2015c). The receptor kinase CERK1 has dual functions in symbiosis and immunity signalling. Plant J. 81: 258– 267. [[Zhang_et+al_ReceptorKinaseCerk1DualFunctions_2015]] [OA](https://api.scholarcy.com/oa_version?query=Zhang%2C%20X.%20Dong%2C%20W.%20Sun%2C%20J.%20Feng%2C%20F.%20The%20receptor%20kinase%20CERK1%20has%20dual%20functions%20in%20symbiosis%20and%20immunity%20signalling%202015) [GScholar](https://scholar.google.co.uk/scholar?q=Zhang%2C%20X.%20Dong%2C%20W.%20Sun%2C%20J.%20Feng%2C%20F.%20The%20receptor%20kinase%20CERK1%20has%20dual%20functions%20in%20symbiosis%20and%20immunity%20signalling%202015) [Scite](https://api.scholarcy.com/scite_url?query=Zhang%2C%20X.%20Dong%2C%20W.%20Sun%2C%20J.%20Feng%2C%20F.%20The%20receptor%20kinase%20CERK1%20has%20dual%20functions%20in%20symbiosis%20and%20immunity%20signalling%202015)

[^Zipfel_2017_a]: Zipfel, C., and Oldroyd, G.E.D. (2017). Plant signalling in symbiosis and immunity. Nature 543: 328–336. [[Zipfel_PlantSignallingSymbiosisImmunity_2017]] [OA](https://api.scholarcy.com/oa_version?query=Zipfel%2C%20C.%20Oldroyd%2C%20G.E.D.%20Plant%20signalling%20in%20symbiosis%20and%20immunity%202017) [GScholar](https://scholar.google.co.uk/scholar?q=Zipfel%2C%20C.%20Oldroyd%2C%20G.E.D.%20Plant%20signalling%20in%20symbiosis%20and%20immunity%202017) [Scite](https://api.scholarcy.com/scite_url?query=Zipfel%2C%20C.%20Oldroyd%2C%20G.E.D.%20Plant%20signalling%20in%20symbiosis%20and%20immunity%202017)

