[[Reeve_et+al_EnsifersinorhizobiumMedicaeStrainWsm419_2010]]

# [Ensifer (Sinorhizobium) medicae strain WSM419](https://dx.doi.org/10.4506/sigs.43526)

## [[Wayne Reeve]]; [[Patrick Chain]]; [[Graham O’Hara]] et al.

### 2010

## Abstract
Editorial note - Readers are advised ==that in Opinion 84 the Judicial Commission of the International Committee on Systematics of Prokaryotes ruled that the genus name Ensifer Casida 1982 has priority over Sinorhizobium Chen et al. 1988== and the names are synonyms [^1]. ==It was further concluded that the transfer of members== of the genus Sinorhizobium to the genus Ensifer, as proposed by Young [^2] would not cause confusion.

## Key concepts
#rhizobium; #bacteria; #rhizobia; #joint_genome_institute; #rhizobium_meliloti; #acid_soil; #nitrogen_fixation; #plant_soil; #symbiosis; #alphaproteobacteria; #taxonomy; #aerobic; #systematics; #ontology; #international_committee; #sinorhizobium_medicae; #sinorhizobium_meliloti; #grains_research_and_development_corporation; #medicago_truncatula

## Quote
> WSM419 is of particular interest because it is saprophytically competent in the acidic, infertile soils of southern Australia, and it is a highly effective nitrogen fixing microsymbiont of a broad range of annual medics of Mediterranean origin


## Key points
- Agricultural systems are nearly always nitrogen deficient, a factor which grossly limits their productivity
- A particular suite of strains isolated from acidic soils on the Italian island of Sardinia proved to be acid soil tolerant [^9], an attribute we know is related to the presence of a unique set of genes required for acid adaptation [^10]. Characterization of these acid-tolerant isolates revealed that they belonged to the species E. medicae and could be symbiotically distinguished from the related species E. meliloti by their unique capacity to fix nitrogen in association with annual acid soil adapted Medicago hosts of worldwide agronomic value [^11], as well as with the perennial forage legume M. sativa [^12]
- WSM419 is of particular interest because it is saprophytically competent in the acidic, infertile soils of southern Australia [^9],[^13], and it is a highly effective nitrogen fixing microsymbiont of a broad range of annual medics of Mediterranean origin [^11],[^12]
- WSM419 is more effective at fixing nitrogen with M. truncatula than the previously sequenced E. meliloti Sm1021, making it an ideal candidate for inoculation of this model legume [^12]
- Genome project history E. medicae WSM419 was selected for sequencing on the basis of its importance as a symbiotic nitrogen fixing bacterium in agriculture, and its tolerance for acidic soils [^9],[^14]
- Genome annotation Automated gene prediction was completed by assessing congruence of gene call results from three independent programs, the Critica [^46], Generation, and Glimmer [^47] modeling packages, and by comparing the translations to the GenBank nonredundant database using the basic local alignment search tool for proteins (BLASTP)

## Synopsis
Agricultural systems are nearly always nitrogen deficient, a factor which grossly limits their productivity.
Characterization of these acid-tolerant isolates revealed that they belonged to the species E. medicae and could be symbiotically distinguished from the related species E. meliloti by their unique capacity to fix nitrogen in association with annual acid soil adapted Medicago hosts of worldwide agronomic value [^11], as well as with the perennial forage legume M. sativa [^12].
One of the acid-tolerant isolates, E. medicae strain WSM419, was isolated in 1981 from a nodule recovered from the roots of an annual medic (M. murex) growing south of Tempio in Sardinia.
WSM419 is of particular interest because it is saprophytically competent in the acidic, infertile soils of southern Australia [^9],[^13], and it is a highly effective nitrogen fixing microsymbiont of a broad range of annual medics of Mediterranean origin [^11],[^12].
The authors present a summary classification and a set of features (Table 1) for E. medicae strain WSM419, together with the description of a complete genome sequence and annotation.
An intragenic fragment of 1,440 bp was chosen since the 16S rRNA gene has not been completely sequenced in many type strains.
Specific symbiotic characteristics that further distinguish E. medicae WSM419 from E. meliloti include its ability to nodulate and fix nitrogen effectively with a wide range of annual Mediterranean medics, including M. polymorpha, M. arabica, M. murex and M. sphaerocarpos.
WSM419 is more effective at fixing nitrogen with M. truncatula than the previously sequenced E. meliloti Sm1021, making it an ideal candidate for inoculation of this model legume [^12].
Genome project history E. medicae WSM419 was selected for sequencing on the basis of its importance as a symbiotic nitrogen fixing bacterium in agriculture, and its tolerance for acidic soils [^9],[^14].
Processing of sequence traces and base calling and assessment of data quality and assembly were performed with the PHRED/PHRAP/CONSED package [^42],[^43],[^44].
Gaps in the sequence were primarily identified by mate-pair sequences and closed by primer walking on gap-spanning library clones or genomic DNA amplified PCR products.
Genome annotation Automated gene prediction was completed by assessing congruence of gene call results from three independent programs, the Critica [^46], Generation, and Glimmer [^47] modeling packages, and by comparing the translations to the GenBank nonredundant database using the basic local alignment search tool for proteins (BLASTP).
Additional gene prediction analysis and functional annotation was performed within the Integrated Microbial Genomes (IMG-ER) platform [^49].


## Study subjects

### 6599 genes
- The genome is 6,817,576 bp long with 61.15% GC content and comprised of four replicons (Table 4); one circular chromosome of size 3,781,904 bp (Figure 3) and three plasmids of size 1,570,951 bp, 1,245,408 bp and 219,313 bp (Figure 4). ==Of the 6,599 genes predicted, 6,518 were protein-coding genes, and 81 RNA only encoding genes==. In addition, 305 pseudogenes were identified

## Findings
- The majority of the genes (70.4%) were assigned a putative function while those remaining were annotated as hypothetical proteins

##  Confirmation of earlier findings
- Consequently, an effort was initiated in the 1980s to discover more acid-tolerant medic microsymbionts from world regions with acidic soils upon which annual medics had evolved. A particular suite of strains isolated from acidic soils on the Italian island of Sardinia proved to be acid soil tolerant [^9], an attribute ==we now know is related to the presence of a unique set of genes required for acid adaptation== [^10]


## References
[^1]: Judicial Commission of the International Committee on Systematics of Prokaryotes. The genus name Sinorhizobium Chen et al. 1988 is a later synonym of Ensifer Casida 1982 and is not conserved over the latter genus name, and the species name 'Sinorhizobium adhaerens' is not validly published. Opinion 84. Int J Syst Evol Microbiol 2008; 58: 1973. PubMed doi:10.1099/ijs.0.2008/005991-0 [[Judicial_CommitteeSystematicsProkaryotesTheGenus_1988]] [OA](https://doi.org/10.1099/ijs.0.2008/005991-0)  [Scite](https://scite.ai/reports/10.1099/ijs.0.2008/005991-0)

[^2]: Young JM. The genus name Ensifer Casida 1982 takes priority over Sinorhizobium Chen et al. 1988, and Sinorhizobium morelense Wang et al. 2002 is a later synonym of Ensifer adhaerens Casida 1982. Is the combination Sinorhizobium adhaerens (Casida 1982) Willems et al. 2003 legitimate? Request for an Opinion. Int J Syst Evol Microbiol 2003; 53:2107-2110. PubMed doi:10.1099/ijs.0.02665-0 [[Young_GenusNameEnsiferCasida1982_2002]] [OA](https://doi.org/10.1099/ijs.0.02665-0)  [Scite](https://scite.ai/reports/10.1099/ijs.0.02665-0)

[^3]: Peoples MB, Hauggaard-Nielsen H, Jensen EE. Chapter 13. The potential environmental benefits and risks derived from legumes in rotations. In: Emerich, DW & Krishnan HB (Eds.), Agronomy Monograph 52. Nitrogen Fixation in Crop Production Am Soc Agron, Crop Sci Soc Am & Soil Sci Soc Am 2009, pp. 349-385 Madison, Wisconsin, USA. [[Peoples_et+al_Chapter13ThePotentialEnvironmental_2009]] [OA](https://api.scholarcy.com/oa_version?query=Peoples%2C%20M.B.%20Hauggaard-Nielsen%2C%20H.%20Jensen%2C%20E.E.%20Chapter%2013.%20The%20potential%20environmental%20benefits%20and%20risks%20derived%20from%20legumes%20in%20rotations%202009) [GScholar](https://scholar.google.co.uk/scholar?q=Peoples%2C%20M.B.%20Hauggaard-Nielsen%2C%20H.%20Jensen%2C%20E.E.%20Chapter%2013.%20The%20potential%20environmental%20benefits%20and%20risks%20derived%20from%20legumes%20in%20rotations%202009) [Scite](https://api.scholarcy.com/scite_url?query=Peoples%2C%20M.B.%20Hauggaard-Nielsen%2C%20H.%20Jensen%2C%20E.E.%20Chapter%2013.%20The%20potential%20environmental%20benefits%20and%20risks%20derived%20from%20legumes%20in%20rotations%202009)

[^4]: Sprent JI. Legume nodulation: a global perspective. 2009. Oxford, Wiley-Blackwell. [[Sprent_LegumeNodulationGlobalPerspective2009_2009]] [OA](https://scholar.google.co.uk/scholar?q=Sprent%20JI%20Legume%20nodulation%20a%20global%20perspective%202009%20Oxford%20WileyBlackwell) [GScholar](https://scholar.google.co.uk/scholar?q=Sprent%20JI%20Legume%20nodulation%20a%20global%20perspective%202009%20Oxford%20WileyBlackwell) 

[^5]: Herridge DF, Peoples MB, Boddey RM. Global inputs of biological nitrogen fixation in agricultural systems. Marschner Review. Plant Soil 2008; 311:1-18. doi:10.1007/s11104-008-9668-3 [[Herridge_et+al_GlobalInputsBiologicalNitrogenFixation_2008]] [OA](https://doi.org/10.1007/s11104-008-9668-3)  [Scite](https://scite.ai/reports/10.1007/s11104-008-9668-3)

[^6]: Robson AD, Loneragan JF. Nodulation and growth of Medicago truncatula on acid soils. II Colonization of acid soils by Rhizobium meliloti. Aust J Agric Res 1970; 21:435445. doi:10.1071/AR9700435 [[Robson_NodulationGrowthMedicagoTruncatulaAcid_1970]] [OA](https://doi.org/10.1071/AR9700435)  [Scite](https://scite.ai/reports/10.1071/AR9700435)

[^7]: Graham PH, Parker CA. Diagnostic features in the characterization of the root nodule bacteria of legumes. Plant Soil 1964; 20:383396. doi:10.1007/BF01373828 [[Graham_DiagnosticFeaturesCharacterizationRootNodule_1964]] [OA](https://doi.org/10.1007/BF01373828)  [Scite](https://scite.ai/reports/10.1007/BF01373828)

[^8]: Howieson JG. Characteristics of an ideotype acid tolerant pasture legume symbiosis in Mediterranean agriculture. Plant Soil 1995; 171:7176. doi:10.1007/BF00009567 [[Howieson_CharacteristicsIdeotypeAcidTolerantPasture_1995]] [OA](https://doi.org/10.1007/BF00009567)  [Scite](https://scite.ai/reports/10.1007/BF00009567)

[^9]: Howieson JG, Ewing MA. Acid tolerance in the Rhizobium meliloti-Medicago symbiosis. Aust J Agric Res 1986; 37:5564. doi:10.1071/AR9860055 [[Howieson_AcidToleranceRhizobiumMelilotimedicagoSymbiosis_1986]] [OA](https://doi.org/10.1071/AR9860055)  [Scite](https://scite.ai/reports/10.1071/AR9860055)

[^10]: Reeve WG, Brau L, Castelli J, Garau G, Sohlenkamp C, Geiger O, Dilworth MJ, Glenn AR, Howieson JG, Tiwari RP. The Sinorhizobium medicae WSM419 lpiA gene is transcriptionally activated by FsrR and required to enhance survival in lethal acid conditions. Microbiology 2006; 152:3049-3059. PubMed doi:10.1099/mic.0.28764-0 [[Reeve_et+al_SinorhizobiumMedicaeWsm419LpiaGene_2006]] [OA](https://doi.org/10.1099/mic.0.28764-0)  [Scite](https://scite.ai/reports/10.1099/mic.0.28764-0)

[^11]: Garau G, Reeve WG, Brau L, Deiana P, Yates RJ, James D, Tiwari RP, O’Hara GW, Howieson JG. The symbiotic requirements of different Medicago spp. suggest the evolution of Sinorhizobium meliloti and E. medicae with hosts differentially adapted to soil pH. Plant Soil 2005; 276:263277. doi:10.1007/s11104-005-0374-0 [[Garau_et+al_SymbioticRequirementsDifferentMedicago_2005]] [OA](https://doi.org/10.1007/s11104-005-0374-0)  [Scite](https://scite.ai/reports/10.1007/s11104-005-0374-0)

[^12]: Terpolilli JJ, O’Hara GW, Tiwari RP, Dilworth MJ, Howieson JG. The model legume Medicago truncatula A17 is poorly matched for N2 fixation with the sequenced microsymbiont Sinorhizobium meliloti 1021. New Phytol 2008; 179:62-66. PubMed doi:10.1111/j.1469-8137.2008.02464.x [[Terpolilli_et+al_ModelLegumeMedicagoTruncatulaA17_2008]] [OA](https://doi.org/10.1111/j.1469-8137.2008.02464.x)  [Scite](https://scite.ai/reports/10.1111/j.1469-8137.2008.02464.x)

[^13]: Howieson JG, Ewing MA, D'Antuono MF. Selection for acid tolerance in Rhizobium meliloti. Plant Soil 1988; 105:179188. doi:10.1007/BF02376781 [[Howieson_et+al_SelectionAcidToleranceRhizobiumMeliloti_1988]] [OA](https://doi.org/10.1007/BF02376781)  [Scite](https://scite.ai/reports/10.1007/BF02376781)

[^14]: Bullard GK, Roughley RJ, Pulsford DJ. The legume inoculant industry and inoculant quality control in Australia: 1953–2003. Aust J Exp Agric 2005; 45:127-140. doi:10.1071/EA03159 [[Bullard_et+al_LegumeInoculantIndustryInoculantQuality_2005]] [OA](https://doi.org/10.1071/EA03159)  [Scite](https://scite.ai/reports/10.1071/EA03159)

[^15]: Dilworth MJ, Howieson JG, Reeve WG, Tiwari RP, Glenn AR. Acid tolerance in legume root nodule bacteria and selecting for it. Aust J Exp Agric 2001; 41:435-446. doi:10.1071/EA99155 [[Dilworth_et+al_AcidToleranceLegumeRootNodule_2001]] [OA](https://doi.org/10.1071/EA99155)  [Scite](https://scite.ai/reports/10.1071/EA99155)

[^16]: Field D, Garrity G, Gray T, Morrison N, Selengut J, Sterk P, Tatusova T, Thomson N, Allen MJ, Angiuoli SV et al. Towards a richer description of our complete collection of genomes and metagenomes: the “Minimum Information about a Genome Sequence” (MIGS) specification. Nat Biotechnol 2008; 26:541-547. doi:10.1038/nbt1360 PubMed doi:10.1038/ [[Field_et+al_TowardsRicherDescriptionCompleteCollection_2008]] [OA](https://doi.org/10.1038/nbt1360)  [Scite](https://scite.ai/reports/10.1038/nbt1360)

[^17]: Woese CR, Kandler O, Wheelis ML. Towards a natural system of organisms: proposal for the domains Archaea, Bacteria, and Eucarya. Proc Natl Acad Sci USA 1990; 87: 4576-4579. PubMed doi:10.1073/pnas.87.12.4576 [[Woese_et+al_TowardsNaturalSystemOrganismsProposal_1990]] [OA](https://doi.org/10.1073/pnas.87.12.4576)  [Scite](https://scite.ai/reports/10.1073/pnas.87.12.4576)

[^18]: Garrity GM, Holt JG. The Road Map to the Manual. In: Garrity GM, Boone DR, Castenholz RW (eds), Bergey's Manual of Systematic Bacteriology, Second Edition, Volume 1, Springer, New York, 2001, p. 119-169. [[Garrity_RoadMapManual_2001]] [OA](https://api.scholarcy.com/oa_version?query=Garrity%2C%20G.M.%20Holt%2C%20J.G.%20The%20Road%20Map%20to%20the%20Manual%202001) [GScholar](https://scholar.google.co.uk/scholar?q=Garrity%2C%20G.M.%20Holt%2C%20J.G.%20The%20Road%20Map%20to%20the%20Manual%202001) [Scite](https://api.scholarcy.com/scite_url?query=Garrity%2C%20G.M.%20Holt%2C%20J.G.%20The%20Road%20Map%20to%20the%20Manual%202001)

[^19]: Garrity GM, Bell JA, Lilburn T. Class I. Alphaproteobacteria class. nov. In: Garrity GM, Brenner DJ, Krieg NR, Staley JT (eds), Bergey's Manual of Systematic Bacteriology, Second Edition, Volume 2, Part C, Springer, New York, 2005, p. 1. [[Garrity_et+al_AlphaproteobacteriaClass_2005]] [OA](https://api.scholarcy.com/oa_version?query=Garrity%2C%20G.M.%20Bell%2C%20J.A.%20Class%2C%20Lilburn%20T.%20I.%20Alphaproteobacteria%20class%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Garrity%2C%20G.M.%20Bell%2C%20J.A.%20Class%2C%20Lilburn%20T.%20I.%20Alphaproteobacteria%20class%202005) [Scite](https://api.scholarcy.com/scite_url?query=Garrity%2C%20G.M.%20Bell%2C%20J.A.%20Class%2C%20Lilburn%20T.%20I.%20Alphaproteobacteria%20class%202005)

[^20]: List editor. Validation List No. 107. List of new names and new combinations previously effectively, but not validly, published. Int J Syst Evol Microbiol 2006; 56: 1-6. PubMed doi:10.1099/ijs.0.64188-0 [[List_ValidationListNo107List_2006]] [OA](https://doi.org/10.1099/ijs.0.64188-0)  [Scite](https://scite.ai/reports/10.1099/ijs.0.64188-0)

[^21]: Kuykendall LD. Order VI. Rhizobiales ord. nov. In: Garrity GM, Brenner DJ, Krieg NR, Staley JT (eds), Bergey's Manual of Systematic Bacteriology, Second Edition, Volume 2, Part C, Springer, New York, 2005, p. 324. [[Order_Rhizobiales_2005]] [OA](https://api.scholarcy.com/oa_version?query=Order%2C%20VI%2C%20Kuykendall%20L.D.%20Rhizobiales%20ord%202005) [GScholar](https://scholar.google.co.uk/scholar?q=Order%2C%20VI%2C%20Kuykendall%20L.D.%20Rhizobiales%20ord%202005) [Scite](https://api.scholarcy.com/scite_url?query=Order%2C%20VI%2C%20Kuykendall%20L.D.%20Rhizobiales%20ord%202005)

[^22]: Skerman VBD, McGowan V, Sneath PHA. Approved Lists of Bacterial Names. Int J Syst Bacteriol 1980; 30: 225-420. [[Skerman_et+al_ApprovedListsBacterialNames_1980]] [OA](https://api.scholarcy.com/oa_version?query=Skerman%2C%20V.B.D.%20McGowan%2C%20V.%20Sneath%2C%20P.H.A.%20Approved%20Lists%20of%20Bacterial%20Names%201980) [GScholar](https://scholar.google.co.uk/scholar?q=Skerman%2C%20V.B.D.%20McGowan%2C%20V.%20Sneath%2C%20P.H.A.%20Approved%20Lists%20of%20Bacterial%20Names%201980) [Scite](https://api.scholarcy.com/scite_url?query=Skerman%2C%20V.B.D.%20McGowan%2C%20V.%20Sneath%2C%20P.H.A.%20Approved%20Lists%20of%20Bacterial%20Names%201980)

[^23]: Conn HJ. Taxonomic relationships of certain nonsporeforming rods in soil. J Bacteriol 1938; 36: 320-321. [[Conn_TaxonomicRelationshipsCertainNonsporeformingRods_1938]] [OA](https://api.scholarcy.com/oa_version?query=Conn%2C%20H.J.%20Taxonomic%20relationships%20of%20certain%20nonsporeforming%20rods%20in%20soil%201938) [GScholar](https://scholar.google.co.uk/scholar?q=Conn%2C%20H.J.%20Taxonomic%20relationships%20of%20certain%20nonsporeforming%20rods%20in%20soil%201938) [Scite](https://api.scholarcy.com/scite_url?query=Conn%2C%20H.J.%20Taxonomic%20relationships%20of%20certain%20nonsporeforming%20rods%20in%20soil%201938)

[^24]: Chen WX, Yan GH, Li JL. Numerical taxonomic study of fast-growing soybean rhizobia and a proposal that Rhizobium fredii be assigned to Sinorhizobium gen. nov. Int J Syst Bacteriol 1988; 38: 392-397. [[Chen_et+al_NumericalTaxonomicStudyFastgrowingSoybean_1988]] [OA](https://api.scholarcy.com/oa_version?query=Chen%2C%20W.X.%20Yan%2C%20G.H.%20Li%2C%20J.L.%20Numerical%20taxonomic%20study%20of%20fast-growing%20soybean%20rhizobia%20and%20a%20proposal%20that%20Rhizobium%20fredii%20be%20assigned%20to%20Sinorhizobium%20gen.%20nov%201988) [GScholar](https://scholar.google.co.uk/scholar?q=Chen%2C%20W.X.%20Yan%2C%20G.H.%20Li%2C%20J.L.%20Numerical%20taxonomic%20study%20of%20fast-growing%20soybean%20rhizobia%20and%20a%20proposal%20that%20Rhizobium%20fredii%20be%20assigned%20to%20Sinorhizobium%20gen.%20nov%201988) [Scite](https://api.scholarcy.com/scite_url?query=Chen%2C%20W.X.%20Yan%2C%20G.H.%20Li%2C%20J.L.%20Numerical%20taxonomic%20study%20of%20fast-growing%20soybean%20rhizobia%20and%20a%20proposal%20that%20Rhizobium%20fredii%20be%20assigned%20to%20Sinorhizobium%20gen.%20nov%201988)

[^25]: De Lajudie P, Willems A, Pot B, Dewettinck D, Maestrojuan G, Neyra M, Collins MD, Dreyfus B, Kersters K, Gillis M. Polyphasic taxonomy of Rhizobia: emendation of the genus Sinorhizobium and description of Sinorhizobium meliloti comb. nov., Sinorhizobium saheli sp. nov., and Sinorhizobium teranga sp. nov. Int J Syst Bacteriol 1994; 44: 715-733. [[de_Lajudie_et+al_PolyphasicTaxonomyRhizobiaEmendationGenus_1994]] [OA](https://api.scholarcy.com/oa_version?query=P%2C%20De%20Lajudie%20A%2C%20Willems%20B%2C%20Pot%20D%2C%20Dewettinck%20Polyphasic%20taxonomy%20of%20Rhizobia%3A%20emendation%20of%20the%20genus%20Sinorhizobium%20and%20description%20of%20Sinorhizobium%20meliloti%20comb.%20nov.%2C%20Sinorhizobium%20saheli%20sp.%20nov.%2C%20and%20Sinorhizobium%20teranga%20sp.%20nov%201994) [GScholar](https://scholar.google.co.uk/scholar?q=P%2C%20De%20Lajudie%20A%2C%20Willems%20B%2C%20Pot%20D%2C%20Dewettinck%20Polyphasic%20taxonomy%20of%20Rhizobia%3A%20emendation%20of%20the%20genus%20Sinorhizobium%20and%20description%20of%20Sinorhizobium%20meliloti%20comb.%20nov.%2C%20Sinorhizobium%20saheli%20sp.%20nov.%2C%20and%20Sinorhizobium%20teranga%20sp.%20nov%201994) [Scite](https://api.scholarcy.com/scite_url?query=P%2C%20De%20Lajudie%20A%2C%20Willems%20B%2C%20Pot%20D%2C%20Dewettinck%20Polyphasic%20taxonomy%20of%20Rhizobia%3A%20emendation%20of%20the%20genus%20Sinorhizobium%20and%20description%20of%20Sinorhizobium%20meliloti%20comb.%20nov.%2C%20Sinorhizobium%20saheli%20sp.%20nov.%2C%20and%20Sinorhizobium%20teranga%20sp.%20nov%201994)

[^26]: Willems A, Fernández-López M, MuñozAdelantado E, Goris J, De Vos P, MartínezRomero E, Toro N, Gillis M. Description of new Ensifer strains from nodules and proposal to transfer Ensifer adhaerens Casida 1982 to Sinorhizobium as Sinorhizobium adhaerens comb. nov. Request for an opinion. Int J Syst Evol Microbiol 2003; 53: 1207-1217. PubMed doi:10.1099/ijs.0.02264-0 [[Willems_et+al_DescriptionEnsiferStrainsFromNodules_2003]] [OA](https://doi.org/10.1099/ijs.0.02264-0)  [Scite](https://scite.ai/reports/10.1099/ijs.0.02264-0)

[^27]: Lindström K, Martinez-Romero ME. International Committee on Systematics of Prokaryotes Subcommittee on the taxonomy of Agrobacterium and Rhizobium. Minutes of the meeting, 4 July 2001, Hamilton, Canada. Int J Syst Evol Microbiol 2002; 52: 2337. doi:10.1099/ijs.0.02524-0 [[Lindstroem_InternationalCommitteeSystematicsProkaryotesSubcommittee_2001]] [OA](https://doi.org/10.1099/ijs.0.02524-0)  [Scite](https://scite.ai/reports/10.1099/ijs.0.02524-0)

[^28]: Rome S, Fernandez MP, Brunel B, Normand P, Cleyet-Marel JC. Sinorhizobium medicae sp. nov., isolated from annual Medicago spp. Int J Syst Bacteriol 1996; 46: 972-980. PubMed [[Rome_et+al_SinorhizobiumMedicaeIsolated_1996]] [OA](https://api.scholarcy.com/oa_version?query=Rome%2C%20S.%20Fernandez%2C%20M.P.%20Brunel%2C%20B.%20Normand%2C%20P.%20Sinorhizobium%20medicae%20sp.%20nov.%2C%20isolated%20from%20annual%20Medicago%20spp%201996) [GScholar](https://scholar.google.co.uk/scholar?q=Rome%2C%20S.%20Fernandez%2C%20M.P.%20Brunel%2C%20B.%20Normand%2C%20P.%20Sinorhizobium%20medicae%20sp.%20nov.%2C%20isolated%20from%20annual%20Medicago%20spp%201996) [Scite](https://api.scholarcy.com/scite_url?query=Rome%2C%20S.%20Fernandez%2C%20M.P.%20Brunel%2C%20B.%20Normand%2C%20P.%20Sinorhizobium%20medicae%20sp.%20nov.%2C%20isolated%20from%20annual%20Medicago%20spp%201996)

[^29]: Kuykendall LD, Hashem F, Wang ET. Genus VII. Sinorhizobium, 2005, pp 358-361. In: Bergey’s Manual of Systematic Bacteriology. Second Edition. Volume 2 The Proteobacteria. Part C The Alpha-, Delta-, and Epsilonproteobacteria. Brenner DJ, Krieg NR, Staley JT (Eds.), Garrity GM (Editor in Chief) Springer Science and Business Media Inc, New York, USA. [[Kuykendall_BergeyManualSystematicBacteriology_2005]] [OA](https://api.scholarcy.com/oa_version?query=Kuykendall%20LD%20Hashem%20F%20Wang%20ET%20Genus%20VII%20Sinorhizobium%202005%20pp%20358361%20In%20Bergeys%20Manual%20of%20Systematic%20Bacteriology%20Second%20Edition%20Volume%202%20The%20Proteobacteria%20Part%20C%20The%20Alpha%20Delta%20and%20Epsilonproteobacteria%20Brenner%20DJ%20Krieg%20NR%20Staley%20JT%20Eds%20Garrity%20GM%20Editor%20in%20Chief%20Springer%20Science%20and%20Business%20Media%20Inc%20New%20York%20USA) [GScholar](https://scholar.google.co.uk/scholar?q=Kuykendall%20LD%20Hashem%20F%20Wang%20ET%20Genus%20VII%20Sinorhizobium%202005%20pp%20358361%20In%20Bergeys%20Manual%20of%20Systematic%20Bacteriology%20Second%20Edition%20Volume%202%20The%20Proteobacteria%20Part%20C%20The%20Alpha%20Delta%20and%20Epsilonproteobacteria%20Brenner%20DJ%20Krieg%20NR%20Staley%20JT%20Eds%20Garrity%20GM%20Editor%20in%20Chief%20Springer%20Science%20and%20Business%20Media%20Inc%20New%20York%20USA) [Scite](https://api.scholarcy.com/scite_url?query=Kuykendall%20LD%20Hashem%20F%20Wang%20ET%20Genus%20VII%20Sinorhizobium%202005%20pp%20358361%20In%20Bergeys%20Manual%20of%20Systematic%20Bacteriology%20Second%20Edition%20Volume%202%20The%20Proteobacteria%20Part%20C%20The%20Alpha%20Delta%20and%20Epsilonproteobacteria%20Brenner%20DJ%20Krieg%20NR%20Staley%20JT%20Eds%20Garrity%20GM%20Editor%20in%20Chief%20Springer%20Science%20and%20Business%20Media%20Inc%20New%20York%20USA)

[^30]: Biological Agents. Technical rules for biological agents www.baua.de TRBA 466. [[Biological_BiologicalAgentsTechnicalRulesBiological_0000]] [OA](http://www.baua.de)  

[^31]: Liolios K, Mavromatis K, Tavernarakis N, Kyrpides NC. The Genomes OnLine Database (GOLD) in 2007: status of genomic and metagenomic projects and their associated metadata. Nucleic Acids Res 2008; 36:D475-D479. PubMed doi:10.1093/nar/gkm884 [[Liolios_et+al_GenomesOnlineDatabasegold2007_2007]] [OA](https://doi.org/10.1093/nar/gkm884)  [Scite](https://scite.ai/reports/10.1093/nar/gkm884)

[^32]: Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, Davis AP, Dolinski K, Dwight SS, Eppig JT, et al. The Gene Ontology Consortium. Gene ontology: tool for the unification of biology. Nat Genet 2000; 25: 5-29. PubMed doi:10.1038/75556 [[Ashburner_et+al_GeneOntologyConsortiumGeneOntology_2000]] [OA](https://doi.org/10.1038/75556)  [Scite](https://scite.ai/reports/10.1038/75556)

[^33]: Reeve WG, Tiwari RP, Dilworth MJ, Glenn AR. Calcium affects the growth and survival of Rhizobium meliloti. Soil Biol Biochem 1993; 25:581586. doi:10.1016/0038-0717(93)90197-J [[Reeve_et+al_CalciumAffectsGrowthSurvivalRhizobium_1993]] [OA](https://doi.org/10.1016/0038-0717(93)90197-J)  [Scite](https://scite.ai/reports/10.1016/0038-0717(93)90197-J)

[^34]: Kumar S, Tamura K, Nei M. MEGA3: integrated software for molecular evolutionary genetics analysis and sequence alignment. Brief Bioinform 2004; 5:150-163. PubMed doi:10.1093/bib/5.2.150 [[Kumar_et+al_Mega3IntegratedSoftwareMolecularEvolutionary_2004]] [OA](https://doi.org/10.1093/bib/5.2.150)  [Scite](https://scite.ai/reports/10.1093/bib/5.2.150)

[^35]: Kimura M. A simple model for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences. J Mol Evol 1980; 16:111-120. doi:10.1007/BF01731581 [[Kimura_SimpleModelEstimatingEvolutionaryRates_1980]] [OA](https://doi.org/10.1007/BF01731581)  [Scite](https://scite.ai/reports/10.1007/BF01731581)

[^36]: Felsenstein J. Confidence limits on phylogenies: an approach using the bootstrap. Evolution 1985; 39:783-791. doi:10.2307/2408678 [[Felsenstein_ConfidenceLimitsPhylogeniesApproachUsing_1985]] [OA](https://doi.org/10.2307/2408678)  [Scite](https://scite.ai/reports/10.2307/2408678)

[^37]: Saitou N, Nei M. Reconstructing phylogenetic trees. Mol Biol Evol 1987; 4:406-425. PubMed [[Saitou_ReconstructingPhylogeneticTrees_1987]] [OA](https://api.scholarcy.com/oa_version?query=Saitou%2C%20N.%20Nei%2C%20M.%20Reconstructing%20phylogenetic%20trees%201987) [GScholar](https://scholar.google.co.uk/scholar?q=Saitou%2C%20N.%20Nei%2C%20M.%20Reconstructing%20phylogenetic%20trees%201987) [Scite](https://api.scholarcy.com/scite_url?query=Saitou%2C%20N.%20Nei%2C%20M.%20Reconstructing%20phylogenetic%20trees%201987)

[^38]: Rome S, Fernandez MP, Brunel B, Normand P, Cleyet-Marel JC. Sinorhizobium medicae sp. nov., isolated from annual Medicago spp. Int J Syst Bacteriol 1996; 46:972-980. PubMed [[Rome_et+al_SinorhizobiumMedicaeIsolated_1996]] [OA](https://api.scholarcy.com/oa_version?query=Rome%2C%20S.%20Fernandez%2C%20M.P.%20Brunel%2C%20B.%20Normand%2C%20P.%20Sinorhizobium%20medicae%20sp.%20nov.%2C%20isolated%20from%20annual%20Medicago%20spp%201996) [GScholar](https://scholar.google.co.uk/scholar?q=Rome%2C%20S.%20Fernandez%2C%20M.P.%20Brunel%2C%20B.%20Normand%2C%20P.%20Sinorhizobium%20medicae%20sp.%20nov.%2C%20isolated%20from%20annual%20Medicago%20spp%201996) [Scite](https://api.scholarcy.com/scite_url?query=Rome%2C%20S.%20Fernandez%2C%20M.P.%20Brunel%2C%20B.%20Normand%2C%20P.%20Sinorhizobium%20medicae%20sp.%20nov.%2C%20isolated%20from%20annual%20Medicago%20spp%201996)

[^39]: Howieson JG, Ewing MA. Annual species of Medicago differ greatly in their ability to nodulate on acid soils. Aust J Agric Res 1989; 40:843850. doi:10.1071/AR9890843 Howieson JG, Evans P, Nutt B. Estimation of hoststrain compatibility for symbiotic N-fixation between Rhizobium meliloti, several annual species of Medicago and Medicago sativa. Plant Soil 2000; 219:49-55. doi:10.1023/A:1004795617375 [[Howieson_AnnualSpeciesMedicagoDifferGreatly_1989]] [OA](https://doi.org/10.1071/AR9890843)  [Scite](https://scite.ai/reports/10.1071/AR9890843)

[^41]: Reeve WG, Tiwari RP, Worsely PS, Dilworth MJ, Glenn AR, Howieson JG. Constructs for insertional mutagenesis, transcriptional signal localisation and gene regulation studies in root nodule and other bacteria. Microbiology 1999; 145:13071316. PubMed doi:10.1099/13500872-145-61307 [[Reeve_et+al_ConstructsInsertionalMutagenesisTranscriptionalSignal_1999]] [OA](https://doi.org/10.1099/13500872-145-61307)  [Scite](https://scite.ai/reports/10.1099/13500872-145-61307)

[^42]: Ewing B, Green P. Base-calling of automated sequencer traces using phred. II. Error probabilities. Genome Res 1998; 8:186-194. PubMed [[Ewing_BasecallingAutomatedSequencerTracesUsing_1998]] [OA](https://api.scholarcy.com/oa_version?query=Ewing%2C%20B.%20Green%2C%20P.%20Base-calling%20of%20automated%20sequencer%20traces%20using%20phred.%20II.%20Error%20probabilities%201998) [GScholar](https://scholar.google.co.uk/scholar?q=Ewing%2C%20B.%20Green%2C%20P.%20Base-calling%20of%20automated%20sequencer%20traces%20using%20phred.%20II.%20Error%20probabilities%201998) [Scite](https://api.scholarcy.com/scite_url?query=Ewing%2C%20B.%20Green%2C%20P.%20Base-calling%20of%20automated%20sequencer%20traces%20using%20phred.%20II.%20Error%20probabilities%201998)

[^43]: Ewing B, Hillier L, Wendl MC, Green P. Basecalling of automated sequencer traces using phred. I. Accuracy assessment. Genome Res 1998; 8:175-185. PubMed [[Ewing_et+al_BasecallingAutomatedSequencerTracesUsing_1998]] [OA](https://api.scholarcy.com/oa_version?query=Ewing%2C%20B.%20Hillier%2C%20L.%20Wendl%2C%20M.C.%20Green%2C%20P.%20Basecalling%20of%20automated%20sequencer%20traces%20using%20phred%201998) [GScholar](https://scholar.google.co.uk/scholar?q=Ewing%2C%20B.%20Hillier%2C%20L.%20Wendl%2C%20M.C.%20Green%2C%20P.%20Basecalling%20of%20automated%20sequencer%20traces%20using%20phred%201998) [Scite](https://api.scholarcy.com/scite_url?query=Ewing%2C%20B.%20Hillier%2C%20L.%20Wendl%2C%20M.C.%20Green%2C%20P.%20Basecalling%20of%20automated%20sequencer%20traces%20using%20phred%201998)

[^44]: Gordon D, Abajian C, Green P. Consed: a graphical tool for sequence finishing. Genome Res 1998; 8:195-202. PubMed [[Gordon_et+al_ConsedGraphicalToolSequenceFinishing_1998]] [OA](https://api.scholarcy.com/oa_version?query=Gordon%2C%20D.%20Abajian%2C%20C.%20Green%2C%20P.%20Consed%3A%20a%20graphical%20tool%20for%20sequence%20finishing%201998) [GScholar](https://scholar.google.co.uk/scholar?q=Gordon%2C%20D.%20Abajian%2C%20C.%20Green%2C%20P.%20Consed%3A%20a%20graphical%20tool%20for%20sequence%20finishing%201998) [Scite](https://api.scholarcy.com/scite_url?query=Gordon%2C%20D.%20Abajian%2C%20C.%20Green%2C%20P.%20Consed%3A%20a%20graphical%20tool%20for%20sequence%20finishing%201998)

[^45]: Chain P, Lamerdin J, Larimer F, Regala W, Lao V, Land M, Hauser L, Hooper A, Klotz M, Norton J, et al. Complete genome sequence of the ammonia-oxidizing bacterium and obligate chemolithoautotroph Nitrosomonas europaea. J Bacteriol 2003; 185:27592773. doi:10.1128/JB.185.9.2759-2773.2003 PubMed [[Chain_et+al_CompleteGenomeSequenceAmmoniaoxidizingBacterium_2003]] [OA](https://doi.org/10.1128/JB.185.9.2759-2773.2003)  [Scite](https://scite.ai/reports/10.1128/JB.185.9.2759-2773.2003)

[^46]: Badger JH, Olsen G. CRITICA: coding region identification tool invoking comparative analysis. Mol Biol Evol 1999; 16:512-524. PubMed [[Badger_CriticaCodingRegionIdentificationTool_1999]] [OA](https://api.scholarcy.com/oa_version?query=Badger%2C%20J.H.%20Olsen%2C%20G.%20CRITICA%3A%20coding%20region%20identification%20tool%20invoking%20comparative%20analysis%201999) [GScholar](https://scholar.google.co.uk/scholar?q=Badger%2C%20J.H.%20Olsen%2C%20G.%20CRITICA%3A%20coding%20region%20identification%20tool%20invoking%20comparative%20analysis%201999) [Scite](https://api.scholarcy.com/scite_url?query=Badger%2C%20J.H.%20Olsen%2C%20G.%20CRITICA%3A%20coding%20region%20identification%20tool%20invoking%20comparative%20analysis%201999)

[^47]: Delcher AL, Harmon D, Kasif S, White O, Salzberg SL. Improved microbial gene identification with GLIMMER. Nucleic Acids Res 1999; 27:4636-4641. PubMed doi:10.1093/nar/27.23.4636 [[Delcher_et+al_ImprovedMicrobialGeneIdentificationWith_1999]] [OA](https://doi.org/10.1093/nar/27.23.4636)  [Scite](https://scite.ai/reports/10.1093/nar/27.23.4636)

[^48]: Lowe TM, Eddy SR. tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence. Nucleic Acids Res 1997; 25:955-964. PubMed doi:10.1093/nar/25.5.955 [[Lowe_TrnascanseProgramImprovedDetectionTransfer_1997]] [OA](https://doi.org/10.1093/nar/25.5.955)  [Scite](https://scite.ai/reports/10.1093/nar/25.5.955)

[^49]: Markowitz VM, Szeto E, Palaniappan K, Grechkin Y, Chu K, Chen IMA, Dubchak I, Anderson I, Lykidis A, Mavromatis K, et al. The Integrated Microbial Genomes (IMG) system in 2007: data content and analysis tool extensions. Nucleic Acids Res 2008; 36:D528-D533. PubMed doi:10.1093/nar/gkm846  [[Markowitz_et+al_IntegratedMicrobialGenomesimgSystem_2007]] [OA](https://doi.org/10.1093/nar/gkm846)  [Scite](https://scite.ai/reports/10.1093/nar/gkm846)

